{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Análise de literatura - PubMed - GENE ELP4\n",
    "Nota:Para a realização de todo o código do trabalho teve-se em conta o tutorial do Biopython. \n",
    "Consoante os códigos que estavam disponíveis no tutorial, tentamos adaptá-los aos genes de interesse, neste caso tentamos adaptar ao gene ELP4."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "1.Numa primeira fase, recorreu-se a algumas funções do Bio.Entrez para se aceder a base de dados PubMed, para o gene ELP4.Ou seja, consultou-se no PubMed todos os artigos relativos ao gene ELP4.\n",
    "Este pequeno código posteriormente apresentado, indica quantos artigos sobre este gene existem no PubMed, 61 artigos."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "61\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "Entrez.email = \"joanafigueiredo2014@gmail.com\" \n",
    "handle = Entrez.egquery(term=\"ELP4\")\n",
    "record = Entrez.read(handle)\n",
    "for row in record[\"eGQueryResult\"]:\n",
    "    if row[\"DbName\"]==\"pubmed\":\n",
    "        print(row[\"Count\"])\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "2.Seguidamente, utilizou-se as funções Bio.Entrez.esearch e Bio.Entrez.read para fazer o download de todos os IDs dos 61 artigos relacionados com o gene ELP4. \n",
    "Posteriormente imprimiu-se a lista que contem todos esses IDs de cada um dos artigos do PubMed.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "Entrez.email = \"joanafigueiredo2014@gmail.com\" \n",
    "handle = Entrez.esearch(db=\"pubmed\", term=\"ELP4\", retmax=61)\n",
    "record = Entrez.read(handle)\n",
    "handle.close()\n",
    "idlist = record[\"IdList\"]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['31743616', '31450992', '30401723', '30087688', '29805303', '29332244', '29217025', '29191170', '28598868', '28474462', '28167615', '27919838', '27386500', '27381094', '26449484', '26419218', '26010655', '25631615', '25628759', '25462850', '25301525', '25194491', '25009170', '24995671', '23505204', '23053802', '22991255', '22854966', '22818593', '22787611', '22556426', '22459513', '22343726', '22266513', '21601275', '21364908', '21321669', '21268240', '20825490', '20546675', '20424645', '20080602', '20054296', '19779648', '19682046', '19500300', '19172991', '18534097', '18282108', '17679951', '15894610', '15473317', '15138274', '12424236', '12386836', '11929532', '11904415', '11889558', '11737649', '11689709', '11435442']\n"
     ]
    }
   ],
   "source": [
    "print(idlist)#obtém-se todos os PMIDs"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "3.Depois de se ter a lista de todos os IDs, dos 61 artigos, pretende-se obter os registos Medline correspondentes para se poder extrair informações a partir deles. \n",
    "Para isso fez se o download dos registos Medline no formato de arquivo simples Medline e utiliza-se o módulo Bio.Medline para analisar esse mesmos registos. Posteriormente imprimiu-se toda a informação contida em cada um dos artigos (records) numa lista.\n",
    "Assim consegue-se saber muito superficialmente, se algum dos artigos relaciona este gene ELP4 com o cancro ou não (ou mais especificamente, se está relacionado com o cancro da mama ou não), analisando algusn dos parâmetros de cada um dos records, por exemplo."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Medline\n",
    "handle = Entrez.efetch(db=\"pubmed\", id=idlist, rettype=\"medline\",retmode=\"text\")#download dos registos no formato de arquivo simples Medline\n",
    "records = Medline.parse(handle)#para analisar os registos\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'PMID': '31743616',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'In-Data-Review',\n",
       "  'LR': '20191222',\n",
       "  'IS': '2162-3279 (Electronic)',\n",
       "  'VI': '9',\n",
       "  'IP': '12',\n",
       "  'DP': '2019 Dec',\n",
       "  'TI': 'Association between ELP4 rs986527 polymorphism and the occurrence and development of intracranial arachnoid cyst.',\n",
       "  'PG': 'e01480',\n",
       "  'LID': '10.1002/brb3.1480 [doi]',\n",
       "  'AB': 'OBJECTIVE: The association between ELP4 rs986527 polymorphism and the occurrence and development of intracranial arachnoid cyst was studied in this paper. METHODS: Eighty-five patients diagnosed with intracranial arachnoid cysts by cerebral computed tomography scan were selected. Sixty-three healthy volunteers for medical examination in hospitals served as controls. The cognition, depressive symptoms, and the likelihood of headache, dizziness, head trauma history, dementia, depression, and epilepsy were assessed. ELP4 genotypes and its allele frequency were determined by PCR, endonuclease restriction analysis, and gel electrophoresis. RESULTS: ELP4 rs986527 had three genotypes: TT, TC, and CC. The intracranial arachnoid cyst group showed no statistically significant difference in genotype frequencies compared with healthy controls. There was no significant correlation between ELP4 rs986527 polymorphism and location of intracranial arachnoid cyst. TC and C genotype frequencies were associated with a higher incidence of clinical symptoms than TT genotype frequencies, and C allele frequencies were associated with a significantly higher incidence of clinical symptoms compared with T allele frequencies. There was no significant difference in TNF-alpha and IL-1beta levels between TT/TC/CC genotypes before treatment. After treatment, the levels of TNF-alpha and IL-1beta were significantly decreased in different genotypes, and the decrease in CC was the greatest. The frequency of TT and TC genotypes was higher than that of CC genotypes. CONCLUSION: ELP4 rs986527 polymorphism affected the incidence of clinical symptoms and the levels of TNF-alpha and IL-1beta in patients with intracranial arachnoid cysts.',\n",
       "  'CI': ['(c) 2019 The Authors. Brain and Behavior published by Wiley Periodicals, Inc.'],\n",
       "  'FAU': ['Li, Kai',\n",
       "   'Kong, De-Sheng',\n",
       "   'Zhang, Jun',\n",
       "   'Wang, Xin-Sheng',\n",
       "   'Ye, Xun',\n",
       "   'Zhao, Yuan-Li'],\n",
       "  'AU': ['Li K', 'Kong DS', 'Zhang J', 'Wang XS', 'Ye X', 'Zhao YL'],\n",
       "  'AD': 'Department of Neurosurgery, Beijing Tian Tan Hospital, Capital Medical University, Beijing, China. Department of Neurosurgery, Peking University International Hospital, Beijing, China. Department of Neurosurgery, Peking University International Hospital, Beijing, China. Department of Neurosurgery, Peking University International Hospital, Beijing, China. Department of Neurosurgery, Beijing Tian Tan Hospital, Capital Medical University, Beijing, China. Department of Neurosurgery, Beijing Tian Tan Hospital, Capital Medical University, Beijing, China. Department of Neurosurgery, Beijing Tian Tan Hospital, Capital Medical University, Beijing, China.',\n",
       "  'AUID': ['ORCID: https://orcid.org/0000-0003-1104-8459'],\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article'],\n",
       "  'DEP': '20191119',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Brain Behav',\n",
       "  'JT': 'Brain and behavior',\n",
       "  'JID': '101570837',\n",
       "  'SB': 'IM',\n",
       "  'PMC': 'PMC6908874',\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['ELP4', 'incidence', 'intracranial arachnoid cyst', 'polymorphism'],\n",
       "  'EDAT': '2019/11/20 06:00',\n",
       "  'MHDA': '2019/11/20 06:00',\n",
       "  'CRDT': ['2019/11/20 06:00'],\n",
       "  'PHST': ['2019/07/04 00:00 [received]',\n",
       "   '2019/10/14 00:00 [accepted]',\n",
       "   '2019/11/20 06:00 [pubmed]',\n",
       "   '2019/11/20 06:00 [medline]',\n",
       "   '2019/11/20 06:00 [entrez]'],\n",
       "  'AID': ['10.1002/brb3.1480 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Brain Behav. 2019 Dec;9(12):e01480. doi: 10.1002/brb3.1480. Epub 2019 Nov 19.'},\n",
       " {'PMID': '31450992',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'Publisher',\n",
       "  'LR': '20190827',\n",
       "  'IS': '1476-4954 (Electronic) 1476-4954 (Linking)',\n",
       "  'DP': '2019 Aug 27',\n",
       "  'TI': 'Genetic testing in fetuses with isolated agenesis of the corpus callosum.',\n",
       "  'PG': '1-201',\n",
       "  'LID': '10.1080/14767058.2019.1660769 [doi]',\n",
       "  'AB': 'Purpose: The objectives of this study were to explore genetics pathogenesis of isolated agenesis of corpus callosum (ACC) and assess the utility of chromosomal microarray analysis (CMA) for genetic diagnosis of Isolated ACC. Methods: We analyzed the genomes of 16 fetuses with isolated ACC using Afymetrix CytoScan HD arrays and conducted further bioinformatic analysis for one proband fetus with an abnormal copy number variation (CNV). Results: Of the 16 fetal samples examined, 2 (12.5%) had pathogenic CNVs and 3 (18.75%) had variants of unknown significance. Two cases, case 2 and case 9, were found to have pathogenic CNVs. Bioinformatic analyses indicated that the CNV of one fetus (case 9) contained 115 annotated coding genes, 5 of which (SLC6A5, BDNF, ELP4, PAX6, and SLC1A2) have been associated with neurodevelopment. Three of these genes (SLC1A2, BDNF, and PAX6) may play a key role in ACC development. GO cluster analysis of the implicated genes revealed strong representations of protein binding and metal ion binding functions. KEGG pathway analysis pointed to four pathways: longevity regulating pathway, amyotrophic lateral sclerosis, cocaine addiction, and autophagy-animal. Conclusions: BDNF, SLC1A2, and PAX6 may be involved in the development of isolated ACC. CMA is a feasible technology for prenatal diagnosis of isolated ACC.',\n",
       "  'FAU': ['She, Qin', 'Fu, Fang', 'Guo, Xiaoyan', 'Tan, Weihe', 'Liao, Can'],\n",
       "  'AU': ['She Q', 'Fu F', 'Guo X', 'Tan W', 'Liao C'],\n",
       "  'AD': \"Zhujiang Hospital, Southern Medical University or The Second School of Clinical Medicine, Southern Medical University , Guangzhou , China. Prenatal Diagnostic Center, Guangzhou Women and Children's Medical Center , Guangzhou , China. Prenatal Diagnostic Center, the Six Affiliated Hospital, Guangzhou Medical University , Qingyuan , China. Prenatal Diagnostic Center, Guangzhou Women and Children's Medical Center , Guangzhou , China. Prenatal Diagnostic Center, the Six Affiliated Hospital, Guangzhou Medical University , Qingyuan , China. Prenatal Diagnostic Center, the Six Affiliated Hospital, Guangzhou Medical University , Qingyuan , China. Zhujiang Hospital, Southern Medical University or The Second School of Clinical Medicine, Southern Medical University , Guangzhou , China. Prenatal Diagnostic Center, Guangzhou Women and Children's Medical Center , Guangzhou , China.\",\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article'],\n",
       "  'DEP': '20190827',\n",
       "  'PL': 'England',\n",
       "  'TA': 'J Matern Fetal Neonatal Med',\n",
       "  'JT': 'The journal of maternal-fetal & neonatal medicine : the official journal of the European Association of Perinatal Medicine, the Federation of Asia and Oceania Perinatal Societies, the International Society of Perinatal Obstetricians',\n",
       "  'JID': '101136916',\n",
       "  'SB': 'IM',\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['CMA',\n",
       "   'Chromosome microarray',\n",
       "   'fetus',\n",
       "   'genetic testing',\n",
       "   'isolated agenesis of corpus callosum'],\n",
       "  'EDAT': '2019/08/28 06:00',\n",
       "  'MHDA': '2019/08/28 06:00',\n",
       "  'CRDT': ['2019/08/28 06:00'],\n",
       "  'PHST': ['2019/08/28 06:00 [entrez]',\n",
       "   '2019/08/28 06:00 [pubmed]',\n",
       "   '2019/08/28 06:00 [medline]'],\n",
       "  'AID': ['10.1080/14767058.2019.1660769 [doi]'],\n",
       "  'PST': 'aheadofprint',\n",
       "  'SO': 'J Matern Fetal Neonatal Med. 2019 Aug 27:1-201. doi: 10.1080/14767058.2019.1660769.'},\n",
       " {'PMID': '30401723',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20190521',\n",
       "  'LR': '20190521',\n",
       "  'IS': '1532-2548 (Electronic) 0032-0889 (Linking)',\n",
       "  'VI': '179',\n",
       "  'IP': '1',\n",
       "  'DP': '2019 Jan',\n",
       "  'TI': 'Elongator Is Required for Root Stem Cell Maintenance by Regulating SHORTROOT Transcription.',\n",
       "  'PG': '220-232',\n",
       "  'LID': '10.1104/pp.18.00534 [doi]',\n",
       "  'AB': 'SHORTROOT (SHR) is essential for stem cell maintenance and radial patterning in Arabidopsis (Arabidopsis thaliana) roots, but how its expression is regulated is unknown. Here, we report that the Elongator complex, which consists of six subunits (ELP1 to ELP6), regulates the transcription of SHR Depletion of Elongator drastically reduced SHR expression and led to defective root stem cell maintenance and radial patterning. The importance of the nuclear localization of Elongator for its functioning, together with the insensitivity of the elp1 mutant to the transcription elongation inhibitor 6-azauracil, and the direct interaction of the ELP4 subunit with the carboxyl-terminal domain of RNA polymerase II, support the notion that Elongator plays important roles in transcription elongation. Indeed, we found that ELP3 associates with the premessenger RNA of SHR and that mutation of Elongator reduces the enrichment of RNA polymerase II on the SHR gene body. Moreover, Elongator interacted in vivo with SUPPRESSOR OF Ty4, a well-established transcription elongation factor that is recruited to the SHR locus. Together, these results demonstrate that Elongator acts in concert with SUPPRESSOR OF Ty4 to regulate the transcription of SHR.',\n",
       "  'CI': ['(c) 2019 American Society of Plant Biologists. All Rights Reserved.'],\n",
       "  'FAU': ['Qi, Linlin',\n",
       "   'Zhang, Xiaoyue',\n",
       "   'Zhai, Huawei',\n",
       "   'Liu, Jian',\n",
       "   'Wu, Fangming',\n",
       "   'Li, Chuanyou',\n",
       "   'Chen, Qian'],\n",
       "  'AU': ['Qi L', 'Zhang X', 'Zhai H', 'Liu J', 'Wu F', 'Li C', 'Chen Q'],\n",
       "  'AUID': ['ORCID: 0000-0001-5187-8401',\n",
       "   'ORCID: 0000-0003-4358-5881',\n",
       "   'ORCID: 0000-0003-0202-3890',\n",
       "   'ORCID: 0000-0002-0300-3931'],\n",
       "  'AD': 'State Key Laboratory of Crop Biology, College of Agronomy, Shandong Agricultural University, Taian, Shandong 271018, China. University of the Chinese Academy of Sciences, Beijing 100049, China. University of the Chinese Academy of Sciences, Beijing 100049, China. State Key Laboratory of Plant Genomics, National Centre for Plant Gene Research, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing 100101, China. University of the Chinese Academy of Sciences, Beijing 100049, China. State Key Laboratory of Plant Genomics, National Centre for Plant Gene Research, Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing 100101, China. State Key Laboratory of Crop Biology, College of Agronomy, Shandong Agricultural University, Taian, Shandong 271018, China. University of the Chinese Academy of Sciences, Beijing 100049, China. University of the Chinese Academy of Sciences, Beijing 100049, China. State Key Laboratory of Crop Biology, College of Agronomy, Shandong Agricultural University, Taian, Shandong 271018, China chenqiangenetics@163.com.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20181106',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Plant Physiol',\n",
       "  'JT': 'Plant physiology',\n",
       "  'JID': '0401224',\n",
       "  'RN': ['0 (Arabidopsis Proteins)',\n",
       "   '0 (SHORT ROOT protein, Arabidopsis)',\n",
       "   '0 (Transcription Factors)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Arabidopsis/cytology/*genetics/growth & development',\n",
       "   'Arabidopsis Proteins/genetics/metabolism/*physiology',\n",
       "   'Meristem/*cytology/genetics/growth & development',\n",
       "   'Plant Roots/cytology/genetics/growth & development',\n",
       "   'Transcription Factors/genetics/*metabolism/physiology',\n",
       "   'Transcription, Genetic'],\n",
       "  'PMC': 'PMC6324240',\n",
       "  'EDAT': '2018/11/08 06:00',\n",
       "  'MHDA': '2019/05/22 06:00',\n",
       "  'CRDT': ['2018/11/08 06:00'],\n",
       "  'PHST': ['2018/05/03 00:00 [received]',\n",
       "   '2018/10/29 00:00 [accepted]',\n",
       "   '2018/11/08 06:00 [pubmed]',\n",
       "   '2019/05/22 06:00 [medline]',\n",
       "   '2018/11/08 06:00 [entrez]'],\n",
       "  'AID': ['pp.18.00534 [pii]', '10.1104/pp.18.00534 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Plant Physiol. 2019 Jan;179(1):220-232. doi: 10.1104/pp.18.00534. Epub 2018 Nov 6.'},\n",
       " {'PMID': '30087688',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'PubMed-not-MEDLINE',\n",
       "  'LR': '20191120',\n",
       "  'IS': '1664-462X (Print) 1664-462X (Linking)',\n",
       "  'VI': '9',\n",
       "  'DP': '2018',\n",
       "  'TI': 'The Arabidopsis Elongator Subunit ELP3 and ELP4 Confer Resistance to Bacterial Speck in Tomato.',\n",
       "  'PG': '1066',\n",
       "  'LID': '10.3389/fpls.2018.01066 [doi]',\n",
       "  'AB': 'Although production of tomato (Solanum lycopersicum) is threatened by a number of major diseases worldwide, it has been difficult to identify effective and durable management measures against these diseases. In this study, we attempted to improve tomato disease resistance by transgenic overexpression of genes encoding the Arabidopsis thaliana Elongator (AtELP) complex subunits AtELP3 and AtELP4. We show that overexpression of AtELP3 and AtELP4 significantly enhanced resistance to tomato bacterial speck caused by the Pseudomonas syringae pv. tomato strain J4 (Pst J4) without clear detrimental effects on plant growth and development. Interestingly, the transgenic plants exhibited resistance to Pst J4 only when inoculated through foliar sprays but not through infiltration into the leaf apoplast. Although this result suggested possible involvement of stomatal immunity, we found that Pst J4 inoculation did not induce stomatal closure and there were no differences in stomatal apertures and conductance between the transgenic and control plants. Further RNA sequencing and real-time quantitative PCR analyses revealed a group of defense-related genes to be induced to higher levels after infection in the AtELP4 transgenic tomato plants than in the control, suggesting that the enhanced disease resistance of the transgenic plants may be attributed to elevated induction of defense responses. Additionally, we show that the tomato genome contains single-copy genes encoding all six Elongator subunits (SlELPs), which share high identities with the AtELP proteins, and that SlELP3 and SlELP4 complemented the Arabidopsis Atelp3 and Atelp4 mutants, respectively, indicating that the function of tomato Elongator is probably conserved. Taken together, our results not only shed new light on the tomato Elongator complex, but also revealed potential candidate genes for engineering disease resistance in tomato.',\n",
       "  'FAU': ['Pereira, Juliana A',\n",
       "   'Yu, Fahong',\n",
       "   'Zhang, Yanping',\n",
       "   'Jones, Jeffrey B',\n",
       "   'Mou, Zhonglin'],\n",
       "  'AU': ['Pereira JA', 'Yu F', 'Zhang Y', 'Jones JB', 'Mou Z'],\n",
       "  'AD': 'Department of Plant Pathology, University of Florida, Gainesville, FL, United States. Interdisciplinary Center for Biotechnology Research, University of Florida, Gainesville, FL, United States. Interdisciplinary Center for Biotechnology Research, University of Florida, Gainesville, FL, United States. Department of Plant Pathology, University of Florida, Gainesville, FL, United States. Department of Microbiology and Cell Science, University of Florida, Gainesville, FL, United States.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article'],\n",
       "  'DEP': '20180724',\n",
       "  'PL': 'Switzerland',\n",
       "  'TA': 'Front Plant Sci',\n",
       "  'JT': 'Frontiers in plant science',\n",
       "  'JID': '101568200',\n",
       "  'PMC': 'PMC6066517',\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['AtELP4',\n",
       "   'Pseudomonas syringae',\n",
       "   'disease resistance',\n",
       "   'the Elongator complex',\n",
       "   'tomato',\n",
       "   'transgenic overexpression'],\n",
       "  'EDAT': '2018/08/09 06:00',\n",
       "  'MHDA': '2018/08/09 06:01',\n",
       "  'CRDT': ['2018/08/09 06:00'],\n",
       "  'PHST': ['2018/05/09 00:00 [received]',\n",
       "   '2018/06/29 00:00 [accepted]',\n",
       "   '2018/08/09 06:00 [entrez]',\n",
       "   '2018/08/09 06:00 [pubmed]',\n",
       "   '2018/08/09 06:01 [medline]'],\n",
       "  'AID': ['10.3389/fpls.2018.01066 [doi]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'Front Plant Sci. 2018 Jul 24;9:1066. doi: 10.3389/fpls.2018.01066. eCollection 2018.'},\n",
       " {'PMID': '29805303',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20190715',\n",
       "  'LR': '20190715',\n",
       "  'IS': '1449-2288 (Electronic) 1449-2288 (Linking)',\n",
       "  'VI': '14',\n",
       "  'IP': '5',\n",
       "  'DP': '2018',\n",
       "  'TI': 'Elongator promotes the migration and invasion of hepatocellular carcinoma cell by the phosphorylation of AKT.',\n",
       "  'PG': '518-530',\n",
       "  'LID': '10.7150/ijbs.23511 [doi]',\n",
       "  'AB': 'The Elongator is a complex with multiple subunits (Elp1-Elp6) which promotes transcript elongation and protein translation. In this study, we investigated the effects of Elongator on the migration and invasion of HCC cells as well as the underlying mechanisms. We showed that overexpression of Elp3 or Elp4 promoted the migration and invasion of HCC cells, which was abolished when either Elp3 or Elp4 was silenced. The expression of matrix metalloproteinase-2 (MMP-2) and MMP-9 were enhanced by phosphorylation of AKT. Elongator-driven migration and invasion and the expression of MMP-2 and MMP-9 were reduced in HCC cells treated with AKT inhibitor LY294002. Depletion of Elp3 also reduced the phosphorylation of AKT induced by growth factors. In vivo assay of lung metastasis in mice demonstrated that overexpression of Elp3 increased tumor nodules metastatic to lung. Importantly, Elp3 was up-regulated in human HCC tissues, which was correlated with the phosphorylation of AKT and expression of MMP-2. Collectively, these results suggested that Elongator activated migration and invasion of HCC cells by promoting the expression of MMP-2 and MMP-9 through the PI3K/AKT signaling pathway. Our work suggests that Elongator might be a potential marker which promotes the metastasis of HCC.',\n",
       "  'FAU': ['Xu, Yi',\n",
       "   'Zhou, Wei',\n",
       "   'Ji, Yuan',\n",
       "   'Shen, Jian',\n",
       "   'Zhu, Xiaxia',\n",
       "   'Yu, Huijun',\n",
       "   'Guo, Jingchun',\n",
       "   'Pang, Zhi',\n",
       "   'Wei, Wenxiang'],\n",
       "  'AU': ['Xu Y',\n",
       "   'Zhou W',\n",
       "   'Ji Y',\n",
       "   'Shen J',\n",
       "   'Zhu X',\n",
       "   'Yu H',\n",
       "   'Guo J',\n",
       "   'Pang Z',\n",
       "   'Wei W'],\n",
       "  'AD': 'Department of Cell Biology and Institute of Bioengineering, School of Medicine, Soochow University, Suzhou, 215123 China. Department of Cell Biology and Institute of Bioengineering, School of Medicine, Soochow University, Suzhou, 215123 China. Department of Cell Biology and Institute of Bioengineering, School of Medicine, Soochow University, Suzhou, 215123 China. Department of Cell Biology and Institute of Bioengineering, School of Medicine, Soochow University, Suzhou, 215123 China. Department of Cell Biology and Institute of Bioengineering, School of Medicine, Soochow University, Suzhou, 215123 China. Department of Cell Biology and Institute of Bioengineering, School of Medicine, Soochow University, Suzhou, 215123 China. State Key laboratory of Medical Neurobiology, Fudan University, Shanghai, 200032 China. Department of Gastroenterology, The North District of the Affiliated Suzhou Hospital of Nanjing Medical University, Suzhou, Jiangsu, 215008 China. Department of Cell Biology and Institute of Bioengineering, School of Medicine, Soochow University, Suzhou, 215123 China.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20180405',\n",
       "  'PL': 'Australia',\n",
       "  'TA': 'Int J Biol Sci',\n",
       "  'JT': 'International journal of biological sciences',\n",
       "  'JID': '101235568',\n",
       "  'RN': ['0 (Chromones)',\n",
       "   '0 (ELP4 protein, human)',\n",
       "   '0 (Morpholines)',\n",
       "   '0 (Nerve Tissue Proteins)',\n",
       "   '31M2U1DVID (2-(4-morpholinyl)-8-phenyl-4H-1-benzopyran-4-one)',\n",
       "   'EC 2.3.1.48 (ELP3 protein, human)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)',\n",
       "   'EC 2.7.11.1 (AKT1 protein, human)',\n",
       "   'EC 2.7.11.1 (Proto-Oncogene Proteins c-akt)',\n",
       "   'EC 3.4.24.24 (MMP2 protein, human)',\n",
       "   'EC 3.4.24.24 (Matrix Metalloproteinase 2)',\n",
       "   'EC 3.4.24.35 (MMP9 protein, human)',\n",
       "   'EC 3.4.24.35 (Matrix Metalloproteinase 9)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Animals',\n",
       "   'Carcinoma, Hepatocellular/*metabolism',\n",
       "   'Cell Movement',\n",
       "   'Cell Proliferation',\n",
       "   'Chromones/pharmacology',\n",
       "   'DNA Damage',\n",
       "   'Female',\n",
       "   'Gene Expression Regulation, Neoplastic',\n",
       "   'Gene Silencing',\n",
       "   'Hep G2 Cells',\n",
       "   'Histone Acetyltransferases/*metabolism',\n",
       "   'Humans',\n",
       "   'Liver Neoplasms/*metabolism',\n",
       "   'Matrix Metalloproteinase 2/metabolism',\n",
       "   'Matrix Metalloproteinase 9/metabolism',\n",
       "   'Mice',\n",
       "   'Mice, Nude',\n",
       "   'Morpholines/pharmacology',\n",
       "   'Neoplasm Invasiveness',\n",
       "   'Neoplasm Metastasis',\n",
       "   'Nerve Tissue Proteins/*metabolism',\n",
       "   'Phosphorylation',\n",
       "   'Proto-Oncogene Proteins c-akt/*metabolism',\n",
       "   'Signal Transduction',\n",
       "   'Wound Healing'],\n",
       "  'PMC': 'PMC5968844',\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['*AKT',\n",
       "   '*Elp3',\n",
       "   '*Elp4',\n",
       "   '*hepatocellular carcinoma',\n",
       "   '*invasion',\n",
       "   '*migration'],\n",
       "  'COIS': ['Competing Interests: The authors have declared that no competing interest exists.'],\n",
       "  'EDAT': '2018/05/29 06:00',\n",
       "  'MHDA': '2019/07/16 06:00',\n",
       "  'CRDT': ['2018/05/29 06:00'],\n",
       "  'PHST': ['2017/10/27 00:00 [received]',\n",
       "   '2018/03/01 00:00 [accepted]',\n",
       "   '2018/05/29 06:00 [entrez]',\n",
       "   '2018/05/29 06:00 [pubmed]',\n",
       "   '2019/07/16 06:00 [medline]'],\n",
       "  'AID': ['10.7150/ijbs.23511 [doi]', 'ijbsv14p0518 [pii]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'Int J Biol Sci. 2018 Apr 5;14(5):518-530. doi: 10.7150/ijbs.23511. eCollection 2018.'},\n",
       " {'PMID': '29332244',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20190102',\n",
       "  'LR': '20190102',\n",
       "  'IS': '1420-9071 (Electronic) 1420-682X (Linking)',\n",
       "  'VI': '75',\n",
       "  'IP': '9',\n",
       "  'DP': '2018 May',\n",
       "  'TI': 'Structural insights into the function of Elongator.',\n",
       "  'PG': '1613-1622',\n",
       "  'LID': '10.1007/s00018-018-2747-6 [doi]',\n",
       "  'AB': \"Conserved from yeast to humans, Elongator is a protein complex implicated in multiple processes including transcription regulation, alpha-tubulin acetylation, and tRNA modification, and its defects have been shown to cause human diseases such as familial dysautonomia. Elongator consists of two copies of six core subunits (Elp1, Elp2, Elp3, Elp4, Elp5, and Elp6) that are organized into two subcomplexes: Elp1/2/3 and Elp4/5/6 and form a stable assembly of ~ 850 kDa in size. Although the catalytic subunit of Elongator is Elp3, which contains a radical S-adenosyl-L-methionine (SAM) domain and a putative histone acetyltransferase domain, the Elp4/5/6 subcomplex also possesses ATP-modulated tRNA binding activity. How at the molecular level, Elongator performs its multiple functions and how the different subunits regulate Elongator's activities remains poorly understood. Here, we provide an overview of the proposed functions of Elongator and describe how recent structural studies provide new insights into the mechanism of action of this multifunctional complex.\",\n",
       "  'FAU': ['Dalwadi, Udit', 'Yip, Calvin K'],\n",
       "  'AU': ['Dalwadi U', 'Yip CK'],\n",
       "  'AD': 'Department of Biochemistry and Molecular Biology, The University of British Columbia, Vancouver, BC, V6T 1Z3, Canada. Department of Biochemistry and Molecular Biology, The University of British Columbia, Vancouver, BC, V6T 1Z3, Canada. calvin.yip@ubc.ca.',\n",
       "  'AUID': ['ORCID: http://orcid.org/0000-0003-1779-9501'],\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['FDN-143228/Canadian Institutes of Health Research/Canada',\n",
       "   '418157-2012/Natural Sciences and Engineering Research Council of Canada'],\n",
       "  'PT': ['Journal Article', 'Review'],\n",
       "  'DEP': '20180113',\n",
       "  'PL': 'Switzerland',\n",
       "  'TA': 'Cell Mol Life Sci',\n",
       "  'JT': 'Cellular and molecular life sciences : CMLS',\n",
       "  'JID': '9705402',\n",
       "  'RN': ['0 (Peptide Elongation Factors)',\n",
       "   '0 (Protein Subunits)',\n",
       "   '9014-25-9 (RNA, Transfer)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Animals',\n",
       "   'Histone Acetyltransferases/metabolism',\n",
       "   'Humans',\n",
       "   'Peptide Elongation Factors/*metabolism',\n",
       "   'Protein Binding/physiology',\n",
       "   'Protein Subunits/metabolism',\n",
       "   'RNA, Transfer/metabolism',\n",
       "   'Transcription, Genetic/physiology'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['Electron microscopy',\n",
       "   'Elongator',\n",
       "   'Familial dysautonomia',\n",
       "   'Transcription',\n",
       "   'X-ray crystallography',\n",
       "   'tRNA modification'],\n",
       "  'EDAT': '2018/01/15 06:00',\n",
       "  'MHDA': '2019/01/03 06:00',\n",
       "  'CRDT': ['2018/01/15 06:00'],\n",
       "  'PHST': ['2017/10/03 00:00 [received]',\n",
       "   '2018/01/08 00:00 [accepted]',\n",
       "   '2017/12/09 00:00 [revised]',\n",
       "   '2018/01/15 06:00 [pubmed]',\n",
       "   '2019/01/03 06:00 [medline]',\n",
       "   '2018/01/15 06:00 [entrez]'],\n",
       "  'AID': ['10.1007/s00018-018-2747-6 [doi]',\n",
       "   '10.1007/s00018-018-2747-6 [pii]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Cell Mol Life Sci. 2018 May;75(9):1613-1622. doi: 10.1007/s00018-018-2747-6. Epub 2018 Jan 13.'},\n",
       " {'PMID': '29217025',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20171226',\n",
       "  'LR': '20171226',\n",
       "  'IS': '1715-3360 (Electronic) 0008-4182 (Linking)',\n",
       "  'VI': '52',\n",
       "  'IP': '6',\n",
       "  'DP': '2017 Dec',\n",
       "  'TI': 'Correlation of novel PAX6 gene abnormalities in aniridia and clinical presentation.',\n",
       "  'PG': '570-577',\n",
       "  'LID': 'S0008-4182(16)30943-7 [pii] 10.1016/j.jcjo.2017.04.006 [doi]',\n",
       "  'AB': 'OBJECTIVE: To describe the clinical presentation and genotype of subjects with aniridia with a particular focus on foveal hypoplasia. DESIGN: Prospective cohort study. PARTICIPANTS: Thirty-three Canadian participants with aniridia and of various ethnic backgrounds residing in British Columbia. METHODS: Full ophthalmic examinations and posterior segment spectral domain-optical coherence tomography (SD-OCT) imaging were performed. Foveal hypoplasia was graded independently by 2 staff ophthalmologists. PAX6 sequencing was performed and chromosomal 11p anomalies investigated. Candidate gene and single-nucleotide polymorphism sequencing in genes functionally related to PAX6 were also studied. RESULTS: Best corrected visual acuities in the cohort ranged from 0.0 logMAR to no light perception. Total absence of iris tissue was seen in the majority (42 of 66 eyes). In those in whom SD-OCT was possible, foveal hypoplasia was seen in the majority (45 of 56 eyes, 80%). Molecular genetic defects involving PAX6 were identified in 30 participants (91%), including 4 novel PAX6 mutations (Gly18Val; Ser65ProfsX14; Met337ArgfsX18; Ser321CysfsX34) and 4 novel chromosome 11p deletions inclusive of PAX6 or a known PAX6 regulatory region. CONCLUSIONS: The number of PAX6 mutations associated with aniridia continues to increase. Variable foveal architecture despite nearly identical anterior segment disease in 4 participants with an Ex9 ELP4-Ex4 DCDC1 deletion suggested that molecular cues causing variation in disease in the posterior segment differ from those at play in the anterior segment. Results in 3 patients without identifiable PAX6 mutations and a review of the literature suggest that such cases be described as phenocopies rather than actual cases of the syndrome of aniridia.',\n",
       "  'CI': ['Copyright (c) 2017. Published by Elsevier Inc.'],\n",
       "  'FAU': ['Sannan, Naif S',\n",
       "   'Gregory-Evans, Cheryl Y',\n",
       "   'Lyons, Christopher J',\n",
       "   'Lehman, Anna M',\n",
       "   'Langlois, Sylvie',\n",
       "   'Warner, Simon J',\n",
       "   'Zakrzewski, Helen',\n",
       "   'Gregory-Evans, Kevin'],\n",
       "  'AU': ['Sannan NS',\n",
       "   'Gregory-Evans CY',\n",
       "   'Lyons CJ',\n",
       "   'Lehman AM',\n",
       "   'Langlois S',\n",
       "   'Warner SJ',\n",
       "   'Zakrzewski H',\n",
       "   'Gregory-Evans K'],\n",
       "  'AD': \"Department of Ophthalmology and Visual Sciences, University of British Columbia, Vancouver, B.C. Department of Ophthalmology and Visual Sciences, University of British Columbia, Vancouver, B.C. Department of Ophthalmology and Visual Sciences, University of British Columbia, Vancouver, B.C; Department of Ophthalmology, BC Children's Hospital, Vancouver, B.C. Department of Medical Genetics, University of British Columbia, Vancouver, B.C. Department of Medical Genetics, University of British Columbia, Vancouver, B.C. Department of Ophthalmology and Visual Sciences, University of British Columbia, Vancouver, B.C. Cumming School of Medicine, University of Calgary, Calgary, Alta. Department of Ophthalmology and Visual Sciences, University of British Columbia, Vancouver, B.C. Electronic address: kge30@mail.ubc.ca.\",\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article'],\n",
       "  'DEP': '20170706',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Can J Ophthalmol',\n",
       "  'JT': \"Canadian journal of ophthalmology. Journal canadien d'ophtalmologie\",\n",
       "  'JID': '0045312',\n",
       "  'RN': ['0 (PAX6 Transcription Factor)',\n",
       "   '0 (PAX6 protein, human)',\n",
       "   'Chromosome 11, deletion 11p'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Adolescent',\n",
       "   'Adult',\n",
       "   'Aged',\n",
       "   'Aged, 80 and over',\n",
       "   'Aniridia/*diagnosis/*genetics',\n",
       "   'Child',\n",
       "   'Chromosome Deletion',\n",
       "   'Chromosomes, Human, Pair 11/genetics',\n",
       "   'Cohort Studies',\n",
       "   'Female',\n",
       "   'Fovea Centralis/*abnormalities',\n",
       "   'Gene Amplification',\n",
       "   'Humans',\n",
       "   'In Situ Hybridization, Fluorescence',\n",
       "   'Male',\n",
       "   'Middle Aged',\n",
       "   '*Mutation',\n",
       "   'PAX6 Transcription Factor/*genetics',\n",
       "   'Phenotype',\n",
       "   '*Polymorphism, Single Nucleotide',\n",
       "   'Prospective Studies',\n",
       "   'Real-Time Polymerase Chain Reaction',\n",
       "   'Tomography, Optical Coherence',\n",
       "   'Visual Acuity/physiology',\n",
       "   'Young Adult'],\n",
       "  'EDAT': '2017/12/09 06:00',\n",
       "  'MHDA': '2017/12/27 06:00',\n",
       "  'CRDT': ['2017/12/09 06:00'],\n",
       "  'PHST': ['2016/10/27 00:00 [received]',\n",
       "   '2017/03/24 00:00 [revised]',\n",
       "   '2017/04/05 00:00 [accepted]',\n",
       "   '2017/12/09 06:00 [entrez]',\n",
       "   '2017/12/09 06:00 [pubmed]',\n",
       "   '2017/12/27 06:00 [medline]'],\n",
       "  'AID': ['S0008-4182(16)30943-7 [pii]', '10.1016/j.jcjo.2017.04.006 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Can J Ophthalmol. 2017 Dec;52(6):570-577. doi: 10.1016/j.jcjo.2017.04.006. Epub 2017 Jul 6.'},\n",
       " {'PMID': '29191170',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20180522',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1471-2229 (Electronic) 1471-2229 (Linking)',\n",
       "  'VI': '17',\n",
       "  'IP': '1',\n",
       "  'DP': '2017 Dec 1',\n",
       "  'TI': 'The Arabidopsis ELP3/ELO3 and ELP4/ELO1 genes enhance disease resistance in Fragaria vesca L.',\n",
       "  'PG': '230',\n",
       "  'LID': '10.1186/s12870-017-1173-5 [doi]',\n",
       "  'AB': \"BACKGROUND: Plant immune response is associated with a large-scale transcriptional reprogramming, which is regulated by numerous transcription regulators such as the Elongator complex. Elongator is a multitasking protein complex involved in diverse cellular processes, including histone modification, DNA methylation, and tRNA modification. In recent years, Elongator is emerging as a key regulator of plant immune responses. However, characterization of Elongator's function in plant immunity has been conducted only in the model plant Arabidopsis thaliana. It is thus unclear whether Elongator's role in plant immunity is conserved in higher plants. The objective of this study is to characterize transgenic woodland strawberry (Fragaria vesca L.) overexpressing the Arabidopsis Elongator (AtELP) genes, AtELP3 and AtELP4, and to determine whether F. vesca carries a functional Elongator complex. METHODS: Transgenic F. vesca and Arabidopsis plants were produced via Agrobacterium-mediated genetic transformation and characterized by morphology, PCR, real-time quantitative PCR, and disease resistance test. The Student's t test was used to analyze the data. RESULTS: Overexpression of AtELP3 and AtELP4 in F. vesca impacts plant growth and development and confers enhanced resistance to anthracnose crown rot, powdery mildew, and angular leaf spot, which are caused by the hemibiotrophic fungal pathogen Colletotrichum gloeosporioides, the obligate biotrophic fungal pathogen Podosphaera aphanis, and the hemibiotrophic bacterial pathogen Xanthomonas fragariae, respectively. Moreover, the F. vesca genome encodes all six Elongator subunits by single-copy genes with the exception of FvELP4, which is encoded by two homologous genes, FvELP4-1 and FvELP4-2. We show that FvELP4-1 complemented the Arabidopsis Atelp4/elo1-1 mutant, indicating that FvELP4 is biologically functional. CONCLUSIONS: This is the first report on overexpression of Elongator genes in plants. Our results indicate that the function of Elongator in plant immunity is most likely conserved in F. vesca and suggest that Elongator genes may hold potential for helping mitigate disease severity and reduce the use of fungicides in strawberry industry.\",\n",
       "  'FAU': ['Silva, Katchen Julliany P',\n",
       "   'Brunings, Asha M',\n",
       "   'Pereira, Juliana A',\n",
       "   'Peres, Natalia A',\n",
       "   'Folta, Kevin M',\n",
       "   'Mou, Zhonglin'],\n",
       "  'AU': ['Silva KJP',\n",
       "   'Brunings AM',\n",
       "   'Pereira JA',\n",
       "   'Peres NA',\n",
       "   'Folta KM',\n",
       "   'Mou Z'],\n",
       "  'AD': 'Department of Microbiology and Cell Science, University of Florida, Gainesville, FL, 32611, USA. Department of Horticultural Sciences, University of Florida, Gainesville, FL, 32611, USA. Department of Horticultural Sciences, University of Florida, Gainesville, FL, 32611, USA. Department of Microbiology and Cell Science, University of Florida, Gainesville, FL, 32611, USA. Department of Plant Pathology, University of Florida, Gainesville, FL, 32611, USA. Department of Plant Pathology, Gulf Coast Research and Education Center, University of Florida, Wimauma, FL, 33598, USA. Department of Horticultural Sciences, University of Florida, Gainesville, FL, 32611, USA. kfolta@ufl.edu. Department of Microbiology and Cell Science, University of Florida, Gainesville, FL, 32611, USA. zhlmou@ufl.edu.',\n",
       "  'AUID': ['ORCID: http://orcid.org/0000-0003-0243-4905'],\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['BEX 5640/10-5/Brazilian National Council for the Improvement of Higher Education'],\n",
       "  'PT': ['Journal Article'],\n",
       "  'DEP': '20171201',\n",
       "  'PL': 'England',\n",
       "  'TA': 'BMC Plant Biol',\n",
       "  'JT': 'BMC plant biology',\n",
       "  'JID': '100967807',\n",
       "  'RN': ['0 (Arabidopsis Proteins)',\n",
       "   '0 (ELO1 protein, Arabidopsis)',\n",
       "   '0 (ELP protein, Arabidopsis)',\n",
       "   'EC 2.3.1.48 (ELO3 protein, Arabidopsis)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Arabidopsis/*genetics',\n",
       "   'Arabidopsis Proteins/*genetics/physiology',\n",
       "   'Disease Resistance/*genetics',\n",
       "   'Fragaria/*genetics/immunology/microbiology',\n",
       "   'Genes, Plant',\n",
       "   'Histone Acetyltransferases/*genetics',\n",
       "   'Plant Diseases/genetics/*immunology/microbiology',\n",
       "   'Plants, Genetically Modified'],\n",
       "  'PMC': 'PMC5709926',\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['AtELP3',\n",
       "   'AtELP4',\n",
       "   'Disease resistance',\n",
       "   'Fragaria vesca L.',\n",
       "   'FvELP4',\n",
       "   'The Elongator complex',\n",
       "   'Transgenic plants'],\n",
       "  'EDAT': '2017/12/02 06:00',\n",
       "  'MHDA': '2018/05/23 06:00',\n",
       "  'CRDT': ['2017/12/02 06:00'],\n",
       "  'PHST': ['2017/04/17 00:00 [received]',\n",
       "   '2017/11/13 00:00 [accepted]',\n",
       "   '2017/12/02 06:00 [entrez]',\n",
       "   '2017/12/02 06:00 [pubmed]',\n",
       "   '2018/05/23 06:00 [medline]'],\n",
       "  'AID': ['10.1186/s12870-017-1173-5 [doi]',\n",
       "   '10.1186/s12870-017-1173-5 [pii]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'BMC Plant Biol. 2017 Dec 1;17(1):230. doi: 10.1186/s12870-017-1173-5.'},\n",
       " {'PMID': '28598868',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20170901',\n",
       "  'LR': '20181202',\n",
       "  'IS': '1531-7021 (Electronic) 1040-8738 (Linking)',\n",
       "  'VI': '28',\n",
       "  'IP': '5',\n",
       "  'DP': '2017 Sep',\n",
       "  'TI': 'PAX6 aniridia syndrome: clinics, genetics, and therapeutics.',\n",
       "  'PG': '436-447',\n",
       "  'LID': '10.1097/ICU.0000000000000405 [doi]',\n",
       "  'AB': 'PURPOSE OF REVIEW: Aniridia is a rare and panocular disorder affecting most of the ocular structures which may have significant impact on vision. The purpose of this review is to describe the clinical features, genetics, and therapeutic options for this disease and to provide an update of current knowledge and latest research findings. RECENT FINDINGS: Aside from the ocular features, a variety of associated systemic abnormalities, including hormonal, metabolic, gastrointestinal, genitourinary, and neurologic pathologies have been reported in children with aniridia. Although mutations in PAX6 are a major cause of aniridia, genetic defects in nearby genes, such as TRIM44 or ELP4, have also been reported to cause aniridia. Recent improvement in genetic testing technique will help more rapid and precise diagnosis for aniridia. A promising therapeutic approach called nonsense suppression therapy has been introduced and successfully used in an animal model. SUMMARY: Aniridia is a challenging disease. The progressive nature of this condition and its potential complications require continuous and life-long ophthalmologic care. Genetic diagnosis for aniridia is important for establishing definitive molecular characterization as well as identifying individuals at high risk for Wilms tumor. Recent advancement in understanding the genetic pathogenesis of this disease offers promise for the approaches to treatment.',\n",
       "  'FAU': ['Lim, Hyun Taek', 'Kim, Dae Hee', 'Kim, Hyuna'],\n",
       "  'AU': ['Lim HT', 'Kim DH', 'Kim H'],\n",
       "  'AD': \"aDepartment of Ophthalmology, Asan Medical Center, University of Ulsan College of Medicine, Seoul, Korea bDepartment of Ophthalmology, Kim's Eye Hospital, College of Medicine, Konyang University, Nonsan, Korea.\",\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', 'Review'],\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Curr Opin Ophthalmol',\n",
       "  'JT': 'Current opinion in ophthalmology',\n",
       "  'JID': '9011108',\n",
       "  'RN': ['0 (PAX6 Transcription Factor)', '0 (PAX6 protein, human)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['*Aniridia/diagnosis/genetics/therapy',\n",
       "   '*Diagnostic Techniques, Ophthalmological',\n",
       "   '*Disease Management',\n",
       "   'Genetic Testing/methods',\n",
       "   'Humans',\n",
       "   '*Mutation',\n",
       "   'PAX6 Transcription Factor/*genetics/metabolism',\n",
       "   'Syndrome'],\n",
       "  'EDAT': '2017/06/10 06:00',\n",
       "  'MHDA': '2017/09/02 06:00',\n",
       "  'CRDT': ['2017/06/10 06:00'],\n",
       "  'PHST': ['2017/06/10 06:00 [pubmed]',\n",
       "   '2017/09/02 06:00 [medline]',\n",
       "   '2017/06/10 06:00 [entrez]'],\n",
       "  'AID': ['10.1097/ICU.0000000000000405 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Curr Opin Ophthalmol. 2017 Sep;28(5):436-447. doi: 10.1097/ICU.0000000000000405.'},\n",
       " {'PMID': '28474462',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20180312',\n",
       "  'LR': '20190113',\n",
       "  'IS': '1751-7915 (Electronic) 1751-7915 (Linking)',\n",
       "  'VI': '10',\n",
       "  'IP': '4',\n",
       "  'DP': '2017 Jul',\n",
       "  'TI': 'Involvement of BcElp4 in vegetative development, various environmental stress response and virulence of Botrytis cinerea.',\n",
       "  'PG': '886-895',\n",
       "  'LID': '10.1111/1751-7915.12720 [doi]',\n",
       "  'AB': 'The Saccharomyces cerevisiae Elongator complex consisting of the six Elp1-Elp6 proteins has been proposed to participate in three distinct cellular processes: transcriptional elongation, polarized exocytosis and formation of modified wobble uridines in tRNA. In this study, we investigated the function of BcElp4 in Botrytis cinerea, which is homologous to S. cerevisiae Elp4. A bcelp4 deletion mutant was significantly impaired in vegetative growth, sclerotia formation and melanin biosynthesis. This mutant exhibited decreased sensitivity to osmotic and oxidative stresses as well as cell way-damaging agent. Pathogenicity assays revealed that BcElp4 is involved in the virulence of B. cinerea. In addition, the deletion of bcelp4 led to increased aerial mycelia development. All these defects were restored by genetic complementation of the bcelp4 deletion mutant with the wild-type bcelp4 gene. The results of this study indicated that BcElp4 is involved in regulation of vegetative development, various environmental stress response and virulence in B. cinerea.',\n",
       "  'CI': ['(c) 2017 The Authors. Microbial Biotechnology published by John Wiley & Sons Ltd',\n",
       "   'and Society for Applied Microbiology.'],\n",
       "  'FAU': ['Shao, Wenyong',\n",
       "   'Lv, Chiyuan',\n",
       "   'Zhang, Yu',\n",
       "   'Wang, Jin',\n",
       "   'Chen, Changjun'],\n",
       "  'AU': ['Shao W', 'Lv C', 'Zhang Y', 'Wang J', 'Chen C'],\n",
       "  'AD': 'College of Plant Protection, Nanjing Agricultural University, Nanjing, 210095, China. College of Plant Protection, Nanjing Agricultural University, Nanjing, 210095, China. College of Plant Protection, Nanjing Agricultural University, Nanjing, 210095, China. College of Plant Protection, Nanjing Agricultural University, Nanjing, 210095, China. College of Plant Protection, Nanjing Agricultural University, Nanjing, 210095, China.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20170505',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Microb Biotechnol',\n",
       "  'JT': 'Microbial biotechnology',\n",
       "  'JID': '101316335',\n",
       "  'RN': ['0 (Melanins)', '0 (RNA-Binding Proteins)', '0 (Virulence Factors)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Botrytis/genetics/growth & development/*physiology',\n",
       "   'Gene Deletion',\n",
       "   'Genetic Complementation Test',\n",
       "   'Melanins/biosynthesis',\n",
       "   'Mycelium/growth & development',\n",
       "   'Osmotic Pressure',\n",
       "   'Oxidative Stress',\n",
       "   'Plant Diseases/microbiology',\n",
       "   'RNA-Binding Proteins/genetics/*metabolism',\n",
       "   '*Stress, Physiological',\n",
       "   'Virulence',\n",
       "   'Virulence Factors/genetics/*metabolism'],\n",
       "  'PMC': 'PMC5481526',\n",
       "  'EDAT': '2017/05/06 06:00',\n",
       "  'MHDA': '2018/03/13 06:00',\n",
       "  'CRDT': ['2017/05/06 06:00'],\n",
       "  'PHST': ['2017/01/27 00:00 [received]',\n",
       "   '2017/03/29 00:00 [revised]',\n",
       "   '2017/03/30 00:00 [accepted]',\n",
       "   '2017/05/06 06:00 [pubmed]',\n",
       "   '2018/03/13 06:00 [medline]',\n",
       "   '2017/05/06 06:00 [entrez]'],\n",
       "  'AID': ['10.1111/1751-7915.12720 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Microb Biotechnol. 2017 Jul;10(4):886-895. doi: 10.1111/1751-7915.12720. Epub 2017 May 5.'},\n",
       " {'PMID': '28167615',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20180620',\n",
       "  'LR': '20191210',\n",
       "  'IS': '1754-8411 (Electronic) 1754-8403 (Linking)',\n",
       "  'VI': '10',\n",
       "  'IP': '5',\n",
       "  'DP': '2017 May 1',\n",
       "  'TI': 'The familial dysautonomia disease gene IKBKAP is required in the developing and adult mouse central nervous system.',\n",
       "  'PG': '605-618',\n",
       "  'LID': '10.1242/dmm.028258 [doi]',\n",
       "  'AB': 'Hereditary sensory and autonomic neuropathies (HSANs) are a genetically and clinically diverse group of disorders defined by peripheral nervous system (PNS) dysfunction. HSAN type III, known as familial dysautonomia (FD), results from a single base mutation in the gene IKBKAP that encodes a scaffolding unit (ELP1) for a multi-subunit complex known as Elongator. Since mutations in other Elongator subunits (ELP2 to ELP4) are associated with central nervous system (CNS) disorders, the goal of this study was to investigate a potential requirement for Ikbkap in the CNS of mice. The sensory and autonomic pathophysiology of FD is fatal, with the majority of patients dying by age 40. While signs and pathology of FD have been noted in the CNS, the clinical and research focus has been on the sensory and autonomic dysfunction, and no genetic model studies have investigated the requirement for Ikbkap in the CNS. Here, we report, using a novel mouse line in which Ikbkap is deleted solely in the nervous system, that not only is Ikbkap widely expressed in the embryonic and adult CNS, but its deletion perturbs both the development of cortical neurons and their survival in adulthood. Primary cilia in embryonic cortical apical progenitors and motile cilia in adult ependymal cells are reduced in number and disorganized. Furthermore, we report that, in the adult CNS, both autonomic and non-autonomic neuronal populations require Ikbkap for survival, including spinal motor and cortical neurons. In addition, the mice developed kyphoscoliosis, an FD hallmark, indicating its neuropathic etiology. Ultimately, these perturbations manifest in a developmental and progressive neurodegenerative condition that includes impairments in learning and memory. Collectively, these data reveal an essential function for Ikbkap that extends beyond the peripheral nervous system to CNS development and function. With the identification of discrete CNS cell types and structures that depend on Ikbkap, novel strategies to thwart the progressive demise of CNS neurons in FD can be developed.',\n",
       "  'CI': ['(c) 2017. Published by The Company of Biologists Ltd.'],\n",
       "  'FAU': ['Chaverra, Marta',\n",
       "   'George, Lynn',\n",
       "   'Mergy, Marc',\n",
       "   'Waller, Hannah',\n",
       "   'Kujawa, Katharine',\n",
       "   'Murnion, Connor',\n",
       "   'Sharples, Ezekiel',\n",
       "   'Thorne, Julian',\n",
       "   'Podgajny, Nathaniel',\n",
       "   'Grindeland, Andrea',\n",
       "   'Ueki, Yumi',\n",
       "   'Eiger, Steven',\n",
       "   'Cusick, Cassie',\n",
       "   'Babcock, A Michael',\n",
       "   'Carlson, George A',\n",
       "   'Lefcort, Frances'],\n",
       "  'AU': ['Chaverra M',\n",
       "   'George L',\n",
       "   'Mergy M',\n",
       "   'Waller H',\n",
       "   'Kujawa K',\n",
       "   'Murnion C',\n",
       "   'Sharples E',\n",
       "   'Thorne J',\n",
       "   'Podgajny N',\n",
       "   'Grindeland A',\n",
       "   'Ueki Y',\n",
       "   'Eiger S',\n",
       "   'Cusick C',\n",
       "   'Babcock AM',\n",
       "   'Carlson GA',\n",
       "   'Lefcort F'],\n",
       "  'AD': 'Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA. Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA. Department of Biological and Physical Sciences, Montana State University Billings, Billings, MT 59101, USA. Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA. Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA. Department of Psychology, Montana State University, Bozeman, MT 59717, USA. Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA. Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA. Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA. University of Washington, School of Medicine, Seattle, WA 98195, USA. Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA. McLaughlin Research Institute, Great Falls, MT 59405, USA. Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA. Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA. Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA. Department of Psychology, Montana State University, Bozeman, MT 59717, USA. McLaughlin Research Institute, Great Falls, MT 59405, USA. Department of Cell Biology and Neuroscience, Montana State University, Bozeman, MT 59717, USA lefcort@montana.edu.',\n",
       "  'AUID': ['ORCID: 0000-0001-5359-2570',\n",
       "   'ORCID: 0000-0001-7703-8865',\n",
       "   'ORCID: 0000-0002-9334-4894',\n",
       "   'ORCID: 0000-0003-0014-0735',\n",
       "   'ORCID: 0000-0001-9666-372X',\n",
       "   'ORCID: 0000-0002-9494-4111',\n",
       "   'ORCID: 0000-0002-2064-8678'],\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['P01 NS041997/NS/NINDS NIH HHS/United States',\n",
       "   'P20 GM103474/GM/NIGMS NIH HHS/United States',\n",
       "   'R01 NS035714/NS/NINDS NIH HHS/United States',\n",
       "   'R01 NS086796/NS/NINDS NIH HHS/United States'],\n",
       "  'PT': ['Journal Article'],\n",
       "  'DEP': '20170206',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Dis Model Mech',\n",
       "  'JT': 'Disease models & mechanisms',\n",
       "  'JID': '101483332',\n",
       "  'RN': ['0 (Carrier Proteins)',\n",
       "   '0 (Ikbkap protein, mouse)',\n",
       "   '0 (Intracellular Signaling Peptides and Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Animals',\n",
       "   'Behavior, Animal',\n",
       "   'Carrier Proteins/*genetics',\n",
       "   'Cell Survival/genetics',\n",
       "   'Central Nervous System/growth & development/*metabolism/pathology',\n",
       "   'Dysautonomia, Familial/*genetics',\n",
       "   'Intracellular Signaling Peptides and Proteins',\n",
       "   'Mice',\n",
       "   'Mice, Knockout',\n",
       "   'Mutation',\n",
       "   'Neurons/pathology'],\n",
       "  'PMC': 'PMC5451171',\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['*Elongator',\n",
       "   '*Familial dysautonomia',\n",
       "   '*Hereditary sensory and autonomic neuropathy'],\n",
       "  'COIS': ['Competing interestsThe authors declare no competing or financial interests.'],\n",
       "  'EDAT': '2017/02/09 06:00',\n",
       "  'MHDA': '2018/06/21 06:00',\n",
       "  'CRDT': ['2017/02/08 06:00'],\n",
       "  'PHST': ['2016/10/26 00:00 [received]',\n",
       "   '2017/01/23 00:00 [accepted]',\n",
       "   '2017/02/09 06:00 [pubmed]',\n",
       "   '2018/06/21 06:00 [medline]',\n",
       "   '2017/02/08 06:00 [entrez]'],\n",
       "  'AID': ['dmm.028258 [pii]', '10.1242/dmm.028258 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Dis Model Mech. 2017 May 1;10(5):605-618. doi: 10.1242/dmm.028258. Epub 2017 Feb 6.'},\n",
       " {'PMID': '27919838',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20171113',\n",
       "  'LR': '20180131',\n",
       "  'IS': '1096-1194 (Electronic) 0890-8508 (Linking)',\n",
       "  'VI': '32',\n",
       "  'DP': '2017 Apr',\n",
       "  'TI': 'A CGH array procedure to detect PAX6 gene structural defects.',\n",
       "  'PG': '65-68',\n",
       "  'LID': 'S0890-8508(16)30117-7 [pii] 10.1016/j.mcp.2016.12.001 [doi]',\n",
       "  'AB': 'Aniridia is a rare congenital disease characterized by eye development defects, in which the more evident clinical manifestation is iris absence or malformation. In most of the patients, aniridia is associated to PAX6 gene point mutations or deletions. When these deletions are large and involve other genes, a more complex disease, named WAGR syndrome, arises. In order to develop a new tool to analyze aniridia and WAGR subjects, a CGH array (CGHa) of the PAX6 genomic region was set up. We generated a custom microarray kit using an oligonucleotide-based platform that allows high resolution molecular profiling of genomic aberrations in 20 Mb of the 11p13 chromosomal region, centered on the PAX6 gene. The average probe spacing was 100 bp. Thirty-five subjects have been analyzed. The major advantage of CGHa compared to MLPA was the knowledge of the deletions borders. Our approach identifies patients harboring deletions including the WT1 gene and, therefore, at risk for kidney tumors. The CGHa assay confirmed that several aniridia patients show a deletion at the level of ELP4 gene, without involvement of the PAX6 exonic regions. In all these patients, deletions include the PAX6 transcriptional enhancer SIMO. This finding further highlights the role of mutation/deletion of long-range enhancers in monogenic human pathology.',\n",
       "  'CI': ['Copyright (c) 2016 Elsevier Ltd. All rights reserved.'],\n",
       "  'FAU': ['Franzoni, Alessandra',\n",
       "   'Russo, Patrizia Dello',\n",
       "   'Baldan, Federica',\n",
       "   \"D'Elia, Angela Valentina\",\n",
       "   'Puppin, Cinzia',\n",
       "   'Penco, Silvana',\n",
       "   'Damante, Giuseppe'],\n",
       "  'AU': ['Franzoni A',\n",
       "   'Russo PD',\n",
       "   'Baldan F',\n",
       "   \"D'Elia AV\",\n",
       "   'Puppin C',\n",
       "   'Penco S',\n",
       "   'Damante G'],\n",
       "  'AD': \"Istituto di Genetica Medica, Azienda Ospedaliero-Universitaria di Udine, Italy. Istituto di Genetica Medica, Azienda Ospedaliero-Universitaria di Udine, Italy. Dipartimento di Scienze Mediche e Biologiche, Universita di Udine, Italy. Istituto di Genetica Medica, Azienda Ospedaliero-Universitaria di Udine, Italy. Dipartimento di Scienze Mediche e Biologiche, Universita di Udine, Italy. Genetica Medica, Dipartimento di Medicina di Laboratorio, Ospedale Niguarda Ca' Granda, Milano, Italy. Istituto di Genetica Medica, Azienda Ospedaliero-Universitaria di Udine, Italy; Dipartimento di Scienze Mediche e Biologiche, Universita di Udine, Italy. Electronic address: giuseppe.damante@uniud.it.\",\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20161202',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Mol Cell Probes',\n",
       "  'JT': 'Molecular and cellular probes',\n",
       "  'JID': '8709751',\n",
       "  'RN': ['0 (PAX6 Transcription Factor)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Comparative Genomic Hybridization/*methods',\n",
       "   'Humans',\n",
       "   'PAX6 Transcription Factor/*genetics',\n",
       "   'Sequence Deletion'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['*Aniridia', '*CGH array', '*PAX6'],\n",
       "  'EDAT': '2016/12/07 06:00',\n",
       "  'MHDA': '2017/11/14 06:00',\n",
       "  'CRDT': ['2016/12/07 06:00'],\n",
       "  'PHST': ['2016/10/07 00:00 [received]',\n",
       "   '2016/12/01 00:00 [revised]',\n",
       "   '2016/12/01 00:00 [accepted]',\n",
       "   '2016/12/07 06:00 [pubmed]',\n",
       "   '2017/11/14 06:00 [medline]',\n",
       "   '2016/12/07 06:00 [entrez]'],\n",
       "  'AID': ['S0890-8508(16)30117-7 [pii]', '10.1016/j.mcp.2016.12.001 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Mol Cell Probes. 2017 Apr;32:65-68. doi: 10.1016/j.mcp.2016.12.001. Epub 2016 Dec 2.'},\n",
       " {'PMID': '27386500',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'PubMed-not-MEDLINE',\n",
       "  'DCOM': '20160707',\n",
       "  'LR': '20181113',\n",
       "  'IS': '2328-9503 (Print) 2328-9503 (Linking)',\n",
       "  'VI': '3',\n",
       "  'IP': '7',\n",
       "  'DP': '2016 Jul',\n",
       "  'TI': 'A microRNA-328 binding site in PAX6 is associated with centrotemporal spikes of rolandic epilepsy.',\n",
       "  'PG': '512-22',\n",
       "  'LID': '10.1002/acn3.320 [doi]',\n",
       "  'AB': \"OBJECTIVE: Rolandic epilepsy is a common genetic focal epilepsy of childhood characterized by centrotemporal sharp waves on electroencephalogram. In previous genome-wide analysis, we had reported linkage of centrotemporal sharp waves to chromosome 11p13, and fine mapping with 44 SNPs identified the ELP4-PAX6 locus in two independent US and Canadian case-control samples. Here, we aimed to find a causative variant for centrotemporal sharp waves using a larger sample and higher resolution genotyping array. METHODS: We fine-mapped the ELP4-PAX6 locus in 186 individuals from rolandic epilepsy families and 1000 population controls of European origin using the Illumina HumanCoreExome-12 v1.0 BeadChip. Controls were matched to cases on ethnicity using principal component analysis. We used generalized estimating equations to assess association, followed up with a bioinformatics survey and literature search to evaluate functional significance. RESULTS: Homozygosity at the T allele of SNP rs662702 in the 3' untranslated region of PAX6 conferred increased risk of CTS: Odds ratio = 12.29 (95% CI: 3.20-47.22), P = 2.6 x 10(-4) and is seen in 3.9% of cases but only 0.3% of controls. INTERPRETATION: The minor T allele of SNP rs662702 disrupts regulation by microRNA-328, which is known to result in increased PAX6 expression in vitro. This study provides, for the first time, evidence of a noncoding genomic variant contributing to the etiology of a common human epilepsy via a posttranscriptional regulatory mechanism.\",\n",
       "  'FAU': ['Panjwani, Naim',\n",
       "   'Wilson, Michael D',\n",
       "   'Addis, Laura',\n",
       "   'Crosbie, Jennifer',\n",
       "   'Wirrell, Elaine',\n",
       "   'Auvin, Stephane',\n",
       "   'Caraballo, Roberto H',\n",
       "   'Kinali, Maria',\n",
       "   'McCormick, David',\n",
       "   'Oren, Caroline',\n",
       "   'Taylor, Jacqueline',\n",
       "   'Trounce, John',\n",
       "   'Clarke, Tara',\n",
       "   'Akman, Cigdem I',\n",
       "   'Kugler, Steven L',\n",
       "   'Mandelbaum, David E',\n",
       "   'McGoldrick, Patricia',\n",
       "   'Wolf, Steven M',\n",
       "   'Arnold, Paul',\n",
       "   'Schachar, Russell',\n",
       "   'Pal, Deb K',\n",
       "   'Strug, Lisa J'],\n",
       "  'AU': ['Panjwani N',\n",
       "   'Wilson MD',\n",
       "   'Addis L',\n",
       "   'Crosbie J',\n",
       "   'Wirrell E',\n",
       "   'Auvin S',\n",
       "   'Caraballo RH',\n",
       "   'Kinali M',\n",
       "   'McCormick D',\n",
       "   'Oren C',\n",
       "   'Taylor J',\n",
       "   'Trounce J',\n",
       "   'Clarke T',\n",
       "   'Akman CI',\n",
       "   'Kugler SL',\n",
       "   'Mandelbaum DE',\n",
       "   'McGoldrick P',\n",
       "   'Wolf SM',\n",
       "   'Arnold P',\n",
       "   'Schachar R',\n",
       "   'Pal DK',\n",
       "   'Strug LJ'],\n",
       "  'AD': 'Program in Genetics and Genome Biology The Hospital for Sick Children Toronto Ontario M5G 0A4 Canada. Program in Genetics and Genome Biology The Hospital for Sick Children Toronto Ontario M5G 0A4 Canada; Department of Molecular Genetics University of Toronto Toronto Ontario M5S 1A1 Canada. Department of Basic and Clinical Neuroscience Institute of Psychiatry, Psychology and Neuroscience King\\'s College London London SE5 9RX United Kingdom; Neuroscience Discovery Research Eli Lilly and Company Erl Wood, Surrey GU20 6PH United Kingdom. Neurosciences and Mental Health Program Research Institute The Hospital for Sick Children Toronto Ontario M5G 0A4 Canada; Department of Psychiatry The Hospital for Sick Children Toronto Ontario M5G 0A4 Canada. Division of Child and Adolescent Neurology Mayo Clinic Rochester Minnesota 55905. Service de neurologie pediatrique/Inserm 1141 Hopital Robert Debre AP-HP, 48 boulevard Serurier Paris 75019 France. Department of Neurology Hospital de Pediatria \"Prof Dr Juan P Garrahan\" Combate de los Pozos 1881 C1245AAM Buenos Aires Argentina. Chelsea and Westminster Hospital London SW10 9NH United Kingdom. King\\'s College Hospital London SE5 9RS United Kingdom. Northwick Park Hospital Middlesex HA1 3UJ United Kingdom. Barnet and Chase Farm Hospitals Enfield, Greater London EN2 8JL United Kingdom. Brighton and Sussex University Hospitals Brighton BN1 6AG United Kingdom. Department of Epidemiology Columbia University New York New York 10027. Neurological Institute Columbia University Medical Centre New York, New York 10032. Children\\'s Hospital of Philadelphia and University of Pennsylvania School of Medicine Philadelphia Pennsylvania 19104. Hasbro Children\\'s Hospital and the Warren Alpert Medical School of Brown University Providence Rhode Island 02903. Beth Israel Medical Center New York, New York 10003. Beth Israel Medical Center New York, New York 10003. Neurosciences and Mental Health Program Research Institute The Hospital for Sick Children Toronto Ontario M5G 0A4 Canada; Department of Psychiatry The Hospital for Sick Children Toronto Ontario M5G 0A4 Canada; Mathison Centre for Mental Health Research and Education University of Calgary Calgary Alberta T2N 4Z6 Canada. Neurosciences and Mental Health Program Research Institute The Hospital for Sick Children Toronto Ontario M5G 0A4 Canada; Department of Psychiatry The Hospital for Sick Children Toronto Ontario M5G 0A4 Canada. Department of Basic and Clinical Neuroscience Institute of Psychiatry, Psychology and Neuroscience King\\'s College London London SE5 9RX United Kingdom; King\\'s College Hospital London SE5 9RS United Kingdom; Evelina London Children\\'s Hospita lLondon SE1 7EH United Kingdom. Program in Genetics and Genome Biology The Hospital for Sick Children Toronto Ontario M5G 0A4 Canada; Division of Biostatistics Dalla Lana School of Public Health University of Toronto Toronto Ontario M5T 3M7 Canada; The Centre for Applied Genomics The Hospital for Sick Children Toronto Ontario M5G 0A4 Canada.',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['R01 NS047530/NS/NINDS NIH HHS/United States'],\n",
       "  'PT': ['Journal Article'],\n",
       "  'DEP': '20160602',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Ann Clin Transl Neurol',\n",
       "  'JT': 'Annals of clinical and translational neurology',\n",
       "  'JID': '101623278',\n",
       "  'PMC': 'PMC4931716',\n",
       "  'EDAT': '2016/07/08 06:00',\n",
       "  'MHDA': '2016/07/08 06:01',\n",
       "  'CRDT': ['2016/07/08 06:00'],\n",
       "  'PHST': ['2016/04/21 00:00 [received]',\n",
       "   '2016/04/28 00:00 [accepted]',\n",
       "   '2016/07/08 06:00 [entrez]',\n",
       "   '2016/07/08 06:00 [pubmed]',\n",
       "   '2016/07/08 06:01 [medline]'],\n",
       "  'AID': ['10.1002/acn3.320 [doi]', 'ACN3320 [pii]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'Ann Clin Transl Neurol. 2016 Jun 2;3(7):512-22. doi: 10.1002/acn3.320. eCollection 2016 Jul.'},\n",
       " {'PMID': '27381094',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20180129',\n",
       "  'LR': '20190115',\n",
       "  'IS': '1476-5438 (Electronic) 1018-4813 (Linking)',\n",
       "  'VI': '24',\n",
       "  'IP': '11',\n",
       "  'DP': '2016 Nov',\n",
       "  'TI': 'Clinical utility gene card for: Aniridia.',\n",
       "  'LID': '10.1038/ejhg.2016.73 [doi]',\n",
       "  'FAU': ['Richardson, Rose',\n",
       "   'Hingorani, Melanie',\n",
       "   'Van Heyningen, Veronica',\n",
       "   'Gregory-Evans, Cheryl',\n",
       "   'Moosajee, Mariya'],\n",
       "  'AU': ['Richardson R',\n",
       "   'Hingorani M',\n",
       "   'Van Heyningen V',\n",
       "   'Gregory-Evans C',\n",
       "   'Moosajee M'],\n",
       "  'AD': 'Department of Ocular Biology and Therapeutics, UCL Institute of Ophthalmology, London, UK. Department of Genetics, Moorfields Eye Hospital NHS Foundation Trust, London, UK. Department of Ocular Biology and Therapeutics, UCL Institute of Ophthalmology, London, UK. Department of Ophthalmology and Visual Sciences, University of British Columbia, Vancouver, British Columbia, Canada. Department of Ocular Biology and Therapeutics, UCL Institute of Ophthalmology, London, UK. Department of Genetics, Moorfields Eye Hospital NHS Foundation Trust, London, UK. Department of Ophthalmology, Great Ormond Street Hospital, London, UK.',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['MC_PC_U127527199/Medical Research Council/United Kingdom',\n",
       "   'MC_U127527199/Medical Research Council/United Kingdom'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20160706',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Eur J Hum Genet',\n",
       "  'JT': 'European journal of human genetics : EJHG',\n",
       "  'JID': '9302235',\n",
       "  'RN': ['0 (ELP4 protein, human)',\n",
       "   '0 (Nerve Tissue Proteins)',\n",
       "   '0 (PAX6 Transcription Factor)',\n",
       "   '0 (PAX6 protein, human)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Aniridia/*diagnosis/*genetics',\n",
       "   'Cost-Benefit Analysis',\n",
       "   'Genetic Testing',\n",
       "   'Genotype',\n",
       "   'Humans',\n",
       "   'Mutation',\n",
       "   'Nerve Tissue Proteins/genetics',\n",
       "   'PAX6 Transcription Factor/genetics',\n",
       "   'Pathology, Molecular',\n",
       "   'Risk Assessment',\n",
       "   'Sensitivity and Specificity'],\n",
       "  'PMC': 'PMC5110069',\n",
       "  'EDAT': '2016/07/07 06:00',\n",
       "  'MHDA': '2018/01/30 06:00',\n",
       "  'CRDT': ['2016/07/07 06:00'],\n",
       "  'PHST': ['2016/01/18 00:00 [received]',\n",
       "   '2016/04/28 00:00 [revised]',\n",
       "   '2016/05/26 00:00 [accepted]',\n",
       "   '2016/07/07 06:00 [pubmed]',\n",
       "   '2018/01/30 06:00 [medline]',\n",
       "   '2016/07/07 06:00 [entrez]'],\n",
       "  'AID': ['ejhg201673 [pii]', '10.1038/ejhg.2016.73 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Eur J Hum Genet. 2016 Nov;24(11). pii: ejhg201673. doi: 10.1038/ejhg.2016.73. Epub 2016 Jul 6.'},\n",
       " {'PMID': '26449484',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20160519',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1756-994X (Electronic) 1756-994X (Linking)',\n",
       "  'VI': '7',\n",
       "  'DP': '2015 Oct 8',\n",
       "  'TI': 'Many obesity-associated SNPs strongly associate with DNA methylation changes at proximal promoters and enhancers.',\n",
       "  'PG': '103',\n",
       "  'LID': '10.1186/s13073-015-0225-4 [doi]',\n",
       "  'AB': 'BACKGROUND: The mechanisms by which genetic variants, such as single nucleotide polymorphisms (SNPs), identified in genome-wide association studies act to influence body mass remain unknown for most of these SNPs, which continue to puzzle the scientific community. Recent evidence points to the epigenetic and chromatin states of the genome as having important roles. METHODS: We genotyped 355 healthy young individuals for 52 known obesity-associated SNPs and obtained DNA methylation levels in their blood using the Illumina 450 K BeadChip. Associations between alleles and methylation at proximal cytosine residues were tested using a linear model adjusted for age, sex, weight category, and a proxy for blood cell type counts. For replication in other tissues, we used two open-access datasets (skin fibroblasts, n = 62; four brain regions, n = 121-133) and an additional dataset in subcutaneous and visceral fat (n = 149). RESULTS: We found that alleles at 28 of these obesity-associated SNPs associate with methylation levels at 107 proximal CpG sites. Out of 107 CpG sites, 38 are located in gene promoters, including genes strongly implicated in obesity (MIR148A, BDNF, PTPMT1, NR1H3, MGAT1, SCGB3A1, HOXC12, PMAIP1, PSIP1, RPS10-NUDT3, RPS10, SKOR1, MAP2K5, SIX5, AGRN, IMMP1L, ELP4, ITIH4, SEMA3G, POMC, ADCY3, SSPN, LGR4, TUFM, MIR4721, SULT1A1, SULT1A2, APOBR, CLN3, SPNS1, SH2B1, ATXN2L, and IL27). Interestingly, the associated SNPs are in known eQTLs for some of these genes. We also found that the 107 CpGs are enriched in enhancers in peripheral blood mononuclear cells. Finally, our results indicate that some of these associations are not blood-specific as we successfully replicated four associations in skin fibroblasts. CONCLUSIONS: Our results strongly suggest that many obesity-associated SNPs are associated with proximal gene regulation, which was reflected by association of obesity risk allele genotypes with differential DNA methylation. This study highlights the importance of DNA methylation and other chromatin marks as a way to understand the molecular basis of genetic variants associated with human diseases and traits.',\n",
       "  'FAU': ['Voisin, Sarah',\n",
       "   'Almen, Markus Sallman',\n",
       "   'Zheleznyakova, Galina Y',\n",
       "   'Lundberg, Lina',\n",
       "   'Zarei, Sanaz',\n",
       "   'Castillo, Sandra',\n",
       "   'Eriksson, Fia Ence',\n",
       "   'Nilsson, Emil K',\n",
       "   'Bluher, Matthias',\n",
       "   'Bottcher, Yvonne',\n",
       "   'Kovacs, Peter',\n",
       "   'Klovins, Janis',\n",
       "   'Rask-Andersen, Mathias',\n",
       "   'Schioth, Helgi B'],\n",
       "  'AU': ['Voisin S',\n",
       "   'Almen MS',\n",
       "   'Zheleznyakova GY',\n",
       "   'Lundberg L',\n",
       "   'Zarei S',\n",
       "   'Castillo S',\n",
       "   'Eriksson FE',\n",
       "   'Nilsson EK',\n",
       "   'Bluher M',\n",
       "   'Bottcher Y',\n",
       "   'Kovacs P',\n",
       "   'Klovins J',\n",
       "   'Rask-Andersen M',\n",
       "   'Schioth HB'],\n",
       "  'AD': 'Department of Neuroscience, Functional Pharmacology, Uppsala University, Uppsala, Sweden. sarah.voisin.aeris@gmail.com. Department of Neuroscience, Functional Pharmacology, Uppsala University, Uppsala, Sweden. markus.almen@imbim.uu.se. Department of Medical Biochemistry and Microbiology, Uppsala University, SE-751 23, Uppsala, Sweden. markus.almen@imbim.uu.se. Department of Neuroscience, Functional Pharmacology, Uppsala University, Uppsala, Sweden. galina.zheleznyakova@neuro.uu.se. Department of Neuroscience, Functional Pharmacology, Uppsala University, Uppsala, Sweden. lina.llundberg@gmail.com. Department of Neuroscience, Functional Pharmacology, Uppsala University, Uppsala, Sweden. sanaz.zr@gmail.com. Department of Neuroscience, Functional Pharmacology, Uppsala University, Uppsala, Sweden. sandra.r.castillo@hotmail.se. Department of Neuroscience, Functional Pharmacology, Uppsala University, Uppsala, Sweden. fia.e@hotmail.com. Department of Neuroscience, Functional Pharmacology, Uppsala University, Uppsala, Sweden. emil.nilsson@neuro.uu.se. Medical Faculty, IFB Adiposity Diseases, University of Leipzig, Liebigstrasse 21, 04103, Leipzig, Germany. Matthias.Blueher@medizin.uni-leipzig.de. Medical Faculty, IFB Adiposity Diseases, University of Leipzig, Liebigstrasse 21, 04103, Leipzig, Germany. Yvonne.Boettcher@medizin.uni-leipzig.de. Medical Faculty, IFB Adiposity Diseases, University of Leipzig, Liebigstrasse 21, 04103, Leipzig, Germany. peter.kovacs@medizin.uni-leipzig.de. Latvian Biomedical Research and Study Center, Ratsupites 1, Riga, LV-1067, Latvia. klovins@biomed.lu.lv. Department of Neuroscience, Functional Pharmacology, Uppsala University, Uppsala, Sweden. mathias.rask-andersen@neuro.uu.se. Department of Neuroscience, Functional Pharmacology, Uppsala University, Uppsala, Sweden. Helgi.Schioth@neuro.uu.se.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20151008',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Genome Med',\n",
       "  'JT': 'Genome medicine',\n",
       "  'JID': '101475844',\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Adipose Tissue/metabolism',\n",
       "   'Adolescent',\n",
       "   'Adult',\n",
       "   'Aged',\n",
       "   'Alleles',\n",
       "   'Brain/metabolism',\n",
       "   'CpG Islands',\n",
       "   '*DNA Methylation',\n",
       "   'Enhancer Elements, Genetic',\n",
       "   'Female',\n",
       "   'Fibroblasts/metabolism',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   'Middle Aged',\n",
       "   'Obesity/blood/*genetics',\n",
       "   'Polymorphism, Single Nucleotide',\n",
       "   'Promoter Regions, Genetic',\n",
       "   'Young Adult'],\n",
       "  'PMC': 'PMC4599317',\n",
       "  'EDAT': '2015/10/10 06:00',\n",
       "  'MHDA': '2016/05/20 06:00',\n",
       "  'CRDT': ['2015/10/10 06:00'],\n",
       "  'PHST': ['2015/07/22 00:00 [received]',\n",
       "   '2015/09/21 00:00 [accepted]',\n",
       "   '2015/10/10 06:00 [entrez]',\n",
       "   '2015/10/10 06:00 [pubmed]',\n",
       "   '2016/05/20 06:00 [medline]'],\n",
       "  'AID': ['10.1186/s13073-015-0225-4 [doi]',\n",
       "   '10.1186/s13073-015-0225-4 [pii]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'Genome Med. 2015 Oct 8;7:103. doi: 10.1186/s13073-015-0225-4.'},\n",
       " {'PMID': '26419218',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20161019',\n",
       "  'LR': '20170103',\n",
       "  'IS': '1552-4833 (Electronic) 1552-4825 (Linking)',\n",
       "  'VI': '170A',\n",
       "  'IP': '1',\n",
       "  'DP': '2016 Jan',\n",
       "  'TI': 'A familial pericentric inversion of chromosome 11 associated with a microdeletion of 163 kb and microduplication of 288 kb at 11p13 and 11q22.3 without aniridia or eye anomalies.',\n",
       "  'PG': '202-9',\n",
       "  'LID': '10.1002/ajmg.a.37388 [doi]',\n",
       "  'AB': \"Interstitial deletions of 11p13 involving MPPED2, DCDC5, DCDC1, DNAJC24, IMMP1L, and ELP4 are previously reported to have downstream transcriptional effects on the expression of PAX6, due to a downstream regulatory region (DRR). Currently, no clear genotype-phenotype correlations have been established allowing for conclusive information regarding the exact location of the PAX6 DRR, though its location has been approximated in mouse models to be within the Elp4 gene. Of the clinical reports currently published examining patients with intact PAX6 genes but harboring deletions identified in genes downstream of PAX6, 100% indicate phenotypes which include aniridia, whereas approximately half report additional eye deformities, autism, or intellectual disability. In this clinical report, we present a 12-year-old male patient, his brother, and mother with pericentric inversions of chromosome 11 associated with submicroscopic interstitial deletions of 11p13 and duplications of 11q22.3. The inversions were identified by standard cytogenetic analysis; microarray and FISH detected the chromosomal imbalance. The patient's phenotype includes intellectual disability, speech abnormalities, and autistic behaviors, but interestingly neither the patient, his brother, nor mother have aniridia or other eye anomalies. To the best of our knowledge, these findings in three family members represent the only reported cases with 11p13 deletions downstream of PAX6 not demonstrating phenotypic characteristics of aniridia or abnormal eye development. Although none of the deleted genes are obvious candidates for the patient's phenotype, the absence of aniridia in the presence of this deletion in all three family members further delineates the location of the DRR for PAX6.\",\n",
       "  'CI': ['(c) 2015 Wiley Periodicals, Inc.'],\n",
       "  'FAU': ['Balay, Lara',\n",
       "   'Totten, Ellen',\n",
       "   'Okada, Luna',\n",
       "   'Zell, Sidney',\n",
       "   'Ticho, Benjamin',\n",
       "   'Israel, Jeannette',\n",
       "   'Kogan, Jillene'],\n",
       "  'AU': ['Balay L',\n",
       "   'Totten E',\n",
       "   'Okada L',\n",
       "   'Zell S',\n",
       "   'Ticho B',\n",
       "   'Israel J',\n",
       "   'Kogan J'],\n",
       "  'AD': \"Center for Genetic Medicine, Northwestern University, Chicago, Illinois. Advocate Medical Group Genetics, Park Ridge, Illinois. St. Alexius Medical Center, Hoffman Estates, Illinois. Advocate Children's Hospital, Oak Lawn, Illinois. Advocate Children's Hospital, Oak Lawn, Illinois. Advocate Children's Hospital, Oak Lawn, Illinois. Advocate Medical Group Genetics, Park Ridge, Illinois. Department of Cytogenetics, ACL Laboratories, Rosemont, Illinois.\",\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Case Reports', 'Journal Article'],\n",
       "  'DEP': '20150930',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Am J Med Genet A',\n",
       "  'JT': 'American journal of medical genetics. Part A',\n",
       "  'JID': '101235741',\n",
       "  'RN': ['0 (Eye Proteins)',\n",
       "   '0 (Homeodomain Proteins)',\n",
       "   '0 (PAX6 Transcription Factor)',\n",
       "   '0 (PAX6 protein, human)',\n",
       "   '0 (Paired Box Transcription Factors)',\n",
       "   '0 (Pax6 protein, mouse)',\n",
       "   '0 (Repressor Proteins)',\n",
       "   'Chromosome 11p, partial deletion'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Aniridia/*genetics',\n",
       "   'Autistic Disorder/genetics',\n",
       "   'Child',\n",
       "   'Chromosome Deletion',\n",
       "   'Chromosome Inversion/*genetics',\n",
       "   'Chromosomes, Human, Pair 11/genetics',\n",
       "   'Eye Proteins/*genetics',\n",
       "   'Gene Duplication/*genetics',\n",
       "   'Homeodomain Proteins/*genetics',\n",
       "   'Humans',\n",
       "   'Intellectual Disability/genetics',\n",
       "   'Male',\n",
       "   'PAX6 Transcription Factor',\n",
       "   'Paired Box Transcription Factors/*genetics',\n",
       "   'Pedigree',\n",
       "   'Phenotype',\n",
       "   'Repressor Proteins/*genetics',\n",
       "   'Sequence Deletion/*genetics',\n",
       "   'Speech Disorders/genetics'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['11p13 deletion',\n",
       "   '11q22.3 duplication',\n",
       "   'PAX6',\n",
       "   'PAX6 downstream regulatory region',\n",
       "   'aniridia'],\n",
       "  'EDAT': '2015/10/01 06:00',\n",
       "  'MHDA': '2016/11/27 06:00',\n",
       "  'CRDT': ['2015/10/01 06:00'],\n",
       "  'PHST': ['2013/10/22 00:00 [received]',\n",
       "   '2015/08/31 00:00 [accepted]',\n",
       "   '2015/10/01 06:00 [entrez]',\n",
       "   '2015/10/01 06:00 [pubmed]',\n",
       "   '2016/11/27 06:00 [medline]'],\n",
       "  'AID': ['10.1002/ajmg.a.37388 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Am J Med Genet A. 2016 Jan;170A(1):202-9. doi: 10.1002/ajmg.a.37388. Epub 2015 Sep 30.'},\n",
       " {'PMID': '26010655',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20160510',\n",
       "  'LR': '20150814',\n",
       "  'IS': '1098-1004 (Electronic) 1059-7794 (Linking)',\n",
       "  'VI': '36',\n",
       "  'IP': '9',\n",
       "  'DP': '2015 Sep',\n",
       "  'TI': 'Microdeletions of ELP4 Are Associated with Language Impairment, Autism Spectrum Disorder, and Mental Retardation.',\n",
       "  'PG': '842-50',\n",
       "  'LID': '10.1002/humu.22816 [doi]',\n",
       "  'AB': 'Copy-number variations (CNVs) are important in the aetiology of neurodevelopmental disorders and show broad phenotypic manifestations. We compared the presence of small CNVs disrupting the ELP4-PAX6 locus in 4,092 UK individuals with a range of neurodevelopmental conditions, clinically referred for array comparative genomic hybridization, with WTCCC controls (n = 4,783). The phenotypic analysis was then extended using the DECIPHER database. We followed up association using an autism patient cohort (n = 3,143) compared with six additional control groups (n = 6,469). In the clinical discovery series, we identified eight cases with ELP4 deletions, and one with a partial duplication of ELP4 and PAX6. These cases were referred for neurological phenotypes including language impairment, developmental delay, autism, and epilepsy. Six further cases with a primary diagnosis of autism spectrum disorder (ASD) and similar secondary phenotypes were identified with ELP4 deletions, as well as another six (out of nine) with neurodevelopmental phenotypes from DECIPHER. CNVs at ELP4 were only present in 1/11,252 controls. We found a significant excess of CNVs in discovery cases compared with controls, P = 7.5 x 10(-3) , as well as for autism, P = 2.7 x 10(-3) . Our results suggest that ELP4 deletions are highly likely to be pathogenic, predisposing to a range of neurodevelopmental phenotypes from ASD to language impairment and epilepsy.',\n",
       "  'CI': ['(c) 2015 WILEY PERIODICALS, INC.'],\n",
       "  'FAU': ['Addis, Laura',\n",
       "   'Ahn, Joo Wook',\n",
       "   'Dobson, Richard',\n",
       "   'Dixit, Abhishek',\n",
       "   'Ogilvie, Caroline M',\n",
       "   'Pinto, Dalila',\n",
       "   'Vaags, Andrea K',\n",
       "   'Coon, Hilary',\n",
       "   'Chaste, Pauline',\n",
       "   'Wilson, Scott',\n",
       "   'Parr, Jeremy R',\n",
       "   'Andrieux, Joris',\n",
       "   'Lenne, Bruno',\n",
       "   'Tumer, Zeynep',\n",
       "   'Leuzzi, Vincenzo',\n",
       "   'Aubell, Kristina',\n",
       "   'Koillinen, Hannele',\n",
       "   'Curran, Sarah',\n",
       "   'Marshall, Christian R',\n",
       "   'Scherer, Stephen W',\n",
       "   'Strug, Lisa J',\n",
       "   'Collier, David A',\n",
       "   'Pal, Deb K'],\n",
       "  'AU': ['Addis L',\n",
       "   'Ahn JW',\n",
       "   'Dobson R',\n",
       "   'Dixit A',\n",
       "   'Ogilvie CM',\n",
       "   'Pinto D',\n",
       "   'Vaags AK',\n",
       "   'Coon H',\n",
       "   'Chaste P',\n",
       "   'Wilson S',\n",
       "   'Parr JR',\n",
       "   'Andrieux J',\n",
       "   'Lenne B',\n",
       "   'Tumer Z',\n",
       "   'Leuzzi V',\n",
       "   'Aubell K',\n",
       "   'Koillinen H',\n",
       "   'Curran S',\n",
       "   'Marshall CR',\n",
       "   'Scherer SW',\n",
       "   'Strug LJ',\n",
       "   'Collier DA',\n",
       "   'Pal DK'],\n",
       "  'AD': \"Department of Basic and Clinical Neuroscience, Institute of Psychiatry, Psychology and Neuroscience, King's College London, London, UK. Neuroscience Discovery Research, Eli Lilly and Company, Erl Wood, Surrey, UK. Department of Cytogenetics, Guy's and St Thomas' NHS Foundation Trust, London, UK. Department of Biostatistics and NIHR BRC for Mental Health, Institute of Psychiatry, Psychology and Neuroscience, King's College London, London, UK. Department of Biostatistics and NIHR BRC for Mental Health, Institute of Psychiatry, Psychology and Neuroscience, King's College London, London, UK. Department of Cytogenetics, Guy's and St Thomas' NHS Foundation Trust, London, UK. Departments of Psychiatry, and Genetics and Genomic Sciences, Seaver Autism Center, The Mindich Child Health & Development Institute, Icahn School of Medicine at Mount Sinai, New York, New York. The Centre for Applied Genomics, The Hospital for Sick Children, Toronto, Ontario, Canada. Department of Psychiatry, University of Utah School of Medicine, Salt Lake City, Utah. Department of Psychiatry, University of Pittsburgh School of Medicine, Pittsburgh, Pennsylvania. Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, Psychology and Neuroscience, King's College London, London, UK. Department of Endocrinology and Diabetes, Sir Charles Gairdner Hospital, Perth, Western Australia, Australia. School of Medicine and Pharmacology, University of Western Australia, Nedlands, Western Australia, Australia. Institute of Neuroscience, Newcastle University, Newcastle upon Tyne, UK. Institut de Genetique Medicale, Hopital Jeanne de Flandre, CHRU de Lille, France. Centre de Genetique Chromosomique, GHICL, Hopital Saint Vincent de Paul, Lille, France. Applied Human Molecular Genetics, Kennedy Center, Department of Clinical Genetics, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark. Department of Pediatrics, Child Neurology and Psychiatry, Sapienza Universita di Roma, Rome, Italy. Institute of Human Genetics, Medical University of Graz, Graz, Austria. Department of Clinical Genetics, Helsinki University Hospital, Helsinki, Finland. Department of Cytogenetics, Guy's and St Thomas' NHS Foundation Trust, London, UK. The Centre for Applied Genomics, The Hospital for Sick Children, Toronto, Ontario, Canada. Department of Molecular Genetics, Paediatric Laboratory Medicine, The Hospital for Sick Children, Toronto, Ontario, Canada. The Centre for Applied Genomics, The Hospital for Sick Children, Toronto, Ontario, Canada. McLaughlin Centre and Department of Molecular Genetics, University of Toronto, Toronto, Ontario, Canada. Program in Genetics and Genome Biology, The Hospital for Sick Children, Division of Biostatistics, Dalla Lana School of Public Health, University of Toronto, Toronto, Ontario, Canada. Neuroscience Discovery Research, Eli Lilly and Company, Erl Wood, Surrey, UK. Social, Genetic and Developmental Psychiatry Centre, Institute of Psychiatry, Psychology and Neuroscience, King's College London, London, UK. Department of Basic and Clinical Neuroscience, Institute of Psychiatry, Psychology and Neuroscience, King's College London, London, UK.\",\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20150630',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Hum Mutat',\n",
       "  'JT': 'Human mutation',\n",
       "  'JID': '9215429',\n",
       "  'RN': ['0 (ELP4 protein, human)', '0 (Nerve Tissue Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Adolescent',\n",
       "   'Adult',\n",
       "   'Autism Spectrum Disorder/*genetics',\n",
       "   'Case-Control Studies',\n",
       "   'Child',\n",
       "   'Child, Preschool',\n",
       "   'Comparative Genomic Hybridization',\n",
       "   'DNA Copy Number Variations',\n",
       "   'Databases, Genetic',\n",
       "   'Datasets as Topic',\n",
       "   'Female',\n",
       "   '*Genetic Association Studies',\n",
       "   'Humans',\n",
       "   'Infant',\n",
       "   'Inheritance Patterns',\n",
       "   'Intellectual Disability/*genetics',\n",
       "   'Language Disorders/*genetics',\n",
       "   'Male',\n",
       "   'Nerve Tissue Proteins/*genetics',\n",
       "   'Phenotype',\n",
       "   '*Sequence Deletion',\n",
       "   'Young Adult'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['CNV',\n",
       "   'copy number variation',\n",
       "   'developmental',\n",
       "   'epilepsy and seizures',\n",
       "   'neurology'],\n",
       "  'EDAT': '2015/05/27 06:00',\n",
       "  'MHDA': '2016/05/11 06:00',\n",
       "  'CRDT': ['2015/05/27 06:00'],\n",
       "  'PHST': ['2015/03/06 00:00 [received]',\n",
       "   '2015/05/15 00:00 [accepted]',\n",
       "   '2015/05/27 06:00 [entrez]',\n",
       "   '2015/05/27 06:00 [pubmed]',\n",
       "   '2016/05/11 06:00 [medline]'],\n",
       "  'AID': ['10.1002/humu.22816 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Hum Mutat. 2015 Sep;36(9):842-50. doi: 10.1002/humu.22816. Epub 2015 Jun 30.'},\n",
       " {'PMID': '25631615',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20150707',\n",
       "  'LR': '20181202',\n",
       "  'IS': '1098-2272 (Electronic) 0741-0395 (Linking)',\n",
       "  'VI': '39',\n",
       "  'IP': '3',\n",
       "  'DP': '2015 Mar',\n",
       "  'TI': 'Meta-analysis of Genome-Wide Association Studies Identifies Novel Loci Associated With Optic Disc Morphology.',\n",
       "  'PG': '207-16',\n",
       "  'LID': '10.1002/gepi.21886 [doi]',\n",
       "  'AB': 'Primary open-angle glaucoma is the most common optic neuropathy and an important cause of irreversible blindness worldwide. The optic nerve head or optic disc is divided in two parts: a central cup (without nerve fibers) surrounded by the neuroretinal rim (containing axons of the retinal ganglion cells). The International Glaucoma Genetics Consortium conducted a meta-analysis of genome-wide association studies consisting of 17,248 individuals of European ancestry and 6,841 individuals of Asian ancestry. The outcomes of the genome-wide association studies were disc area and cup area. These specific measurements describe optic nerve morphology in another way than the vertical cup-disc ratio, which is a clinically used measurement, and may shed light on new glaucoma mechanisms. We identified 10 new loci associated with disc area (CDC42BPA, F5, DIRC3, RARB, ABI3BP, DCAF4L2, ELP4, TMTC2, NR2F2, and HORMAD2) and another 10 new loci associated with cup area (DHRS3, TRIB2, EFEMP1, FLNB, FAM101, DDHD1, ASB7, KPNB1, BCAS3, and TRIOBP). The new genes participate in a number of pathways and future work is likely to identify more functions related to the pathogenesis of glaucoma.',\n",
       "  'CI': ['(c) 2015 Wiley Periodicals, Inc.'],\n",
       "  'FAU': ['Springelkamp, Henriet',\n",
       "   'Mishra, Aniket',\n",
       "   'Hysi, Pirro G',\n",
       "   'Gharahkhani, Puya',\n",
       "   'Hohn, Rene',\n",
       "   'Khor, Chiea-Chuen',\n",
       "   'Cooke Bailey, Jessica N',\n",
       "   'Luo, Xiaoyan',\n",
       "   'Ramdas, Wishal D',\n",
       "   'Vithana, Eranga',\n",
       "   'Koh, Victor',\n",
       "   'Yazar, Seyhan',\n",
       "   'Xu, Liang',\n",
       "   'Forward, Hannah',\n",
       "   'Kearns, Lisa S',\n",
       "   'Amin, Najaf',\n",
       "   'Iglesias, Adriana I',\n",
       "   'Sim, Kar-Seng',\n",
       "   'van Leeuwen, Elisabeth M',\n",
       "   'Demirkan, Ayse',\n",
       "   'van der Lee, Sven',\n",
       "   'Loon, Seng-Chee',\n",
       "   'Rivadeneira, Fernando',\n",
       "   'Nag, Abhishek',\n",
       "   'Sanfilippo, Paul G',\n",
       "   'Schillert, Arne',\n",
       "   'de Jong, Paulus T V M',\n",
       "   'Oostra, Ben A',\n",
       "   'Uitterlinden, Andre G',\n",
       "   'Hofman, Albert',\n",
       "   'Zhou, Tiger',\n",
       "   'Burdon, Kathryn P',\n",
       "   'Spector, Timothy D',\n",
       "   'Lackner, Karl J',\n",
       "   'Saw, Seang-Mei',\n",
       "   'Vingerling, Johannes R',\n",
       "   'Teo, Yik-Ying',\n",
       "   'Pasquale, Louis R',\n",
       "   'Wolfs, Roger C W',\n",
       "   'Lemij, Hans G',\n",
       "   'Tai, E-Shyong',\n",
       "   'Jonas, Jost B',\n",
       "   'Cheng, Ching-Yu',\n",
       "   'Aung, Tin',\n",
       "   'Jansonius, Nomdo M',\n",
       "   'Klaver, Caroline C W',\n",
       "   'Craig, Jamie E',\n",
       "   'Young, Terri L',\n",
       "   'Haines, Jonathan L',\n",
       "   'MacGregor, Stuart',\n",
       "   'Mackey, David A',\n",
       "   'Pfeiffer, Norbert',\n",
       "   'Wong, Tien-Yin',\n",
       "   'Wiggs, Janey L',\n",
       "   'Hewitt, Alex W',\n",
       "   'van Duijn, Cornelia M',\n",
       "   'Hammond, Christopher J'],\n",
       "  'AU': ['Springelkamp H',\n",
       "   'Mishra A',\n",
       "   'Hysi PG',\n",
       "   'Gharahkhani P',\n",
       "   'Hohn R',\n",
       "   'Khor CC',\n",
       "   'Cooke Bailey JN',\n",
       "   'Luo X',\n",
       "   'Ramdas WD',\n",
       "   'Vithana E',\n",
       "   'Koh V',\n",
       "   'Yazar S',\n",
       "   'Xu L',\n",
       "   'Forward H',\n",
       "   'Kearns LS',\n",
       "   'Amin N',\n",
       "   'Iglesias AI',\n",
       "   'Sim KS',\n",
       "   'van Leeuwen EM',\n",
       "   'Demirkan A',\n",
       "   'van der Lee S',\n",
       "   'Loon SC',\n",
       "   'Rivadeneira F',\n",
       "   'Nag A',\n",
       "   'Sanfilippo PG',\n",
       "   'Schillert A',\n",
       "   'de Jong PT',\n",
       "   'Oostra BA',\n",
       "   'Uitterlinden AG',\n",
       "   'Hofman A',\n",
       "   'Zhou T',\n",
       "   'Burdon KP',\n",
       "   'Spector TD',\n",
       "   'Lackner KJ',\n",
       "   'Saw SM',\n",
       "   'Vingerling JR',\n",
       "   'Teo YY',\n",
       "   'Pasquale LR',\n",
       "   'Wolfs RC',\n",
       "   'Lemij HG',\n",
       "   'Tai ES',\n",
       "   'Jonas JB',\n",
       "   'Cheng CY',\n",
       "   'Aung T',\n",
       "   'Jansonius NM',\n",
       "   'Klaver CC',\n",
       "   'Craig JE',\n",
       "   'Young TL',\n",
       "   'Haines JL',\n",
       "   'MacGregor S',\n",
       "   'Mackey DA',\n",
       "   'Pfeiffer N',\n",
       "   'Wong TY',\n",
       "   'Wiggs JL',\n",
       "   'Hewitt AW',\n",
       "   'van Duijn CM',\n",
       "   'Hammond CJ'],\n",
       "  'AD': 'Department of Epidemiology, Erasmus Medical Center, Rotterdam, the Netherlands; Department of Ophthalmology, Erasmus Medical Center, Rotterdam, the Netherlands.',\n",
       "  'CN': ['NEIGHBORHOOD Consortium'],\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['R01 EY004859/EY/NEI NIH HHS/United States',\n",
       "   'R01 EY013178/EY/NEI NIH HHS/United States',\n",
       "   'R01 EY022305/EY/NEI NIH HHS/United States',\n",
       "   'R01 EY015473/EY/NEI NIH HHS/United States',\n",
       "   'R01 EY011405/EY/NEI NIH HHS/United States',\n",
       "   'R01 EY022306/EY/NEI NIH HHS/United States',\n",
       "   'P30 EY014104/EY/NEI NIH HHS/United States',\n",
       "   'SRF/01/010/Department of Health/United Kingdom',\n",
       "   'R01 EY023754/EY/NEI NIH HHS/United States'],\n",
       "  'PT': ['Journal Article', 'Meta-Analysis'],\n",
       "  'DEP': '20150128',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Genet Epidemiol',\n",
       "  'JT': 'Genetic epidemiology',\n",
       "  'JID': '8411723',\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Asian Continental Ancestry Group/genetics',\n",
       "   'European Continental Ancestry Group/genetics',\n",
       "   '*Genome-Wide Association Study',\n",
       "   'Glaucoma/ethnology/*genetics/pathology',\n",
       "   'Humans',\n",
       "   'Optic Disk/*pathology',\n",
       "   'Optic Nerve Diseases/ethnology/*genetics/pathology',\n",
       "   'Quantitative Trait Loci/*genetics'],\n",
       "  'PMC': 'PMC4480365',\n",
       "  'MID': ['NIHMS700469'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['GWAS', 'cup area', 'disc area', 'glaucoma'],\n",
       "  'IR': ['Allingham RR',\n",
       "   'Brilliant MH',\n",
       "   'Budenz DL',\n",
       "   'Cooke Bailey JN',\n",
       "   'Christen WG',\n",
       "   'Fingert J',\n",
       "   'Gaasterland D',\n",
       "   'Gaasterland T',\n",
       "   'Haines JL',\n",
       "   'Hauser MA',\n",
       "   'Kang JH',\n",
       "   'Kraft P',\n",
       "   'Lee RK',\n",
       "   'Lichter PA',\n",
       "   'Liu YT',\n",
       "   'Loomis SJ',\n",
       "   'Moroi SE',\n",
       "   'Pasquale LR',\n",
       "   'Pericak-Vance MA',\n",
       "   'Realini A',\n",
       "   'Richards JE',\n",
       "   'Schuman JS',\n",
       "   'Scott WK',\n",
       "   'Singh K',\n",
       "   'Sit AJ',\n",
       "   'Vollrath D',\n",
       "   'Weinreb RN',\n",
       "   'Wiggs JL',\n",
       "   'Wollstein G',\n",
       "   'Zack DJ',\n",
       "   'Zhang K'],\n",
       "  'FIR': ['Allingham, R Rand',\n",
       "   'Brilliant, Murray H',\n",
       "   'Budenz, Donald L',\n",
       "   'Cooke Bailey, Jessica N',\n",
       "   'Christen, William G',\n",
       "   'Fingert, John',\n",
       "   'Gaasterland, Douglas',\n",
       "   'Gaasterland, Terry',\n",
       "   'Haines, Jonathan L',\n",
       "   'Hauser, Michael A',\n",
       "   'Kang, Jae H',\n",
       "   'Kraft, Peter',\n",
       "   'Lee, Richard K',\n",
       "   'Lichter, Paul A',\n",
       "   'Liu, Yu-tao',\n",
       "   'Loomis, Stephanie J',\n",
       "   'Moroi, Sayoko E',\n",
       "   'Pasquale, Louis R',\n",
       "   'Pericak-Vance, Margaret A',\n",
       "   'Realini, Anthony',\n",
       "   'Richards, Julia E',\n",
       "   'Schuman, Joel S',\n",
       "   'Scott, William K',\n",
       "   'Singh, Kuldev',\n",
       "   'Sit, Arthur J',\n",
       "   'Vollrath, Douglas',\n",
       "   'Weinreb, Robert N',\n",
       "   'Wiggs, Janey L',\n",
       "   'Wollstein, Gadi',\n",
       "   'Zack, Donald J',\n",
       "   'Zhang, Kang'],\n",
       "  'EDAT': '2015/01/30 06:00',\n",
       "  'MHDA': '2015/07/08 06:00',\n",
       "  'CRDT': ['2015/01/30 06:00'],\n",
       "  'PHST': ['2014/11/07 00:00 [received]',\n",
       "   '2014/12/05 00:00 [revised]',\n",
       "   '2014/12/11 00:00 [accepted]',\n",
       "   '2015/01/30 06:00 [entrez]',\n",
       "   '2015/01/30 06:00 [pubmed]',\n",
       "   '2015/07/08 06:00 [medline]'],\n",
       "  'AID': ['10.1002/gepi.21886 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Genet Epidemiol. 2015 Mar;39(3):207-16. doi: 10.1002/gepi.21886. Epub 2015 Jan 28.'},\n",
       " {'PMID': '25628759',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'PubMed-not-MEDLINE',\n",
       "  'DCOM': '20150128',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1755-8166 (Print) 1755-8166 (Linking)',\n",
       "  'VI': '8',\n",
       "  'IP': '1',\n",
       "  'DP': '2015',\n",
       "  'TI': 'A novel 11p13 microdeletion encompassing PAX6 in a Chinese Han family with aniridia, ptosis and mental retardation.',\n",
       "  'PG': '3',\n",
       "  'LID': '10.1186/s13039-015-0110-2 [doi]',\n",
       "  'AB': 'PURPOSE: To explore possible genetic aberrations in a Chinese family with aniridia, ptosis and mental retardation, and provide genetic evidence for the prenatal diagnosis. METHODS: 14 exons of PAX6 in the proband were sequenced by the Sanger sequencing technique. Multiplex ligation-dependent probe amplification (MLPA) technique was employed to further explore gene alterations of PAX6. Single nucleotide polymorphisms-array (SNP-array) assay was applied to screen potential pathologic genome-wide copy number variations (CNV). RESULTS: There were no detectable pathogenic mutations in the 14 exons of PAX6 in the proband. MLPA indicated a heterozygous deletion encompassing all PAX6 gene regions covered and a partial upstream region. SNP-array assay detected a heterozygous 11p13 microdeletion with a length of 518 kb in the proband, spanning two whole annotated genes, elongation factor protein 4 (ELP4), the paired box gene 6 (PAX6), and partial IMP1 inner-mitochondrial membrane (IMMP1L) gene. SNP-array revealed her affected brother carried the identical deletion. CONCLUSIONS: The 518 kb heterozygous deletion in 11p13 encompassing PAX6 should be the genetic etiology for the familial aniridia.',\n",
       "  'FAU': ['Hu, Ping',\n",
       "   'Meng, Lulu',\n",
       "   'Ma, Dingyuan',\n",
       "   'Qiao, Fengchang',\n",
       "   'Wang, Yan',\n",
       "   'Zhou, Jing',\n",
       "   'Yi, Long',\n",
       "   'Xu, Zhengfeng'],\n",
       "  'AU': ['Hu P',\n",
       "   'Meng L',\n",
       "   'Ma D',\n",
       "   'Qiao F',\n",
       "   'Wang Y',\n",
       "   'Zhou J',\n",
       "   'Yi L',\n",
       "   'Xu Z'],\n",
       "  'AD': 'State key Laboratory of Reproductive Medicine, Department of Prenatal Diagnosis, Nanjing Maternity and Child Health Care Hospital Affiliated to Nanjing Medical University, 123# Tianfei Street, Baixia District Nanjing, 210029 China. State key Laboratory of Reproductive Medicine, Department of Prenatal Diagnosis, Nanjing Maternity and Child Health Care Hospital Affiliated to Nanjing Medical University, 123# Tianfei Street, Baixia District Nanjing, 210029 China. State key Laboratory of Reproductive Medicine, Department of Prenatal Diagnosis, Nanjing Maternity and Child Health Care Hospital Affiliated to Nanjing Medical University, 123# Tianfei Street, Baixia District Nanjing, 210029 China. State key Laboratory of Reproductive Medicine, Department of Prenatal Diagnosis, Nanjing Maternity and Child Health Care Hospital Affiliated to Nanjing Medical University, 123# Tianfei Street, Baixia District Nanjing, 210029 China. State key Laboratory of Reproductive Medicine, Department of Prenatal Diagnosis, Nanjing Maternity and Child Health Care Hospital Affiliated to Nanjing Medical University, 123# Tianfei Street, Baixia District Nanjing, 210029 China. State key Laboratory of Reproductive Medicine, Department of Prenatal Diagnosis, Nanjing Maternity and Child Health Care Hospital Affiliated to Nanjing Medical University, 123# Tianfei Street, Baixia District Nanjing, 210029 China. Department of Pathology, Nanjing University Medical School, Nanjing, China. State key Laboratory of Reproductive Medicine, Department of Prenatal Diagnosis, Nanjing Maternity and Child Health Care Hospital Affiliated to Nanjing Medical University, 123# Tianfei Street, Baixia District Nanjing, 210029 China.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article'],\n",
       "  'DEP': '20150122',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Mol Cytogenet',\n",
       "  'JT': 'Molecular cytogenetics',\n",
       "  'JID': '101317942',\n",
       "  'PMC': 'PMC4307215',\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['11p13 deletion',\n",
       "   'Aniridia',\n",
       "   'Mental retardation',\n",
       "   'PAX6',\n",
       "   'SNP-array'],\n",
       "  'EDAT': '2015/01/30 06:00',\n",
       "  'MHDA': '2015/01/30 06:01',\n",
       "  'CRDT': ['2015/01/29 06:00'],\n",
       "  'PHST': ['2014/09/01 00:00 [received]',\n",
       "   '2015/01/09 00:00 [accepted]',\n",
       "   '2015/01/29 06:00 [entrez]',\n",
       "   '2015/01/30 06:00 [pubmed]',\n",
       "   '2015/01/30 06:01 [medline]'],\n",
       "  'AID': ['10.1186/s13039-015-0110-2 [doi]', '110 [pii]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'Mol Cytogenet. 2015 Jan 22;8(1):3. doi: 10.1186/s13039-015-0110-2. eCollection 2015.'},\n",
       " {'PMID': '25462850',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20150916',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1878-7568 (Electronic) 1742-7061 (Linking)',\n",
       "  'VI': '13',\n",
       "  'DP': '2015 Feb',\n",
       "  'TI': 'Electrospun fiber constructs for vocal fold tissue engineering: effects of alignment and elastomeric polypeptide coating.',\n",
       "  'PG': '111-20',\n",
       "  'LID': '10.1016/j.actbio.2014.10.039 [doi] S1742-7061(14)00490-5 [pii]',\n",
       "  'AB': \"Vocal fold lamina propria extracellular matrix (ECM) is highly aligned and when injured, becomes disorganized with loss of the tissue's critical biomechanical properties. This study examines the effects of electrospun fiber scaffold architecture and elastin-like polypeptide (ELP4) coating on human vocal fold fibroblast (HVFF) behavior for applications toward tissue engineering the vocal fold lamina propria. Electrospun Tecoflex scaffolds were made with aligned and unaligned fibers, and were characterized using scanning electron microscopy and uniaxial tensile testing. ELP4 was successfully adsorbed onto the scaffolds; HVFFs were seeded and their viability, proliferation, morphology and gene expression were characterized. Aligned and unaligned scaffolds had initial elastic moduli of approximately 14 MPa, approximately 5 MPa and approximately 0.3 MPa, approximately 0.6 MPa in the preferred and cross-preferred directions, respectively. Scaffold topography had an effect on the orientation of the cells, with HVFFs seeded on aligned scaffolds having a significantly different (p<0.001) angle of orientation than HVFFs cultured on unaligned scaffolds. This same effect and significant difference (p<0.001) was seen on aligned and unaligned scaffolds coated with ELP4. Scaffold alignment and ELP4 coating impacted ECM gene expression. ELP4 coating, and aligned scaffolds upregulated elastin synthesis when tested on day 7 without a concomitant upregulation of collagen III synthesis. Collectively, results indicate that aligned electrospun scaffolds and ELP4 coating are promising candidates in the development of biodegradeable vocal fold lamina propria constructs.\",\n",
       "  'CI': ['Copyright (c) 2014 Acta Materialia Inc. Published by Elsevier Ltd. All rights',\n",
       "   'reserved.'],\n",
       "  'FAU': ['Hughes, Lindsay A',\n",
       "   'Gaston, Joel',\n",
       "   'McAlindon, Katherine',\n",
       "   'Woodhouse, Kimberly A',\n",
       "   'Thibeault, Susan L'],\n",
       "  'AU': ['Hughes LA',\n",
       "   'Gaston J',\n",
       "   'McAlindon K',\n",
       "   'Woodhouse KA',\n",
       "   'Thibeault SL'],\n",
       "  'AD': \"Department of Chemical Engineering, Queen's University, 201 Dupuis Hall, 19 Division Street, Kingston, ON K7L 3N6, Canada. Department of Surgery and Biomedical Engineering, University of Wisconsin-Madison, 5118 WIMR, 1111 Highland Ave, Madison, WI 53705, USA. Department of Chemical Engineering, Queen's University, 201 Dupuis Hall, 19 Division Street, Kingston, ON K7L 3N6, Canada. Department of Chemical Engineering, Queen's University, 201 Dupuis Hall, 19 Division Street, Kingston, ON K7L 3N6, Canada. Departments of Surgery, Biomedical Engineering and Communication Sciences and Disorders, University of Wisconsin-Madison, 5107 WIMR, 1111 Highland Ave, Madison, WI 53705, USA. Electronic address: thibeault@surgery.wisc.edu.\",\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['R01 DC004336/DC/NIDCD NIH HHS/United States',\n",
       "   'R01 DC013508/DC/NIDCD NIH HHS/United States',\n",
       "   'R01 4336/PHS HHS/United States'],\n",
       "  'PT': ['Journal Article',\n",
       "   'Research Support, N.I.H., Extramural',\n",
       "   \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20141104',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Acta Biomater',\n",
       "  'JT': 'Acta biomaterialia',\n",
       "  'JID': '101233144',\n",
       "  'RN': ['0 (ELP4 protein, human)', '0 (Nerve Tissue Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Cell Line, Transformed',\n",
       "   '*Cell Proliferation',\n",
       "   'Cell Survival',\n",
       "   'Extracellular Matrix/*chemistry',\n",
       "   'Fibroblasts/cytology/*metabolism',\n",
       "   'Humans',\n",
       "   'Materials Testing',\n",
       "   'Mucous Membrane',\n",
       "   'Nerve Tissue Proteins',\n",
       "   '*Tissue Engineering',\n",
       "   'Tissue Scaffolds/*chemistry',\n",
       "   '*Vocal Cords'],\n",
       "  'PMC': 'PMC4293333',\n",
       "  'MID': ['NIHMS640429'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['Elastin',\n",
       "   'Electrospinning',\n",
       "   'Fibroblast alignment',\n",
       "   'Gene expression',\n",
       "   'Vocal folds'],\n",
       "  'EDAT': '2014/12/03 06:00',\n",
       "  'MHDA': '2015/09/17 06:00',\n",
       "  'CRDT': ['2014/12/03 06:00'],\n",
       "  'PHST': ['2014/07/28 00:00 [received]',\n",
       "   '2014/10/03 00:00 [revised]',\n",
       "   '2014/10/28 00:00 [accepted]',\n",
       "   '2014/12/03 06:00 [entrez]',\n",
       "   '2014/12/03 06:00 [pubmed]',\n",
       "   '2015/09/17 06:00 [medline]'],\n",
       "  'AID': ['S1742-7061(14)00490-5 [pii]', '10.1016/j.actbio.2014.10.039 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Acta Biomater. 2015 Feb;13:111-20. doi: 10.1016/j.actbio.2014.10.039. Epub 2014 Nov 4.'},\n",
       " {'PMID': '25301525',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20150804',\n",
       "  'LR': '20180816',\n",
       "  'IS': '1872-6844 (Electronic) 0920-1211 (Linking)',\n",
       "  'VI': '108',\n",
       "  'IP': '10',\n",
       "  'DP': '2014 Dec',\n",
       "  'TI': 'Association of brain-derived neurotrophic factor (BDNF) and elongator protein complex 4 (ELP4) polymorphisms with benign epilepsy with centrotemporal spikes in a Greek population.',\n",
       "  'PG': '1734-9',\n",
       "  'LID': '10.1016/j.eplepsyres.2014.09.005 [doi] S0920-1211(14)00238-1 [pii]',\n",
       "  'AB': 'PURPOSE: Benign epilepsy with centrotemporal spikes (BECTS) is considered to be the most common childhood epileptic syndrome. Different mutations in genes that control the excitability of neurons have been described. Recent reports on the involvement of the BDNF and ELP4 genes in cell motility, migration, and adhesion raise the possibility that these genes are involved in pathogenesis of BECTS. MATERIALS AND METHODS: We conducted a case-control association study on 60 patients with BECTS and 60 control participants to assess the influence of the BDNF and ELP4 polymorphisms on BECTS. The polymorphisms were detected with a PCR-RFLP method. Moreover, we explored the possible association of these polymorphisms with clinical and electroencephalographic parameters of patients with BECTS. RESULTS: Our results show no difference in BDNF and ELP4 genotype frequencies between patients and controls. Haplotype analysis also revealed no statistical difference. CONCLUSION: The role of BDNF and ELP4 polymorphisms remains controversial.',\n",
       "  'CI': ['Copyright (c) 2014 Elsevier B.V. All rights reserved.'],\n",
       "  'FAU': ['Gkampeta, Anastasia',\n",
       "   'Fidani, Liana',\n",
       "   'Clarimon, Jordi',\n",
       "   'Kalinderi, Kallirhoe',\n",
       "   'Katopodi, Teodora',\n",
       "   'Zafeiriou, Dimitrios',\n",
       "   'Pavlou, Evangelos'],\n",
       "  'AU': ['Gkampeta A',\n",
       "   'Fidani L',\n",
       "   'Clarimon J',\n",
       "   'Kalinderi K',\n",
       "   'Katopodi T',\n",
       "   'Zafeiriou D',\n",
       "   'Pavlou E'],\n",
       "  'AD': \"2nd Department of Pediatrics, Aristotle University of Thessaloniki, 'AHEPA' General Hospital, Thessaloniki, Greece. Electronic address: anastagab@yahoo.gr. Department of General Biology, Medical School, Aristotle University of Thessaloniki, Thessaloniki, Greece. Memory Unit, Alzheimer's Laboratory, and Centro de Investigacion Biomedica en Red (CIBERNED), Neurology Department, Sant Pau Hospital, Barcelona, Spain. Department of General Biology, Medical School, Aristotle University of Thessaloniki, Thessaloniki, Greece. Department of General Biology, Medical School, Aristotle University of Thessaloniki, Thessaloniki, Greece. 1st Department of Pediatrics, Aristotle University of Thessaloniki, Thessaloniki, Greece. 2nd Department of Pediatrics, Aristotle University of Thessaloniki, 'AHEPA' General Hospital, Thessaloniki, Greece.\",\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article'],\n",
       "  'DEP': '20140922',\n",
       "  'PL': 'Netherlands',\n",
       "  'TA': 'Epilepsy Res',\n",
       "  'JT': 'Epilepsy research',\n",
       "  'JID': '8703089',\n",
       "  'RN': ['0 (Brain-Derived Neurotrophic Factor)',\n",
       "   '0 (ELP4 protein, human)',\n",
       "   '0 (Nerve Tissue Proteins)',\n",
       "   '7171WSG8A2 (BDNF protein, human)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Brain-Derived Neurotrophic Factor/*genetics',\n",
       "   'Case-Control Studies',\n",
       "   'Child',\n",
       "   'Child, Preschool',\n",
       "   'Electroencephalography',\n",
       "   'Epilepsy, Rolandic/*genetics/physiopathology',\n",
       "   'Female',\n",
       "   'Gene Frequency',\n",
       "   'Genetic Association Studies',\n",
       "   '*Genetic Predisposition to Disease',\n",
       "   'Greece',\n",
       "   'Haplotypes',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   'Nerve Tissue Proteins/*genetics',\n",
       "   '*Polymorphism, Single Nucleotide'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['BDNF gene',\n",
       "   'BECTS',\n",
       "   'ELP4 gene',\n",
       "   'Val66Met polymorphism',\n",
       "   'rs11031434 polymorphism',\n",
       "   'rs964112 polymorphism'],\n",
       "  'EDAT': '2014/10/11 06:00',\n",
       "  'MHDA': '2015/08/05 06:00',\n",
       "  'CRDT': ['2014/10/11 06:00'],\n",
       "  'PHST': ['2014/04/29 00:00 [received]',\n",
       "   '2014/07/02 00:00 [revised]',\n",
       "   '2014/09/02 00:00 [accepted]',\n",
       "   '2014/10/11 06:00 [entrez]',\n",
       "   '2014/10/11 06:00 [pubmed]',\n",
       "   '2015/08/05 06:00 [medline]'],\n",
       "  'AID': ['S0920-1211(14)00238-1 [pii]',\n",
       "   '10.1016/j.eplepsyres.2014.09.005 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Epilepsy Res. 2014 Dec;108(10):1734-9. doi: 10.1016/j.eplepsyres.2014.09.005. Epub 2014 Sep 22.'},\n",
       " {'PMID': '25194491',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20150406',\n",
       "  'LR': '20161019',\n",
       "  'IS': '1875-7855 (Electronic) 0079-6123 (Linking)',\n",
       "  'VI': '213',\n",
       "  'DP': '2014',\n",
       "  'TI': 'Remind me again what disease we are studying? A population genetics, genetic analysis, and real data perspective on why progress on identifying genetic influences on common epilepsies has been so slow.',\n",
       "  'PG': '199-221',\n",
       "  'LID': '10.1016/B978-0-444-63326-2.00011-9 [doi] B978-0-444-63326-2.00011-9 [pii]',\n",
       "  'AB': 'In this chapter, we will use the concepts and perspective of population genetics and genetic analysis to discuss the trends that are currently popular in the epilepsy genetics literature. We will also show that some of the recent approaches to understanding epilepsy genetics are unlikely to lead to insight into the causes of the most common epilepsies. Much effort has been directed toward very rare influences on epilepsy expression and away from approaches that could shed light on the most heritable epilepsies. The latest genetic technology, e.g., exome sequencing and copy number variants (CNV) analysis, has been applied without proper consideration of the important principles of population genetics, inheritance, or phenotype definition that are critical in common disease studies. We will show that ignoring these issues has led to unlikely suppositions about etiology and about which genes should be pursued. We also suggest that, taken together, the existing evidence supports a neurodevelopmental origin for common epilepsies. We recommend a return to thorough phenotype definition and family studies and a more careful and thoughtful application of the new technology.',\n",
       "  'CI': ['(c) 2014 Elsevier B.V. All rights reserved.'],\n",
       "  'FAU': ['Greenberg, David A', 'Stewart, William L'],\n",
       "  'AU': ['Greenberg DA', 'Stewart WL'],\n",
       "  'AD': \"Battelle Center for Mathematical Medicine, Nationwide Children's Hospital and Pediatrics Department, Wexner Medical Center, Ohio State University, Columbus, OH, USA. Electronic address: david.greenberg@nationwidechildrens.org. Battelle Center for Mathematical Medicine, Nationwide Children's Hospital and Pediatrics Department, Wexner Medical Center, Ohio State University, Columbus, OH, USA.\",\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['R01 NS061829/NS/NINDS NIH HHS/United States',\n",
       "   'R21 NS070323/NS/NINDS NIH HHS/United States',\n",
       "   'NS61829/NS/NINDS NIH HHS/United States'],\n",
       "  'PT': ['Journal Article',\n",
       "   'Research Support, N.I.H., Extramural',\n",
       "   \"Research Support, Non-U.S. Gov't\",\n",
       "   'Review'],\n",
       "  'PL': 'Netherlands',\n",
       "  'TA': 'Prog Brain Res',\n",
       "  'JT': 'Progress in brain research',\n",
       "  'JID': '0376441',\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Epilepsy/*genetics', 'Genetics, Population/*trends', 'Humans'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['BRD2',\n",
       "   'EFHC1',\n",
       "   'ELP4',\n",
       "   'copy number variant',\n",
       "   'genetic association',\n",
       "   'idiopathic generalized epilepsy',\n",
       "   'juvenile myoclonic epilepsy',\n",
       "   'linkage analysis',\n",
       "   'neurodevelopment',\n",
       "   'rolandic epilepsy'],\n",
       "  'EDAT': '2014/09/10 06:00',\n",
       "  'MHDA': '2015/04/07 06:00',\n",
       "  'CRDT': ['2014/09/08 06:00'],\n",
       "  'PHST': ['2014/09/08 06:00 [entrez]',\n",
       "   '2014/09/10 06:00 [pubmed]',\n",
       "   '2015/04/07 06:00 [medline]'],\n",
       "  'AID': ['B978-0-444-63326-2.00011-9 [pii]',\n",
       "   '10.1016/B978-0-444-63326-2.00011-9 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Prog Brain Res. 2014;213:199-221. doi: 10.1016/B978-0-444-63326-2.00011-9.'},\n",
       " {'PMID': '25009170',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20150609',\n",
       "  'LR': '20181113',\n",
       "  'IS': '2160-1836 (Electronic) 2160-1836 (Linking)',\n",
       "  'VI': '4',\n",
       "  'IP': '9',\n",
       "  'DP': '2014 Jul 9',\n",
       "  'TI': 'Genetic basis for Saccharomyces cerevisiae biofilm in liquid medium.',\n",
       "  'PG': '1671-80',\n",
       "  'LID': '10.1534/g3.114.010892 [doi]',\n",
       "  'AB': 'Biofilm-forming microorganisms switch between two forms: free-living planktonic and sessile multicellular. Sessile communities of yeast biofilms in liquid medium provide a primitive example of multicellularity and are clinically important because biofilms tend to have other growth characteristics than free-living cells. We investigated the genetic basis for yeast, Saccharomyces cerevisiae, biofilm on solid surfaces in liquid medium by screening a comprehensive deletion mutant collection in the Sigma1278b background and found 71 genes that were essential for biofilm development. Quantitative northern blots further revealed that AIM1, ASG1, AVT1, DRN1, ELP4, FLO8, FMP10, HMT1, KAR5, MIT1, MRPL32, MSS11, NCP1, NPR1, PEP5, PEX25, RIM8, RIM101, RGT1, SNF8, SPC2, STB6, STP22, TEC1, VID24, VPS20, VTC3, YBL029W, YBL029C-A, YFL054C, YGR161W-C, YIL014C-A, YIR024C, YKL151C, YNL200C, YOR034C-A, and YOR223W controlled biofilm through FLO11 induction. Almost all deletion mutants that were unable to form biofilms in liquid medium also lost the ability to form surface-spreading biofilm colonies (mats) on agar and 69% also lost the ability to grow invasively. The protein kinase A isoform Tpk3p functioned specifically in biofilm and mat formation. In a tpk3 mutant, transcription of FLO11 was induced three-fold compared with wild-type, but biofilm development and cell-cell adhesion was absent, suggesting that Tpk3p regulates FLO11 positive posttranscriptionally and negative transcriptionally.The study provides a resource of biofilm-influencing genes for additional research on biofilm development and suggests that the regulation of FLO11 is more complex than previously anticipated.',\n",
       "  'CI': ['Copyright (c) 2014 Andersen et al.'],\n",
       "  'FAU': ['Scherz, Kaj',\n",
       "   'Andersen',\n",
       "   'Bojsen, Rasmus',\n",
       "   'Gro, Laura',\n",
       "   'Rejkjaer',\n",
       "   'Sorensen',\n",
       "   'Weiss, Martin',\n",
       "   'Nielsen',\n",
       "   'Lisby, Michael',\n",
       "   'Folkesson, Anders',\n",
       "   'Regenberg, Birgitte'],\n",
       "  'AU': ['Scherz K',\n",
       "   'Andersen',\n",
       "   'Bojsen R',\n",
       "   'Gro L',\n",
       "   'Rejkjaer',\n",
       "   'Sorensen',\n",
       "   'Weiss M',\n",
       "   'Nielsen',\n",
       "   'Lisby M',\n",
       "   'Folkesson A',\n",
       "   'Regenberg B'],\n",
       "  'AD': 'Department of Biology, University of Copenhagen, Copenhagen, Denmark Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark. Department of Biology, University of Copenhagen, Copenhagen, Denmark. Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark. Department of Biology, University of Copenhagen, Copenhagen, Denmark Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark. Department of Biology, University of Copenhagen, Copenhagen, Denmark Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark. Department of Biology, University of Copenhagen, Copenhagen, Denmark. Department of Biology, University of Copenhagen, Copenhagen, Denmark Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark. Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark. Department of Biology, University of Copenhagen, Copenhagen, Denmark. Department of Systems Biology, Technical University of Denmark, Copenhagen, Denmark. Department of Biology, University of Copenhagen, Copenhagen, Denmark bregenberg@bio.ku.dk.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20140709',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'G3 (Bethesda)',\n",
       "  'JT': 'G3 (Bethesda, Md.)',\n",
       "  'JID': '101566598',\n",
       "  'RN': ['0 (Culture Media)',\n",
       "   '0 (RNA, Messenger)',\n",
       "   '0 (Saccharomyces cerevisiae Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['*Biofilms',\n",
       "   'Culture Media',\n",
       "   'Genes, Fungal/genetics',\n",
       "   'RNA, Messenger/metabolism',\n",
       "   'Saccharomyces cerevisiae/*genetics/physiology',\n",
       "   'Saccharomyces cerevisiae Proteins/*genetics'],\n",
       "  'PMC': 'PMC4169159',\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['Biofilm', 'PKA', 'adhesion', 'genome-wide screen', 'multicellular'],\n",
       "  'EDAT': '2014/07/11 06:00',\n",
       "  'MHDA': '2015/06/10 06:00',\n",
       "  'CRDT': ['2014/07/11 06:00'],\n",
       "  'PHST': ['2014/07/11 06:00 [entrez]',\n",
       "   '2014/07/11 06:00 [pubmed]',\n",
       "   '2015/06/10 06:00 [medline]'],\n",
       "  'AID': ['g3.114.010892 [pii]', '10.1534/g3.114.010892 [doi]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'G3 (Bethesda). 2014 Jul 9;4(9):1671-80. doi: 10.1534/g3.114.010892.'},\n",
       " {'PMID': '24995671',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20140929',\n",
       "  'LR': '20191211',\n",
       "  'IS': '1528-1167 (Electronic) 0013-9580 (Linking)',\n",
       "  'VI': '55',\n",
       "  'IP': '8',\n",
       "  'DP': '2014 Aug',\n",
       "  'TI': 'Analysis of ELP4, SRPX2, and interacting genes in typical and atypical rolandic epilepsy.',\n",
       "  'PG': 'e89-93',\n",
       "  'LID': '10.1111/epi.12712 [doi]',\n",
       "  'AB': 'Rolandic epilepsy (RE) and its atypical variants (atypical rolandic epilepsy, ARE) along the spectrum of epilepsy-aphasia disorders are characterized by a strong but largely unknown genetic basis. Two genes with a putative (ELP4) or a proven (SRPX2) function in neuronal migration were postulated to confer susceptibility to parts of the disease spectrum: the ELP4 gene to centrotemporal spikes and SRPX2 to ARE. To reexamine these findings, we investigated a cohort of 280 patients of European ancestry with RE/ARE for the etiological contribution of these genes and their close interaction partners. We performed next-generation sequencing and single-nucleotide polymorphism (SNP)-array based genotyping to screen for sequence and structural variants. In comparison to European controls we could not detect an enrichment of rare deleterious variants of ELP4, SRPX2, or their interaction partners in affected individuals. The previously described functional p.N327S variant in the X chromosomal SRPX2 gene was detected in two affected individuals (0.81%) and also in controls (0.26%), with some preponderance of male patients. We did not detect an association of SNPs in the ELP4 gene with centrotemporal spikes as previously reported. In conclusion our data do not support a major role of ELP4 and SRPX2 in the etiology of RE/ARE.',\n",
       "  'CI': ['Wiley Periodicals, Inc. (c) 2014 International League Against Epilepsy.'],\n",
       "  'FAU': ['Reinthaler, Eva M',\n",
       "   'Lal, Dennis',\n",
       "   'Jurkowski, Wiktor',\n",
       "   'Feucht, Martha',\n",
       "   'Steinbock, Hannelore',\n",
       "   'Gruber-Sedlmayr, Ursula',\n",
       "   'Ronen, Gabriel M',\n",
       "   'Geldner, Julia',\n",
       "   'Haberlandt, Edda',\n",
       "   'Neophytou, Birgit',\n",
       "   'Hahn, Andreas',\n",
       "   'Altmuller, Janine',\n",
       "   'Thiele, Holger',\n",
       "   'Toliat, Mohammad R',\n",
       "   'Lerche, Holger',\n",
       "   'Nurnberg, Peter',\n",
       "   'Sander, Thomas',\n",
       "   'Neubauer, Bernd A',\n",
       "   'Zimprich, Fritz'],\n",
       "  'AU': ['Reinthaler EM',\n",
       "   'Lal D',\n",
       "   'Jurkowski W',\n",
       "   'Feucht M',\n",
       "   'Steinbock H',\n",
       "   'Gruber-Sedlmayr U',\n",
       "   'Ronen GM',\n",
       "   'Geldner J',\n",
       "   'Haberlandt E',\n",
       "   'Neophytou B',\n",
       "   'Hahn A',\n",
       "   'Altmuller J',\n",
       "   'Thiele H',\n",
       "   'Toliat MR',\n",
       "   'Lerche H',\n",
       "   'Nurnberg P',\n",
       "   'Sander T',\n",
       "   'Neubauer BA',\n",
       "   'Zimprich F'],\n",
       "  'AD': 'Department of Neurology, Medical University of Vienna, Vienna, Austria.',\n",
       "  'CN': ['EuroEPINOMICS Consortium'],\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['HL-103010/HL/NHLBI NIH HHS/United States',\n",
       "   'HL-102924/HL/NHLBI NIH HHS/United States',\n",
       "   'I 643/FWF_/Austrian Science Fund FWF/Austria',\n",
       "   'HL-102926/HL/NHLBI NIH HHS/United States',\n",
       "   'HL-102925/HL/NHLBI NIH HHS/United States',\n",
       "   'HL-102923/HL/NHLBI NIH HHS/United States'],\n",
       "  'PT': ['Journal Article',\n",
       "   'Research Support, N.I.H., Extramural',\n",
       "   \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20140703',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Epilepsia',\n",
       "  'JT': 'Epilepsia',\n",
       "  'JID': '2983306R',\n",
       "  'RN': ['0 (ELP4 protein, human)',\n",
       "   '0 (Nerve Tissue Proteins)',\n",
       "   '0 (SRPX2 protein, human)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Austria/epidemiology',\n",
       "   'Canada/epidemiology',\n",
       "   'Child',\n",
       "   'Epilepsy, Rolandic/*diagnosis/epidemiology/*genetics',\n",
       "   'Female',\n",
       "   'Genetic Variation/genetics',\n",
       "   'Germany/epidemiology',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   'Nerve Tissue Proteins/*genetics',\n",
       "   'Polymorphism, Single Nucleotide/*genetics'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['Association',\n",
       "   'CNV',\n",
       "   'Gene',\n",
       "   'Idiopathic focal childhood epilepsy',\n",
       "   'Mutation',\n",
       "   'SNP'],\n",
       "  'EDAT': '2014/07/06 06:00',\n",
       "  'MHDA': '2014/09/30 06:00',\n",
       "  'CRDT': ['2014/07/05 06:00'],\n",
       "  'PHST': ['2014/05/30 00:00 [accepted]',\n",
       "   '2014/07/05 06:00 [entrez]',\n",
       "   '2014/07/06 06:00 [pubmed]',\n",
       "   '2014/09/30 06:00 [medline]'],\n",
       "  'AID': ['10.1111/epi.12712 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Epilepsia. 2014 Aug;55(8):e89-93. doi: 10.1111/epi.12712. Epub 2014 Jul 3.'},\n",
       " {'PMID': '23505204',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20140818',\n",
       "  'LR': '20161125',\n",
       "  'IS': '1552-4965 (Electronic) 1549-3296 (Linking)',\n",
       "  'VI': '102',\n",
       "  'IP': '2',\n",
       "  'DP': '2014 Feb',\n",
       "  'TI': 'Evaluation of the bulk platelet response and fibrinogen interaction to elastin-like polypeptide coatings.',\n",
       "  'PG': '540-51',\n",
       "  'LID': '10.1002/jbm.a.34699 [doi]',\n",
       "  'AB': 'In this work, we expand on our understanding of the thrombogenicity of coatings prepared with three different recombinant elastin-like polypeptides (ELPs). The bulk platelet response of the ELP coatings was characterized following whole blood contact under physiological shear flow (300 s(-1) ) using flow cytometry. Prolonged exposure to shear flow (1-h) indicated that materials coated with the longer ELP coatings (ELP2 and ELP4) had less bulk platelet activation and microparticle formation than materials coated with the shorter ELP1. Quartz crystal microbalance with dissipation (QCM-D) was used to monitor the binding of the platelet membrane receptor GPIIb/IIIa to ELP-adsorbed fibrinogen (Fg) surfaces. Compared to the shorter ELPs, a lower amount of Fg adsorbed to the ELP4 coated material and ELP4 appeared to form a softer, more structurally flexible coating layer. When Fg was adsorbed to the ELP coated surface it demonstrated an altered binding for GPIIb/IIIa that was inhibited in the presence of an AGDV-containing peptide but not an RGD-containing peptide. Conversely, on the shorter ELP coatings, binding of GPIIb/IIIa to an adsorbed Fg layer was partially inhibited in the presence of an RGD-containing peptide. These results indicate that both the quantity and conformational state of Fg varies when adsorbed to surfaces coated with ELPs of varying sequence length, which may be mediating their platelet response. Collectively, the findings reinforce the applicability of the ELPs as potential thromboresistant coatings, especially with the use of the longer polypeptide-ELP4.',\n",
       "  'CI': ['(c) 2013 Wiley Periodicals, Inc.'],\n",
       "  'FAU': ['Srokowski, Elizabeth M', 'Woodhouse, Kimberly A'],\n",
       "  'AU': ['Srokowski EM', 'Woodhouse KA'],\n",
       "  'AD': 'Department of Chemical Engineering and Applied Chemistry, University of Toronto, Ontario, Canada; Institute of Biomaterials and Biomedical Engineering, University of Toronto, Ontario, Canada.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Clinical Trial',\n",
       "   'Journal Article',\n",
       "   \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20131004',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'J Biomed Mater Res A',\n",
       "  'JT': 'Journal of biomedical materials research. Part A',\n",
       "  'JID': '101234237',\n",
       "  'RN': ['0 (Coated Materials, Biocompatible)',\n",
       "   '0 (Oligopeptides)',\n",
       "   '0 (Platelet Glycoprotein GPIIb-IIIa Complex)',\n",
       "   '78VO7F77PN (arginyl-glycyl-aspartic acid)',\n",
       "   '9001-32-5 (Fibrinogen)',\n",
       "   '9007-58-3 (Elastin)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Blood Platelets/cytology/*metabolism',\n",
       "   'Coated Materials, Biocompatible/*chemistry',\n",
       "   'Elastin/*chemistry',\n",
       "   'Female',\n",
       "   'Fibrinogen/*chemistry',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   '*Materials Testing',\n",
       "   'Oligopeptides/chemistry',\n",
       "   'Platelet Glycoprotein GPIIb-IIIa Complex/*metabolism'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['blood compatibility',\n",
       "   'elastin-like polypeptide',\n",
       "   'fibrinogen',\n",
       "   'platelet activation',\n",
       "   'quartz crystal microbalance with dissipation'],\n",
       "  'EDAT': '2013/03/19 06:00',\n",
       "  'MHDA': '2014/08/19 06:00',\n",
       "  'CRDT': ['2013/03/19 06:00'],\n",
       "  'PHST': ['2012/10/24 00:00 [received]',\n",
       "   '2013/02/18 00:00 [revised]',\n",
       "   '2013/02/21 00:00 [accepted]',\n",
       "   '2013/03/19 06:00 [entrez]',\n",
       "   '2013/03/19 06:00 [pubmed]',\n",
       "   '2014/08/19 06:00 [medline]'],\n",
       "  'AID': ['10.1002/jbm.a.34699 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'J Biomed Mater Res A. 2014 Feb;102(2):540-51. doi: 10.1002/jbm.a.34699. Epub 2013 Oct 4.'},\n",
       " {'PMID': '23053802',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20130621',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1573-4838 (Electronic) 0957-4530 (Linking)',\n",
       "  'VI': '24',\n",
       "  'IP': '1',\n",
       "  'DP': '2013 Jan',\n",
       "  'TI': 'Surface and adsorption characteristics of three elastin-like polypeptide coatings with varying sequence lengths.',\n",
       "  'PG': '71-84',\n",
       "  'LID': '10.1007/s10856-012-4772-6 [doi]',\n",
       "  'AB': 'The surface properties of a family of elastin-like polypeptides (ELPs), differing in molecular weight and sequence length, were investigated to understand how the nature of the polypeptide film might contribute to their thrombogenic profile. Physical adsorption of the ELPs onto Mylar increased surface wettability as the sequence length decreased while X-ray spectroscopy analysis showed an increasing amide content with sequence length. Chemical force microscopy analysis revealed that the ELP-coated surfaces displayed purely hydrophilic adhesion forces that increased as the ELP sequence length decreased. Adsorption isotherms performed using the quartz crystal microbalance with dissipation, showed that the surface coverage increased with ELP sequence length. The longer polypeptides (ELP2 and ELP4) also displayed higher specific dissipation values indicating that they established films with greater structural flexibility and associated water content than the shorter polypeptide, ELP1. Additionally, the stability of the ELP coating was lower with the shorter polypeptides. This study highlights the different surface properties of the ELP coatings as well as the dynamic nature of the ELP adsorbed layer wherein the conformational state may be an important factor contributing to their blood response.',\n",
       "  'FAU': ['Srokowski, Elizabeth M', 'Woodhouse, Kimberly A'],\n",
       "  'AU': ['Srokowski EM', 'Woodhouse KA'],\n",
       "  'AD': 'Department of Chemical Engineering and Applied Science, University of Toronto, Toronto, ON, Canada. elizabeth.srokowski@mail.utoronto.ca',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['Canadian Institutes of Health Research/Canada'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20120930',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'J Mater Sci Mater Med',\n",
       "  'JT': 'Journal of materials science. Materials in medicine',\n",
       "  'JID': '9013087',\n",
       "  'RN': ['0 (Peptides)', '9007-58-3 (Elastin)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Adsorption',\n",
       "   'Amino Acid Sequence',\n",
       "   'Elastin/*chemistry',\n",
       "   'Exons',\n",
       "   'Humans',\n",
       "   'Microscopy, Atomic Force',\n",
       "   'Molecular Sequence Data',\n",
       "   'Peptides/*chemistry',\n",
       "   'Spectrum Analysis/methods',\n",
       "   'Surface Properties'],\n",
       "  'PMC': 'PMC3540362',\n",
       "  'EDAT': '2012/10/12 06:00',\n",
       "  'MHDA': '2013/06/26 06:00',\n",
       "  'CRDT': ['2012/10/12 06:00'],\n",
       "  'PHST': ['2012/06/19 00:00 [received]',\n",
       "   '2012/09/17 00:00 [accepted]',\n",
       "   '2012/10/12 06:00 [entrez]',\n",
       "   '2012/10/12 06:00 [pubmed]',\n",
       "   '2013/06/26 06:00 [medline]'],\n",
       "  'AID': ['10.1007/s10856-012-4772-6 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'J Mater Sci Mater Med. 2013 Jan;24(1):71-84. doi: 10.1007/s10856-012-4772-6. Epub 2012 Sep 30.'},\n",
       " {'PMID': '22991255',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20130621',\n",
       "  'LR': '20121019',\n",
       "  'IS': '1552-4833 (Electronic) 1552-4825 (Linking)',\n",
       "  'VI': '158A',\n",
       "  'IP': '11',\n",
       "  'DP': '2012 Nov',\n",
       "  'TI': 'Insertional translocation of 15q25-q26 into 11p13 and duplication at 8p23.1 characterized by high resolution arrays in a boy with congenital malformations and aniridia.',\n",
       "  'PG': '2905-10',\n",
       "  'LID': '10.1002/ajmg.a.35603 [doi]',\n",
       "  'AB': 'We report on a boy presenting submucous cleft palate, hydronephrosis, ventriculoseptal defect, aniridia, and developmental delay. Additional material on 11p13 was cytogenetically visible and array analyses identified a duplicated segment on 15q25-26 chromosome region; further, array analyses revealed a small deletion (49 kb) at 11p13 region involving the ELP4 gene and a duplication at 8p23.1. Results were confirmed with both molecular and molecular cytogenetics techniques. Possibilities for etiological basis of clinical phenotype are discussed.',\n",
       "  'CI': ['Copyright (c) 2012 Wiley Periodicals, Inc.'],\n",
       "  'FAU': ['Simioni, Milena',\n",
       "   'Vieira, Tarsis Paiva',\n",
       "   'Sgardioli, Ilaria Cristina',\n",
       "   'Freitas, Erika Lopes',\n",
       "   'Rosenberg, Carla',\n",
       "   'Maurer-Morelli, Claudia Vianna',\n",
       "   'Lopes-Cendes, Iscia',\n",
       "   'Fett-Conte, Agnes Cristina',\n",
       "   'Gil-da-Silva-Lopes, Vera Lucia'],\n",
       "  'AU': ['Simioni M',\n",
       "   'Vieira TP',\n",
       "   'Sgardioli IC',\n",
       "   'Freitas EL',\n",
       "   'Rosenberg C',\n",
       "   'Maurer-Morelli CV',\n",
       "   'Lopes-Cendes I',\n",
       "   'Fett-Conte AC',\n",
       "   'Gil-da-Silva-Lopes VL'],\n",
       "  'AD': 'Department of Medical Genetics, Faculty of Medical Sciences, University of Campinas - UNICAMP, Campinas, SP, Brazil.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Case Reports',\n",
       "   'Journal Article',\n",
       "   \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20120918',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Am J Med Genet A',\n",
       "  'JT': 'American journal of medical genetics. Part A',\n",
       "  'JID': '101235741',\n",
       "  'RN': ['Chromosome 8, trisomy'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Abnormalities, Multiple/diagnosis/*genetics',\n",
       "   'Aniridia/diagnosis/*genetics',\n",
       "   'Child, Preschool',\n",
       "   '*Chromosomes, Human, Pair 11',\n",
       "   '*Chromosomes, Human, Pair 15',\n",
       "   'Chromosomes, Human, Pair 8',\n",
       "   'Comparative Genomic Hybridization',\n",
       "   'Humans',\n",
       "   'In Situ Hybridization, Fluorescence',\n",
       "   'Karyotype',\n",
       "   'Male',\n",
       "   'Phenotype',\n",
       "   '*Translocation, Genetic',\n",
       "   '*Trisomy'],\n",
       "  'EDAT': '2012/09/20 06:00',\n",
       "  'MHDA': '2013/06/25 06:00',\n",
       "  'CRDT': ['2012/09/20 06:00'],\n",
       "  'PHST': ['2011/12/21 00:00 [received]',\n",
       "   '2012/06/29 00:00 [accepted]',\n",
       "   '2012/09/20 06:00 [entrez]',\n",
       "   '2012/09/20 06:00 [pubmed]',\n",
       "   '2013/06/25 06:00 [medline]'],\n",
       "  'AID': ['10.1002/ajmg.a.35603 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Am J Med Genet A. 2012 Nov;158A(11):2905-10. doi: 10.1002/ajmg.a.35603. Epub 2012 Sep 18.'},\n",
       " {'PMID': '22854966',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20121207',\n",
       "  'LR': '20191210',\n",
       "  'IS': '1083-351X (Electronic) 0021-9258 (Linking)',\n",
       "  'VI': '287',\n",
       "  'IP': '39',\n",
       "  'DP': '2012 Sep 21',\n",
       "  'TI': 'DERP6 (ELP5) and C3ORF75 (ELP6) regulate tumorigenicity and migration of melanoma cells as subunits of Elongator.',\n",
       "  'PG': '32535-45',\n",
       "  'AB': 'The Elongator complex is composed of 6 subunits (Elp1-Elp6) and promotes RNAPII transcript elongation through histone acetylation in the nucleus as well as tRNA modification in the cytoplasm. This acetyltransferase complex directly or indirectly regulates numerous biological processes ranging from exocytosis and resistance to heat shock in yeast to cell migration and neuronal differentiation in higher eukaryotes. The identity of human ELP1 through ELP4 has been reported but human ELP5 and ELP6 have remained uncharacterized. Here, we report that DERP6 (ELP5) and C3ORF75 (ELP6) encode these subunits of human Elongator. We further investigated the importance and function of these two subunits by a combination of biochemical analysis and cellular assays. Our results show that DERP6/ELP5 is required for the integrity of Elongator and directly connects ELP3 to ELP4. Importantly, the migration and tumorigenicity of melanoma-derived cells are significantly decreased upon Elongator depletion through ELP1 or ELP3. Strikingly, DERP6/ELP5 and C3ORF75/ELP6-depleted melanoma cells have similar defects, further supporting the idea that DERP6/ELP5 and C3ORF75/ELP6 are essential for Elongator function. Together, our data identify DERP6/ELP5 and C3ORF75/ELP6 as key players for migration, invasion and tumorigenicity of melanoma cells, as integral subunits of Elongator.',\n",
       "  'FAU': ['Close, Pierre',\n",
       "   'Gillard, Magali',\n",
       "   'Ladang, Aurelie',\n",
       "   'Jiang, Zheshen',\n",
       "   'Papuga, Jessica',\n",
       "   'Hawkes, Nicola',\n",
       "   'Nguyen, Laurent',\n",
       "   'Chapelle, Jean-Paul',\n",
       "   'Bouillenne, Fabrice',\n",
       "   'Svejstrup, Jesper',\n",
       "   'Fillet, Marianne',\n",
       "   'Chariot, Alain'],\n",
       "  'AU': ['Close P',\n",
       "   'Gillard M',\n",
       "   'Ladang A',\n",
       "   'Jiang Z',\n",
       "   'Papuga J',\n",
       "   'Hawkes N',\n",
       "   'Nguyen L',\n",
       "   'Chapelle JP',\n",
       "   'Bouillenne F',\n",
       "   'Svejstrup J',\n",
       "   'Fillet M',\n",
       "   'Chariot A'],\n",
       "  'AD': 'Interdisciplinary Cluster for Applied Genoproteomics (GIGA-R), University of Liege, CHU, Sart-Tilman, B-4000 Liege, Belgium. Pierre.Close@ulg.ac.be',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['Cancer Research UK/United Kingdom'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20120801',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'J Biol Chem',\n",
       "  'JT': 'The Journal of biological chemistry',\n",
       "  'JID': '2985121R',\n",
       "  'RN': ['0 (Carrier Proteins)',\n",
       "   '0 (ELP5 protein, human)',\n",
       "   '0 (ELP6 protein, human)',\n",
       "   '0 (Multiprotein Complexes)',\n",
       "   '0 (Neoplasm Proteins)',\n",
       "   '0 (Proteins)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)',\n",
       "   'EC 2.7.7.- (RNA Polymerase II)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Carrier Proteins/genetics/*metabolism',\n",
       "   'Cell Line, Tumor',\n",
       "   '*Cell Movement',\n",
       "   'Gene Deletion',\n",
       "   'HEK293 Cells',\n",
       "   'Histone Acetyltransferases',\n",
       "   'Humans',\n",
       "   'Melanoma/genetics/*metabolism/pathology',\n",
       "   'Multiprotein Complexes/genetics/*metabolism',\n",
       "   'Neoplasm Invasiveness',\n",
       "   'Neoplasm Proteins/genetics/*metabolism',\n",
       "   'Proteins/genetics/*metabolism',\n",
       "   'RNA Polymerase II/genetics/metabolism'],\n",
       "  'PMC': 'PMC3463322',\n",
       "  'EDAT': '2012/08/03 06:00',\n",
       "  'MHDA': '2012/12/12 06:00',\n",
       "  'CRDT': ['2012/08/03 06:00'],\n",
       "  'PHST': ['2012/08/03 06:00 [entrez]',\n",
       "   '2012/08/03 06:00 [pubmed]',\n",
       "   '2012/12/12 06:00 [medline]'],\n",
       "  'AID': ['M112.402727 [pii]', '10.1074/jbc.M112.402727 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'J Biol Chem. 2012 Sep 21;287(39):32535-45. doi: 10.1074/jbc.M112.402727. Epub 2012 Aug 1.'},\n",
       " {'PMID': '22818593',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20130222',\n",
       "  'LR': '20120917',\n",
       "  'IS': '1872-6844 (Electronic) 0920-1211 (Linking)',\n",
       "  'VI': '101',\n",
       "  'IP': '3',\n",
       "  'DP': '2012 Sep',\n",
       "  'TI': 'Emerging genetic influences in benign epilepsy with centro-temporal spikes - BECTS.',\n",
       "  'PG': '197-201',\n",
       "  'LID': '10.1016/j.eplepsyres.2012.06.011 [doi] S0920-1211(12)00179-9 [pii]',\n",
       "  'AB': 'BECTS is considered to be the most common childhood epileptic syndrome. Multifactorial inheritance is the most important model accounting for the genetic behavior of the common epilepsies. In recent years, different mutations in genes that control the excitability of neurons have been described. Recent reports on the involvement of the BDNF and ELP4 genes with possible roles in cell motility, migration, and adhesion have provided first insights into the complex molecular bases of childhood focal epilepsies. However, in the most common idiopathic benign childhood epilepsies (BECTS and occipital epilepsies), major breakthroughs are still awaited.',\n",
       "  'CI': ['Copyright (c) 2012 Elsevier B.V. All rights reserved.'],\n",
       "  'FAU': ['Gkampeta, Anastasia', 'Pavlou, Evangelos'],\n",
       "  'AU': ['Gkampeta A', 'Pavlou E'],\n",
       "  'AD': '2nd Department of Pediatrics, Aristotle University of Thessaloniki, AHEPA General Hospital, Thessaloniki, Greece. anastagab@yahoo.gr',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', 'Review'],\n",
       "  'DEP': '20120719',\n",
       "  'PL': 'Netherlands',\n",
       "  'TA': 'Epilepsy Res',\n",
       "  'JT': 'Epilepsy research',\n",
       "  'JID': '8703089',\n",
       "  'RN': ['0 (Brain-Derived Neurotrophic Factor)',\n",
       "   '0 (ELP4 protein, human)',\n",
       "   '0 (Nerve Tissue Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Brain-Derived Neurotrophic Factor/*genetics',\n",
       "   'Epilepsy, Rolandic/*genetics',\n",
       "   'Humans',\n",
       "   'Mutation',\n",
       "   'Nerve Tissue Proteins/*genetics'],\n",
       "  'EDAT': '2012/07/24 06:00',\n",
       "  'MHDA': '2013/02/23 06:00',\n",
       "  'CRDT': ['2012/07/24 06:00'],\n",
       "  'PHST': ['2012/03/04 00:00 [received]',\n",
       "   '2012/06/06 00:00 [revised]',\n",
       "   '2012/06/30 00:00 [accepted]',\n",
       "   '2012/07/24 06:00 [entrez]',\n",
       "   '2012/07/24 06:00 [pubmed]',\n",
       "   '2013/02/23 06:00 [medline]'],\n",
       "  'AID': ['S0920-1211(12)00179-9 [pii]',\n",
       "   '10.1016/j.eplepsyres.2012.06.011 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Epilepsy Res. 2012 Sep;101(3):197-201. doi: 10.1016/j.eplepsyres.2012.06.011. Epub 2012 Jul 19.'},\n",
       " {'PMID': '22787611',\n",
       "  'STAT': 'Publisher',\n",
       "  'DA': '20120713',\n",
       "  'PB': ['National Center for Biotechnology Information (US)'],\n",
       "  'ED': ['4th',\n",
       "   'Noebels JL',\n",
       "   'Avoli M',\n",
       "   'Rogawski MA',\n",
       "   'Olsen RW',\n",
       "   'Delgado-Escueta AV'],\n",
       "  'DP': '2012',\n",
       "  'TI': 'Major Susceptibility Genes for Common Idiopathic Epilepsies: ELP4 in Rolandic Epilepsy and BRD2 in Juvenile Myoclonic Epilepsy',\n",
       "  'BTI': [\"Jasper's Basic Mechanisms of the Epilepsies\"],\n",
       "  'AB': 'Clinical observations confirm that the idiopathic epilepsies are neurodevelopmental disorders strongly influenced by genetic factors. Genetic epidemiological studies prove major genetic influences on two common forms of idiopathic epilepsy to be discussed here: rolandic epilepsy (RE) and juvenile myoclonic epilepsy (JME). The common forms of these idiopathic syndromes have a complex genetic inheritance and this fact complicates finding and elucidating the susceptibility genes as well as proving their pathogenic role. Genetic heterogeneity and phenotype definition are much more serious factors in studying common complex epilepsies than they are in the study of densely affected epilepsy pedigrees showing Mendelian inheritance. In complex disorders, several genes and sometimes environmental factors are believed to contribute to disease etiology; as a result, proving a causative role for any one gene, in a genetic model involving multiple genes, can present a challenge. In this chapter, we first review the methodological issues associated with analyzing complex genetic disorders, then explain the strategy that we have used in investigating these two particular syndromes. Genomewide linkage analyses and fine-mapping have resulted in the identification of susceptibility genes ELP4 in Rolandic epilepsy and BRD2 in juvenile myoclonic epilepsy. We discuss the evidence in favor of a pathogenic role for each gene and the further steps that are required to establish causality.',\n",
       "  'CI': ['Copyright (c) 2012, Michael A Rogawski, Antonio V Delgado-Escueta, Jeffrey L',\n",
       "   'Noebels, Massimo Avoli and Richard W Olsen.'],\n",
       "  'FED': ['Noebels, Jeffrey L',\n",
       "   'Avoli, Massimo',\n",
       "   'Rogawski, Michael A',\n",
       "   'Olsen, Richard W',\n",
       "   'Delgado-Escueta, Antonio V'],\n",
       "  'FAU': ['Pal, Deb K', 'Greenberg, David A'],\n",
       "  'AU': ['Pal DK', 'Greenberg DA'],\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Review', 'Book Chapter'],\n",
       "  'PL': 'Bethesda (MD)',\n",
       "  'EDAT': '2012/07/13 06:00',\n",
       "  'MHDA': '2012/07/13 06:00',\n",
       "  'CDAT': ['2012/07/13 06:00'],\n",
       "  'AID': ['NBK98203 [bookaccession]']},\n",
       " {'PMID': '22556426',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20120824',\n",
       "  'LR': '20190108',\n",
       "  'IS': '1083-351X (Electronic) 0021-9258 (Linking)',\n",
       "  'VI': '287',\n",
       "  'IP': '25',\n",
       "  'DP': '2012 Jun 15',\n",
       "  'TI': 'Crystal structure of elongator subcomplex Elp4-6.',\n",
       "  'PG': '21501-8',\n",
       "  'LID': '10.1074/jbc.M112.341560 [doi]',\n",
       "  'AB': 'Elongator is a multiprotein complex composed of two subcomplexes, Elp1-3 and Elp4-6. Elongator is highly conserved between yeast and humans and plays an important role in RNA polymerase II-mediated transcriptional elongation and many other processes, including cytoskeleton organization, exocytosis, and tRNA modification. Here, we determined the crystal structure of the Elp4-6 subcomplex of yeast. The overall structure of Elp4-6 revealed that Elp6 acts as a bridge to assemble Elp4 and Elp5. Detailed structural and sequence analyses revealed that each subunit in the Elp4-6 subcomplex forms a RecA-ATPase-like fold, although it lacks the key sequence signature of ATPases. Site-directed mutagenesis and biochemical analyses indicated that the Elp4-6 subcomplex can assemble into a hexameric ring-shaped structure in vitro and in vivo. Furthermore, GST pulldown assays showed that the ring-shaped assembly of the Elp4-6 subcomplex is important for its specific histone H3 binding. Our results may shed light on the substrate recognition and assembly of the holo-Elongator complex.',\n",
       "  'FAU': ['Lin, Zhijie',\n",
       "   'Zhao, Weijing',\n",
       "   'Diao, Wentao',\n",
       "   'Xie, Xingqiao',\n",
       "   'Wang, Zheng',\n",
       "   'Zhang, Jinxiu',\n",
       "   'Shen, Yuequan',\n",
       "   'Long, Jiafu'],\n",
       "  'AU': ['Lin Z',\n",
       "   'Zhao W',\n",
       "   'Diao W',\n",
       "   'Xie X',\n",
       "   'Wang Z',\n",
       "   'Zhang J',\n",
       "   'Shen Y',\n",
       "   'Long J'],\n",
       "  'AD': 'State Key Laboratory of Medicinal Chemical Biology, Nankai University, 94 Weijin Road, Tianjin 300071, China.',\n",
       "  'LA': ['eng'],\n",
       "  'SI': ['PDB/4EJS'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20120502',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'J Biol Chem',\n",
       "  'JT': 'The Journal of biological chemistry',\n",
       "  'JID': '2985121R',\n",
       "  'RN': ['0 (Elp4 protein, S cerevisiae)',\n",
       "   '0 (Elp6 protein, S cerevisiae)',\n",
       "   '0 (Multiprotein Complexes)',\n",
       "   '0 (RNA-Binding Proteins)',\n",
       "   '0 (Saccharomyces cerevisiae Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Crystallography, X-Ray',\n",
       "   'Multiprotein Complexes/*chemistry/genetics/metabolism',\n",
       "   'Mutagenesis, Site-Directed',\n",
       "   'Protein Structure, Quaternary',\n",
       "   'RNA-Binding Proteins/*chemistry/genetics/metabolism',\n",
       "   'Saccharomyces cerevisiae/*chemistry/genetics/metabolism',\n",
       "   'Saccharomyces cerevisiae Proteins/*chemistry/genetics/metabolism'],\n",
       "  'PMC': 'PMC3375571',\n",
       "  'EDAT': '2012/05/05 06:00',\n",
       "  'MHDA': '2012/08/25 06:00',\n",
       "  'CRDT': ['2012/05/05 06:00'],\n",
       "  'PHST': ['2012/05/05 06:00 [entrez]',\n",
       "   '2012/05/05 06:00 [pubmed]',\n",
       "   '2012/08/25 06:00 [medline]'],\n",
       "  'AID': ['M112.341560 [pii]', '10.1074/jbc.M112.341560 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'J Biol Chem. 2012 Jun 15;287(25):21501-8. doi: 10.1074/jbc.M112.341560. Epub 2012 May 2.'},\n",
       " {'PMID': '22459513',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20121001',\n",
       "  'LR': '20151119',\n",
       "  'IS': '1878-7568 (Electronic) 1742-7061 (Linking)',\n",
       "  'VI': '8',\n",
       "  'IP': '7',\n",
       "  'DP': '2012 Jul',\n",
       "  'TI': 'Electrospun elastin-like polypeptide enriched polyurethanes and their interactions with vascular smooth muscle cells.',\n",
       "  'PG': '2493-503',\n",
       "  'LID': '10.1016/j.actbio.2012.03.032 [doi]',\n",
       "  'AB': 'In vascular tissue, elastin is an essential extracellular matrix protein that plays an important biomechanical and biological signalling role. Native elastin is insoluble and is difficult to extract from tissues, which results in its relatively rare use for the fabrication of vascular tissue engineering scaffolds. Recombinant elastin-like polypeptide-4 (ELP4), which mimics the structure and function of native tropoelastin, represents a practical alternative to the native elastic fibre for vascular applications. In this study, electrospinning was utilized to fabricate fibrous scaffolds which were subsequently surface modified with ELP4 and used as substrates for smooth muscle cell culture. ELP4 surface modified materials demonstrated enhanced smooth muscle cell (SMC) adhesion and maintenance of cell numbers over a 1-week period relative to controls. SMCs seeded on the ELP4 surface modified materials were also shown to exhibit the cell morphology and biological markers of a contractile phenotype including a spindle-like morphology, actin filament organization and smooth muscle myosin heavy chain expression. Competitive inhibition experiments demonstrated that the elastin-laminin cell surface receptor and its affinity for the VGVAPG peptide sequence on ELP4 molecules are likely involved in the initial SMC contact with the ELP4 modified materials. Elastin-like polypeptides show promise as surface modifiers for candidate scaffolds for engineering contractile vascular tissues.',\n",
       "  'CI': ['Copyright (c) 2012 Acta Materialia Inc. Published by Elsevier Ltd. All rights',\n",
       "   'reserved.'],\n",
       "  'FAU': ['Blit, Patrick H',\n",
       "   'Battiston, Kyle G',\n",
       "   'Yang, Meilin',\n",
       "   'Paul Santerre, J',\n",
       "   'Woodhouse, Kimberly A'],\n",
       "  'AU': ['Blit PH',\n",
       "   'Battiston KG',\n",
       "   'Yang M',\n",
       "   'Paul Santerre J',\n",
       "   'Woodhouse KA'],\n",
       "  'AD': 'Department of Chemical Engineering and Applied Chemistry, University of Toronto, Ontario, Canada.',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['82388/Canadian Institutes of Health Research/Canada'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20120327',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Acta Biomater',\n",
       "  'JT': 'Acta biomaterialia',\n",
       "  'JID': '101233144',\n",
       "  'RN': ['0 (Actins)',\n",
       "   '0 (Biomarkers)',\n",
       "   '0 (Peptides)',\n",
       "   '0 (Polyurethanes)',\n",
       "   '9007-58-3 (Elastin)',\n",
       "   'EC 3.6.4.1 (Myosin Heavy Chains)',\n",
       "   'J2B2A4N98G (Lactose)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Actin Cytoskeleton/drug effects/metabolism',\n",
       "   'Actins/metabolism',\n",
       "   'Amino Acid Sequence',\n",
       "   'Biomarkers/metabolism',\n",
       "   'Cell Adhesion/drug effects',\n",
       "   'Cell Count',\n",
       "   'Cell Shape/drug effects',\n",
       "   'Cell Survival/drug effects',\n",
       "   'Elastin/*pharmacology',\n",
       "   'Humans',\n",
       "   'Lactose/pharmacology',\n",
       "   'Materials Testing/*methods',\n",
       "   'Microscopy, Confocal',\n",
       "   'Molecular Sequence Data',\n",
       "   'Muscle, Smooth, Vascular/*cytology',\n",
       "   'Myocytes, Smooth Muscle/*cytology/*drug effects/ultrastructure',\n",
       "   'Myosin Heavy Chains/metabolism',\n",
       "   'Peptides/chemistry/pharmacology',\n",
       "   'Phenotype',\n",
       "   'Polyurethanes/*pharmacology',\n",
       "   'Surface Properties/drug effects',\n",
       "   'Time Factors'],\n",
       "  'EDAT': '2012/03/31 06:00',\n",
       "  'MHDA': '2012/10/02 06:00',\n",
       "  'CRDT': ['2012/03/31 06:00'],\n",
       "  'PHST': ['2012/01/17 00:00 [received]',\n",
       "   '2012/03/01 00:00 [revised]',\n",
       "   '2012/03/20 00:00 [accepted]',\n",
       "   '2012/03/31 06:00 [entrez]',\n",
       "   '2012/03/31 06:00 [pubmed]',\n",
       "   '2012/10/02 06:00 [medline]'],\n",
       "  'AID': ['S1742-7061(12)00127-4 [pii]', '10.1016/j.actbio.2012.03.032 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Acta Biomater. 2012 Jul;8(7):2493-503. doi: 10.1016/j.actbio.2012.03.032. Epub 2012 Mar 27.'},\n",
       " {'PMID': '22343726',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20120420',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1545-9985 (Electronic) 1545-9985 (Linking)',\n",
       "  'VI': '19',\n",
       "  'IP': '3',\n",
       "  'DP': '2012 Feb 19',\n",
       "  'TI': 'The Elongator subcomplex Elp456 is a hexameric RecA-like ATPase.',\n",
       "  'PG': '314-20',\n",
       "  'LID': '10.1038/nsmb.2234 [doi]',\n",
       "  'AB': 'Elongator was initially described as an RNA polymerase II-associated factor but has since been associated with a broad range of cellular activities. It has also attracted clinical attention because of its role in certain neurodegenerative diseases. Here we describe the crystal structure of the Saccharomyces cerevisiae subcomplex of Elongator proteins 4, 5 and 6 (Elp456). The subunits each show almost identical RecA folds that form a heterohexameric ring-like structure resembling hexameric RecA-like ATPases. This structural finding is supported by different complementary in vitro and in vivo approaches, including the specific binding of the hexameric Elp456 subcomplex to tRNAs in a manner regulated by ATP. Our results support a role of Elongator in tRNA modification, explain the importance of each of the Elp4, Elp5 and Elp6 subunits for complex integrity and suggest a model for the overall architecture of the holo-Elongator complex.',\n",
       "  'FAU': ['Glatt, Sebastian',\n",
       "   'Letoquart, Juliette',\n",
       "   'Faux, Celine',\n",
       "   'Taylor, Nicholas M I',\n",
       "   'Seraphin, Bertrand',\n",
       "   'Muller, Christoph W'],\n",
       "  'AU': ['Glatt S',\n",
       "   'Letoquart J',\n",
       "   'Faux C',\n",
       "   'Taylor NM',\n",
       "   'Seraphin B',\n",
       "   'Muller CW'],\n",
       "  'AD': 'European Molecular Biology Laboratory (EMBL), Structural and Computational Biology Unit, Heidelberg, Germany.',\n",
       "  'LA': ['eng'],\n",
       "  'SI': ['PDB/4A8J'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20120219',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Nat Struct Mol Biol',\n",
       "  'JT': 'Nature structural & molecular biology',\n",
       "  'JID': '101186374',\n",
       "  'RN': ['0 (Elp4 protein, S cerevisiae)',\n",
       "   '0 (Elp5 protein, S cerevisiae)',\n",
       "   '0 (Elp6 protein, S cerevisiae)',\n",
       "   '0 (Protein Subunits)',\n",
       "   '0 (RNA-Binding Proteins)',\n",
       "   '0 (Saccharomyces cerevisiae Proteins)',\n",
       "   '9014-25-9 (RNA, Transfer)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Models, Molecular',\n",
       "   'Protein Binding',\n",
       "   'Protein Structure, Quaternary',\n",
       "   'Protein Structure, Tertiary',\n",
       "   'Protein Subunits/chemistry/metabolism',\n",
       "   'RNA, Transfer/chemistry/metabolism',\n",
       "   'RNA-Binding Proteins/*chemistry/metabolism',\n",
       "   'Saccharomyces cerevisiae/*enzymology',\n",
       "   'Saccharomyces cerevisiae Proteins/*chemistry/metabolism',\n",
       "   'Substrate Specificity'],\n",
       "  'EDAT': '2012/02/22 06:00',\n",
       "  'MHDA': '2012/04/21 06:00',\n",
       "  'CRDT': ['2012/02/21 06:00'],\n",
       "  'PHST': ['2011/06/07 00:00 [received]',\n",
       "   '2011/12/22 00:00 [accepted]',\n",
       "   '2012/02/21 06:00 [entrez]',\n",
       "   '2012/02/22 06:00 [pubmed]',\n",
       "   '2012/04/21 06:00 [medline]'],\n",
       "  'AID': ['nsmb.2234 [pii]', '10.1038/nsmb.2234 [doi]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'Nat Struct Mol Biol. 2012 Feb 19;19(3):314-20. doi: 10.1038/nsmb.2234.'},\n",
       " {'PMID': '22266513',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20120503',\n",
       "  'LR': '20161125',\n",
       "  'IS': '1552-5783 (Electronic) 0146-0404 (Linking)',\n",
       "  'VI': '53',\n",
       "  'IP': '3',\n",
       "  'DP': '2012 Mar 21',\n",
       "  'TI': 'A genetic epidemiologic study of candidate genes involved in the optic nerve head morphology.',\n",
       "  'PG': '1485-91',\n",
       "  'LID': '10.1167/iovs.11-7384 [doi]',\n",
       "  'AB': 'PURPOSE: The size of the optic nerve head, referred to as disc area (DA), and the vertical cup-disc ratio (VCDR), are clinically relevant parameters for glaucomatous optic neuropathy. Although these measures have a high heritability, little is known about the underlying genes. Previously, the genes SALL1 and SIX1 were found to be genome-wide significantly associated with DA and VCDR. The purpose of the present study was to investigate whether genes encoding protein known to interact with protein encoded by SALL1 and SIX1 are also associated with either DA or VCDR. METHODS: A total of 38 candidate genes were chosen covering all known proteins interacting with SALL1 and SIX1. These were initially studied in the Rotterdam Study (RS)-I, including 5312 Caucasian subjects characterized for DA and VCDR. Positive findings were further investigated in two independent cohorts (RS-II and RS-III) and finally replicated in a fourth population (ERF). Bonferroni correction was applied to the meta-analyses. RESULTS: Three loci were found to be associated with DA. The only locus significant after correcting for multiple testing is located on chromosome 11p13. Three single nucleotide polymorphisms (SNPs) in ELP4, a gene which neighbors and plays a crucial role in the expression of PAX6, show association in meta-analysis of the four cohorts yielding P values of respectively 4.79 x 10(-6), 3.92 x 10(-6), and 4.88 x 10(-6) which is below the threshold dictated by the most conservative Bonferroni correction (P = 5.2 x 10(-6)). CONCLUSIONS: This study suggests that the ELP4-PAX6 region plays a role in the DA. Further research to confirm this finding is needed.',\n",
       "  'FAU': ['Gasten, Andrea C',\n",
       "   'Ramdas, Wishal D',\n",
       "   'Broer, Linda',\n",
       "   'van Koolwijk, Leonieke M E',\n",
       "   'Ikram, M Kamran',\n",
       "   'de Jong, Paulus T V M',\n",
       "   'Aulchenko, Yurii S',\n",
       "   'Wolfs, Roger C',\n",
       "   'Hofman, Albert',\n",
       "   'Rivadeneira, Fernando',\n",
       "   'Uitterlinden, Andre G',\n",
       "   'Oostra, Ben A',\n",
       "   'Lemij, Hans G',\n",
       "   'Klaver, Caroline C W',\n",
       "   'Jansonius, Nomdo M',\n",
       "   'Vingerling, Johannes R',\n",
       "   'van Duijn, Cornelia M'],\n",
       "  'AU': ['Gasten AC',\n",
       "   'Ramdas WD',\n",
       "   'Broer L',\n",
       "   'van Koolwijk LM',\n",
       "   'Ikram MK',\n",
       "   'de Jong PT',\n",
       "   'Aulchenko YS',\n",
       "   'Wolfs RC',\n",
       "   'Hofman A',\n",
       "   'Rivadeneira F',\n",
       "   'Uitterlinden AG',\n",
       "   'Oostra BA',\n",
       "   'Lemij HG',\n",
       "   'Klaver CC',\n",
       "   'Jansonius NM',\n",
       "   'Vingerling JR',\n",
       "   'van Duijn CM'],\n",
       "  'AD': 'Department of Epidemiology, Erasmus Medical Center, Rotterdam, The Netherlands.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20120321',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Invest Ophthalmol Vis Sci',\n",
       "  'JT': 'Investigative ophthalmology & visual science',\n",
       "  'JID': '7703701',\n",
       "  'RN': ['0 (ELP4 protein, human)',\n",
       "   '0 (Eye Proteins)',\n",
       "   '0 (Homeodomain Proteins)',\n",
       "   '0 (Nerve Tissue Proteins)',\n",
       "   '0 (PAX6 Transcription Factor)',\n",
       "   '0 (PAX6 protein, human)',\n",
       "   '0 (Paired Box Transcription Factors)',\n",
       "   '0 (Repressor Proteins)',\n",
       "   '0 (SALL1 protein, human)',\n",
       "   '0 (SIX1 protein, human)',\n",
       "   '0 (Transcription Factors)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Aged',\n",
       "   'Aged, 80 and over',\n",
       "   'Chromosomes, Human, Pair 11/*genetics',\n",
       "   'Cohort Studies',\n",
       "   'Eye Proteins/genetics',\n",
       "   'Female',\n",
       "   'Genome-Wide Association Study',\n",
       "   'Homeodomain Proteins/genetics',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   'Middle Aged',\n",
       "   'Nerve Tissue Proteins/genetics',\n",
       "   'Netherlands',\n",
       "   'Optic Disk/*anatomy & histology',\n",
       "   'PAX6 Transcription Factor',\n",
       "   'Paired Box Transcription Factors/genetics',\n",
       "   'Polymorphism, Single Nucleotide/genetics',\n",
       "   'Repressor Proteins/genetics',\n",
       "   'Transcription Factors/genetics'],\n",
       "  'EDAT': '2012/01/24 06:00',\n",
       "  'MHDA': '2012/05/04 06:00',\n",
       "  'CRDT': ['2012/01/24 06:00'],\n",
       "  'PHST': ['2012/01/24 06:00 [entrez]',\n",
       "   '2012/01/24 06:00 [pubmed]',\n",
       "   '2012/05/04 06:00 [medline]'],\n",
       "  'AID': ['iovs.11-7384 [pii]', '10.1167/iovs.11-7384 [doi]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'Invest Ophthalmol Vis Sci. 2012 Mar 21;53(3):1485-91. doi: 10.1167/iovs.11-7384.'},\n",
       " {'PMID': '21601275',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20110926',\n",
       "  'LR': '20110607',\n",
       "  'IS': '1878-5905 (Electronic) 0142-9612 (Linking)',\n",
       "  'VI': '32',\n",
       "  'IP': '25',\n",
       "  'DP': '2011 Sep',\n",
       "  'TI': 'Platelet inhibition and endothelial cell adhesion on elastin-like polypeptide surface modified materials.',\n",
       "  'PG': '5790-800',\n",
       "  'LID': '10.1016/j.biomaterials.2011.04.067 [doi]',\n",
       "  'AB': 'Platelet adhesion and activation are important early markers of biomaterial blood compatibility, while surfaces that promote enhanced endothelial cell adhesion and eNOS expression are strategic targets for long term vascular graft applications. Materials surface modified with fluorinated surface modifiers, containing peptides inspired from elastin cross-linking domains, have been used for the cross-linking of elastin-like polypeptide 4 (ELP4) macromolecules onto polyurethane surfaces. In the present study, ELP4 modified polyurethanes were evaluated in vitro to assess platelet adhesion, microparticle formation and bulk platelet activation following blood-material interactions. Reduced platelet adhesion and bulk platelet activation were observed following contact between reconstituted human blood and the ELP4 materials, relative to the uncoated base polyurethane controls. ELP4 modified materials also promoted endothelial cell adhesion and retention over a period of one week and showed that the endothelial cells exhibited an organized actin cytoskeleton and enhanced endothelial nitric oxide synthase (eNOS) expression relative to the control surfaces. These results indicate that polyurethane elastomers modified with ELP4 covalently bound to fluorinated surface modifiers provide a promising approach for endowing synthetic elastomers with both reduced blood platelet activation properties and enhanced endothelial cell adhesion for potential use in vascular graft applications.',\n",
       "  'CI': ['Copyright (c) 2011 Elsevier Ltd. All rights reserved.'],\n",
       "  'FAU': ['Blit, Patrick H',\n",
       "   'McClung, W Glenn',\n",
       "   'Brash, John L',\n",
       "   'Woodhouse, Kimberly A',\n",
       "   'Santerre, J Paul'],\n",
       "  'AU': ['Blit PH', 'McClung WG', 'Brash JL', 'Woodhouse KA', 'Santerre JP'],\n",
       "  'AD': 'Department of Chemical Engineering and Applied Chemistry, University of Toronto, Toronto, Ontario, Canada.',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['82388/Canadian Institutes of Health Research/Canada'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20110520',\n",
       "  'PL': 'Netherlands',\n",
       "  'TA': 'Biomaterials',\n",
       "  'JT': 'Biomaterials',\n",
       "  'JID': '8100316',\n",
       "  'RN': ['0 (Biocompatible Materials)', '9007-58-3 (Elastin)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Adsorption',\n",
       "   '*Biocompatible Materials',\n",
       "   'Blood Platelets/chemistry/*cytology',\n",
       "   '*Cell Adhesion',\n",
       "   'Cells, Cultured',\n",
       "   'Elastin/*chemistry',\n",
       "   'Endothelium, Vascular/chemistry/*cytology',\n",
       "   'Humans',\n",
       "   'Microscopy, Electron, Scanning',\n",
       "   'Surface Properties'],\n",
       "  'EDAT': '2011/05/24 06:00',\n",
       "  'MHDA': '2011/09/29 06:00',\n",
       "  'CRDT': ['2011/05/24 06:00'],\n",
       "  'PHST': ['2011/04/06 00:00 [received]',\n",
       "   '2011/04/23 00:00 [accepted]',\n",
       "   '2011/05/24 06:00 [entrez]',\n",
       "   '2011/05/24 06:00 [pubmed]',\n",
       "   '2011/09/29 06:00 [medline]'],\n",
       "  'AID': ['S0142-9612(11)00489-3 [pii]',\n",
       "   '10.1016/j.biomaterials.2011.04.067 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Biomaterials. 2011 Sep;32(25):5790-800. doi: 10.1016/j.biomaterials.2011.04.067. Epub 2011 May 20.'},\n",
       " {'PMID': '21364908',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20110616',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1090-0535 (Electronic) 1090-0535 (Linking)',\n",
       "  'VI': '17',\n",
       "  'DP': '2011 Feb 19',\n",
       "  'TI': 'Large novel deletions detected in Chinese families with aniridia: correlation between genotype and phenotype.',\n",
       "  'PG': '548-57',\n",
       "  'AB': 'PURPOSE: To describe the clinical and genetic findings in two Chinese families with aniridia and other ocular abnormalities. METHODS: Two unrelated families were examined clinically. After informed consent was obtained, genomic DNA was extracted from the venous blood of all participants. Mutation screening of all exons of the PAX6 (paired box gene 6) gene was performed by direct sequencing of PCR-amplified DNA fragments. Multiplex ligation-dependent probe amplification (MLPA) was performed to detect large deletions. Linkage analysis was used to validate the large deletions revealed by MLPA in all available family members. RESULTS: Clinical examination and pedigree analysis revealed one four-generation family (85) and one three- generation family (86) with total aniridia, congenital cataracts, foveal hypoplasia, and glaucoma. No mutation in PAX6 was identified after PCR-sequencing. Through MLPA analysis, a large deletion including the whole PAX6 gene, DKFZp686k1684 (hypothetical LOC440034), and the RCN1 (reticulocalbin 1) gene was detected in family 85; a 3\\' deletion to the PAX6 gene including the ELP4 (elongator complex protein 4) and the DCDC1 (doublecortin domain containing 1) gene was identified in family 86.The two large deletions were confirmed with linkage analysis and the \"loss of heterozygous\" in the different PAX6 regions were co-segregated with the phenotype of the two families, respectively. CONCLUSIONS: Patients with the PAX6 contiguous gene deletion, including the RCN1 gene, presented more severe vision impairments than those carrying the PAX6 3\\' deletion. Large deletions may account for several Chinese families and sporadic cases with aniridia and screening for these kinds of alterations should be included in aniridia patients\\' analyses.',\n",
       "  'FAU': ['Zhang, Xiaohui',\n",
       "   'Zhang, Qingsheng',\n",
       "   'Tong, Yi',\n",
       "   'Dai, Hanjun',\n",
       "   'Zhao, Xin',\n",
       "   'Bai, Fengge',\n",
       "   'Xu, Liang',\n",
       "   'Li, Yang'],\n",
       "  'AU': ['Zhang X',\n",
       "   'Zhang Q',\n",
       "   'Tong Y',\n",
       "   'Dai H',\n",
       "   'Zhao X',\n",
       "   'Bai F',\n",
       "   'Xu L',\n",
       "   'Li Y'],\n",
       "  'AD': 'Beijing Institute of Ophthalmology, Beijing Tongren Eye Center, Beijing Tongren Hospital, Capital Medical University, Beijing Ophthalmology & Visual Sciences Key Laboratory, Beijing, China.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20110219',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Mol Vis',\n",
       "  'JT': 'Molecular vision',\n",
       "  'JID': '9605351',\n",
       "  'RN': ['0 (Eye Proteins)',\n",
       "   '0 (Homeodomain Proteins)',\n",
       "   '0 (PAX6 Transcription Factor)',\n",
       "   '0 (PAX6 protein, human)',\n",
       "   '0 (Paired Box Transcription Factors)',\n",
       "   '0 (Repressor Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Adolescent',\n",
       "   'Adult',\n",
       "   'Aged',\n",
       "   'Aniridia/*genetics/pathology',\n",
       "   'Asian Continental Ancestry Group/*genetics',\n",
       "   'Child',\n",
       "   'Child, Preschool',\n",
       "   'China',\n",
       "   'Chromosomes, Human, Pair 11/genetics',\n",
       "   'DNA Mutational Analysis',\n",
       "   'Eye Proteins/genetics',\n",
       "   'Family',\n",
       "   'Female',\n",
       "   '*Genetic Association Studies',\n",
       "   'Genotype',\n",
       "   'Haplotypes/genetics',\n",
       "   'Homeodomain Proteins/genetics',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   'Microsatellite Repeats/genetics',\n",
       "   'Middle Aged',\n",
       "   'PAX6 Transcription Factor',\n",
       "   'Paired Box Transcription Factors/genetics',\n",
       "   'Pedigree',\n",
       "   'Repressor Proteins/genetics',\n",
       "   'Reverse Transcriptase Polymerase Chain Reaction',\n",
       "   'Sequence Deletion/*genetics'],\n",
       "  'PMC': 'PMC3044699',\n",
       "  'EDAT': '2011/03/03 06:00',\n",
       "  'MHDA': '2011/06/17 06:00',\n",
       "  'CRDT': ['2011/03/03 06:00'],\n",
       "  'PHST': ['2010/09/29 00:00 [received]',\n",
       "   '2011/02/14 00:00 [accepted]',\n",
       "   '2011/03/03 06:00 [entrez]',\n",
       "   '2011/03/03 06:00 [pubmed]',\n",
       "   '2011/06/17 06:00 [medline]'],\n",
       "  'AID': ['63 [pii]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'Mol Vis. 2011 Feb 19;17:548-57.'},\n",
       " {'PMID': '21321669',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20110526',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1090-0535 (Electronic) 1090-0535 (Linking)',\n",
       "  'VI': '17',\n",
       "  'DP': '2011 Feb 10',\n",
       "  'TI': 'A 556 kb deletion in the downstream region of the PAX6 gene causes familial aniridia and other eye anomalies in a Chinese family.',\n",
       "  'PG': '448-55',\n",
       "  'AB': 'PURPOSE: The paired box gene 6 (PAX6) on human chromosome 11p13 is an essential transcription factor for eye formation in animals. Mutations in PAX6 can lead to varieties of autosomal-dominant ocular malformations with aniridia as the major clinical signs. Known genetic alterations causing haplo-insufficiency of PAX6 include nonsense mutations, frame-shift mutations, splicing errors, or genomic deletions. The purpose of this study was to identify genetic defects as the underlying cause of familial aniridia in a large Chinese family. METHODS: All exons of PAX6 in the proband were sequenced by the Sanger sequencing technique. The genome of the proband was evaluated by a microarray-based comparative genomic hybridization (aCGH). Quantitative real-time PCR was applied to verify the abnormal aCGH findings in the proband and to test five other family members. RESULTS: There were no detectable pathogenic mutations in the exons of PAX6 in the proband. The aCGH analysis showed two copies of PAX6 but revealed a 566 kb hemizygous deletion of chromosome 11p13, including four annotated genes doublecortin domain containing 1 (DCDC1), DnaJ homolog subfamily C member 24 (DNAJC24), IMP1 inner mitochondrial membrane(IMMP1L), andelongation factor protein 4 (ELP4) downstream of PAX6. Quantitative real-time PCR verified the deletion in the proband and further identified the deletion in a blind fashion in four affected family members but not in the one with a normal phenotype. CONCLUSIONS: The 566 kb hemizygous deletion of chromosome 11p13 downstream of PAX6 should be the cause of the familial aniridia in this Chinese family, although two copies of PAX6 are intact. aCGH evaluation should be applied if there is a negative result for the mutation detection of PAX6 in patients with aniridia.',\n",
       "  'FAU': ['Cheng, Fang',\n",
       "   'Song, Wulian',\n",
       "   'Kang, Yang',\n",
       "   'Yu, Shihui',\n",
       "   'Yuan, Huiping'],\n",
       "  'AU': ['Cheng F', 'Song W', 'Kang Y', 'Yu S', 'Yuan H'],\n",
       "  'AD': 'Department of Ophthalmology, the 2nd Affiliated Hospital of Harbin Medical University, Harbin, China.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Case Reports',\n",
       "   'Journal Article',\n",
       "   \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20110210',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Mol Vis',\n",
       "  'JT': 'Molecular vision',\n",
       "  'JID': '9605351',\n",
       "  'RN': ['0 (Eye Proteins)',\n",
       "   '0 (Homeodomain Proteins)',\n",
       "   '0 (PAX6 Transcription Factor)',\n",
       "   '0 (PAX6 protein, human)',\n",
       "   '0 (Paired Box Transcription Factors)',\n",
       "   '0 (Repressor Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Aniridia/ethnology/*genetics',\n",
       "   'China',\n",
       "   'Chromosomes, Human, Pair 11/genetics',\n",
       "   'Exons',\n",
       "   'Eye Abnormalities/ethnology/*genetics',\n",
       "   'Eye Proteins/*genetics',\n",
       "   'Family Health',\n",
       "   'Female',\n",
       "   '*Gene Deletion',\n",
       "   'Homeodomain Proteins/*genetics',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   'Mutation',\n",
       "   'PAX6 Transcription Factor',\n",
       "   'Paired Box Transcription Factors/*genetics',\n",
       "   'Pedigree',\n",
       "   'Phenotype',\n",
       "   'Repressor Proteins/*genetics',\n",
       "   'Reverse Transcriptase Polymerase Chain Reaction',\n",
       "   'Sequence Analysis, DNA'],\n",
       "  'PMC': 'PMC3038207',\n",
       "  'EDAT': '2011/02/16 06:00',\n",
       "  'MHDA': '2011/05/27 06:00',\n",
       "  'CRDT': ['2011/02/16 06:00'],\n",
       "  'PHST': ['2010/12/10 00:00 [received]',\n",
       "   '2011/02/05 00:00 [accepted]',\n",
       "   '2011/02/16 06:00 [entrez]',\n",
       "   '2011/02/16 06:00 [pubmed]',\n",
       "   '2011/05/27 06:00 [medline]'],\n",
       "  'AID': ['51 [pii]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'Mol Vis. 2011 Feb 10;17:448-55.'},\n",
       " {'PMID': '21268240',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20110516',\n",
       "  'LR': '20191210',\n",
       "  'IS': '1552-4965 (Electronic) 1549-3296 (Linking)',\n",
       "  'VI': '96',\n",
       "  'IP': '4',\n",
       "  'DP': '2011 Mar 15',\n",
       "  'TI': 'Surface immobilization of elastin-like polypeptides using fluorinated surface modifying additives.',\n",
       "  'PG': '648-62',\n",
       "  'LID': '10.1002/jbm.a.33022 [doi]',\n",
       "  'AB': 'Elastin-like polypeptide (ELP) surface modification represents a valuable approach for the development of biomaterials in a wide range of medical applications. In this study, ELP surface modification has been achieved through the use of elastin cross-linking peptide (ECP) bioactive fluorinated surface modifiers (ECP-BFSMs). The synthesis of low molecular weight fluorinated additives was described and their subsequent blending with a base polycarbonate urethane (PCNU) was shown to successfully enrich the surface to allow for ELP surface cross-linking via lysine moieties on the peptide segments of the ECP-BFSMs. The kinetics for the surface migration of fluorescent ECP-BFSMs was studied over a 2-week period by two-photon confocal microscopy. A decrease in advancing contact angle from 87.9 degrees to 75.3 degrees was observed for ECP-BFSM modified PCNU and was associated with the presence of ECP peptides on the surface. X-ray photoelectron spectroscopy demonstrated an increase in surface atomic percent of fluorine (from 0.2 to 7.2%) and nitrogen (from 1.0 to 3.0%) associated with the surface localization of fluoro groups and amide groups associated with the peptides in the ECP-BFSMs. A further increase in surface atomic percent of nitrogen (from 3.0 to 8.3%) was observed after ELP surface cross-linking. These ELP-modified surfaces were shown to promote increased smooth muscle cell adhesion, spreading and retention over a 7-day culture period relative to their non-ELP4 analogs. This novel surface modifying additive approach may be used for various biomimetic applications since it generates a stable ECM-like surface retained onto a relatively inert fluorinated background.',\n",
       "  'CI': ['Copyright (c) 2011 Wiley Periodicals, Inc.'],\n",
       "  'FAU': ['Blit, Patrick H',\n",
       "   'Battiston, Kyle G',\n",
       "   'Woodhouse, Kimberly A',\n",
       "   'Santerre, J Paul'],\n",
       "  'AU': ['Blit PH', 'Battiston KG', 'Woodhouse KA', 'Santerre JP'],\n",
       "  'AD': 'Institute of Biomaterials and Biomedical Engineering, University of Toronto, Toronto, Ontario, Canada.',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['82388/Canadian Institutes of Health Research/Canada'],\n",
       "  'PT': ['Evaluation Study',\n",
       "   'Journal Article',\n",
       "   \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20110125',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'J Biomed Mater Res A',\n",
       "  'JT': 'Journal of biomedical materials research. Part A',\n",
       "  'JID': '101234237',\n",
       "  'RN': ['0 (Biocompatible Materials)',\n",
       "   '0 (Cross-Linking Reagents)',\n",
       "   '0 (Peptides)',\n",
       "   '284SYP0193 (Fluorine)',\n",
       "   '9007-58-3 (Elastin)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Biocompatible Materials/chemistry',\n",
       "   'Cross-Linking Reagents/chemistry',\n",
       "   'Elastin/*chemistry',\n",
       "   'Fluorine/*chemistry',\n",
       "   'Halogenation',\n",
       "   'Materials Testing',\n",
       "   'Microscopy, Electron, Scanning',\n",
       "   'Molecular Structure',\n",
       "   'Molecular Weight',\n",
       "   'Peptides/*chemistry',\n",
       "   'Surface Properties'],\n",
       "  'EDAT': '2011/01/27 06:00',\n",
       "  'MHDA': '2011/05/17 06:00',\n",
       "  'CRDT': ['2011/01/27 06:00'],\n",
       "  'PHST': ['2010/08/14 00:00 [received]',\n",
       "   '2010/11/08 00:00 [revised]',\n",
       "   '2010/12/01 00:00 [accepted]',\n",
       "   '2011/01/27 06:00 [entrez]',\n",
       "   '2011/01/27 06:00 [pubmed]',\n",
       "   '2011/05/17 06:00 [medline]'],\n",
       "  'AID': ['10.1002/jbm.a.33022 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'J Biomed Mater Res A. 2011 Mar 15;96(4):648-62. doi: 10.1002/jbm.a.33022. Epub 2011 Jan 25.'},\n",
       " {'PMID': '20825490',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20110302',\n",
       "  'LR': '20101108',\n",
       "  'IS': '1601-183X (Electronic) 1601-183X (Linking)',\n",
       "  'VI': '9',\n",
       "  'IP': '8',\n",
       "  'DP': '2010 Nov',\n",
       "  'TI': 'Pleiotropic effects of the 11p13 locus on developmental verbal dyspraxia and EEG centrotemporal sharp waves.',\n",
       "  'PG': '1004-12',\n",
       "  'LID': '10.1111/j.1601-183X.2010.00648.x [doi]',\n",
       "  'AB': 'We recently showed genomewide linkage of centrotemporal sharp waves (CTS) in classic Rolandic epilepsy (RE) families to chromosome 11p13, and fine-mapped this locus to variants in the ELP4 gene. Speech sound disorder (SSD) is a common comorbidity in RE subjects, of unknown etiology, which co-aggregates in family members in a manner that could hypothetically be explained by shared underlying genetic risk with CTS. Furthermore, the neural mechanism of SSD is unknown, although individuals with rare, Mendelian forms of RE are described with severe verbal and oromotor apraxia. We therefore first performed genomewide linkage analysis for SSD, operationally defined as clinical history consistent with ICD-10 speech articulation disorder, in 38 families singly ascertained through a proband with RE. We tested the hypothesis of shared genetic risk with CTS at the 11p13 locus. In the second part of the study we used computerized acoustic analysis of recorded speech to test the hypothesis of dyspraxia as a mechanism for SSD in a smaller subset of RE probands and relatives. In two-point and multipoint LOD score analysis, we found that evidence for linkage to the 11p13 locus increased substantially when the phenotype was broadened from CTS to CTS/SSD. In multipoint analysis, the LOD score rose by 3.2 to HLOD 7.54 at D11S914 for CTS/SSD, the same marker at which multipoint linkage maximized for CTS alone. Non-parametric, affected-only methods in a sub-set of the data provide further confirmatory evidence for pleiotropy. In acoustic analysis there were voice-onset time abnormalities in 10/18 RE probands, 8/16 siblings and 5/15 parents, providing evidence of breakdown in the spatial/temporal properties of speech articulation consistent with a dyspraxic mechanism. The results from genetic and physiological studies suggest a pleiotropic role for the 11p13 locus in the development of both SSD and CTS, and also indicate a dyspraxic mechanism for the SSD linked to 11p13. Taken together, these data strongly support a neurodevelopmental origin for classic RE.',\n",
       "  'CI': ['(c) 2010 The Authors. Genes, Brain and Behavior (c) 2010 Blackwell Publishing Ltd',\n",
       "   'and International Behavioural and Neural Genetics Society.'],\n",
       "  'FAU': ['Pal, D K', 'Li, W', 'Clarke, T', 'Lieberman, P', 'Strug, L J'],\n",
       "  'AU': ['Pal DK', 'Li W', 'Clarke T', 'Lieberman P', 'Strug LJ'],\n",
       "  'AD': \"King's College London, Institute of Psychiatry, Department of Clinical Neuroscience, London, UK. deb.pal@kcl.ac.uk\",\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['HG004314/HG/NHGRI NIH HHS/United States',\n",
       "   'NS047530/NS/NINDS NIH HHS/United States',\n",
       "   'NS27941/NS/NINDS NIH HHS/United States'],\n",
       "  'PT': ['Journal Article',\n",
       "   'Research Support, N.I.H., Extramural',\n",
       "   \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20101018',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Genes Brain Behav',\n",
       "  'JT': 'Genes, brain, and behavior',\n",
       "  'JID': '101129617',\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Adolescent',\n",
       "   'Adult',\n",
       "   'Apraxias/complications/*genetics',\n",
       "   'Articulation Disorders/complications/*genetics',\n",
       "   'Child',\n",
       "   'Chromosomes, Human, Pair 11/*genetics',\n",
       "   'Electroencephalography',\n",
       "   'Epilepsy, Rolandic/*complications/genetics',\n",
       "   'Female',\n",
       "   'Genetic Linkage',\n",
       "   'Genetic Pleiotropy',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   'Middle Aged',\n",
       "   'Pedigree',\n",
       "   'Phonetics',\n",
       "   'Quantitative Trait Loci/*genetics'],\n",
       "  'EDAT': '2010/09/10 06:00',\n",
       "  'MHDA': '2011/03/03 06:00',\n",
       "  'CRDT': ['2010/09/10 06:00'],\n",
       "  'PHST': ['2010/09/10 06:00 [entrez]',\n",
       "   '2010/09/10 06:00 [pubmed]',\n",
       "   '2011/03/03 06:00 [medline]'],\n",
       "  'AID': ['GBB648 [pii]', '10.1111/j.1601-183X.2010.00648.x [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Genes Brain Behav. 2010 Nov;9(8):1004-12. doi: 10.1111/j.1601-183X.2010.00648.x. Epub 2010 Oct 18.'},\n",
       " {'PMID': '20546675',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20160526',\n",
       "  'LR': '20150828',\n",
       "  'IS': '1568-5624 (Electronic) 0920-5063 (Linking)',\n",
       "  'VI': '22',\n",
       "  'IP': '1-3',\n",
       "  'DP': '2011',\n",
       "  'TI': 'Platelet Adhesion and Fibrinogen Accretion on a Family of Elastin-Like Polypeptides.',\n",
       "  'PG': '41-57',\n",
       "  'LID': '10.1163/092050609X12578498935594 [doi]',\n",
       "  'AB': 'Previous work in our laboratory showed the potential of using a human recombinant elastin-like polypeptide (ELP) as a thromboresistant coating. In this work we investigate the use of three particular ELPs (ELP1, ELP2 and ELP4), that differ by molecular weight and number of repeating hydrophobic and cross-linking domains, as coatings to improve blood-contacting properties. All three ELPs were passively adsorbed on Mylar surfaces. Differences in water contact angle and surface concentration were found among the three ELP coatings, with the shortest polypeptide, ELP1, being the most hydrophilic and abundant on the surface (55 degrees , 0.76 mug/cm(2)), followed by ELP2 (55 degrees , 0.35 mug/cm(2)) and ELP4, the longest of the three (66 degrees , 0.25 mug/cm(2)), respectively. The blood interactions of the ELP coatings were investigated by measuring fibrinogen adsorption and platelet adhesion in whole blood under laminar flow in a cone and plate viscometer configuration. In general, platelet adhesion to the ELP-coated surfaces was found to correlate with fibrinogen adsorption. Decreases in fibrinogen accretion and platelet adhesion were observed for ELP-coated compared to uncoated surfaces. The magnitude of the decreases was found to depend on the ELP sequence length, with ELP4 exhibiting the lowest levels of fibrinogen adsorption and platelet adhesion at 43 +/- 24 ng/cm(2) and 113 +/- 77 platelets/mm(2), respectively.',\n",
       "  'FAU': ['Srokowski, E M',\n",
       "   'Blit, P H',\n",
       "   'McClung, W G',\n",
       "   'Brash, J L',\n",
       "   'Santerre, J P',\n",
       "   'Woodhouse, K A'],\n",
       "  'AU': ['Srokowski EM',\n",
       "   'Blit PH',\n",
       "   'McClung WG',\n",
       "   'Brash JL',\n",
       "   'Santerre JP',\n",
       "   'Woodhouse KA'],\n",
       "  'AD': 'a Department of Chemical Engineering and Applied Chemistry, University of Toronto, Toronto, ON, Canada; Institute of Biomaterials and Biomedical Engineering, University of Toronto, Toronto, ON, Canada.',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['Canadian Institutes of Health Research/Canada'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'PL': 'England',\n",
       "  'TA': 'J Biomater Sci Polym Ed',\n",
       "  'JT': 'Journal of biomaterials science. Polymer edition',\n",
       "  'JID': '9007393',\n",
       "  'RN': ['0 (Peptides)',\n",
       "   '0 (Recombinant Proteins)',\n",
       "   '059QF0KO0R (Water)',\n",
       "   '9001-32-5 (Fibrinogen)',\n",
       "   '9007-58-3 (Elastin)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Adsorption',\n",
       "   'Amino Acid Sequence',\n",
       "   'Blood Platelets/physiology',\n",
       "   'Elastin/chemistry/genetics',\n",
       "   'Fibrinogen/*chemistry',\n",
       "   'Humans',\n",
       "   'Hydrophobic and Hydrophilic Interactions',\n",
       "   'Materials Testing',\n",
       "   'Microscopy, Electron, Scanning',\n",
       "   'Molecular Weight',\n",
       "   '*Peptides/chemistry',\n",
       "   '*Platelet Adhesiveness',\n",
       "   'Recombinant Proteins/chemistry/genetics',\n",
       "   'Surface Properties',\n",
       "   'Water/chemistry'],\n",
       "  'OTO': ['NOTNLM'],\n",
       "  'OT': ['BLOOD COMPATIBILITY',\n",
       "   'ELASTIN-LIKE POLYPEPTIDES',\n",
       "   'FIBRINOGEN ADSORPTION',\n",
       "   'PLATELET ADHESION'],\n",
       "  'EDAT': '2010/06/16 06:00',\n",
       "  'MHDA': '2016/05/27 06:00',\n",
       "  'CRDT': ['2010/06/16 06:00'],\n",
       "  'PHST': ['2010/06/16 06:00 [entrez]',\n",
       "   '2010/06/16 06:00 [pubmed]',\n",
       "   '2016/05/27 06:00 [medline]'],\n",
       "  'AID': ['jbs3083 [pii]', '10.1163/092050609X12578498935594 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'J Biomater Sci Polym Ed. 2011;22(1-3):41-57. doi: 10.1163/092050609X12578498935594.'},\n",
       " {'PMID': '20424645',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20101101',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1476-5438 (Electronic) 1018-4813 (Linking)',\n",
       "  'VI': '18',\n",
       "  'IP': '8',\n",
       "  'DP': '2010 Aug',\n",
       "  'TI': 'A pure likelihood approach to the analysis of genetic association data: an alternative to Bayesian and frequentist analysis.',\n",
       "  'PG': '933-41',\n",
       "  'LID': '10.1038/ejhg.2010.47 [doi]',\n",
       "  'AB': \"Investigators performing genetic association studies grapple with how to measure strength of association evidence, choose sample size, and adjust for multiple testing. We apply the evidential paradigm (EP) to genetic association studies, highlighting its strengths. The EP uses likelihood ratios (LRs), as opposed to P-values or Bayes' factors, to measure strength of association evidence. We derive EP methodology to estimate sample size, adjust for multiple testing, and provide informative graphics for drawing inferences, as illustrated with a Rolandic Epilepsy (RE) fine-mapping study. We focus on controlling the probability of observing weak evidence for or against association (W) rather than type I errors (M). For example, for LR> or =32 representing strong evidence, at one locus with n=200 cases, n=200 controls, W=0.134, whereas M=0.005. For n=300 cases and controls, W=0.039 and M=0.004. These calculations are based on detecting an OR=1.5. Despite the common misconception, one is not tied to this planning value for analysis; rather one calculates the likelihood at all possible values to assess evidence for association. We provide methodology to adjust for multiple tests across m loci, which adjusts M and W for m. We do so for (a) single-stage designs, (b) two-stage designs, and (c) simultaneously controlling family-wise error rate (FWER) and W. Method (c) chooses larger sample sizes than (a) or (b), whereas (b) has smaller bounds on the FWER than (a). The EP, using our innovative graphical display, identifies important SNPs in elongator protein complex 4 (ELP4) associated with RE that may not have been identified using standard approaches.\",\n",
       "  'FAU': ['Strug, Lisa J',\n",
       "   'Hodge, Susan E',\n",
       "   'Chiang, Theodore',\n",
       "   'Pal, Deb K',\n",
       "   'Corey, Paul N',\n",
       "   'Rohde, Charles'],\n",
       "  'AU': ['Strug LJ', 'Hodge SE', 'Chiang T', 'Pal DK', 'Corey PN', 'Rohde C'],\n",
       "  'AD': 'Child Health Evaluative Sciences, The Hospital for Sick Children, 555 University Avenue, Toronto, Ontario, Canada. lisa.strug@utoronto.ca',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['R03 HG004314-02/HG/NHGRI NIH HHS/United States',\n",
       "   'R03 HG004314/HG/NHGRI NIH HHS/United States',\n",
       "   'R01 DK031813/DK/NIDDK NIH HHS/United States',\n",
       "   'DK-31813/DK/NIDDK NIH HHS/United States',\n",
       "   'R01 NS047530/NS/NINDS NIH HHS/United States',\n",
       "   'R03 HG004314-01/HG/NHGRI NIH HHS/United States',\n",
       "   'HG-004314/HG/NHGRI NIH HHS/United States',\n",
       "   'R01 MH048858/MH/NIMH NIH HHS/United States',\n",
       "   'NS047530/NS/NINDS NIH HHS/United States',\n",
       "   'MH-48858/MH/NIMH NIH HHS/United States'],\n",
       "  'PT': ['Journal Article',\n",
       "   'Research Support, N.I.H., Extramural',\n",
       "   \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20100428',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Eur J Hum Genet',\n",
       "  'JT': 'European journal of human genetics : EJHG',\n",
       "  'JID': '9302235',\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Bayes Theorem',\n",
       "   'Canada',\n",
       "   'Case-Control Studies',\n",
       "   'Chromosome Mapping/*statistics & numerical data',\n",
       "   'Data Interpretation, Statistical',\n",
       "   'Epilepsy, Rolandic/*genetics',\n",
       "   'Genome-Wide Association Study/methods/*statistics & numerical data',\n",
       "   'Humans',\n",
       "   '*Likelihood Functions',\n",
       "   '*Models, Genetic',\n",
       "   'Probability'],\n",
       "  'PMC': 'PMC2911506',\n",
       "  'MID': ['NIHMS185413'],\n",
       "  'EDAT': '2010/04/29 06:00',\n",
       "  'MHDA': '2010/11/03 06:00',\n",
       "  'CRDT': ['2010/04/29 06:00'],\n",
       "  'PHST': ['2010/04/29 06:00 [entrez]',\n",
       "   '2010/04/29 06:00 [pubmed]',\n",
       "   '2010/11/03 06:00 [medline]'],\n",
       "  'AID': ['ejhg201047 [pii]', '10.1038/ejhg.2010.47 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Eur J Hum Genet. 2010 Aug;18(8):933-41. doi: 10.1038/ejhg.2010.47. Epub 2010 Apr 28.'},\n",
       " {'PMID': '20080602',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20100312',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1091-6490 (Electronic) 0027-8424 (Linking)',\n",
       "  'VI': '107',\n",
       "  'IP': '4',\n",
       "  'DP': '2010 Jan 26',\n",
       "  'TI': 'Plant Elongator regulates auxin-related genes during RNA polymerase II transcription elongation.',\n",
       "  'PG': '1678-83',\n",
       "  'LID': '10.1073/pnas.0913559107 [doi]',\n",
       "  'AB': 'In eukaryotes, transcription of protein-encoding genes is strongly regulated by posttranslational modifications of histones that affect the accessibility of the DNA by RNA polymerase II (RNAPII). The Elongator complex was originally identified in yeast as a histone acetyltransferase (HAT) complex that activates RNAPII-mediated transcription. In Arabidopsis thaliana, the Elongator mutants elo1, elo2, and elo3 with decreased leaf and primary root growth due to reduced cell proliferation identified homologs of components of the yeast Elongator complex, Elp4, Elp1, and Elp3, respectively. Here we show that the Elongator complex was purified from plant cell cultures as a six-component complex. The role of plant Elongator in transcription elongation was supported by colocalization of the HAT enzyme, ELO3, with euchromatin and the phosphorylated form of RNAPII, and reduced histone H3 lysine 14 acetylation at the coding region of the SHORT HYPOCOTYL 2 auxin repressor and the LAX2 auxin influx carrier gene with reduced expression levels in the elo3 mutant. Additional auxin-related genes were down-regulated in the transcriptome of elo mutants but not targeted by the Elongator HAT activity showing specificity in target gene selection. Biological relevance was apparent by auxin-related phenotypes and marker gene analysis. Ethylene and jasmonic acid signaling and abiotic stress responses were up-regulated in the elo transcriptome and might contribute to the pleiotropic elo phenotype. Thus, although the structure of Elongator and its substrate are conserved, target gene selection has diverged, showing that auxin signaling and influx are under chromatin control.',\n",
       "  'FAU': ['Nelissen, Hilde',\n",
       "   'De Groeve, Steven',\n",
       "   'Fleury, Delphine',\n",
       "   'Neyt, Pia',\n",
       "   'Bruno, Leonardo',\n",
       "   'Bitonti, Maria Beatrice',\n",
       "   'Vandenbussche, Filip',\n",
       "   'Van der Straeten, Dominique',\n",
       "   'Yamaguchi, Takahiro',\n",
       "   'Tsukaya, Hirokazu',\n",
       "   'Witters, Erwin',\n",
       "   'De Jaeger, Geert',\n",
       "   'Houben, Andreas',\n",
       "   'Van Lijsebettens, Mieke'],\n",
       "  'AU': ['Nelissen H',\n",
       "   'De Groeve S',\n",
       "   'Fleury D',\n",
       "   'Neyt P',\n",
       "   'Bruno L',\n",
       "   'Bitonti MB',\n",
       "   'Vandenbussche F',\n",
       "   'Van der Straeten D',\n",
       "   'Yamaguchi T',\n",
       "   'Tsukaya H',\n",
       "   'Witters E',\n",
       "   'De Jaeger G',\n",
       "   'Houben A',\n",
       "   'Van Lijsebettens M'],\n",
       "  'AD': 'Department of Plant Systems Biology, Flanders Institute for Biotechnology, 9052 Ghent, Belgium.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20100108',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Proc Natl Acad Sci U S A',\n",
       "  'JT': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'JID': '7505876',\n",
       "  'RN': ['0 (Arabidopsis Proteins)',\n",
       "   '0 (ELO1 protein, Arabidopsis)',\n",
       "   '0 (Indoleacetic Acids)',\n",
       "   'EC 2.3.1.48 (ELO3 protein, Arabidopsis)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)',\n",
       "   'EC 2.7.7.- (RNA Polymerase II)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Arabidopsis/cytology/genetics/growth & development/*metabolism',\n",
       "   'Arabidopsis Proteins/genetics/*metabolism',\n",
       "   'Gene Expression Regulation, Plant',\n",
       "   'Histone Acetyltransferases/genetics/*metabolism',\n",
       "   'Indoleacetic Acids/*metabolism',\n",
       "   'Interphase',\n",
       "   'Meristem/genetics/growth & development/metabolism',\n",
       "   'Protein Binding',\n",
       "   'RNA Polymerase II/*metabolism',\n",
       "   '*Transcription, Genetic'],\n",
       "  'PMC': 'PMC2824411',\n",
       "  'EDAT': '2010/01/19 06:00',\n",
       "  'MHDA': '2010/03/13 06:00',\n",
       "  'CRDT': ['2010/01/19 06:00'],\n",
       "  'PHST': ['2010/01/19 06:00 [entrez]',\n",
       "   '2010/01/19 06:00 [pubmed]',\n",
       "   '2010/03/13 06:00 [medline]'],\n",
       "  'AID': ['0913559107 [pii]', '10.1073/pnas.0913559107 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Proc Natl Acad Sci U S A. 2010 Jan 26;107(4):1678-83. doi: 10.1073/pnas.0913559107. Epub 2010 Jan 8.'},\n",
       " {'PMID': '20054296',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20100217',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1476-4687 (Electronic) 0028-0836 (Linking)',\n",
       "  'VI': '463',\n",
       "  'IP': '7280',\n",
       "  'DP': '2010 Jan 28',\n",
       "  'TI': 'A role for the elongator complex in zygotic paternal genome demethylation.',\n",
       "  'PG': '554-8',\n",
       "  'LID': '10.1038/nature08732 [doi]',\n",
       "  'AB': 'The life cycle of mammals begins when a sperm enters an egg. Immediately after fertilization, both the maternal and paternal genomes undergo dramatic reprogramming to prepare for the transition from germ cell to somatic cell transcription programs. One of the molecular events that takes place during this transition is the demethylation of the paternal genome. Despite extensive efforts, the factors responsible for paternal DNA demethylation have not been identified. To search for such factors, we developed a live cell imaging system that allows us to monitor the paternal DNA methylation state in zygotes. Through short-interfering-RNA-mediated knockdown in mouse zygotes, we identified Elp3 (also called KAT9), a component of the elongator complex, to be important for paternal DNA demethylation. We demonstrate that knockdown of Elp3 impairs paternal DNA demethylation as indicated by reporter binding, immunostaining and bisulphite sequencing. Similar results were also obtained when other elongator components, Elp1 and Elp4, were knocked down. Importantly, injection of messenger RNA encoding the Elp3 radical SAM domain mutant, but not the HAT domain mutant, into MII oocytes before fertilization also impaired paternal DNA demethylation, indicating that the SAM radical domain is involved in the demethylation process. Our study not only establishes a critical role for the elongator complex in zygotic paternal genome demethylation, but also indicates that the demethylation process may be mediated through a reaction that requires an intact radical SAM domain.',\n",
       "  'FAU': ['Okada, Yuki',\n",
       "   'Yamagata, Kazuo',\n",
       "   'Hong, Kwonho',\n",
       "   'Wakayama, Teruhiko',\n",
       "   'Zhang, Yi'],\n",
       "  'AU': ['Okada Y', 'Yamagata K', 'Hong K', 'Wakayama T', 'Zhang Y'],\n",
       "  'AD': 'Howard Hughes Medical Institute, Chapel Hill, North Carolina 27599-7295, USA.',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['Howard Hughes Medical Institute/United States',\n",
       "   'R01 GM068804/GM/NIGMS NIH HHS/United States',\n",
       "   'R01 GM068804-07/GM/NIGMS NIH HHS/United States'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20100106',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Nature',\n",
       "  'JT': 'Nature',\n",
       "  'JID': '0410462',\n",
       "  'RN': ['EC 2.3.1.48 (Elp3 protein, mouse)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Animals',\n",
       "   'Cell Line',\n",
       "   'Cells, Cultured',\n",
       "   '*DNA Methylation',\n",
       "   'Embryonic Development/*genetics',\n",
       "   'Female',\n",
       "   'Gene Knockdown Techniques',\n",
       "   'Genome/*genetics',\n",
       "   'Histone Acetyltransferases/*metabolism',\n",
       "   'Male',\n",
       "   'Mice',\n",
       "   'Mice, Inbred C57BL',\n",
       "   'Mutation/genetics',\n",
       "   'Protein Structure, Tertiary/genetics',\n",
       "   'Zygote/*metabolism'],\n",
       "  'PMC': 'PMC2834414',\n",
       "  'MID': ['NIHMS165690'],\n",
       "  'EDAT': '2010/01/08 06:00',\n",
       "  'MHDA': '2010/02/18 06:00',\n",
       "  'CRDT': ['2010/01/08 06:00'],\n",
       "  'PHST': ['2009/09/30 00:00 [received]',\n",
       "   '2009/12/09 00:00 [accepted]',\n",
       "   '2010/01/08 06:00 [entrez]',\n",
       "   '2010/01/08 06:00 [pubmed]',\n",
       "   '2010/02/18 06:00 [medline]'],\n",
       "  'AID': ['nature08732 [pii]', '10.1038/nature08732 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Nature. 2010 Jan 28;463(7280):554-8. doi: 10.1038/nature08732. Epub 2010 Jan 6.'},\n",
       " {'PMID': '19779648',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20100202',\n",
       "  'LR': '20190513',\n",
       "  'IS': '1745-7270 (Electronic) 1672-9145 (Linking)',\n",
       "  'VI': '41',\n",
       "  'IP': '10',\n",
       "  'DP': '2009 Oct',\n",
       "  'TI': 'Human Elongator complex is involved in cell cycle and suppresses cell growth in 293T human embryonic kidney cells.',\n",
       "  'PG': '831-8',\n",
       "  'AB': 'Elongator complex has been associated with hyperphosphorylated RNA polymerase II and is known to play critical roles in transcriptional elongation, as well as in tRNA modification and exocytosis. However, the specific mechanism of how human Elongator complex regulates cell growth and cell cycle remains unclear. To investigate the composition of human Elongator complex and its effects on cell growth, 293T cells were established that stably overexpressed Flag-Elp3 and Flag-Elp4. By using anti-Flag M2 antibody-bound resin, a core Elongator complex was purified from cells that stably overexpressed Flag-Elp3. No Elongator complex was purified from cells stably transfected with pFlagCMV4-Elp4. Interestingly, the cell growth was inhibited in 293T cells transfected with pFlagCMV4- Elp3. Flow cytometry analysis showed that most of the cells stably overexpressing Flag-Elp3 were found in G1 stage, indicating a role of the core Elongator in the G1 checkpoint for the regulation of cell cycle. We observed increased basal transcription and remarkably enhanced transcription stimulated by VP16 in 293T cells overexpressing Flag-Elp3. The transcription could also be synergistically activated by overexpressing both Elp3 and Elp4. Taken together, our results suggested that the core Elongator complex formed by Elp1, Elp2, and Elp3 was rather stable, whereas Elp4, Elp5, and Elp6 might loosely contact and work together with the core Elongator to regulate cell functions.',\n",
       "  'FAU': ['Gu, Junxia',\n",
       "   'Sun, Dongmei',\n",
       "   'Zheng, Qiping',\n",
       "   'Wang, Xiaochun',\n",
       "   'Yang, Huicui',\n",
       "   'Miao, Jingcheng',\n",
       "   'Jiang, Jingting',\n",
       "   'Wei, Wenxiang'],\n",
       "  'AU': ['Gu J',\n",
       "   'Sun D',\n",
       "   'Zheng Q',\n",
       "   'Wang X',\n",
       "   'Yang H',\n",
       "   'Miao J',\n",
       "   'Jiang J',\n",
       "   'Wei W'],\n",
       "  'AD': 'Department of Cellular and Molecular Biology, School of Medicine, Suzhou University, Suzhou, China.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'PL': 'China',\n",
       "  'TA': 'Acta Biochim Biophys Sin (Shanghai)',\n",
       "  'JT': 'Acta biochimica et biophysica Sinica',\n",
       "  'JID': '101206716',\n",
       "  'RN': ['0 (ELP4 protein, human)',\n",
       "   '0 (Nerve Tissue Proteins)',\n",
       "   'EC 2.3.1.48 (ELP3 protein, human)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Cell Cycle/*physiology',\n",
       "   'Cell Proliferation',\n",
       "   'Histone Acetyltransferases/*physiology',\n",
       "   'Humans',\n",
       "   'Kidney/*cytology/embryology',\n",
       "   'Nerve Tissue Proteins/*physiology',\n",
       "   'Phosphorylation'],\n",
       "  'EDAT': '2009/09/26 06:00',\n",
       "  'MHDA': '2010/02/03 06:00',\n",
       "  'CRDT': ['2009/09/26 06:00'],\n",
       "  'PHST': ['2009/09/26 06:00 [entrez]',\n",
       "   '2009/09/26 06:00 [pubmed]',\n",
       "   '2010/02/03 06:00 [medline]'],\n",
       "  'AID': ['10.1093/abbs/gmp072 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Acta Biochim Biophys Sin (Shanghai). 2009 Oct;41(10):831-8. doi: 10.1093/abbs/gmp072.'},\n",
       " {'PMID': '19682046',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20090911',\n",
       "  'LR': '20090817',\n",
       "  'IS': '1528-1167 (Electronic) 0013-9580 (Linking)',\n",
       "  'VI': '50 Suppl 7',\n",
       "  'DP': '2009 Aug',\n",
       "  'TI': 'From rolandic epilepsy to continuous spike-and-waves during sleep and Landau-Kleffner syndromes: insights into possible genetic factors.',\n",
       "  'PG': '25-8',\n",
       "  'LID': '10.1111/j.1528-1167.2009.02214.x [doi]',\n",
       "  'AB': 'Epilepsy is a frequent neurologic disease in childhood, characterized by recurrent seizures and sometimes with major effects on social, behavioral, and cognitive development. Childhood focal epilepsies particularly are age-related diseases mainly occurring during developmental critical periods. A complex interplay between brain development and maturation processes and susceptibility genes may contribute to the development of various childhood epileptic syndromes associated with language and cognitive deficits. Indeed, the Landau-Kleffner syndrome (LKS), the continuous spike-and-waves during sleep syndrome (CSWS), and the benign childhood epilepsy with centrotemporal spikes (BCECTS) or benign rolandic epilepsy, are different entities that are considered as part of a single continuous spectrum of disorders. Genetic predisposition with simple to complex modes of inheritance has long been suspected for this wide group of childhood focal epilepsies. Recent reports on the involvement of the SRPX2 and ELP4 genes with possible roles in cell motility, migration, and adhesion have provided first insights into the complex molecular bases of childhood focal epilepsies.',\n",
       "  'FAU': ['Rudolf, Gabrielle',\n",
       "   'Valenti, Maria P',\n",
       "   'Hirsch, Edouard',\n",
       "   'Szepetowski, Pierre'],\n",
       "  'AU': ['Rudolf G', 'Valenti MP', 'Hirsch E', 'Szepetowski P'],\n",
       "  'AD': \"Service de Neurologie, Hopitaux Universitaires de Strasbourg, 1 place de l'Hopital BP 426, Strasbourg cedex, France. Gabrielle.Rudolf@chrustrasbourg.fr\",\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Epilepsia',\n",
       "  'JT': 'Epilepsia',\n",
       "  'JID': '2983306R',\n",
       "  'RN': ['0 (ELP4 protein, human)',\n",
       "   '0 (Nerve Tissue Proteins)',\n",
       "   '0 (SRPX2 protein, human)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Cognition Disorders/genetics',\n",
       "   'Electroencephalography',\n",
       "   'Epilepsies, Partial/*genetics',\n",
       "   'Epilepsy, Rolandic/*genetics',\n",
       "   'Humans',\n",
       "   'Landau-Kleffner Syndrome/*genetics',\n",
       "   'Nerve Tissue Proteins/genetics',\n",
       "   'Prognosis',\n",
       "   'Sleep/physiology'],\n",
       "  'EDAT': '2009/08/18 09:00',\n",
       "  'MHDA': '2009/09/12 06:00',\n",
       "  'CRDT': ['2009/08/18 09:00'],\n",
       "  'PHST': ['2009/08/18 09:00 [entrez]',\n",
       "   '2009/08/18 09:00 [pubmed]',\n",
       "   '2009/09/12 06:00 [medline]'],\n",
       "  'AID': ['EPI2214 [pii]', '10.1111/j.1528-1167.2009.02214.x [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Epilepsia. 2009 Aug;50 Suppl 7:25-8. doi: 10.1111/j.1528-1167.2009.02214.x.'},\n",
       " {'PMID': '19500300',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20091123',\n",
       "  'LR': '20140115',\n",
       "  'IS': '1365-313X (Electronic) 0960-7412 (Linking)',\n",
       "  'VI': '60',\n",
       "  'IP': '1',\n",
       "  'DP': '2009 Oct',\n",
       "  'TI': 'Elongator mediates ABA responses, oxidative stress resistance and anthocyanin biosynthesis in Arabidopsis.',\n",
       "  'PG': '79-90',\n",
       "  'LID': '10.1111/j.1365-313X.2009.03931.x [doi]',\n",
       "  'AB': 'Elongator is a histone acetyl-transferase complex consisting of six subunits, and is highly conserved in eukaryotic organisms. Here, we isolated two novel mutants, elp2 and elp6, during a genetic screening for ABA-hypersensitive Arabidopsis mutants. Map-based cloning identified ELP2 and ELP6, which encode the orthologs of the yeast Elongator subunits, ELP2 and ELP6, respectively. Another Elongator subunit mutant, elp4/elo1, was obtained from the SALK T-DNA collection. The elp1/abo1/elo2 mutant was isolated in a previous study. All four of the Elongator mutants had narrow leaves, reduced root growth, ABA hypersensitivity and an increased accumulation of anthocyanins. Mutations in the core subcomplex subunits ELP1/ABO1 and ELP2, but not in the accessory subcomplex subunits ELP4/ELO1 and ELP6, caused stomatal closing to be supersensitive to ABA. In addition, the four mutants were all more resistant than the wild type to oxidative stress produced by methyl viologen, and to CsCl. Gene expression analysis indicated that the four mutants had increased transcript levels of CAT3 under normal conditions, increased transcript levels of ZAT10 when treated with ABA and reduced transcript levels of MYBL2, which encodes a single-repeat MYB protein, acting as a negative regulator of anthocyanin biosynthesis. Our results suggest that Elongator plays crucial roles in regulating plant responses to ABA, oxidative stress resistance and anthocyanin biosynthesis in Arabidopsis.',\n",
       "  'FAU': ['Zhou, Xiaofeng',\n",
       "   'Hua, Deping',\n",
       "   'Chen, Zhizhong',\n",
       "   'Zhou, Zhengjun',\n",
       "   'Gong, Zhizhong'],\n",
       "  'AU': ['Zhou X', 'Hua D', 'Chen Z', 'Zhou Z', 'Gong Z'],\n",
       "  'AD': 'State Key Laboratory of Plant Physiology and Biochemistry, College of Biological Sciences, China Agricultural University, Beijing 100193, China.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20090523',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Plant J',\n",
       "  'JT': 'The Plant journal : for cell and molecular biology',\n",
       "  'JID': '9207397',\n",
       "  'RN': ['0 (Anthocyanins)',\n",
       "   '0 (Arabidopsis Proteins)',\n",
       "   '0 (DNA, Bacterial)',\n",
       "   '0 (RNA, Plant)',\n",
       "   '0 (T-DNA)',\n",
       "   '72S9A8J5GW (Abscisic Acid)',\n",
       "   'EC 2.3.1.48 (ELP2 protein, Arabidopsis)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Abscisic Acid/*metabolism',\n",
       "   'Anthocyanins/*biosynthesis',\n",
       "   'Arabidopsis/enzymology/*genetics/growth & development',\n",
       "   'Arabidopsis Proteins/genetics/*metabolism',\n",
       "   'Chromosome Mapping',\n",
       "   'Cloning, Molecular',\n",
       "   'DNA, Bacterial/genetics',\n",
       "   'Gene Expression Regulation, Plant',\n",
       "   'Genetic Complementation Test',\n",
       "   'Germination',\n",
       "   'Histone Acetyltransferases/genetics/*metabolism',\n",
       "   'Mutagenesis, Insertional',\n",
       "   'Mutation',\n",
       "   '*Oxidative Stress',\n",
       "   'RNA, Plant/genetics',\n",
       "   'Seeds/genetics/growth & development',\n",
       "   'Sequence Analysis, DNA'],\n",
       "  'EDAT': '2009/06/09 09:00',\n",
       "  'MHDA': '2009/12/16 06:00',\n",
       "  'CRDT': ['2009/06/09 09:00'],\n",
       "  'PHST': ['2009/06/09 09:00 [entrez]',\n",
       "   '2009/06/09 09:00 [pubmed]',\n",
       "   '2009/12/16 06:00 [medline]'],\n",
       "  'AID': ['TPJ3931 [pii]', '10.1111/j.1365-313X.2009.03931.x [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Plant J. 2009 Oct;60(1):79-90. doi: 10.1111/j.1365-313X.2009.03931.x. Epub 2009 May 23.'},\n",
       " {'PMID': '19172991',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20100113',\n",
       "  'LR': '20181113',\n",
       "  'IS': '1476-5438 (Electronic) 1018-4813 (Linking)',\n",
       "  'VI': '17',\n",
       "  'IP': '9',\n",
       "  'DP': '2009 Sep',\n",
       "  'TI': 'Centrotemporal sharp wave EEG trait in rolandic epilepsy maps to Elongator Protein Complex 4 (ELP4).',\n",
       "  'PG': '1171-81',\n",
       "  'LID': '10.1038/ejhg.2008.267 [doi]',\n",
       "  'AB': 'Rolandic epilepsy (RE) is the most common human epilepsy, affecting children between 3 and 12 years of age, boys more often than girls (3:2). Focal sharp waves in the centrotemporal area define the electroencephalographic (EEG) trait for the syndrome, are a feature of several related childhood epilepsies and are frequently observed in common developmental disorders (eg, speech dyspraxia, attention deficit hyperactivity disorder and developmental coordination disorder). Here we report the first genome-wide linkage scan in RE for the EEG trait, centrotemporal sharp waves (CTS), with genome-wide linkage of CTS to 11p13 (HLOD 4.30). Pure likelihood statistical analysis refined our linkage peak by fine mapping CTS to variants in Elongator Protein Complex 4 (ELP4) in two independent data sets; the strongest evidence was with rs986527 in intron 9 of ELP4, providing a likelihood ratio of 629:1 (P=0.0002) in favor of an association. Resequencing of ELP4 coding, flanking and promoter regions revealed no significant exonic polymorphisms. This is the first report of a gene implicated in a common focal epilepsy and the first human disease association of ELP4. ELP4 is a component of the Elongator complex, involved in transcription and tRNA modification. Elongator depletion results in the brain-specific downregulation of genes implicated in cell motility and migration. We hypothesize that a non-coding mutation in ELP4 impairs brain-specific Elongator-mediated interaction of genes implicated in brain development, resulting in susceptibility to seizures and neurodevelopmental disorders.',\n",
       "  'FAU': ['Strug, Lisa J',\n",
       "   'Clarke, Tara',\n",
       "   'Chiang, Theodore',\n",
       "   'Chien, Minchen',\n",
       "   'Baskurt, Zeynep',\n",
       "   'Li, Weili',\n",
       "   'Dorfman, Ruslan',\n",
       "   'Bali, Bhavna',\n",
       "   'Wirrell, Elaine',\n",
       "   'Kugler, Steven L',\n",
       "   'Mandelbaum, David E',\n",
       "   'Wolf, Steven M',\n",
       "   'McGoldrick, Patricia',\n",
       "   'Hardison, Huntley',\n",
       "   'Novotny, Edward J',\n",
       "   'Ju, Jingyue',\n",
       "   'Greenberg, David A',\n",
       "   'Russo, James J',\n",
       "   'Pal, Deb K'],\n",
       "  'AU': ['Strug LJ',\n",
       "   'Clarke T',\n",
       "   'Chiang T',\n",
       "   'Chien M',\n",
       "   'Baskurt Z',\n",
       "   'Li W',\n",
       "   'Dorfman R',\n",
       "   'Bali B',\n",
       "   'Wirrell E',\n",
       "   'Kugler SL',\n",
       "   'Mandelbaum DE',\n",
       "   'Wolf SM',\n",
       "   'McGoldrick P',\n",
       "   'Hardison H',\n",
       "   'Novotny EJ',\n",
       "   'Ju J',\n",
       "   'Greenberg DA',\n",
       "   'Russo JJ',\n",
       "   'Pal DK'],\n",
       "  'AD': 'Child Health Evaluative Sciences, The Hospital for Sick Children, Toronto, Ontario, Canada.',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['R01 NS047530/NS/NINDS NIH HHS/United States',\n",
       "   'R01 NS047530-04/NS/NINDS NIH HHS/United States',\n",
       "   'R03 HG004314/HG/NHGRI NIH HHS/United States',\n",
       "   'NS047530/NS/NINDS NIH HHS/United States'],\n",
       "  'PT': ['Journal Article',\n",
       "   'Multicenter Study',\n",
       "   'Research Support, N.I.H., Extramural',\n",
       "   \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20090128',\n",
       "  'PL': 'England',\n",
       "  'TA': 'Eur J Hum Genet',\n",
       "  'JT': 'European journal of human genetics : EJHG',\n",
       "  'JID': '9302235',\n",
       "  'RN': ['0 (ELP4 protein, human)', '0 (Nerve Tissue Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Child',\n",
       "   'Child, Preschool',\n",
       "   'Chromosomes, Human, Pair 11/genetics',\n",
       "   'DNA Mutational Analysis',\n",
       "   '*Electroencephalography',\n",
       "   'Epilepsy, Rolandic/*genetics/pathology/*physiopathology',\n",
       "   'Female',\n",
       "   'Genetic Linkage',\n",
       "   'Genetic Predisposition to Disease',\n",
       "   'Genome-Wide Association Study',\n",
       "   'Genotype',\n",
       "   'Humans',\n",
       "   'Lod Score',\n",
       "   'Male',\n",
       "   'Microsatellite Repeats/genetics',\n",
       "   'Nerve Tissue Proteins/*genetics',\n",
       "   'Polymorphism, Single Nucleotide',\n",
       "   'Temporal Lobe/physiopathology'],\n",
       "  'PMC': 'PMC2729813',\n",
       "  'MID': ['NIHMS82545'],\n",
       "  'EDAT': '2009/01/29 09:00',\n",
       "  'MHDA': '2010/01/14 06:00',\n",
       "  'CRDT': ['2009/01/29 09:00'],\n",
       "  'PHST': ['2009/01/29 09:00 [entrez]',\n",
       "   '2009/01/29 09:00 [pubmed]',\n",
       "   '2010/01/14 06:00 [medline]'],\n",
       "  'AID': ['ejhg2008267 [pii]', '10.1038/ejhg.2008.267 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Eur J Hum Genet. 2009 Sep;17(9):1171-81. doi: 10.1038/ejhg.2008.267. Epub 2009 Jan 28.'},\n",
       " {'PMID': '18534097',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20080826',\n",
       "  'LR': '20131121',\n",
       "  'IS': '0920-5063 (Print) 0920-5063 (Linking)',\n",
       "  'VI': '19',\n",
       "  'IP': '6',\n",
       "  'DP': '2008',\n",
       "  'TI': 'Development and characterisation of novel cross-linked bio-elastomeric materials.',\n",
       "  'PG': '785-99',\n",
       "  'LID': '10.1163/156856208784522038 [doi]',\n",
       "  'AB': 'Recombinantly-engineered elastin-like polypeptides (ELPs) possess many of the favourable attributes of the native elastin protein, making them an attractive option for designing biomaterials for tissue-engineering applications. The focus of this study was to synthesize and characterise the bulk material properties of two ELP sequences, ELP2 and ELP4, cross-linked with lysine diisocyanate (LDI). The two distinct ELPs consist of repeating hydrophobic and hydrophilic cross-linking domains in a block co-polymer structure, however, differ by the number of respective domains. Depending on the conditions sets for the cross-linking reactions, two different ELP-based materials were synthesized: a gel-like relatively non-porous material and a porous foam-like material, from both ELP sequences. The physical material properties were characterised by scanning electron microscopy, compression testing, differential calorimetry analysis and swelling analysis. The bulk material properties were found to vary depending on the ELP sequence investigated. ELP gels were also found to have a more dense solidified morphology, lower compressive moduli, higher melting temperature and greater swelling capacity than the porous ELP foams. These novel cross-linked bio-elastomeric materials show promising properties for soft tissue replacement, particularly in load-bearing applications.',\n",
       "  'FAU': ['Srokowski, Elizabeth M', 'Woodhouse, Kimberly A'],\n",
       "  'AU': ['Srokowski EM', 'Woodhouse KA'],\n",
       "  'AD': 'Department of Chemical Engineering and Applied Chemistry, University of Toronto, Toronto, ON, Canada M5S 3E5.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'PL': 'England',\n",
       "  'TA': 'J Biomater Sci Polym Ed',\n",
       "  'JT': 'Journal of biomaterials science. Polymer edition',\n",
       "  'JID': '9007393',\n",
       "  'RN': ['0 (Cross-Linking Reagents)',\n",
       "   '0 (Gels)',\n",
       "   '0 (Isocyanates)',\n",
       "   '0 (Peptides)',\n",
       "   '0 (lysine diisocyanate)',\n",
       "   '9007-58-3 (Elastin)',\n",
       "   'K3Z4F929H6 (Lysine)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Amino Acid Sequence',\n",
       "   'Biomimetic Materials/chemical synthesis/chemistry',\n",
       "   'Calorimetry, Differential Scanning',\n",
       "   'Cross-Linking Reagents/*chemistry',\n",
       "   'Elasticity',\n",
       "   'Elastin/*chemistry',\n",
       "   'Gels/*chemical synthesis/chemistry',\n",
       "   'Isocyanates/*chemistry',\n",
       "   'Lysine/*analogs & derivatives/chemistry',\n",
       "   'Microscopy, Electron, Scanning',\n",
       "   'Molecular Sequence Data',\n",
       "   'Peptides/*chemical synthesis/chemistry',\n",
       "   'Surface Properties',\n",
       "   'Transition Temperature'],\n",
       "  'EDAT': '2008/06/07 09:00',\n",
       "  'MHDA': '2008/08/30 09:00',\n",
       "  'CRDT': ['2008/06/07 09:00'],\n",
       "  'PHST': ['2008/06/07 09:00 [pubmed]',\n",
       "   '2008/08/30 09:00 [medline]',\n",
       "   '2008/06/07 09:00 [entrez]'],\n",
       "  'AID': ['10.1163/156856208784522038 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'J Biomater Sci Polym Ed. 2008;19(6):785-99. doi: 10.1163/156856208784522038.'},\n",
       " {'PMID': '18282108',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20080604',\n",
       "  'LR': '20191008',\n",
       "  'IS': '1553-7404 (Electronic) 1553-7390 (Linking)',\n",
       "  'VI': '4',\n",
       "  'IP': '2',\n",
       "  'DP': '2008 Feb',\n",
       "  'TI': 'Subfunctionalization of duplicated zebrafish pax6 genes by cis-regulatory divergence.',\n",
       "  'PG': 'e29',\n",
       "  'LID': '10.1371/journal.pgen.0040029 [doi]',\n",
       "  'AB': \"Gene duplication is a major driver of evolutionary divergence. In most vertebrates a single PAX6 gene encodes a transcription factor required for eye, brain, olfactory system, and pancreas development. In zebrafish, following a postulated whole-genome duplication event in an ancestral teleost, duplicates pax6a and pax6b jointly fulfill these roles. Mapping of the homozygously viable eye mutant sunrise identified a homeodomain missense change in pax6b, leading to loss of target binding. The mild phenotype emphasizes role-sharing between the co-orthologues. Meticulous mapping of isolated BACs identified perturbed synteny relationships around the duplicates. This highlights the functional conservation of pax6 downstream (3') control sequences, which in most vertebrates reside within the introns of a ubiquitously expressed neighbour gene, ELP4, whose pax6a-linked exons have been lost in zebrafish. Reporter transgenic studies in both mouse and zebrafish, combined with analysis of vertebrate sequence conservation, reveal loss and retention of specific cis-regulatory elements, correlating strongly with the diverged expression of co-orthologues, and providing clear evidence for evolution by subfunctionalization.\",\n",
       "  'FAU': ['Kleinjan, Dirk A',\n",
       "   'Bancewicz, Ruth M',\n",
       "   'Gautier, Philippe',\n",
       "   'Dahm, Ralf',\n",
       "   'Schonthaler, Helia B',\n",
       "   'Damante, Giuseppe',\n",
       "   'Seawright, Anne',\n",
       "   'Hever, Ann M',\n",
       "   'Yeyati, Patricia L',\n",
       "   'van Heyningen, Veronica',\n",
       "   'Coutinho, Pedro'],\n",
       "  'AU': ['Kleinjan DA',\n",
       "   'Bancewicz RM',\n",
       "   'Gautier P',\n",
       "   'Dahm R',\n",
       "   'Schonthaler HB',\n",
       "   'Damante G',\n",
       "   'Seawright A',\n",
       "   'Hever AM',\n",
       "   'Yeyati PL',\n",
       "   'van Heyningen V',\n",
       "   'Coutinho P'],\n",
       "  'AD': 'Medical Research Council Human Genetics Unit, Western General Hospital, Edinburgh, United Kingdom.',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['MC_U127527199/Medical Research Council/United Kingdom'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'PL': 'United States',\n",
       "  'TA': 'PLoS Genet',\n",
       "  'JT': 'PLoS genetics',\n",
       "  'JID': '101239074',\n",
       "  'RN': ['0 (DNA Primers)',\n",
       "   '0 (Eye Proteins)',\n",
       "   '0 (Homeodomain Proteins)',\n",
       "   '0 (PAX6 Transcription Factor)',\n",
       "   '0 (PAX6 protein, human)',\n",
       "   '0 (Paired Box Transcription Factors)',\n",
       "   '0 (Pax6 protein, mouse)',\n",
       "   '0 (Repressor Proteins)',\n",
       "   '0 (Zebrafish Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Animals',\n",
       "   'Animals, Genetically Modified',\n",
       "   'Base Sequence',\n",
       "   'Chromosomes, Artificial, Bacterial/genetics',\n",
       "   'Computational Biology',\n",
       "   'DNA Primers/genetics',\n",
       "   'Enhancer Elements, Genetic',\n",
       "   'Evolution, Molecular',\n",
       "   'Eye Abnormalities/embryology/genetics',\n",
       "   'Eye Proteins/*genetics',\n",
       "   '*Gene Duplication',\n",
       "   'Gene Expression Regulation, Developmental',\n",
       "   'Genes, Homeobox',\n",
       "   'Genes, Reporter',\n",
       "   'Genetic Complementation Test',\n",
       "   'Genetic Linkage',\n",
       "   'Homeodomain Proteins/*genetics',\n",
       "   'Mice',\n",
       "   'Mice, Transgenic',\n",
       "   'Models, Genetic',\n",
       "   'Molecular Sequence Data',\n",
       "   'Mutation, Missense',\n",
       "   'PAX6 Transcription Factor',\n",
       "   'Paired Box Transcription Factors/*genetics',\n",
       "   'Phenotype',\n",
       "   'Repressor Proteins/*genetics',\n",
       "   'Sequence Homology, Nucleic Acid',\n",
       "   'Zebrafish/abnormalities/embryology/*genetics',\n",
       "   'Zebrafish Proteins/*genetics'],\n",
       "  'PMC': 'PMC2242813',\n",
       "  'COIS': ['Competing interests. The authors have declared that no competing interests exist.'],\n",
       "  'EDAT': '2008/02/20 09:00',\n",
       "  'MHDA': '2008/06/05 09:00',\n",
       "  'CRDT': ['2008/02/20 09:00'],\n",
       "  'PHST': ['2006/06/27 00:00 [received]',\n",
       "   '2007/12/21 00:00 [accepted]',\n",
       "   '2008/02/20 09:00 [pubmed]',\n",
       "   '2008/06/05 09:00 [medline]',\n",
       "   '2008/02/20 09:00 [entrez]'],\n",
       "  'AID': ['06-PLGE-RA-0260 [pii]', '10.1371/journal.pgen.0040029 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'PLoS Genet. 2008 Feb;4(2):e29. doi: 10.1371/journal.pgen.0040029.'},\n",
       " {'PMID': '17679951',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20070918',\n",
       "  'LR': '20161124',\n",
       "  'IS': '1090-0535 (Electronic) 1090-0535 (Linking)',\n",
       "  'VI': '13',\n",
       "  'DP': '2007 Jul 23',\n",
       "  'TI': \"A deletion 3' to the PAX6 gene in familial aniridia cases.\",\n",
       "  'PG': '1245-50',\n",
       "  'AB': \"PURPOSE: PAX6 mutations cause aniridia as well as other various congenital eye abnormalities. Aniridia can be due to both point mutations and chromosomal deletions/rearrangements. Therefore, a complete search for PAX6 gene alterations in aniridia subjects requires a technically complex approach involving the comprehension of fluorescence in situ hybridization (FISH) analysis. In the present study, an Italian casistic of aniridia patients has been investigated and a quantitative polymerase chain reaction (PCR) assay to detect PAX6 gene deletions was set up. METHODS: Twenty-one aniridia patients were screened for point mutations (missense, nonsense, splicing-affecting, and short insertion/deletion) by using single-stranded conformational polymorphism (SSCP) and denaturing high performance liquid chromatography (dHPLC). To reveal deletions not detectable by SSCP or dHPLC, a quantitative PCR approach was set up for the PAX6 structural gene and for regions 5' and 3' to it at the level of WT1 and ELP4, respectively. RESULTS: Point mutations were found in 7 out of 21 patients. Three out of twenty-one patients showed deletions at the level of the PAX6 structural gene. In addition, two familial cases showed an undamaged PAX6 gene but a deletion in the region 3' to it at level of the ELP4 gene. In one of the families, the presence of the deletion has been confirmed by linkage analysis of polymorphic markers. CONCLUSIONS: In our casistic, a significant fraction of familial aniridia patients appears to be caused by a 3' deletion to PAX6, suggesting that evaluation of this alteration should be included in routine procedures of aniridia patients analysis. The quantitative PCR assay described here represents a simple approach to accomplish this task.\",\n",
       "  'FAU': [\"D'Elia, Angela Valentina\",\n",
       "   'Pellizzari, Lucia',\n",
       "   'Fabbro, Dora',\n",
       "   'Pianta, Annalisa',\n",
       "   'Divizia, Maria Teresa',\n",
       "   'Rinaldi, Rosanna',\n",
       "   'Grammatico, Barbara',\n",
       "   'Grammatico, Paola',\n",
       "   'Arduino, Carlo',\n",
       "   'Damante, Giuseppe'],\n",
       "  'AU': [\"D'Elia AV\",\n",
       "   'Pellizzari L',\n",
       "   'Fabbro D',\n",
       "   'Pianta A',\n",
       "   'Divizia MT',\n",
       "   'Rinaldi R',\n",
       "   'Grammatico B',\n",
       "   'Grammatico P',\n",
       "   'Arduino C',\n",
       "   'Damante G'],\n",
       "  'AD': 'Dipartimento di Scienze e Tecnologie Biomediche, Universita di Udine, Udine, Italy.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20070723',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Mol Vis',\n",
       "  'JT': 'Molecular vision',\n",
       "  'JID': '9605351',\n",
       "  'RN': ['0 (ELP4 protein, human)',\n",
       "   '0 (Eye Proteins)',\n",
       "   '0 (Homeodomain Proteins)',\n",
       "   '0 (Nerve Tissue Proteins)',\n",
       "   '0 (PAX6 Transcription Factor)',\n",
       "   '0 (PAX6 protein, human)',\n",
       "   '0 (Paired Box Transcription Factors)',\n",
       "   '0 (Repressor Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Aniridia/*genetics',\n",
       "   'Databases, Nucleic Acid',\n",
       "   'Eye Proteins/*genetics',\n",
       "   'Female',\n",
       "   'Genetic Linkage',\n",
       "   'Genome, Human/genetics',\n",
       "   'Homeodomain Proteins/*genetics',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   'Nerve Tissue Proteins/genetics',\n",
       "   'PAX6 Transcription Factor',\n",
       "   'Paired Box Transcription Factors/*genetics',\n",
       "   'Pedigree',\n",
       "   'Point Mutation/genetics',\n",
       "   'Repressor Proteins/*genetics',\n",
       "   'Reverse Transcriptase Polymerase Chain Reaction',\n",
       "   '*Sequence Deletion'],\n",
       "  'EDAT': '2007/08/08 09:00',\n",
       "  'MHDA': '2007/09/19 09:00',\n",
       "  'CRDT': ['2007/08/08 09:00'],\n",
       "  'PHST': ['2007/08/08 09:00 [pubmed]',\n",
       "   '2007/09/19 09:00 [medline]',\n",
       "   '2007/08/08 09:00 [entrez]'],\n",
       "  'AID': ['v13/a135 [pii]'],\n",
       "  'PST': 'epublish',\n",
       "  'SO': 'Mol Vis. 2007 Jul 23;13:1245-50.'},\n",
       " {'PMID': '15894610',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20050804',\n",
       "  'LR': '20181113',\n",
       "  'IS': '0027-8424 (Print) 0027-8424 (Linking)',\n",
       "  'VI': '102',\n",
       "  'IP': '21',\n",
       "  'DP': '2005 May 24',\n",
       "  'TI': 'The elongata mutants identify a functional Elongator complex in plants with a role in cell proliferation during organ growth.',\n",
       "  'PG': '7754-9',\n",
       "  'AB': 'The key enzyme for transcription of protein-encoding genes in eukaryotes is RNA polymerase II (RNAPII). The recruitment of this enzyme during transcription initiation and its passage along the template during transcription elongation is regulated through the association and dissociation of several complexes. Elongator is a histone acetyl transferase complex, consisting of six subunits (ELP1-ELP6), that copurifies with the elongating RNAPII in yeast and humans. We demonstrate that point mutations in three Arabidopsis thaliana genes, encoding homologs of the yeast Elongator subunits ELP1, ELP3 (histone acetyl transferase), and ELP4 are responsible for the phenotypes of the elongata2 (elo2), elo3, and elo1 mutants, respectively. The elo mutants are characterized by narrow leaves and reduced root growth that results from a decreased cell division rate. Morphological and molecular phenotypes show that the ELONGATA (ELO) genes function in the same biological process and the epistatic interactions between the ELO genes can be explained by the model of complex formation in yeast. Furthermore, the plant Elongator complex is genetically positioned in the process of RNAPII-mediated transcription downstream of Mediator. Our data indicate that the Elongator complex is evolutionarily conserved in structure and function but reveal that the mechanism by which it stimulates cell proliferation is different in yeast and plants.',\n",
       "  'FAU': ['Nelissen, Hilde',\n",
       "   'Fleury, Delphine',\n",
       "   'Bruno, Leonardo',\n",
       "   'Robles, Pedro',\n",
       "   'De Veylder, Lieven',\n",
       "   'Traas, Jan',\n",
       "   'Micol, Jose Luis',\n",
       "   'Van Montagu, Marc',\n",
       "   'Inze, Dirk',\n",
       "   'Van Lijsebettens, Mieke'],\n",
       "  'AU': ['Nelissen H',\n",
       "   'Fleury D',\n",
       "   'Bruno L',\n",
       "   'Robles P',\n",
       "   'De Veylder L',\n",
       "   'Traas J',\n",
       "   'Micol JL',\n",
       "   'Van Montagu M',\n",
       "   'Inze D',\n",
       "   'Van Lijsebettens M'],\n",
       "  'AD': 'Department of Plant Systems Biology, Flanders Interuniversity Institute for Biotechnology, Ghent University, Technologiepark 927, B-9052 Gent, Belgium.',\n",
       "  'LA': ['eng'],\n",
       "  'SI': ['GENBANK/AJ964957', 'GENBANK/AJ964958'],\n",
       "  'PT': ['Comparative Study',\n",
       "   'Journal Article',\n",
       "   \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20050513',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Proc Natl Acad Sci U S A',\n",
       "  'JT': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'JID': '7505876',\n",
       "  'RN': ['0 (Arabidopsis Proteins)',\n",
       "   '0 (DNA, Complementary)',\n",
       "   '0 (ELO1 protein, Arabidopsis)',\n",
       "   '0 (Multiprotein Complexes)',\n",
       "   'EC 2.3.1.- (Acetyltransferases)',\n",
       "   'EC 2.3.1.48 (ELO3 protein, Arabidopsis)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)',\n",
       "   'EC 2.7.7.- (RNA Polymerase II)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Acetyltransferases/*genetics/metabolism',\n",
       "   'Arabidopsis/anatomy & histology/*genetics/growth & development',\n",
       "   'Arabidopsis Proteins/*genetics/metabolism',\n",
       "   'Base Sequence',\n",
       "   'Cell Division/genetics',\n",
       "   'Conserved Sequence/genetics',\n",
       "   'DNA, Complementary/genetics',\n",
       "   'Epistasis, Genetic',\n",
       "   'Flow Cytometry',\n",
       "   'Gene Components',\n",
       "   'Histone Acetyltransferases',\n",
       "   'Molecular Sequence Data',\n",
       "   'Multiprotein Complexes/*genetics/metabolism',\n",
       "   'Mutation/genetics',\n",
       "   'Nucleic Acid Amplification Techniques',\n",
       "   '*Phenotype',\n",
       "   'Plant Leaves/cytology/genetics',\n",
       "   'Plant Roots/genetics/growth & development',\n",
       "   'Polymorphism, Restriction Fragment Length',\n",
       "   'Protein Structure, Tertiary',\n",
       "   'RNA Polymerase II/*metabolism',\n",
       "   'Sequence Analysis, DNA'],\n",
       "  'PMC': 'PMC1140448',\n",
       "  'EDAT': '2005/05/17 09:00',\n",
       "  'MHDA': '2005/08/05 09:00',\n",
       "  'CRDT': ['2005/05/17 09:00'],\n",
       "  'PHST': ['2005/05/17 09:00 [pubmed]',\n",
       "   '2005/08/05 09:00 [medline]',\n",
       "   '2005/05/17 09:00 [entrez]'],\n",
       "  'AID': ['0502600102 [pii]', '10.1073/pnas.0502600102 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Proc Natl Acad Sci U S A. 2005 May 24;102(21):7754-9. doi: 10.1073/pnas.0502600102. Epub 2005 May 13.'},\n",
       " {'PMID': '15473317',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20041116',\n",
       "  'LR': '20091119',\n",
       "  'IS': '0379-4172 (Print) 0379-4172 (Linking)',\n",
       "  'VI': '31',\n",
       "  'IP': '7',\n",
       "  'DP': '2004 Jul',\n",
       "  'TI': '[The Elp4 subunit of human Elongator complex partially complements the growth defects of yeast ELP4 deletion strain].',\n",
       "  'PG': '668-74',\n",
       "  'AB': 'In this study, we performed in vivo experiments to determine the function of human Elongator subunit Elp4 by using a yeast complementary system. Our results indicated that though human ELP4 was not able to complement the growth defects of the ELP4 deletion mutant strain to high concentration salt, it partially reduced the sensitivity of mutant strain to caffeine, high temperature and 6-AU. Gene expression analysis indicated that human ELP4 partially resumed the slow activation of the PHO5 gene caused by the deletion of yELP4 under the low phosphate concentration. Meanwhile, under the condition of heat shock treatment, hELP4 increased the expression level of SSA3 gene. All these data demonstrated that human ELP4 can partially complement the growth defects and restore the slow activation of certain genes of the yELP4 deletion strain. These results indicate that human Elp4 subunit has similar functions to that of the yeast.',\n",
       "  'FAU': ['Li, Fen',\n",
       "   'Han, Qiu-Ju',\n",
       "   'Luo, Dian-Hui',\n",
       "   'Lu, Jun',\n",
       "   'Huang, Bai-Qu'],\n",
       "  'AU': ['Li F', 'Han QJ', 'Luo DH', 'Lu J', 'Huang BQ'],\n",
       "  'AD': 'Institute of Genetics and Cytology, Northeast Normal University, Changchun 130024, China. lifen2001@yahoo.com',\n",
       "  'LA': ['chi'],\n",
       "  'PT': ['English Abstract',\n",
       "   'Journal Article',\n",
       "   \"Research Support, Non-U.S. Gov't\"],\n",
       "  'PL': 'China',\n",
       "  'TA': 'Yi Chuan Xue Bao',\n",
       "  'JT': 'Yi chuan xue bao = Acta genetica Sinica',\n",
       "  'JID': '7900784',\n",
       "  'RN': ['0 (Carrier Proteins)',\n",
       "   '0 (ELP4 protein, human)',\n",
       "   '0 (HSP70 Heat-Shock Proteins)',\n",
       "   '0 (Nerve Tissue Proteins)',\n",
       "   '0 (Protein Subunits)',\n",
       "   '0 (SSA3 protein, S cerevisiae)',\n",
       "   '0 (Saccharomyces cerevisiae Proteins)',\n",
       "   '0 (thiamine-binding protein)',\n",
       "   'EC 3.1.3.2 (Acid Phosphatase)',\n",
       "   'EC 3.1.3.2 (PHO3 protein, S cerevisiae)',\n",
       "   'EC 3.1.3.2 (PHO5 protein, S cerevisiae)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Acid Phosphatase',\n",
       "   'Amino Acid Sequence',\n",
       "   'Carrier Proteins/genetics',\n",
       "   'HSP70 Heat-Shock Proteins/genetics',\n",
       "   'Hot Temperature',\n",
       "   'Humans',\n",
       "   'Molecular Sequence Data',\n",
       "   'Nerve Tissue Proteins/chemistry/genetics/*physiology',\n",
       "   'Protein Subunits',\n",
       "   'Saccharomyces cerevisiae/*growth & development',\n",
       "   'Saccharomyces cerevisiae Proteins/genetics'],\n",
       "  'EDAT': '2004/10/12 09:00',\n",
       "  'MHDA': '2004/11/17 09:00',\n",
       "  'CRDT': ['2004/10/12 09:00'],\n",
       "  'PHST': ['2004/10/12 09:00 [pubmed]',\n",
       "   '2004/11/17 09:00 [medline]',\n",
       "   '2004/10/12 09:00 [entrez]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Yi Chuan Xue Bao. 2004 Jul;31(7):668-74.'},\n",
       " {'PMID': '15138274',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20040910',\n",
       "  'LR': '20091103',\n",
       "  'IS': '0021-9258 (Print) 0021-9258 (Linking)',\n",
       "  'VI': '279',\n",
       "  'IP': '31',\n",
       "  'DP': '2004 Jul 30',\n",
       "  'TI': 'Molecular architecture, structure-function relationship, and importance of the Elp3 subunit for the RNA binding of holo-elongator.',\n",
       "  'PG': '32087-92',\n",
       "  'AB': 'The molecular architecture of six-subunit yeast holo-Elongator complex was investigated by the use of immunoprecipitation, two-hybrid interaction mapping, and in vitro studies of binary interactions between individual subunits. Surprisingly, Elp2 is dispensable for the integrity of the holo-Elongator complex, and a purified five-subunit elp2 Delta Elongator complex retains histone acetyltransferase activity in vitro. These results indicate that the WD40 repeats in Elp2 are required neither for subunit-subunit interactions within Elongator nor for Elongator interaction with histones during catalysis. Elp2 and Elp4 were largely dispensable for the association of Elongator with nascent RNA transcript in vivo. In contrast, Elongator-RNA interaction requires the Elp3 protein. Together, these data shed light on the structure-function relationship of the Elongator complex.',\n",
       "  'FAU': ['Petrakis, Thodoris G',\n",
       "   'Wittschieben, Birgitte O',\n",
       "   'Svejstrup, Jesper Q'],\n",
       "  'AU': ['Petrakis TG', 'Wittschieben BO', 'Svejstrup JQ'],\n",
       "  'AD': 'Cancer Research UK, London Research Institute, Clare Hall Laboratories, Blanche Lane, South Mimms, Hertfordshire EN6 3LD, United Kingdom.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20040511',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'J Biol Chem',\n",
       "  'JT': 'The Journal of biological chemistry',\n",
       "  'JID': '2985121R',\n",
       "  'RN': ['0 (Histones)',\n",
       "   '0 (Recombinant Proteins)',\n",
       "   '63231-63-0 (RNA)',\n",
       "   'EC 2.3.1.- (Acetyltransferases)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)',\n",
       "   'EC 2.5.1.18 (Glutathione Transferase)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Acetylation',\n",
       "   'Acetyltransferases/*chemistry/*metabolism',\n",
       "   'Bacteria/metabolism',\n",
       "   'Blotting, Western',\n",
       "   'Genotype',\n",
       "   'Glutathione Transferase/metabolism',\n",
       "   'Histone Acetyltransferases',\n",
       "   'Histones/chemistry',\n",
       "   'Models, Biological',\n",
       "   'Phenotype',\n",
       "   'Precipitin Tests',\n",
       "   'Protein Binding',\n",
       "   'Protein Biosynthesis',\n",
       "   'Protein Structure, Tertiary',\n",
       "   'RNA/chemistry',\n",
       "   'Recombinant Proteins/chemistry',\n",
       "   'Saccharomyces cerevisiae/metabolism',\n",
       "   'Structure-Activity Relationship',\n",
       "   'Transcription, Genetic',\n",
       "   'Two-Hybrid System Techniques'],\n",
       "  'EDAT': '2004/05/13 05:00',\n",
       "  'MHDA': '2004/09/11 05:00',\n",
       "  'CRDT': ['2004/05/13 05:00'],\n",
       "  'PHST': ['2004/05/13 05:00 [pubmed]',\n",
       "   '2004/09/11 05:00 [medline]',\n",
       "   '2004/05/13 05:00 [entrez]'],\n",
       "  'AID': ['10.1074/jbc.M403361200 [doi]', 'M403361200 [pii]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'J Biol Chem. 2004 Jul 30;279(31):32087-92. doi: 10.1074/jbc.M403361200. Epub 2004 May 11.'},\n",
       " {'PMID': '12424236',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20030307',\n",
       "  'LR': '20091119',\n",
       "  'IS': '0021-9258 (Print) 0021-9258 (Linking)',\n",
       "  'VI': '278',\n",
       "  'IP': '2',\n",
       "  'DP': '2003 Jan 10',\n",
       "  'TI': 'Subunit communications crucial for the functional integrity of the yeast RNA polymerase II elongator (gamma-toxin target (TOT)) complex.',\n",
       "  'PG': '956-61',\n",
       "  'AB': 'In response to the Kluyveromyces lactis zymocin, the gamma-toxin target (TOT) function of the Saccharomyces cerevisiae RNA polymerase II (pol II) Elongator complex prevents sensitive strains from cell cycle progression. Studying Elongator subunit communications, Tot1p (Elp1p), the yeast homologue of human IKK-associated protein, was found to be essentially involved in maintaining the structural integrity of Elongator. Thus, the ability of Tot2p (Elp2p) to interact with the HAT subunit Tot3p (Elp3p) of Elongator and with subunit Tot5p (Elp5p) is dependent on Tot1p (Elp1p). Also, the association of core-Elongator (Tot1-3p/Elp1-3p) with HAP (Elp4-6p/Tot5-7p), the second three-subunit subcomplex of Elongator, was found to be sensitive to loss of TOT1 (ELP1) gene function. Structural integrity of the HAP complex itself requires the ELP4/TOT7, ELP5/TOT5, and ELP6/TOT6 genes, and elp6Delta/tot6Delta as well as elp4Delta/tot7Delta cells can no longer promote interaction between Tot5p (Elp5p) and Tot2p (Elp2p). The association between Elongator and Tot4p (Kti12p), a factor that may modulate the TOT activity of Elongator, requires Tot1-3p (Elp1-3p) and Tot5p (Elp5p), indicating that this contact requires a preassembled holo-Elongator complex. Tot4p also binds pol II hyperphosphorylated at its C-terminal domain Ser(5) raising the possibility that Tot4p bridges the contact between Elongator and pol II.',\n",
       "  'FAU': ['Frohloff, Frank',\n",
       "   'Jablonowski, Daniel',\n",
       "   'Fichtner, Lars',\n",
       "   'Schaffrath, Raffael'],\n",
       "  'AU': ['Frohloff F', 'Jablonowski D', 'Fichtner L', 'Schaffrath R'],\n",
       "  'AD': 'Institut fur Genetik, Biologicum, Martin-Luther-Universitat Halle-Wittenberg, Weinbergweg 10, Germany.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'DEP': '20021106',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'J Biol Chem',\n",
       "  'JT': 'The Journal of biological chemistry',\n",
       "  'JID': '2985121R',\n",
       "  'RN': ['0 (Killer Factors, Yeast)',\n",
       "   '0 (Mycotoxins)',\n",
       "   '0 (Protein Subunits)',\n",
       "   '0 (Saccharomyces cerevisiae Proteins)',\n",
       "   '0 (zymocin)',\n",
       "   'EC 2.3.1.- (Acetyltransferases)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)',\n",
       "   'EC 2.7.11.1 (Protein-Serine-Threonine Kinases)',\n",
       "   'EC 2.7.11.10 (I-kappa B Kinase)',\n",
       "   'EC 2.7.7.- (RNA Polymerase II)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Acetyltransferases/chemistry/*metabolism',\n",
       "   'Histone Acetyltransferases',\n",
       "   'I-kappa B Kinase',\n",
       "   'Killer Factors, Yeast',\n",
       "   'Mycotoxins/*metabolism',\n",
       "   'Protein Subunits',\n",
       "   'Protein-Serine-Threonine Kinases/metabolism',\n",
       "   'RNA Polymerase II/*metabolism',\n",
       "   'Saccharomyces cerevisiae Proteins/chemistry/*metabolism'],\n",
       "  'EDAT': '2002/11/09 04:00',\n",
       "  'MHDA': '2003/03/08 04:00',\n",
       "  'CRDT': ['2002/11/09 04:00'],\n",
       "  'PHST': ['2002/11/09 04:00 [pubmed]',\n",
       "   '2003/03/08 04:00 [medline]',\n",
       "   '2002/11/09 04:00 [entrez]'],\n",
       "  'AID': ['10.1074/jbc.M210060200 [doi]', 'M210060200 [pii]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'J Biol Chem. 2003 Jan 10;278(2):956-61. doi: 10.1074/jbc.M210060200. Epub 2002 Nov 6.'},\n",
       " {'PMID': '12386836',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20030331',\n",
       "  'LR': '20181113',\n",
       "  'IS': '0002-9297 (Print) 0002-9297 (Linking)',\n",
       "  'VI': '71',\n",
       "  'IP': '5',\n",
       "  'DP': '2002 Nov',\n",
       "  'TI': 'Frequent chromosome aberrations revealed by molecular cytogenetic studies in patients with aniridia.',\n",
       "  'PG': '1138-49',\n",
       "  'AB': \"Seventy-seven patients with aniridia, referred for cytogenetic analysis predominantly to assess Wilms tumor risk, were studied by fluorescence in situ hybridization (FISH), through use of a panel of cosmids encompassing the aniridia-associated PAX6 gene, the Wilms tumor predisposition gene WT1, and flanking markers, in distal chromosome 11p13. Thirty patients were found to be chromosomally abnormal. Cytogenetically visible interstitial deletions involving 11p13 were found in 13 patients, 11 of which included WT1. A further 13 patients had cryptic deletions detectable only by FISH, 3 of which included WT1. Six of these, with deletions <500 kb, share a similar proximal breakpoint within a cosmid containing the last 10 exons of PAX6 and part of the neighboring gene, ELP4. Two of these six patients were mosaic for the deletion. The remaining four had chromosomal rearrangements: an unbalanced translocation, t(11;13), with a deletion including the WAGR (Wilms' tumor, aniridia, genitourinary abnormalities, and mental retardation) region, and three balanced rearrangements with what appear to be position effect breakpoints 3' of PAX6: (a) a t(7;11) with the 11p13 breakpoint approximately 30 kb downstream of PAX6, (b) a dir ins(12;11) with a breakpoint >50 kb from PAX6, and (c) an inv(11)(p13q13) with a breakpoint >75 kb downstream of PAX6. The proportion and spectrum of chromosome anomalies in familial (4/14, or 28.5%) and sporadic (26/63, or 41%) cases are not significantly different. An unexpectedly high frequency of chromosomal rearrangements is associated with both sporadic and familial aniridia in this cohort.\",\n",
       "  'FAU': ['Crolla, John A', 'van Heyningen, Veronica'],\n",
       "  'AU': ['Crolla JA', 'van Heyningen V'],\n",
       "  'AD': 'Wessex Regional Genetics Laboratory, Salisbury District Hospital, United Kingdom. John.Crolla@salisbury.nhs.uk',\n",
       "  'LA': ['eng'],\n",
       "  'SI': ['OMIM/106200', 'OMIM/194072', 'RefSeq/NT_009237'],\n",
       "  'GR': ['MC_U127527199/Medical Research Council/United Kingdom'],\n",
       "  'PT': ['Journal Article'],\n",
       "  'DEP': '20021017',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Am J Hum Genet',\n",
       "  'JT': 'American journal of human genetics',\n",
       "  'JID': '0370475',\n",
       "  'RN': ['0 (Eye Proteins)',\n",
       "   '0 (Homeodomain Proteins)',\n",
       "   '0 (PAX6 Transcription Factor)',\n",
       "   '0 (PAX6 protein, human)',\n",
       "   '0 (Paired Box Transcription Factors)',\n",
       "   '0 (Repressor Proteins)',\n",
       "   '0 (WT1 Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Aniridia/*genetics',\n",
       "   'Child, Preschool',\n",
       "   '*Chromosome Aberrations',\n",
       "   'Chromosome Deletion',\n",
       "   'Chromosomes, Human, Pair 11',\n",
       "   'Eye Proteins/genetics',\n",
       "   'Gene Rearrangement',\n",
       "   'Homeodomain Proteins/genetics',\n",
       "   'Humans',\n",
       "   'In Situ Hybridization, Fluorescence',\n",
       "   'Infant',\n",
       "   'Molecular Sequence Data',\n",
       "   'PAX6 Transcription Factor',\n",
       "   'Paired Box Transcription Factors',\n",
       "   'Repressor Proteins',\n",
       "   'Risk Factors',\n",
       "   'Sequence Deletion',\n",
       "   'WT1 Proteins/genetics',\n",
       "   'Wilms Tumor/genetics'],\n",
       "  'PMC': 'PMC385089',\n",
       "  'EDAT': '2002/10/19 04:00',\n",
       "  'MHDA': '2003/04/01 05:00',\n",
       "  'CRDT': ['2002/10/19 04:00'],\n",
       "  'PHST': ['2002/06/17 00:00 [received]',\n",
       "   '2002/08/21 00:00 [accepted]',\n",
       "   '2002/10/19 04:00 [pubmed]',\n",
       "   '2003/04/01 05:00 [medline]',\n",
       "   '2002/10/19 04:00 [entrez]'],\n",
       "  'AID': ['S0002-9297(07)60406-6 [pii]', '10.1086/344396 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Am J Hum Genet. 2002 Nov;71(5):1138-49. doi: 10.1086/344396. Epub 2002 Oct 17.'},\n",
       " {'PMID': '11929532',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20020604',\n",
       "  'LR': '20190725',\n",
       "  'IS': '0950-382X (Print) 0950-382X (Linking)',\n",
       "  'VI': '43',\n",
       "  'IP': '3',\n",
       "  'DP': '2002 Feb',\n",
       "  'TI': 'Molecular analysis of KTI12/TOT4, a Saccharomyces cerevisiae gene required for Kluyveromyces lactis zymocin action.',\n",
       "  'PG': '783-91',\n",
       "  'AB': 'TOT, the putative Kluyveromyces lactis zymocin target complex from Saccharomyces cerevisiae, is encoded by TOT1-7, six loci of which are isoallelic to RNA polymerase II (RNAPII) Elongator genes (ELP1-6). Unlike TOT1-3 (ELP1-3) and TOT5-7 (ELP5, ELP6 and ELP4 respectively), which display zymocin resistance when deleted, TOT4 (KTI12) also renders cells refractory to zymocin when maintained in multicopy or overexpressed from the GAL10 promoter. Elevated TOT4 copy number results in an intermediate tot phenotype, which includes mild sensitivities towards caffeine, Calcofluor white and elevated growth temperature, suggesting that TOT4 influences TOT/Elongator function. Tot4p interacts with Elongator, as shown by co-immunoprecipitation, and cell fractionation studies demonstrate partial co-migration with RNAPII and Elongator. As Elongator subunit interaction is not affected by either deletion of TOT4 or multicopy TOT4, Tot4p may not be a structural Elongator subunit but, rather, may regulate TOT/Elongator in a fashion that requires transient physical contact with TOT/Elongator. Consistent with a regulatory role, the presence of a potential P-loop motif conserved between yeast and human TOT4 homologues suggests capability of ATP or GTP binding and P-loop deletion renders Tot4p biologically inactive.',\n",
       "  'FAU': ['Fichtner, Lars',\n",
       "   'Frohloff, Frank',\n",
       "   'Burkner, Konstanze',\n",
       "   'Larsen, Michael',\n",
       "   'Breunig, Karin D',\n",
       "   'Schaffrath, Raffael'],\n",
       "  'AU': ['Fichtner L',\n",
       "   'Frohloff F',\n",
       "   'Burkner K',\n",
       "   'Larsen M',\n",
       "   'Breunig KD',\n",
       "   'Schaffrath R'],\n",
       "  'AD': 'Institut fur Genetik, Martin-Luther-Universitat Halle-Wittenberg, Weinbergweg 10, D-06120 Halle (Saale), Germany.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'PL': 'England',\n",
       "  'TA': 'Mol Microbiol',\n",
       "  'JT': 'Molecular microbiology',\n",
       "  'JID': '8712028',\n",
       "  'RN': ['0 (Adaptor Proteins, Signal Transducing)',\n",
       "   '0 (Fungal Proteins)',\n",
       "   '0 (KTI12 protein, S cerevisiae)',\n",
       "   '0 (Killer Factors, Yeast)',\n",
       "   '0 (Mycotoxins)',\n",
       "   '0 (Saccharomyces cerevisiae Proteins)',\n",
       "   '0 (zymocin)',\n",
       "   'EC 2.1.1.- (Histone Methyltransferases)',\n",
       "   'EC 2.1.1.- (Methyltransferases)',\n",
       "   'EC 2.1.1.- (Protein Methyltransferases)',\n",
       "   'EC 2.1.1.43 (Histone-Lysine N-Methyltransferase)',\n",
       "   'EC 2.7.7.- (RNA Polymerase II)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Adaptor Proteins, Signal Transducing',\n",
       "   'Amino Acid Sequence',\n",
       "   'Fungal Proteins/*genetics/metabolism',\n",
       "   'Gene Dosage',\n",
       "   'Histone Methyltransferases',\n",
       "   '*Histone-Lysine N-Methyltransferase',\n",
       "   'Killer Factors, Yeast',\n",
       "   'Kluyveromyces/genetics/*metabolism',\n",
       "   'Methyltransferases/genetics/metabolism',\n",
       "   'Molecular Sequence Data',\n",
       "   'Mycotoxins/genetics/*metabolism',\n",
       "   'Protein Methyltransferases',\n",
       "   'RNA Polymerase II/genetics/metabolism',\n",
       "   'Saccharomyces cerevisiae/*genetics',\n",
       "   '*Saccharomyces cerevisiae Proteins',\n",
       "   'Sequence Homology, Amino Acid'],\n",
       "  'EDAT': '2002/04/04 10:00',\n",
       "  'MHDA': '2002/06/05 10:01',\n",
       "  'CRDT': ['2002/04/04 10:00'],\n",
       "  'PHST': ['2002/04/04 10:00 [pubmed]',\n",
       "   '2002/06/05 10:01 [medline]',\n",
       "   '2002/04/04 10:00 [entrez]'],\n",
       "  'AID': ['2794 [pii]', '10.1046/j.1365-2958.2002.02794.x [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Mol Microbiol. 2002 Feb;43(3):783-91. doi: 10.1046/j.1365-2958.2002.02794.x.'},\n",
       " {'PMID': '11904415',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20020424',\n",
       "  'LR': '20181113',\n",
       "  'IS': '0027-8424 (Print) 0027-8424 (Linking)',\n",
       "  'VI': '99',\n",
       "  'IP': '6',\n",
       "  'DP': '2002 Mar 19',\n",
       "  'TI': 'Elongator is a histone H3 and H4 acetyltransferase important for normal histone acetylation levels in vivo.',\n",
       "  'PG': '3517-22',\n",
       "  'AB': 'The elongating, hyperphosphorylated form of RNA polymerase II is associated with the Elongator complex, which has the histone acetyltransferase (HAT) Elp3 as a subunit. Here we show that, in contrast to the isolated Elp3 subunit, the activity of intact Elongator complex is directed specifically toward the amino-terminal tails of histone H3 and H4, and that Elongator can acetylate both core histones and nucleosomal substrates. The predominant acetylation sites are lysine-14 of histone H3 and lysine-8 of histone H4. The three smallest Elongator subunits--Elp4, Elp5, and Elp6--are required for HAT activity, and Elongator binds to both naked and nucleosomal DNA. By using chromatin immunoprecipitation, we show that the levels of multiply acetylated histone H3 and H4 in chromatin are decreased in vivo in yeast cells lacking ELP3.',\n",
       "  'FAU': ['Winkler, G Sebastiaan',\n",
       "   'Kristjuhan, Arnold',\n",
       "   'Erdjument-Bromage, Hediye',\n",
       "   'Tempst, Paul',\n",
       "   'Svejstrup, Jesper Q'],\n",
       "  'AU': ['Winkler GS',\n",
       "   'Kristjuhan A',\n",
       "   'Erdjument-Bromage H',\n",
       "   'Tempst P',\n",
       "   'Svejstrup JQ'],\n",
       "  'AD': 'Cancer Research UK., London Research Institute, Clare Hall Laboratories, South Mimms, Hertfordshire EN6 3LD, United Kingdom.',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['P30 CA008748/CA/NCI NIH HHS/United States',\n",
       "   'P30 CA08748/CA/NCI NIH HHS/United States'],\n",
       "  'PT': ['Journal Article',\n",
       "   \"Research Support, Non-U.S. Gov't\",\n",
       "   \"Research Support, U.S. Gov't, P.H.S.\"],\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Proc Natl Acad Sci U S A',\n",
       "  'JT': 'Proceedings of the National Academy of Sciences of the United States of America',\n",
       "  'JID': '7505876',\n",
       "  'RN': ['0 (DNA, Fungal)',\n",
       "   '0 (Histones)',\n",
       "   '0 (Holoenzymes)',\n",
       "   '0 (Macromolecular Substances)',\n",
       "   '0 (Nucleosomes)',\n",
       "   '0 (Protein Subunits)',\n",
       "   '0 (Saccharomyces cerevisiae Proteins)',\n",
       "   'EC 2.3.1.- (Acetyltransferases)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)',\n",
       "   'K3Z4F929H6 (Lysine)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Acetylation',\n",
       "   'Acetyltransferases/*chemistry/genetics/*metabolism',\n",
       "   'DNA, Fungal/chemistry/genetics/metabolism',\n",
       "   'Gene Deletion',\n",
       "   'Histone Acetyltransferases',\n",
       "   'Histones/chemistry/*metabolism',\n",
       "   'Holoenzymes/chemistry/genetics/metabolism',\n",
       "   'Lysine/metabolism',\n",
       "   'Macromolecular Substances',\n",
       "   'Nucleic Acid Conformation',\n",
       "   'Nucleosomes/chemistry/*enzymology/*metabolism',\n",
       "   'Precipitin Tests',\n",
       "   'Protein Subunits',\n",
       "   'Saccharomyces cerevisiae/*enzymology/genetics',\n",
       "   '*Saccharomyces cerevisiae Proteins',\n",
       "   'Substrate Specificity'],\n",
       "  'PMC': 'PMC122555',\n",
       "  'EDAT': '2002/03/21 10:00',\n",
       "  'MHDA': '2002/04/25 10:01',\n",
       "  'CRDT': ['2002/03/21 10:00'],\n",
       "  'PHST': ['2002/03/21 10:00 [pubmed]',\n",
       "   '2002/04/25 10:01 [medline]',\n",
       "   '2002/03/21 10:00 [entrez]'],\n",
       "  'AID': ['10.1073/pnas.022042899 [doi]', '99/6/3517 [pii]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Proc Natl Acad Sci U S A. 2002 Mar 19;99(6):3517-22. doi: 10.1073/pnas.022042899.'},\n",
       " {'PMID': '11889558',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20020624',\n",
       "  'LR': '20170922',\n",
       "  'IS': '0938-8990 (Print) 0938-8990 (Linking)',\n",
       "  'VI': '13',\n",
       "  'IP': '2',\n",
       "  'DP': '2002 Feb',\n",
       "  'TI': 'Characterization of a novel gene adjacent to PAX6, revealing synteny conservation with functional significance.',\n",
       "  'PG': '102-7',\n",
       "  'AB': 'The human eye anomaly aniridia is normally caused by intragenic mutations of PAX6. Several cases of aniridia are, however, associated with chromosomal rearrangements that leave the PAX6 gene intact. We have identified and characterized a novel gene, PAXNEB (C11orf19), downstream (telomeric) of PAX6. Sequence analysis, including interspecies comparisons, show this gene to consist of 10 exons, with an unusually large final intron spanning 134 kb in human and 18 kb in Fugu. This intron is disrupted by each chromosomal rearrangement. The 2-kb PAXNEB transcript, encoding a 424-amino acid protein, is expressed in all cell lines tested. The homologous mouse cDNA is broadly expressed in mouse embryos. PAXNEB is highly conserved from mammals to fish, with some regions of the protein showing conservation to invertebrates, yeast, and plants. The possible role of PAXNEB in aniridia was assessed. Using a transgenic mouse model, we show that the aniridia phenotype of the chromosomal rearrangement cases is not due to the heterozygous loss of PAXNEB function.',\n",
       "  'FAU': ['Kleinjan, Dirk A',\n",
       "   'Seawright, Anne',\n",
       "   'Elgar, Greg',\n",
       "   'van Heyningen, Veronica'],\n",
       "  'AU': ['Kleinjan DA', 'Seawright A', 'Elgar G', 'van Heyningen V'],\n",
       "  'AD': 'MRC Human Genetics Unit, Western General Hospital, Crewe Road, Edinburgh EH4 2XU, UK. d.kleinjan@hgu.mrc.ac.uk',\n",
       "  'LA': ['eng'],\n",
       "  'SI': ['GENBANK/AJ276004', 'GENBANK/AJ414048'],\n",
       "  'GR': ['MC_U127527199/Medical Research Council/United Kingdom'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Mamm Genome',\n",
       "  'JT': 'Mammalian genome : official journal of the International Mammalian Genome Society',\n",
       "  'JID': '9100916',\n",
       "  'RN': ['0 (ELP4 protein, human)',\n",
       "   '0 (Eye Proteins)',\n",
       "   '0 (Homeodomain Proteins)',\n",
       "   '0 (Nerve Tissue Proteins)',\n",
       "   '0 (PAX6 Transcription Factor)',\n",
       "   '0 (PAX6 protein, human)',\n",
       "   '0 (Paired Box Transcription Factors)',\n",
       "   '0 (Pax6 protein, mouse)',\n",
       "   '0 (Repressor Proteins)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Amino Acid Sequence',\n",
       "   'Animals',\n",
       "   'Aniridia/genetics',\n",
       "   'Base Sequence',\n",
       "   '*Chromosome Mapping',\n",
       "   'Eye Proteins',\n",
       "   'Female',\n",
       "   'Homeodomain Proteins/*genetics',\n",
       "   'Humans',\n",
       "   'Male',\n",
       "   'Mice',\n",
       "   'Molecular Sequence Data',\n",
       "   'Nerve Tissue Proteins/*genetics',\n",
       "   'PAX6 Transcription Factor',\n",
       "   'Paired Box Transcription Factors',\n",
       "   'Repressor Proteins',\n",
       "   'Sequence Analysis, DNA',\n",
       "   '*Synteny'],\n",
       "  'EDAT': '2002/03/13 10:00',\n",
       "  'MHDA': '2002/06/25 10:01',\n",
       "  'CRDT': ['2002/03/13 10:00'],\n",
       "  'PHST': ['2001/07/16 00:00 [received]',\n",
       "   '2001/09/28 00:00 [accepted]',\n",
       "   '2002/03/13 10:00 [pubmed]',\n",
       "   '2002/06/25 10:01 [medline]',\n",
       "   '2002/03/13 10:00 [entrez]'],\n",
       "  'AID': ['10.1007/s00335-001-3058-y [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Mamm Genome. 2002 Feb;13(2):102-7. doi: 10.1007/s00335-001-3058-y.'},\n",
       " {'PMID': '11737649',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20020301',\n",
       "  'LR': '20190725',\n",
       "  'IS': '0950-382X (Print) 0950-382X (Linking)',\n",
       "  'VI': '42',\n",
       "  'IP': '4',\n",
       "  'DP': '2001 Nov',\n",
       "  'TI': 'Kluyveromyces lactis zymocin mode of action is linked to RNA polymerase II function via Elongator.',\n",
       "  'PG': '1095-105',\n",
       "  'AB': 'The putative Kluyveromyces lactis zymocin target complex, TOT, from Saccharomyces cerevisiae comprises five Tot proteins, four of which are RNA polymerase II (RNAP II) Elongator subunits. Recently, two more Elongator subunit genes, ELP6 (TOT6) and ELP4 (TOT7), have been identified. Deletions of both TOT6 and TOT7 result in the complex tot phenotype, including resistance to zymocin, thermosensitivity, slow growth and hypersensitivity towards drugs, thus reinforcing the notion that TOT/Elongator may be crucial in signalling zymocicity. Mutagenesis of ELP3/TOT3, the Elongator histone acetyltransferase (HAT) gene, revealed that zymocin sensitivity could be uncoupled from Elongator wild-type function, indicating that TOT interacts genetically with zymocin. To test the possibility that zymocin functions by affecting RNAP II activity in a TOT/Elongator-dependent manner, global poly(A)+ mRNA levels were found to decline drastically on zymocin treatment. Moreover, cells overexpressing Fcp1p, the RNAP II carboxy-terminal domain phosphatase, acquired partial zymocin resistance, whereas cells underproducing RNAP II became zymocin hypersensitive. This suggests that zymocin may convert TOT/Elongator into a cellular poison toxic for RNAP II function and eventually leading to the observed G1 cell cycle arrest.',\n",
       "  'FAU': ['Jablonowski, D',\n",
       "   'Frohloff, F',\n",
       "   'Fichtner, L',\n",
       "   'Stark, M J',\n",
       "   'Schaffrath, R'],\n",
       "  'AU': ['Jablonowski D',\n",
       "   'Frohloff F',\n",
       "   'Fichtner L',\n",
       "   'Stark MJ',\n",
       "   'Schaffrath R'],\n",
       "  'AD': 'Institut fur Genetik, Martin-Luther-Universitat Halle-Wittenberg, Weinbergweg 10, D-06120 Halle (Saale), Germany.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'PL': 'England',\n",
       "  'TA': 'Mol Microbiol',\n",
       "  'JT': 'Molecular microbiology',\n",
       "  'JID': '8712028',\n",
       "  'RN': ['0 (Benzenesulfonates)',\n",
       "   '0 (Fluorescent Dyes)',\n",
       "   '0 (Fungal Proteins)',\n",
       "   '0 (Histones)',\n",
       "   '0 (Killer Factors, Yeast)',\n",
       "   '0 (Mycotoxins)',\n",
       "   '0 (Protein Subunits)',\n",
       "   '0 (RNA, Messenger)',\n",
       "   '0 (Saccharomyces cerevisiae Proteins)',\n",
       "   '0 (zymocin)',\n",
       "   '3G6A5W338E (Caffeine)',\n",
       "   '7S9P0Y4313 (C.I. Fluorescent Brightening Agent 28)',\n",
       "   'EC 2.3.1.- (Acetyltransferases)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)',\n",
       "   'EC 2.7.7.- (RNA Polymerase II)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Acetyltransferases/genetics/*metabolism',\n",
       "   'Benzenesulfonates/pharmacology',\n",
       "   'Caffeine/pharmacology',\n",
       "   'Fluorescent Dyes/pharmacology',\n",
       "   'Fungal Proteins/genetics/*metabolism',\n",
       "   '*Genes, Fungal',\n",
       "   'Histone Acetyltransferases',\n",
       "   'Histones/genetics/metabolism',\n",
       "   'Killer Factors, Yeast',\n",
       "   'Kluyveromyces/drug effects/genetics/*metabolism',\n",
       "   'Mutagenesis',\n",
       "   'Mycotoxins/*metabolism',\n",
       "   'Phenotype',\n",
       "   'Protein Subunits',\n",
       "   'RNA Polymerase II/*metabolism',\n",
       "   'RNA, Messenger/metabolism',\n",
       "   '*Saccharomyces cerevisiae Proteins'],\n",
       "  'EDAT': '2001/12/12 10:00',\n",
       "  'MHDA': '2002/03/02 10:01',\n",
       "  'CRDT': ['2001/12/12 10:00'],\n",
       "  'PHST': ['2001/12/12 10:00 [pubmed]',\n",
       "   '2002/03/02 10:01 [medline]',\n",
       "   '2001/12/12 10:00 [entrez]'],\n",
       "  'AID': ['2705 [pii]', '10.1046/j.1365-2958.2001.02705.x [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Mol Microbiol. 2001 Nov;42(4):1095-105. doi: 10.1046/j.1365-2958.2001.02705.x.'},\n",
       " {'PMID': '11689709',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20011205',\n",
       "  'LR': '20181113',\n",
       "  'IS': '0270-7306 (Print) 0270-7306 (Linking)',\n",
       "  'VI': '21',\n",
       "  'IP': '23',\n",
       "  'DP': '2001 Dec',\n",
       "  'TI': 'Characterization of a six-subunit holo-elongator complex required for the regulated expression of a group of genes in Saccharomyces cerevisiae.',\n",
       "  'PG': '8203-12',\n",
       "  'AB': 'The Elongator complex associated with elongating RNA polymerase II in Saccharomyces cerevisiae was originally reported to have three subunits, Elp1, Elp2, and Elp3. Using the tandem affinity purification (TAP) procedure, we have purified a six-subunit yeast Holo-Elongator complex containing three additional polypeptides, which we have named Elp4, Elp5, and Elp6. TAP tapping and subsequent purification of any one of the six subunits result in the isolation of all six components. Purification of Elongator in higher salt concentrations served to demonstrate that the complex could be separated into two subcomplexes: one consisted of Elp1, -2, and -3, and the other consisted of Elp4, -5, and -6. Deletions of the individual genes encoding the new Elongator subunits showed that only the ELP5 gene is essential for growth. Disruption of the two nonessential new Elongator-encoding genes, ELP4 and ELP6, caused the same phenotypes observed with knockouts of the original Elongator-encoding genes. Results of microarray analyses demonstrated that the gene expression profiles of strains containing deletions of genes encoding subunits of either Elongator subcomplex, in which we detected significantly altered mRNA expression levels for 96 genes, are very similar, implying that all the Elongator subunits likely function together to regulate a group of S. cerevisiae genes in vivo.',\n",
       "  'FAU': ['Krogan, N J', 'Greenblatt, J F'],\n",
       "  'AU': ['Krogan NJ', 'Greenblatt JF'],\n",
       "  'AD': 'Banting and Best Department of Medical Research and Department of Molecular and Medical Genetics, University of Toronto, Toronto, Ontario M5G 1L6, Canada.',\n",
       "  'LA': ['eng'],\n",
       "  'PT': ['Journal Article', \"Research Support, Non-U.S. Gov't\"],\n",
       "  'PL': 'United States',\n",
       "  'TA': 'Mol Cell Biol',\n",
       "  'JT': 'Molecular and cellular biology',\n",
       "  'JID': '8109087',\n",
       "  'RN': ['0 (Fungal Proteins)',\n",
       "   '0 (Macromolecular Substances)',\n",
       "   '0 (Protein Subunits)',\n",
       "   '0 (RNA, Messenger)',\n",
       "   '0 (Transcription Factors)',\n",
       "   'EC 2.7.7.- (RNA Polymerase II)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Chromatography, Affinity',\n",
       "   'Fungal Proteins/*chemistry/*metabolism',\n",
       "   'Gene Deletion',\n",
       "   'Gene Expression Profiling',\n",
       "   'Gene Expression Regulation, Fungal/*physiology',\n",
       "   'Macromolecular Substances',\n",
       "   'Oligonucleotide Array Sequence Analysis',\n",
       "   'Peptide Mapping',\n",
       "   'Phenotype',\n",
       "   '*Protein Subunits',\n",
       "   'RNA Polymerase II/metabolism',\n",
       "   'RNA, Messenger/biosynthesis',\n",
       "   'Saccharomyces cerevisiae',\n",
       "   'Spectrometry, Mass, Matrix-Assisted Laser Desorption-Ionization',\n",
       "   'Transcription Factors/*physiology',\n",
       "   'Up-Regulation'],\n",
       "  'PMC': 'PMC99985',\n",
       "  'EDAT': '2001/11/02 10:00',\n",
       "  'MHDA': '2002/01/05 10:01',\n",
       "  'CRDT': ['2001/11/02 10:00'],\n",
       "  'PHST': ['2001/11/02 10:00 [pubmed]',\n",
       "   '2002/01/05 10:01 [medline]',\n",
       "   '2001/11/02 10:00 [entrez]'],\n",
       "  'AID': ['10.1128/MCB.21.23.8203-8212.2001 [doi]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'Mol Cell Biol. 2001 Dec;21(23):8203-12. doi: 10.1128/MCB.21.23.8203-8212.2001.'},\n",
       " {'PMID': '11435442',\n",
       "  'OWN': 'NLM',\n",
       "  'STAT': 'MEDLINE',\n",
       "  'DCOM': '20011018',\n",
       "  'LR': '20071114',\n",
       "  'IS': '0021-9258 (Print) 0021-9258 (Linking)',\n",
       "  'VI': '276',\n",
       "  'IP': '35',\n",
       "  'DP': '2001 Aug 31',\n",
       "  'TI': 'RNA polymerase II elongator holoenzyme is composed of two discrete subcomplexes.',\n",
       "  'PG': '32743-9',\n",
       "  'AB': 'Elongator is a histone acetyltransferase complex that associates with the elongating form of RNA polymerase II. We purified Elongator to virtual homogeneity via a rapid three-step procedure based largely on affinity chromatography. The purified factor, holo-Elongator, is a labile six-subunit factor composed of two discrete subcomplexes: one comprised of the previously identified Elp1, Elp2, and Elp3 proteins and another comprised of three novel polypeptides, termed Elp4, Elp5, and Elp6. Disruption of the yeast genes encoding the new Elongator proteins confers phenotypes indistinguishable from those previously described for the other elp mutants, and concomitant disruption of genes encoding proteins in either subcomplex does not confer new phenotypes. Taken together, our results indicate that holo-Elongator is a functional entity in vitro as well as in vivo. Metazoan homologues of Elp1 and Elp3 have previously been reported. We cloned the human homologue of yeast ELP4 and show that this gene is ubiquitously expressed in human tissues.',\n",
       "  'FAU': ['Winkler, G S',\n",
       "   'Petrakis, T G',\n",
       "   'Ethelberg, S',\n",
       "   'Tokunaga, M',\n",
       "   'Erdjument-Bromage, H',\n",
       "   'Tempst, P',\n",
       "   'Svejstrup, J Q'],\n",
       "  'AU': ['Winkler GS',\n",
       "   'Petrakis TG',\n",
       "   'Ethelberg S',\n",
       "   'Tokunaga M',\n",
       "   'Erdjument-Bromage H',\n",
       "   'Tempst P',\n",
       "   'Svejstrup JQ'],\n",
       "  'AD': 'Mechanisms of Gene Transcription Laboratory, Imperial Cancer Research Fund, Clare Hall Laboratories, Blanche Lane, South Mimms, Herts, EN6 3LD, United Kingdom.',\n",
       "  'LA': ['eng'],\n",
       "  'GR': ['P30 CA08748/CA/NCI NIH HHS/United States'],\n",
       "  'PT': ['Journal Article',\n",
       "   \"Research Support, Non-U.S. Gov't\",\n",
       "   \"Research Support, U.S. Gov't, P.H.S.\"],\n",
       "  'DEP': '20010702',\n",
       "  'PL': 'United States',\n",
       "  'TA': 'J Biol Chem',\n",
       "  'JT': 'The Journal of biological chemistry',\n",
       "  'JID': '2985121R',\n",
       "  'RN': ['0 (Macromolecular Substances)',\n",
       "   '0 (Peptide Fragments)',\n",
       "   '0 (Protein Subunits)',\n",
       "   '0 (Recombinant Proteins)',\n",
       "   '0 (Saccharomyces cerevisiae Proteins)',\n",
       "   'EC 2.3.1.- (Acetyltransferases)',\n",
       "   'EC 2.3.1.48 (Histone Acetyltransferases)',\n",
       "   'EC 2.7.7.- (RNA Polymerase II)'],\n",
       "  'SB': 'IM',\n",
       "  'MH': ['Acetyltransferases/*chemistry/genetics/isolation & purification/*metabolism',\n",
       "   'Amino Acid Sequence',\n",
       "   'Animals',\n",
       "   'Caenorhabditis elegans/genetics',\n",
       "   'Cloning, Molecular',\n",
       "   'Conserved Sequence',\n",
       "   'Drosophila melanogaster/genetics',\n",
       "   'Histone Acetyltransferases',\n",
       "   'Humans',\n",
       "   'Macromolecular Substances',\n",
       "   'Mice',\n",
       "   'Molecular Sequence Data',\n",
       "   'Peptide Fragments/chemistry',\n",
       "   'Phenotype',\n",
       "   'Protein Subunits',\n",
       "   'RNA Polymerase II/*metabolism',\n",
       "   'Recombinant Proteins/chemistry/metabolism',\n",
       "   'Saccharomyces cerevisiae/*genetics/metabolism',\n",
       "   '*Saccharomyces cerevisiae Proteins',\n",
       "   'Schizosaccharomyces/genetics',\n",
       "   'Sequence Alignment',\n",
       "   'Sequence Homology, Amino Acid',\n",
       "   'Transcription, Genetic'],\n",
       "  'EDAT': '2001/07/04 10:00',\n",
       "  'MHDA': '2001/10/19 10:01',\n",
       "  'CRDT': ['2001/07/04 10:00'],\n",
       "  'PHST': ['2001/07/04 10:00 [pubmed]',\n",
       "   '2001/10/19 10:01 [medline]',\n",
       "   '2001/07/04 10:00 [entrez]'],\n",
       "  'AID': ['10.1074/jbc.M105303200 [doi]', 'M105303200 [pii]'],\n",
       "  'PST': 'ppublish',\n",
       "  'SO': 'J Biol Chem. 2001 Aug 31;276(35):32743-9. doi: 10.1074/jbc.M105303200. Epub 2001 Jul 2.'}]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "records = list(records)\n",
    "# obtém-se toda a informação contida em cada um dos artigos\n",
    "#Posteriormente, explorando alguns parâmetros em cada um dos artigos conseguimos ter uma perceção rápida em relação a que temas são abordados nos artigos. \n",
    "records "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "4.Posteriormente com o ciclo for que se encontra no código abaixo, obtém-se informações mais relevantes sobre cada um dos artigos do PubMed como: o título do artigo, os autores do artigo e ainda, a fonte do artigo (para cada um dos 61 artigos que se encontram no PubMed relacionados com este gene ELP4)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "title: Association between ELP4 rs986527 polymorphism and the occurrence and development of intracranial arachnoid cyst.\n",
      "authors: ['Li K', 'Kong DS', 'Zhang J', 'Wang XS', 'Ye X', 'Zhao YL']\n",
      "source: Brain Behav. 2019 Dec;9(12):e01480. doi: 10.1002/brb3.1480. Epub 2019 Nov 19.\n",
      "\n",
      "title: Genetic testing in fetuses with isolated agenesis of the corpus callosum.\n",
      "authors: ['She Q', 'Fu F', 'Guo X', 'Tan W', 'Liao C']\n",
      "source: J Matern Fetal Neonatal Med. 2019 Aug 27:1-201. doi: 10.1080/14767058.2019.1660769.\n",
      "\n",
      "title: Elongator Is Required for Root Stem Cell Maintenance by Regulating SHORTROOT Transcription.\n",
      "authors: ['Qi L', 'Zhang X', 'Zhai H', 'Liu J', 'Wu F', 'Li C', 'Chen Q']\n",
      "source: Plant Physiol. 2019 Jan;179(1):220-232. doi: 10.1104/pp.18.00534. Epub 2018 Nov 6.\n",
      "\n",
      "title: The Arabidopsis Elongator Subunit ELP3 and ELP4 Confer Resistance to Bacterial Speck in Tomato.\n",
      "authors: ['Pereira JA', 'Yu F', 'Zhang Y', 'Jones JB', 'Mou Z']\n",
      "source: Front Plant Sci. 2018 Jul 24;9:1066. doi: 10.3389/fpls.2018.01066. eCollection 2018.\n",
      "\n",
      "title: Elongator promotes the migration and invasion of hepatocellular carcinoma cell by the phosphorylation of AKT.\n",
      "authors: ['Xu Y', 'Zhou W', 'Ji Y', 'Shen J', 'Zhu X', 'Yu H', 'Guo J', 'Pang Z', 'Wei W']\n",
      "source: Int J Biol Sci. 2018 Apr 5;14(5):518-530. doi: 10.7150/ijbs.23511. eCollection 2018.\n",
      "\n",
      "title: Structural insights into the function of Elongator.\n",
      "authors: ['Dalwadi U', 'Yip CK']\n",
      "source: Cell Mol Life Sci. 2018 May;75(9):1613-1622. doi: 10.1007/s00018-018-2747-6. Epub 2018 Jan 13.\n",
      "\n",
      "title: Correlation of novel PAX6 gene abnormalities in aniridia and clinical presentation.\n",
      "authors: ['Sannan NS', 'Gregory-Evans CY', 'Lyons CJ', 'Lehman AM', 'Langlois S', 'Warner SJ', 'Zakrzewski H', 'Gregory-Evans K']\n",
      "source: Can J Ophthalmol. 2017 Dec;52(6):570-577. doi: 10.1016/j.jcjo.2017.04.006. Epub 2017 Jul 6.\n",
      "\n",
      "title: The Arabidopsis ELP3/ELO3 and ELP4/ELO1 genes enhance disease resistance in Fragaria vesca L.\n",
      "authors: ['Silva KJP', 'Brunings AM', 'Pereira JA', 'Peres NA', 'Folta KM', 'Mou Z']\n",
      "source: BMC Plant Biol. 2017 Dec 1;17(1):230. doi: 10.1186/s12870-017-1173-5.\n",
      "\n",
      "title: PAX6 aniridia syndrome: clinics, genetics, and therapeutics.\n",
      "authors: ['Lim HT', 'Kim DH', 'Kim H']\n",
      "source: Curr Opin Ophthalmol. 2017 Sep;28(5):436-447. doi: 10.1097/ICU.0000000000000405.\n",
      "\n",
      "title: Involvement of BcElp4 in vegetative development, various environmental stress response and virulence of Botrytis cinerea.\n",
      "authors: ['Shao W', 'Lv C', 'Zhang Y', 'Wang J', 'Chen C']\n",
      "source: Microb Biotechnol. 2017 Jul;10(4):886-895. doi: 10.1111/1751-7915.12720. Epub 2017 May 5.\n",
      "\n",
      "title: The familial dysautonomia disease gene IKBKAP is required in the developing and adult mouse central nervous system.\n",
      "authors: ['Chaverra M', 'George L', 'Mergy M', 'Waller H', 'Kujawa K', 'Murnion C', 'Sharples E', 'Thorne J', 'Podgajny N', 'Grindeland A', 'Ueki Y', 'Eiger S', 'Cusick C', 'Babcock AM', 'Carlson GA', 'Lefcort F']\n",
      "source: Dis Model Mech. 2017 May 1;10(5):605-618. doi: 10.1242/dmm.028258. Epub 2017 Feb 6.\n",
      "\n",
      "title: A CGH array procedure to detect PAX6 gene structural defects.\n",
      "authors: ['Franzoni A', 'Russo PD', 'Baldan F', \"D'Elia AV\", 'Puppin C', 'Penco S', 'Damante G']\n",
      "source: Mol Cell Probes. 2017 Apr;32:65-68. doi: 10.1016/j.mcp.2016.12.001. Epub 2016 Dec 2.\n",
      "\n",
      "title: A microRNA-328 binding site in PAX6 is associated with centrotemporal spikes of rolandic epilepsy.\n",
      "authors: ['Panjwani N', 'Wilson MD', 'Addis L', 'Crosbie J', 'Wirrell E', 'Auvin S', 'Caraballo RH', 'Kinali M', 'McCormick D', 'Oren C', 'Taylor J', 'Trounce J', 'Clarke T', 'Akman CI', 'Kugler SL', 'Mandelbaum DE', 'McGoldrick P', 'Wolf SM', 'Arnold P', 'Schachar R', 'Pal DK', 'Strug LJ']\n",
      "source: Ann Clin Transl Neurol. 2016 Jun 2;3(7):512-22. doi: 10.1002/acn3.320. eCollection 2016 Jul.\n",
      "\n",
      "title: Clinical utility gene card for: Aniridia.\n",
      "authors: ['Richardson R', 'Hingorani M', 'Van Heyningen V', 'Gregory-Evans C', 'Moosajee M']\n",
      "source: Eur J Hum Genet. 2016 Nov;24(11). pii: ejhg201673. doi: 10.1038/ejhg.2016.73. Epub 2016 Jul 6.\n",
      "\n",
      "title: Many obesity-associated SNPs strongly associate with DNA methylation changes at proximal promoters and enhancers.\n",
      "authors: ['Voisin S', 'Almen MS', 'Zheleznyakova GY', 'Lundberg L', 'Zarei S', 'Castillo S', 'Eriksson FE', 'Nilsson EK', 'Bluher M', 'Bottcher Y', 'Kovacs P', 'Klovins J', 'Rask-Andersen M', 'Schioth HB']\n",
      "source: Genome Med. 2015 Oct 8;7:103. doi: 10.1186/s13073-015-0225-4.\n",
      "\n",
      "title: A familial pericentric inversion of chromosome 11 associated with a microdeletion of 163 kb and microduplication of 288 kb at 11p13 and 11q22.3 without aniridia or eye anomalies.\n",
      "authors: ['Balay L', 'Totten E', 'Okada L', 'Zell S', 'Ticho B', 'Israel J', 'Kogan J']\n",
      "source: Am J Med Genet A. 2016 Jan;170A(1):202-9. doi: 10.1002/ajmg.a.37388. Epub 2015 Sep 30.\n",
      "\n",
      "title: Microdeletions of ELP4 Are Associated with Language Impairment, Autism Spectrum Disorder, and Mental Retardation.\n",
      "authors: ['Addis L', 'Ahn JW', 'Dobson R', 'Dixit A', 'Ogilvie CM', 'Pinto D', 'Vaags AK', 'Coon H', 'Chaste P', 'Wilson S', 'Parr JR', 'Andrieux J', 'Lenne B', 'Tumer Z', 'Leuzzi V', 'Aubell K', 'Koillinen H', 'Curran S', 'Marshall CR', 'Scherer SW', 'Strug LJ', 'Collier DA', 'Pal DK']\n",
      "source: Hum Mutat. 2015 Sep;36(9):842-50. doi: 10.1002/humu.22816. Epub 2015 Jun 30.\n",
      "\n",
      "title: Meta-analysis of Genome-Wide Association Studies Identifies Novel Loci Associated With Optic Disc Morphology.\n",
      "authors: ['Springelkamp H', 'Mishra A', 'Hysi PG', 'Gharahkhani P', 'Hohn R', 'Khor CC', 'Cooke Bailey JN', 'Luo X', 'Ramdas WD', 'Vithana E', 'Koh V', 'Yazar S', 'Xu L', 'Forward H', 'Kearns LS', 'Amin N', 'Iglesias AI', 'Sim KS', 'van Leeuwen EM', 'Demirkan A', 'van der Lee S', 'Loon SC', 'Rivadeneira F', 'Nag A', 'Sanfilippo PG', 'Schillert A', 'de Jong PT', 'Oostra BA', 'Uitterlinden AG', 'Hofman A', 'Zhou T', 'Burdon KP', 'Spector TD', 'Lackner KJ', 'Saw SM', 'Vingerling JR', 'Teo YY', 'Pasquale LR', 'Wolfs RC', 'Lemij HG', 'Tai ES', 'Jonas JB', 'Cheng CY', 'Aung T', 'Jansonius NM', 'Klaver CC', 'Craig JE', 'Young TL', 'Haines JL', 'MacGregor S', 'Mackey DA', 'Pfeiffer N', 'Wong TY', 'Wiggs JL', 'Hewitt AW', 'van Duijn CM', 'Hammond CJ']\n",
      "source: Genet Epidemiol. 2015 Mar;39(3):207-16. doi: 10.1002/gepi.21886. Epub 2015 Jan 28.\n",
      "\n",
      "title: A novel 11p13 microdeletion encompassing PAX6 in a Chinese Han family with aniridia, ptosis and mental retardation.\n",
      "authors: ['Hu P', 'Meng L', 'Ma D', 'Qiao F', 'Wang Y', 'Zhou J', 'Yi L', 'Xu Z']\n",
      "source: Mol Cytogenet. 2015 Jan 22;8(1):3. doi: 10.1186/s13039-015-0110-2. eCollection 2015.\n",
      "\n",
      "title: Electrospun fiber constructs for vocal fold tissue engineering: effects of alignment and elastomeric polypeptide coating.\n",
      "authors: ['Hughes LA', 'Gaston J', 'McAlindon K', 'Woodhouse KA', 'Thibeault SL']\n",
      "source: Acta Biomater. 2015 Feb;13:111-20. doi: 10.1016/j.actbio.2014.10.039. Epub 2014 Nov 4.\n",
      "\n",
      "title: Association of brain-derived neurotrophic factor (BDNF) and elongator protein complex 4 (ELP4) polymorphisms with benign epilepsy with centrotemporal spikes in a Greek population.\n",
      "authors: ['Gkampeta A', 'Fidani L', 'Clarimon J', 'Kalinderi K', 'Katopodi T', 'Zafeiriou D', 'Pavlou E']\n",
      "source: Epilepsy Res. 2014 Dec;108(10):1734-9. doi: 10.1016/j.eplepsyres.2014.09.005. Epub 2014 Sep 22.\n",
      "\n",
      "title: Remind me again what disease we are studying? A population genetics, genetic analysis, and real data perspective on why progress on identifying genetic influences on common epilepsies has been so slow.\n",
      "authors: ['Greenberg DA', 'Stewart WL']\n",
      "source: Prog Brain Res. 2014;213:199-221. doi: 10.1016/B978-0-444-63326-2.00011-9.\n",
      "\n",
      "title: Genetic basis for Saccharomyces cerevisiae biofilm in liquid medium.\n",
      "authors: ['Scherz K', 'Andersen', 'Bojsen R', 'Gro L', 'Rejkjaer', 'Sorensen', 'Weiss M', 'Nielsen', 'Lisby M', 'Folkesson A', 'Regenberg B']\n",
      "source: G3 (Bethesda). 2014 Jul 9;4(9):1671-80. doi: 10.1534/g3.114.010892.\n",
      "\n",
      "title: Analysis of ELP4, SRPX2, and interacting genes in typical and atypical rolandic epilepsy.\n",
      "authors: ['Reinthaler EM', 'Lal D', 'Jurkowski W', 'Feucht M', 'Steinbock H', 'Gruber-Sedlmayr U', 'Ronen GM', 'Geldner J', 'Haberlandt E', 'Neophytou B', 'Hahn A', 'Altmuller J', 'Thiele H', 'Toliat MR', 'Lerche H', 'Nurnberg P', 'Sander T', 'Neubauer BA', 'Zimprich F']\n",
      "source: Epilepsia. 2014 Aug;55(8):e89-93. doi: 10.1111/epi.12712. Epub 2014 Jul 3.\n",
      "\n",
      "title: Evaluation of the bulk platelet response and fibrinogen interaction to elastin-like polypeptide coatings.\n",
      "authors: ['Srokowski EM', 'Woodhouse KA']\n",
      "source: J Biomed Mater Res A. 2014 Feb;102(2):540-51. doi: 10.1002/jbm.a.34699. Epub 2013 Oct 4.\n",
      "\n",
      "title: Surface and adsorption characteristics of three elastin-like polypeptide coatings with varying sequence lengths.\n",
      "authors: ['Srokowski EM', 'Woodhouse KA']\n",
      "source: J Mater Sci Mater Med. 2013 Jan;24(1):71-84. doi: 10.1007/s10856-012-4772-6. Epub 2012 Sep 30.\n",
      "\n",
      "title: Insertional translocation of 15q25-q26 into 11p13 and duplication at 8p23.1 characterized by high resolution arrays in a boy with congenital malformations and aniridia.\n",
      "authors: ['Simioni M', 'Vieira TP', 'Sgardioli IC', 'Freitas EL', 'Rosenberg C', 'Maurer-Morelli CV', 'Lopes-Cendes I', 'Fett-Conte AC', 'Gil-da-Silva-Lopes VL']\n",
      "source: Am J Med Genet A. 2012 Nov;158A(11):2905-10. doi: 10.1002/ajmg.a.35603. Epub 2012 Sep 18.\n",
      "\n",
      "title: DERP6 (ELP5) and C3ORF75 (ELP6) regulate tumorigenicity and migration of melanoma cells as subunits of Elongator.\n",
      "authors: ['Close P', 'Gillard M', 'Ladang A', 'Jiang Z', 'Papuga J', 'Hawkes N', 'Nguyen L', 'Chapelle JP', 'Bouillenne F', 'Svejstrup J', 'Fillet M', 'Chariot A']\n",
      "source: J Biol Chem. 2012 Sep 21;287(39):32535-45. doi: 10.1074/jbc.M112.402727. Epub 2012 Aug 1.\n",
      "\n",
      "title: Emerging genetic influences in benign epilepsy with centro-temporal spikes - BECTS.\n",
      "authors: ['Gkampeta A', 'Pavlou E']\n",
      "source: Epilepsy Res. 2012 Sep;101(3):197-201. doi: 10.1016/j.eplepsyres.2012.06.011. Epub 2012 Jul 19.\n",
      "\n",
      "title: Major Susceptibility Genes for Common Idiopathic Epilepsies: ELP4 in Rolandic Epilepsy and BRD2 in Juvenile Myoclonic Epilepsy\n",
      "authors: ['Pal DK', 'Greenberg DA']\n",
      "source: ?\n",
      "\n",
      "title: Crystal structure of elongator subcomplex Elp4-6.\n",
      "authors: ['Lin Z', 'Zhao W', 'Diao W', 'Xie X', 'Wang Z', 'Zhang J', 'Shen Y', 'Long J']\n",
      "source: J Biol Chem. 2012 Jun 15;287(25):21501-8. doi: 10.1074/jbc.M112.341560. Epub 2012 May 2.\n",
      "\n",
      "title: Electrospun elastin-like polypeptide enriched polyurethanes and their interactions with vascular smooth muscle cells.\n",
      "authors: ['Blit PH', 'Battiston KG', 'Yang M', 'Paul Santerre J', 'Woodhouse KA']\n",
      "source: Acta Biomater. 2012 Jul;8(7):2493-503. doi: 10.1016/j.actbio.2012.03.032. Epub 2012 Mar 27.\n",
      "\n",
      "title: The Elongator subcomplex Elp456 is a hexameric RecA-like ATPase.\n",
      "authors: ['Glatt S', 'Letoquart J', 'Faux C', 'Taylor NM', 'Seraphin B', 'Muller CW']\n",
      "source: Nat Struct Mol Biol. 2012 Feb 19;19(3):314-20. doi: 10.1038/nsmb.2234.\n",
      "\n",
      "title: A genetic epidemiologic study of candidate genes involved in the optic nerve head morphology.\n",
      "authors: ['Gasten AC', 'Ramdas WD', 'Broer L', 'van Koolwijk LM', 'Ikram MK', 'de Jong PT', 'Aulchenko YS', 'Wolfs RC', 'Hofman A', 'Rivadeneira F', 'Uitterlinden AG', 'Oostra BA', 'Lemij HG', 'Klaver CC', 'Jansonius NM', 'Vingerling JR', 'van Duijn CM']\n",
      "source: Invest Ophthalmol Vis Sci. 2012 Mar 21;53(3):1485-91. doi: 10.1167/iovs.11-7384.\n",
      "\n",
      "title: Platelet inhibition and endothelial cell adhesion on elastin-like polypeptide surface modified materials.\n",
      "authors: ['Blit PH', 'McClung WG', 'Brash JL', 'Woodhouse KA', 'Santerre JP']\n",
      "source: Biomaterials. 2011 Sep;32(25):5790-800. doi: 10.1016/j.biomaterials.2011.04.067. Epub 2011 May 20.\n",
      "\n",
      "title: Large novel deletions detected in Chinese families with aniridia: correlation between genotype and phenotype.\n",
      "authors: ['Zhang X', 'Zhang Q', 'Tong Y', 'Dai H', 'Zhao X', 'Bai F', 'Xu L', 'Li Y']\n",
      "source: Mol Vis. 2011 Feb 19;17:548-57.\n",
      "\n",
      "title: A 556 kb deletion in the downstream region of the PAX6 gene causes familial aniridia and other eye anomalies in a Chinese family.\n",
      "authors: ['Cheng F', 'Song W', 'Kang Y', 'Yu S', 'Yuan H']\n",
      "source: Mol Vis. 2011 Feb 10;17:448-55.\n",
      "\n",
      "title: Surface immobilization of elastin-like polypeptides using fluorinated surface modifying additives.\n",
      "authors: ['Blit PH', 'Battiston KG', 'Woodhouse KA', 'Santerre JP']\n",
      "source: J Biomed Mater Res A. 2011 Mar 15;96(4):648-62. doi: 10.1002/jbm.a.33022. Epub 2011 Jan 25.\n",
      "\n",
      "title: Pleiotropic effects of the 11p13 locus on developmental verbal dyspraxia and EEG centrotemporal sharp waves.\n",
      "authors: ['Pal DK', 'Li W', 'Clarke T', 'Lieberman P', 'Strug LJ']\n",
      "source: Genes Brain Behav. 2010 Nov;9(8):1004-12. doi: 10.1111/j.1601-183X.2010.00648.x. Epub 2010 Oct 18.\n",
      "\n",
      "title: Platelet Adhesion and Fibrinogen Accretion on a Family of Elastin-Like Polypeptides.\n",
      "authors: ['Srokowski EM', 'Blit PH', 'McClung WG', 'Brash JL', 'Santerre JP', 'Woodhouse KA']\n",
      "source: J Biomater Sci Polym Ed. 2011;22(1-3):41-57. doi: 10.1163/092050609X12578498935594.\n",
      "\n",
      "title: A pure likelihood approach to the analysis of genetic association data: an alternative to Bayesian and frequentist analysis.\n",
      "authors: ['Strug LJ', 'Hodge SE', 'Chiang T', 'Pal DK', 'Corey PN', 'Rohde C']\n",
      "source: Eur J Hum Genet. 2010 Aug;18(8):933-41. doi: 10.1038/ejhg.2010.47. Epub 2010 Apr 28.\n",
      "\n",
      "title: Plant Elongator regulates auxin-related genes during RNA polymerase II transcription elongation.\n",
      "authors: ['Nelissen H', 'De Groeve S', 'Fleury D', 'Neyt P', 'Bruno L', 'Bitonti MB', 'Vandenbussche F', 'Van der Straeten D', 'Yamaguchi T', 'Tsukaya H', 'Witters E', 'De Jaeger G', 'Houben A', 'Van Lijsebettens M']\n",
      "source: Proc Natl Acad Sci U S A. 2010 Jan 26;107(4):1678-83. doi: 10.1073/pnas.0913559107. Epub 2010 Jan 8.\n",
      "\n",
      "title: A role for the elongator complex in zygotic paternal genome demethylation.\n",
      "authors: ['Okada Y', 'Yamagata K', 'Hong K', 'Wakayama T', 'Zhang Y']\n",
      "source: Nature. 2010 Jan 28;463(7280):554-8. doi: 10.1038/nature08732. Epub 2010 Jan 6.\n",
      "\n",
      "title: Human Elongator complex is involved in cell cycle and suppresses cell growth in 293T human embryonic kidney cells.\n",
      "authors: ['Gu J', 'Sun D', 'Zheng Q', 'Wang X', 'Yang H', 'Miao J', 'Jiang J', 'Wei W']\n",
      "source: Acta Biochim Biophys Sin (Shanghai). 2009 Oct;41(10):831-8. doi: 10.1093/abbs/gmp072.\n",
      "\n",
      "title: From rolandic epilepsy to continuous spike-and-waves during sleep and Landau-Kleffner syndromes: insights into possible genetic factors.\n",
      "authors: ['Rudolf G', 'Valenti MP', 'Hirsch E', 'Szepetowski P']\n",
      "source: Epilepsia. 2009 Aug;50 Suppl 7:25-8. doi: 10.1111/j.1528-1167.2009.02214.x.\n",
      "\n",
      "title: Elongator mediates ABA responses, oxidative stress resistance and anthocyanin biosynthesis in Arabidopsis.\n",
      "authors: ['Zhou X', 'Hua D', 'Chen Z', 'Zhou Z', 'Gong Z']\n",
      "source: Plant J. 2009 Oct;60(1):79-90. doi: 10.1111/j.1365-313X.2009.03931.x. Epub 2009 May 23.\n",
      "\n",
      "title: Centrotemporal sharp wave EEG trait in rolandic epilepsy maps to Elongator Protein Complex 4 (ELP4).\n",
      "authors: ['Strug LJ', 'Clarke T', 'Chiang T', 'Chien M', 'Baskurt Z', 'Li W', 'Dorfman R', 'Bali B', 'Wirrell E', 'Kugler SL', 'Mandelbaum DE', 'Wolf SM', 'McGoldrick P', 'Hardison H', 'Novotny EJ', 'Ju J', 'Greenberg DA', 'Russo JJ', 'Pal DK']\n",
      "source: Eur J Hum Genet. 2009 Sep;17(9):1171-81. doi: 10.1038/ejhg.2008.267. Epub 2009 Jan 28.\n",
      "\n",
      "title: Development and characterisation of novel cross-linked bio-elastomeric materials.\n",
      "authors: ['Srokowski EM', 'Woodhouse KA']\n",
      "source: J Biomater Sci Polym Ed. 2008;19(6):785-99. doi: 10.1163/156856208784522038.\n",
      "\n",
      "title: Subfunctionalization of duplicated zebrafish pax6 genes by cis-regulatory divergence.\n",
      "authors: ['Kleinjan DA', 'Bancewicz RM', 'Gautier P', 'Dahm R', 'Schonthaler HB', 'Damante G', 'Seawright A', 'Hever AM', 'Yeyati PL', 'van Heyningen V', 'Coutinho P']\n",
      "source: PLoS Genet. 2008 Feb;4(2):e29. doi: 10.1371/journal.pgen.0040029.\n",
      "\n",
      "title: A deletion 3' to the PAX6 gene in familial aniridia cases.\n",
      "authors: [\"D'Elia AV\", 'Pellizzari L', 'Fabbro D', 'Pianta A', 'Divizia MT', 'Rinaldi R', 'Grammatico B', 'Grammatico P', 'Arduino C', 'Damante G']\n",
      "source: Mol Vis. 2007 Jul 23;13:1245-50.\n",
      "\n",
      "title: The elongata mutants identify a functional Elongator complex in plants with a role in cell proliferation during organ growth.\n",
      "authors: ['Nelissen H', 'Fleury D', 'Bruno L', 'Robles P', 'De Veylder L', 'Traas J', 'Micol JL', 'Van Montagu M', 'Inze D', 'Van Lijsebettens M']\n",
      "source: Proc Natl Acad Sci U S A. 2005 May 24;102(21):7754-9. doi: 10.1073/pnas.0502600102. Epub 2005 May 13.\n",
      "\n",
      "title: [The Elp4 subunit of human Elongator complex partially complements the growth defects of yeast ELP4 deletion strain].\n",
      "authors: ['Li F', 'Han QJ', 'Luo DH', 'Lu J', 'Huang BQ']\n",
      "source: Yi Chuan Xue Bao. 2004 Jul;31(7):668-74.\n",
      "\n",
      "title: Molecular architecture, structure-function relationship, and importance of the Elp3 subunit for the RNA binding of holo-elongator.\n",
      "authors: ['Petrakis TG', 'Wittschieben BO', 'Svejstrup JQ']\n",
      "source: J Biol Chem. 2004 Jul 30;279(31):32087-92. doi: 10.1074/jbc.M403361200. Epub 2004 May 11.\n",
      "\n",
      "title: Subunit communications crucial for the functional integrity of the yeast RNA polymerase II elongator (gamma-toxin target (TOT)) complex.\n",
      "authors: ['Frohloff F', 'Jablonowski D', 'Fichtner L', 'Schaffrath R']\n",
      "source: J Biol Chem. 2003 Jan 10;278(2):956-61. doi: 10.1074/jbc.M210060200. Epub 2002 Nov 6.\n",
      "\n",
      "title: Frequent chromosome aberrations revealed by molecular cytogenetic studies in patients with aniridia.\n",
      "authors: ['Crolla JA', 'van Heyningen V']\n",
      "source: Am J Hum Genet. 2002 Nov;71(5):1138-49. doi: 10.1086/344396. Epub 2002 Oct 17.\n",
      "\n",
      "title: Molecular analysis of KTI12/TOT4, a Saccharomyces cerevisiae gene required for Kluyveromyces lactis zymocin action.\n",
      "authors: ['Fichtner L', 'Frohloff F', 'Burkner K', 'Larsen M', 'Breunig KD', 'Schaffrath R']\n",
      "source: Mol Microbiol. 2002 Feb;43(3):783-91. doi: 10.1046/j.1365-2958.2002.02794.x.\n",
      "\n",
      "title: Elongator is a histone H3 and H4 acetyltransferase important for normal histone acetylation levels in vivo.\n",
      "authors: ['Winkler GS', 'Kristjuhan A', 'Erdjument-Bromage H', 'Tempst P', 'Svejstrup JQ']\n",
      "source: Proc Natl Acad Sci U S A. 2002 Mar 19;99(6):3517-22. doi: 10.1073/pnas.022042899.\n",
      "\n",
      "title: Characterization of a novel gene adjacent to PAX6, revealing synteny conservation with functional significance.\n",
      "authors: ['Kleinjan DA', 'Seawright A', 'Elgar G', 'van Heyningen V']\n",
      "source: Mamm Genome. 2002 Feb;13(2):102-7. doi: 10.1007/s00335-001-3058-y.\n",
      "\n",
      "title: Kluyveromyces lactis zymocin mode of action is linked to RNA polymerase II function via Elongator.\n",
      "authors: ['Jablonowski D', 'Frohloff F', 'Fichtner L', 'Stark MJ', 'Schaffrath R']\n",
      "source: Mol Microbiol. 2001 Nov;42(4):1095-105. doi: 10.1046/j.1365-2958.2001.02705.x.\n",
      "\n",
      "title: Characterization of a six-subunit holo-elongator complex required for the regulated expression of a group of genes in Saccharomyces cerevisiae.\n",
      "authors: ['Krogan NJ', 'Greenblatt JF']\n",
      "source: Mol Cell Biol. 2001 Dec;21(23):8203-12. doi: 10.1128/MCB.21.23.8203-8212.2001.\n",
      "\n",
      "title: RNA polymerase II elongator holoenzyme is composed of two discrete subcomplexes.\n",
      "authors: ['Winkler GS', 'Petrakis TG', 'Ethelberg S', 'Tokunaga M', 'Erdjument-Bromage H', 'Tempst P', 'Svejstrup JQ']\n",
      "source: J Biol Chem. 2001 Aug 31;276(35):32743-9. doi: 10.1074/jbc.M105303200. Epub 2001 Jul 2.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for record in records:\n",
    "    print(\"title:\", record.get(\"TI\", \"?\"))\n",
    "    print(\"authors:\", record.get(\"AU\", \"?\"))\n",
    "    print(\"source:\", record.get(\"SO\", \"?\"))\n",
    "    print(\"\")\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "5.O objetivo que agora era pretendido, era procurar em cada um dos 61 artigos do PubMed, qual/quais dos artigos se relacionavam com cancro.Porém como estávamos a falar do cancro numa zona espefífica do corpo, se se obtivessem resultados na procura anterior, ía-se procurar todos os artigos relacionados com o cancro da mama.\n",
    "Raciocínio inicial que se pretendia implementar:\n",
    "ID->título->abstract->Progonóstico (CANCRO ou CANCRO DA MAMA)\n",
    "lista_artigos = lista com os 61 IDs\n",
    "artigos_para_reter = lista vazia # lista dos IDs dos artigos que contém informação sobre o cancro ou mais especificamente, sobre o cancro da mama \n",
    "for a in lista_artigos: # o \"a\" é um ID da lista_artigos\n",
    "    if “prognostic” in a.abstract:\n",
    "\t\tartigo_para_reter.append(a)\n",
    "    \n",
    "PROBLEMA que encontamos: Não reconhece Strings no Abstract!\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Abstract: OBJECTIVE: The association between ELP4 rs986527 polymorphism and the occurrence and development of intracranial arachnoid cyst was studied in this paper. METHODS: Eighty-five patients diagnosed with intracranial arachnoid cysts by cerebral computed tomography scan were selected. Sixty-three healthy volunteers for medical examination in hospitals served as controls. The cognition, depressive symptoms, and the likelihood of headache, dizziness, head trauma history, dementia, depression, and epilepsy were assessed. ELP4 genotypes and its allele frequency were determined by PCR, endonuclease restriction analysis, and gel electrophoresis. RESULTS: ELP4 rs986527 had three genotypes: TT, TC, and CC. The intracranial arachnoid cyst group showed no statistically significant difference in genotype frequencies compared with healthy controls. There was no significant correlation between ELP4 rs986527 polymorphism and location of intracranial arachnoid cyst. TC and C genotype frequencies were associated with a higher incidence of clinical symptoms than TT genotype frequencies, and C allele frequencies were associated with a significantly higher incidence of clinical symptoms compared with T allele frequencies. There was no significant difference in TNF-alpha and IL-1beta levels between TT/TC/CC genotypes before treatment. After treatment, the levels of TNF-alpha and IL-1beta were significantly decreased in different genotypes, and the decrease in CC was the greatest. The frequency of TT and TC genotypes was higher than that of CC genotypes. CONCLUSION: ELP4 rs986527 polymorphism affected the incidence of clinical symptoms and the levels of TNF-alpha and IL-1beta in patients with intracranial arachnoid cysts.\n",
      "Abstract: Purpose: The objectives of this study were to explore genetics pathogenesis of isolated agenesis of corpus callosum (ACC) and assess the utility of chromosomal microarray analysis (CMA) for genetic diagnosis of Isolated ACC. Methods: We analyzed the genomes of 16 fetuses with isolated ACC using Afymetrix CytoScan HD arrays and conducted further bioinformatic analysis for one proband fetus with an abnormal copy number variation (CNV). Results: Of the 16 fetal samples examined, 2 (12.5%) had pathogenic CNVs and 3 (18.75%) had variants of unknown significance. Two cases, case 2 and case 9, were found to have pathogenic CNVs. Bioinformatic analyses indicated that the CNV of one fetus (case 9) contained 115 annotated coding genes, 5 of which (SLC6A5, BDNF, ELP4, PAX6, and SLC1A2) have been associated with neurodevelopment. Three of these genes (SLC1A2, BDNF, and PAX6) may play a key role in ACC development. GO cluster analysis of the implicated genes revealed strong representations of protein binding and metal ion binding functions. KEGG pathway analysis pointed to four pathways: longevity regulating pathway, amyotrophic lateral sclerosis, cocaine addiction, and autophagy-animal. Conclusions: BDNF, SLC1A2, and PAX6 may be involved in the development of isolated ACC. CMA is a feasible technology for prenatal diagnosis of isolated ACC.\n",
      "Abstract: SHORTROOT (SHR) is essential for stem cell maintenance and radial patterning in Arabidopsis (Arabidopsis thaliana) roots, but how its expression is regulated is unknown. Here, we report that the Elongator complex, which consists of six subunits (ELP1 to ELP6), regulates the transcription of SHR Depletion of Elongator drastically reduced SHR expression and led to defective root stem cell maintenance and radial patterning. The importance of the nuclear localization of Elongator for its functioning, together with the insensitivity of the elp1 mutant to the transcription elongation inhibitor 6-azauracil, and the direct interaction of the ELP4 subunit with the carboxyl-terminal domain of RNA polymerase II, support the notion that Elongator plays important roles in transcription elongation. Indeed, we found that ELP3 associates with the premessenger RNA of SHR and that mutation of Elongator reduces the enrichment of RNA polymerase II on the SHR gene body. Moreover, Elongator interacted in vivo with SUPPRESSOR OF Ty4, a well-established transcription elongation factor that is recruited to the SHR locus. Together, these results demonstrate that Elongator acts in concert with SUPPRESSOR OF Ty4 to regulate the transcription of SHR.\n",
      "Abstract: Although production of tomato (Solanum lycopersicum) is threatened by a number of major diseases worldwide, it has been difficult to identify effective and durable management measures against these diseases. In this study, we attempted to improve tomato disease resistance by transgenic overexpression of genes encoding the Arabidopsis thaliana Elongator (AtELP) complex subunits AtELP3 and AtELP4. We show that overexpression of AtELP3 and AtELP4 significantly enhanced resistance to tomato bacterial speck caused by the Pseudomonas syringae pv. tomato strain J4 (Pst J4) without clear detrimental effects on plant growth and development. Interestingly, the transgenic plants exhibited resistance to Pst J4 only when inoculated through foliar sprays but not through infiltration into the leaf apoplast. Although this result suggested possible involvement of stomatal immunity, we found that Pst J4 inoculation did not induce stomatal closure and there were no differences in stomatal apertures and conductance between the transgenic and control plants. Further RNA sequencing and real-time quantitative PCR analyses revealed a group of defense-related genes to be induced to higher levels after infection in the AtELP4 transgenic tomato plants than in the control, suggesting that the enhanced disease resistance of the transgenic plants may be attributed to elevated induction of defense responses. Additionally, we show that the tomato genome contains single-copy genes encoding all six Elongator subunits (SlELPs), which share high identities with the AtELP proteins, and that SlELP3 and SlELP4 complemented the Arabidopsis Atelp3 and Atelp4 mutants, respectively, indicating that the function of tomato Elongator is probably conserved. Taken together, our results not only shed new light on the tomato Elongator complex, but also revealed potential candidate genes for engineering disease resistance in tomato.\n",
      "Abstract: The Elongator is a complex with multiple subunits (Elp1-Elp6) which promotes transcript elongation and protein translation. In this study, we investigated the effects of Elongator on the migration and invasion of HCC cells as well as the underlying mechanisms. We showed that overexpression of Elp3 or Elp4 promoted the migration and invasion of HCC cells, which was abolished when either Elp3 or Elp4 was silenced. The expression of matrix metalloproteinase-2 (MMP-2) and MMP-9 were enhanced by phosphorylation of AKT. Elongator-driven migration and invasion and the expression of MMP-2 and MMP-9 were reduced in HCC cells treated with AKT inhibitor LY294002. Depletion of Elp3 also reduced the phosphorylation of AKT induced by growth factors. In vivo assay of lung metastasis in mice demonstrated that overexpression of Elp3 increased tumor nodules metastatic to lung. Importantly, Elp3 was up-regulated in human HCC tissues, which was correlated with the phosphorylation of AKT and expression of MMP-2. Collectively, these results suggested that Elongator activated migration and invasion of HCC cells by promoting the expression of MMP-2 and MMP-9 through the PI3K/AKT signaling pathway. Our work suggests that Elongator might be a potential marker which promotes the metastasis of HCC.\n",
      "Abstract: Conserved from yeast to humans, Elongator is a protein complex implicated in multiple processes including transcription regulation, alpha-tubulin acetylation, and tRNA modification, and its defects have been shown to cause human diseases such as familial dysautonomia. Elongator consists of two copies of six core subunits (Elp1, Elp2, Elp3, Elp4, Elp5, and Elp6) that are organized into two subcomplexes: Elp1/2/3 and Elp4/5/6 and form a stable assembly of ~ 850 kDa in size. Although the catalytic subunit of Elongator is Elp3, which contains a radical S-adenosyl-L-methionine (SAM) domain and a putative histone acetyltransferase domain, the Elp4/5/6 subcomplex also possesses ATP-modulated tRNA binding activity. How at the molecular level, Elongator performs its multiple functions and how the different subunits regulate Elongator's activities remains poorly understood. Here, we provide an overview of the proposed functions of Elongator and describe how recent structural studies provide new insights into the mechanism of action of this multifunctional complex.\n",
      "Abstract: OBJECTIVE: To describe the clinical presentation and genotype of subjects with aniridia with a particular focus on foveal hypoplasia. DESIGN: Prospective cohort study. PARTICIPANTS: Thirty-three Canadian participants with aniridia and of various ethnic backgrounds residing in British Columbia. METHODS: Full ophthalmic examinations and posterior segment spectral domain-optical coherence tomography (SD-OCT) imaging were performed. Foveal hypoplasia was graded independently by 2 staff ophthalmologists. PAX6 sequencing was performed and chromosomal 11p anomalies investigated. Candidate gene and single-nucleotide polymorphism sequencing in genes functionally related to PAX6 were also studied. RESULTS: Best corrected visual acuities in the cohort ranged from 0.0 logMAR to no light perception. Total absence of iris tissue was seen in the majority (42 of 66 eyes). In those in whom SD-OCT was possible, foveal hypoplasia was seen in the majority (45 of 56 eyes, 80%). Molecular genetic defects involving PAX6 were identified in 30 participants (91%), including 4 novel PAX6 mutations (Gly18Val; Ser65ProfsX14; Met337ArgfsX18; Ser321CysfsX34) and 4 novel chromosome 11p deletions inclusive of PAX6 or a known PAX6 regulatory region. CONCLUSIONS: The number of PAX6 mutations associated with aniridia continues to increase. Variable foveal architecture despite nearly identical anterior segment disease in 4 participants with an Ex9 ELP4-Ex4 DCDC1 deletion suggested that molecular cues causing variation in disease in the posterior segment differ from those at play in the anterior segment. Results in 3 patients without identifiable PAX6 mutations and a review of the literature suggest that such cases be described as phenocopies rather than actual cases of the syndrome of aniridia.\n",
      "Abstract: BACKGROUND: Plant immune response is associated with a large-scale transcriptional reprogramming, which is regulated by numerous transcription regulators such as the Elongator complex. Elongator is a multitasking protein complex involved in diverse cellular processes, including histone modification, DNA methylation, and tRNA modification. In recent years, Elongator is emerging as a key regulator of plant immune responses. However, characterization of Elongator's function in plant immunity has been conducted only in the model plant Arabidopsis thaliana. It is thus unclear whether Elongator's role in plant immunity is conserved in higher plants. The objective of this study is to characterize transgenic woodland strawberry (Fragaria vesca L.) overexpressing the Arabidopsis Elongator (AtELP) genes, AtELP3 and AtELP4, and to determine whether F. vesca carries a functional Elongator complex. METHODS: Transgenic F. vesca and Arabidopsis plants were produced via Agrobacterium-mediated genetic transformation and characterized by morphology, PCR, real-time quantitative PCR, and disease resistance test. The Student's t test was used to analyze the data. RESULTS: Overexpression of AtELP3 and AtELP4 in F. vesca impacts plant growth and development and confers enhanced resistance to anthracnose crown rot, powdery mildew, and angular leaf spot, which are caused by the hemibiotrophic fungal pathogen Colletotrichum gloeosporioides, the obligate biotrophic fungal pathogen Podosphaera aphanis, and the hemibiotrophic bacterial pathogen Xanthomonas fragariae, respectively. Moreover, the F. vesca genome encodes all six Elongator subunits by single-copy genes with the exception of FvELP4, which is encoded by two homologous genes, FvELP4-1 and FvELP4-2. We show that FvELP4-1 complemented the Arabidopsis Atelp4/elo1-1 mutant, indicating that FvELP4 is biologically functional. CONCLUSIONS: This is the first report on overexpression of Elongator genes in plants. Our results indicate that the function of Elongator in plant immunity is most likely conserved in F. vesca and suggest that Elongator genes may hold potential for helping mitigate disease severity and reduce the use of fungicides in strawberry industry.\n",
      "Abstract: PURPOSE OF REVIEW: Aniridia is a rare and panocular disorder affecting most of the ocular structures which may have significant impact on vision. The purpose of this review is to describe the clinical features, genetics, and therapeutic options for this disease and to provide an update of current knowledge and latest research findings. RECENT FINDINGS: Aside from the ocular features, a variety of associated systemic abnormalities, including hormonal, metabolic, gastrointestinal, genitourinary, and neurologic pathologies have been reported in children with aniridia. Although mutations in PAX6 are a major cause of aniridia, genetic defects in nearby genes, such as TRIM44 or ELP4, have also been reported to cause aniridia. Recent improvement in genetic testing technique will help more rapid and precise diagnosis for aniridia. A promising therapeutic approach called nonsense suppression therapy has been introduced and successfully used in an animal model. SUMMARY: Aniridia is a challenging disease. The progressive nature of this condition and its potential complications require continuous and life-long ophthalmologic care. Genetic diagnosis for aniridia is important for establishing definitive molecular characterization as well as identifying individuals at high risk for Wilms tumor. Recent advancement in understanding the genetic pathogenesis of this disease offers promise for the approaches to treatment.\n",
      "Abstract: The Saccharomyces cerevisiae Elongator complex consisting of the six Elp1-Elp6 proteins has been proposed to participate in three distinct cellular processes: transcriptional elongation, polarized exocytosis and formation of modified wobble uridines in tRNA. In this study, we investigated the function of BcElp4 in Botrytis cinerea, which is homologous to S. cerevisiae Elp4. A bcelp4 deletion mutant was significantly impaired in vegetative growth, sclerotia formation and melanin biosynthesis. This mutant exhibited decreased sensitivity to osmotic and oxidative stresses as well as cell way-damaging agent. Pathogenicity assays revealed that BcElp4 is involved in the virulence of B. cinerea. In addition, the deletion of bcelp4 led to increased aerial mycelia development. All these defects were restored by genetic complementation of the bcelp4 deletion mutant with the wild-type bcelp4 gene. The results of this study indicated that BcElp4 is involved in regulation of vegetative development, various environmental stress response and virulence in B. cinerea.\n",
      "Abstract: Hereditary sensory and autonomic neuropathies (HSANs) are a genetically and clinically diverse group of disorders defined by peripheral nervous system (PNS) dysfunction. HSAN type III, known as familial dysautonomia (FD), results from a single base mutation in the gene IKBKAP that encodes a scaffolding unit (ELP1) for a multi-subunit complex known as Elongator. Since mutations in other Elongator subunits (ELP2 to ELP4) are associated with central nervous system (CNS) disorders, the goal of this study was to investigate a potential requirement for Ikbkap in the CNS of mice. The sensory and autonomic pathophysiology of FD is fatal, with the majority of patients dying by age 40. While signs and pathology of FD have been noted in the CNS, the clinical and research focus has been on the sensory and autonomic dysfunction, and no genetic model studies have investigated the requirement for Ikbkap in the CNS. Here, we report, using a novel mouse line in which Ikbkap is deleted solely in the nervous system, that not only is Ikbkap widely expressed in the embryonic and adult CNS, but its deletion perturbs both the development of cortical neurons and their survival in adulthood. Primary cilia in embryonic cortical apical progenitors and motile cilia in adult ependymal cells are reduced in number and disorganized. Furthermore, we report that, in the adult CNS, both autonomic and non-autonomic neuronal populations require Ikbkap for survival, including spinal motor and cortical neurons. In addition, the mice developed kyphoscoliosis, an FD hallmark, indicating its neuropathic etiology. Ultimately, these perturbations manifest in a developmental and progressive neurodegenerative condition that includes impairments in learning and memory. Collectively, these data reveal an essential function for Ikbkap that extends beyond the peripheral nervous system to CNS development and function. With the identification of discrete CNS cell types and structures that depend on Ikbkap, novel strategies to thwart the progressive demise of CNS neurons in FD can be developed.\n",
      "Abstract: Aniridia is a rare congenital disease characterized by eye development defects, in which the more evident clinical manifestation is iris absence or malformation. In most of the patients, aniridia is associated to PAX6 gene point mutations or deletions. When these deletions are large and involve other genes, a more complex disease, named WAGR syndrome, arises. In order to develop a new tool to analyze aniridia and WAGR subjects, a CGH array (CGHa) of the PAX6 genomic region was set up. We generated a custom microarray kit using an oligonucleotide-based platform that allows high resolution molecular profiling of genomic aberrations in 20 Mb of the 11p13 chromosomal region, centered on the PAX6 gene. The average probe spacing was 100 bp. Thirty-five subjects have been analyzed. The major advantage of CGHa compared to MLPA was the knowledge of the deletions borders. Our approach identifies patients harboring deletions including the WT1 gene and, therefore, at risk for kidney tumors. The CGHa assay confirmed that several aniridia patients show a deletion at the level of ELP4 gene, without involvement of the PAX6 exonic regions. In all these patients, deletions include the PAX6 transcriptional enhancer SIMO. This finding further highlights the role of mutation/deletion of long-range enhancers in monogenic human pathology.\n",
      "Abstract: OBJECTIVE: Rolandic epilepsy is a common genetic focal epilepsy of childhood characterized by centrotemporal sharp waves on electroencephalogram. In previous genome-wide analysis, we had reported linkage of centrotemporal sharp waves to chromosome 11p13, and fine mapping with 44 SNPs identified the ELP4-PAX6 locus in two independent US and Canadian case-control samples. Here, we aimed to find a causative variant for centrotemporal sharp waves using a larger sample and higher resolution genotyping array. METHODS: We fine-mapped the ELP4-PAX6 locus in 186 individuals from rolandic epilepsy families and 1000 population controls of European origin using the Illumina HumanCoreExome-12 v1.0 BeadChip. Controls were matched to cases on ethnicity using principal component analysis. We used generalized estimating equations to assess association, followed up with a bioinformatics survey and literature search to evaluate functional significance. RESULTS: Homozygosity at the T allele of SNP rs662702 in the 3' untranslated region of PAX6 conferred increased risk of CTS: Odds ratio = 12.29 (95% CI: 3.20-47.22), P = 2.6 x 10(-4) and is seen in 3.9% of cases but only 0.3% of controls. INTERPRETATION: The minor T allele of SNP rs662702 disrupts regulation by microRNA-328, which is known to result in increased PAX6 expression in vitro. This study provides, for the first time, evidence of a noncoding genomic variant contributing to the etiology of a common human epilepsy via a posttranscriptional regulatory mechanism.\n",
      "Abstract: ?\n",
      "Abstract: BACKGROUND: The mechanisms by which genetic variants, such as single nucleotide polymorphisms (SNPs), identified in genome-wide association studies act to influence body mass remain unknown for most of these SNPs, which continue to puzzle the scientific community. Recent evidence points to the epigenetic and chromatin states of the genome as having important roles. METHODS: We genotyped 355 healthy young individuals for 52 known obesity-associated SNPs and obtained DNA methylation levels in their blood using the Illumina 450 K BeadChip. Associations between alleles and methylation at proximal cytosine residues were tested using a linear model adjusted for age, sex, weight category, and a proxy for blood cell type counts. For replication in other tissues, we used two open-access datasets (skin fibroblasts, n = 62; four brain regions, n = 121-133) and an additional dataset in subcutaneous and visceral fat (n = 149). RESULTS: We found that alleles at 28 of these obesity-associated SNPs associate with methylation levels at 107 proximal CpG sites. Out of 107 CpG sites, 38 are located in gene promoters, including genes strongly implicated in obesity (MIR148A, BDNF, PTPMT1, NR1H3, MGAT1, SCGB3A1, HOXC12, PMAIP1, PSIP1, RPS10-NUDT3, RPS10, SKOR1, MAP2K5, SIX5, AGRN, IMMP1L, ELP4, ITIH4, SEMA3G, POMC, ADCY3, SSPN, LGR4, TUFM, MIR4721, SULT1A1, SULT1A2, APOBR, CLN3, SPNS1, SH2B1, ATXN2L, and IL27). Interestingly, the associated SNPs are in known eQTLs for some of these genes. We also found that the 107 CpGs are enriched in enhancers in peripheral blood mononuclear cells. Finally, our results indicate that some of these associations are not blood-specific as we successfully replicated four associations in skin fibroblasts. CONCLUSIONS: Our results strongly suggest that many obesity-associated SNPs are associated with proximal gene regulation, which was reflected by association of obesity risk allele genotypes with differential DNA methylation. This study highlights the importance of DNA methylation and other chromatin marks as a way to understand the molecular basis of genetic variants associated with human diseases and traits.\n",
      "Abstract: Interstitial deletions of 11p13 involving MPPED2, DCDC5, DCDC1, DNAJC24, IMMP1L, and ELP4 are previously reported to have downstream transcriptional effects on the expression of PAX6, due to a downstream regulatory region (DRR). Currently, no clear genotype-phenotype correlations have been established allowing for conclusive information regarding the exact location of the PAX6 DRR, though its location has been approximated in mouse models to be within the Elp4 gene. Of the clinical reports currently published examining patients with intact PAX6 genes but harboring deletions identified in genes downstream of PAX6, 100% indicate phenotypes which include aniridia, whereas approximately half report additional eye deformities, autism, or intellectual disability. In this clinical report, we present a 12-year-old male patient, his brother, and mother with pericentric inversions of chromosome 11 associated with submicroscopic interstitial deletions of 11p13 and duplications of 11q22.3. The inversions were identified by standard cytogenetic analysis; microarray and FISH detected the chromosomal imbalance. The patient's phenotype includes intellectual disability, speech abnormalities, and autistic behaviors, but interestingly neither the patient, his brother, nor mother have aniridia or other eye anomalies. To the best of our knowledge, these findings in three family members represent the only reported cases with 11p13 deletions downstream of PAX6 not demonstrating phenotypic characteristics of aniridia or abnormal eye development. Although none of the deleted genes are obvious candidates for the patient's phenotype, the absence of aniridia in the presence of this deletion in all three family members further delineates the location of the DRR for PAX6.\n",
      "Abstract: Copy-number variations (CNVs) are important in the aetiology of neurodevelopmental disorders and show broad phenotypic manifestations. We compared the presence of small CNVs disrupting the ELP4-PAX6 locus in 4,092 UK individuals with a range of neurodevelopmental conditions, clinically referred for array comparative genomic hybridization, with WTCCC controls (n = 4,783). The phenotypic analysis was then extended using the DECIPHER database. We followed up association using an autism patient cohort (n = 3,143) compared with six additional control groups (n = 6,469). In the clinical discovery series, we identified eight cases with ELP4 deletions, and one with a partial duplication of ELP4 and PAX6. These cases were referred for neurological phenotypes including language impairment, developmental delay, autism, and epilepsy. Six further cases with a primary diagnosis of autism spectrum disorder (ASD) and similar secondary phenotypes were identified with ELP4 deletions, as well as another six (out of nine) with neurodevelopmental phenotypes from DECIPHER. CNVs at ELP4 were only present in 1/11,252 controls. We found a significant excess of CNVs in discovery cases compared with controls, P = 7.5 x 10(-3) , as well as for autism, P = 2.7 x 10(-3) . Our results suggest that ELP4 deletions are highly likely to be pathogenic, predisposing to a range of neurodevelopmental phenotypes from ASD to language impairment and epilepsy.\n",
      "Abstract: Primary open-angle glaucoma is the most common optic neuropathy and an important cause of irreversible blindness worldwide. The optic nerve head or optic disc is divided in two parts: a central cup (without nerve fibers) surrounded by the neuroretinal rim (containing axons of the retinal ganglion cells). The International Glaucoma Genetics Consortium conducted a meta-analysis of genome-wide association studies consisting of 17,248 individuals of European ancestry and 6,841 individuals of Asian ancestry. The outcomes of the genome-wide association studies were disc area and cup area. These specific measurements describe optic nerve morphology in another way than the vertical cup-disc ratio, which is a clinically used measurement, and may shed light on new glaucoma mechanisms. We identified 10 new loci associated with disc area (CDC42BPA, F5, DIRC3, RARB, ABI3BP, DCAF4L2, ELP4, TMTC2, NR2F2, and HORMAD2) and another 10 new loci associated with cup area (DHRS3, TRIB2, EFEMP1, FLNB, FAM101, DDHD1, ASB7, KPNB1, BCAS3, and TRIOBP). The new genes participate in a number of pathways and future work is likely to identify more functions related to the pathogenesis of glaucoma.\n",
      "Abstract: PURPOSE: To explore possible genetic aberrations in a Chinese family with aniridia, ptosis and mental retardation, and provide genetic evidence for the prenatal diagnosis. METHODS: 14 exons of PAX6 in the proband were sequenced by the Sanger sequencing technique. Multiplex ligation-dependent probe amplification (MLPA) technique was employed to further explore gene alterations of PAX6. Single nucleotide polymorphisms-array (SNP-array) assay was applied to screen potential pathologic genome-wide copy number variations (CNV). RESULTS: There were no detectable pathogenic mutations in the 14 exons of PAX6 in the proband. MLPA indicated a heterozygous deletion encompassing all PAX6 gene regions covered and a partial upstream region. SNP-array assay detected a heterozygous 11p13 microdeletion with a length of 518 kb in the proband, spanning two whole annotated genes, elongation factor protein 4 (ELP4), the paired box gene 6 (PAX6), and partial IMP1 inner-mitochondrial membrane (IMMP1L) gene. SNP-array revealed her affected brother carried the identical deletion. CONCLUSIONS: The 518 kb heterozygous deletion in 11p13 encompassing PAX6 should be the genetic etiology for the familial aniridia.\n",
      "Abstract: Vocal fold lamina propria extracellular matrix (ECM) is highly aligned and when injured, becomes disorganized with loss of the tissue's critical biomechanical properties. This study examines the effects of electrospun fiber scaffold architecture and elastin-like polypeptide (ELP4) coating on human vocal fold fibroblast (HVFF) behavior for applications toward tissue engineering the vocal fold lamina propria. Electrospun Tecoflex scaffolds were made with aligned and unaligned fibers, and were characterized using scanning electron microscopy and uniaxial tensile testing. ELP4 was successfully adsorbed onto the scaffolds; HVFFs were seeded and their viability, proliferation, morphology and gene expression were characterized. Aligned and unaligned scaffolds had initial elastic moduli of approximately 14 MPa, approximately 5 MPa and approximately 0.3 MPa, approximately 0.6 MPa in the preferred and cross-preferred directions, respectively. Scaffold topography had an effect on the orientation of the cells, with HVFFs seeded on aligned scaffolds having a significantly different (p<0.001) angle of orientation than HVFFs cultured on unaligned scaffolds. This same effect and significant difference (p<0.001) was seen on aligned and unaligned scaffolds coated with ELP4. Scaffold alignment and ELP4 coating impacted ECM gene expression. ELP4 coating, and aligned scaffolds upregulated elastin synthesis when tested on day 7 without a concomitant upregulation of collagen III synthesis. Collectively, results indicate that aligned electrospun scaffolds and ELP4 coating are promising candidates in the development of biodegradeable vocal fold lamina propria constructs.\n",
      "Abstract: PURPOSE: Benign epilepsy with centrotemporal spikes (BECTS) is considered to be the most common childhood epileptic syndrome. Different mutations in genes that control the excitability of neurons have been described. Recent reports on the involvement of the BDNF and ELP4 genes in cell motility, migration, and adhesion raise the possibility that these genes are involved in pathogenesis of BECTS. MATERIALS AND METHODS: We conducted a case-control association study on 60 patients with BECTS and 60 control participants to assess the influence of the BDNF and ELP4 polymorphisms on BECTS. The polymorphisms were detected with a PCR-RFLP method. Moreover, we explored the possible association of these polymorphisms with clinical and electroencephalographic parameters of patients with BECTS. RESULTS: Our results show no difference in BDNF and ELP4 genotype frequencies between patients and controls. Haplotype analysis also revealed no statistical difference. CONCLUSION: The role of BDNF and ELP4 polymorphisms remains controversial.\n",
      "Abstract: In this chapter, we will use the concepts and perspective of population genetics and genetic analysis to discuss the trends that are currently popular in the epilepsy genetics literature. We will also show that some of the recent approaches to understanding epilepsy genetics are unlikely to lead to insight into the causes of the most common epilepsies. Much effort has been directed toward very rare influences on epilepsy expression and away from approaches that could shed light on the most heritable epilepsies. The latest genetic technology, e.g., exome sequencing and copy number variants (CNV) analysis, has been applied without proper consideration of the important principles of population genetics, inheritance, or phenotype definition that are critical in common disease studies. We will show that ignoring these issues has led to unlikely suppositions about etiology and about which genes should be pursued. We also suggest that, taken together, the existing evidence supports a neurodevelopmental origin for common epilepsies. We recommend a return to thorough phenotype definition and family studies and a more careful and thoughtful application of the new technology.\n",
      "Abstract: Biofilm-forming microorganisms switch between two forms: free-living planktonic and sessile multicellular. Sessile communities of yeast biofilms in liquid medium provide a primitive example of multicellularity and are clinically important because biofilms tend to have other growth characteristics than free-living cells. We investigated the genetic basis for yeast, Saccharomyces cerevisiae, biofilm on solid surfaces in liquid medium by screening a comprehensive deletion mutant collection in the Sigma1278b background and found 71 genes that were essential for biofilm development. Quantitative northern blots further revealed that AIM1, ASG1, AVT1, DRN1, ELP4, FLO8, FMP10, HMT1, KAR5, MIT1, MRPL32, MSS11, NCP1, NPR1, PEP5, PEX25, RIM8, RIM101, RGT1, SNF8, SPC2, STB6, STP22, TEC1, VID24, VPS20, VTC3, YBL029W, YBL029C-A, YFL054C, YGR161W-C, YIL014C-A, YIR024C, YKL151C, YNL200C, YOR034C-A, and YOR223W controlled biofilm through FLO11 induction. Almost all deletion mutants that were unable to form biofilms in liquid medium also lost the ability to form surface-spreading biofilm colonies (mats) on agar and 69% also lost the ability to grow invasively. The protein kinase A isoform Tpk3p functioned specifically in biofilm and mat formation. In a tpk3 mutant, transcription of FLO11 was induced three-fold compared with wild-type, but biofilm development and cell-cell adhesion was absent, suggesting that Tpk3p regulates FLO11 positive posttranscriptionally and negative transcriptionally.The study provides a resource of biofilm-influencing genes for additional research on biofilm development and suggests that the regulation of FLO11 is more complex than previously anticipated.\n",
      "Abstract: Rolandic epilepsy (RE) and its atypical variants (atypical rolandic epilepsy, ARE) along the spectrum of epilepsy-aphasia disorders are characterized by a strong but largely unknown genetic basis. Two genes with a putative (ELP4) or a proven (SRPX2) function in neuronal migration were postulated to confer susceptibility to parts of the disease spectrum: the ELP4 gene to centrotemporal spikes and SRPX2 to ARE. To reexamine these findings, we investigated a cohort of 280 patients of European ancestry with RE/ARE for the etiological contribution of these genes and their close interaction partners. We performed next-generation sequencing and single-nucleotide polymorphism (SNP)-array based genotyping to screen for sequence and structural variants. In comparison to European controls we could not detect an enrichment of rare deleterious variants of ELP4, SRPX2, or their interaction partners in affected individuals. The previously described functional p.N327S variant in the X chromosomal SRPX2 gene was detected in two affected individuals (0.81%) and also in controls (0.26%), with some preponderance of male patients. We did not detect an association of SNPs in the ELP4 gene with centrotemporal spikes as previously reported. In conclusion our data do not support a major role of ELP4 and SRPX2 in the etiology of RE/ARE.\n",
      "Abstract: In this work, we expand on our understanding of the thrombogenicity of coatings prepared with three different recombinant elastin-like polypeptides (ELPs). The bulk platelet response of the ELP coatings was characterized following whole blood contact under physiological shear flow (300 s(-1) ) using flow cytometry. Prolonged exposure to shear flow (1-h) indicated that materials coated with the longer ELP coatings (ELP2 and ELP4) had less bulk platelet activation and microparticle formation than materials coated with the shorter ELP1. Quartz crystal microbalance with dissipation (QCM-D) was used to monitor the binding of the platelet membrane receptor GPIIb/IIIa to ELP-adsorbed fibrinogen (Fg) surfaces. Compared to the shorter ELPs, a lower amount of Fg adsorbed to the ELP4 coated material and ELP4 appeared to form a softer, more structurally flexible coating layer. When Fg was adsorbed to the ELP coated surface it demonstrated an altered binding for GPIIb/IIIa that was inhibited in the presence of an AGDV-containing peptide but not an RGD-containing peptide. Conversely, on the shorter ELP coatings, binding of GPIIb/IIIa to an adsorbed Fg layer was partially inhibited in the presence of an RGD-containing peptide. These results indicate that both the quantity and conformational state of Fg varies when adsorbed to surfaces coated with ELPs of varying sequence length, which may be mediating their platelet response. Collectively, the findings reinforce the applicability of the ELPs as potential thromboresistant coatings, especially with the use of the longer polypeptide-ELP4.\n",
      "Abstract: The surface properties of a family of elastin-like polypeptides (ELPs), differing in molecular weight and sequence length, were investigated to understand how the nature of the polypeptide film might contribute to their thrombogenic profile. Physical adsorption of the ELPs onto Mylar increased surface wettability as the sequence length decreased while X-ray spectroscopy analysis showed an increasing amide content with sequence length. Chemical force microscopy analysis revealed that the ELP-coated surfaces displayed purely hydrophilic adhesion forces that increased as the ELP sequence length decreased. Adsorption isotherms performed using the quartz crystal microbalance with dissipation, showed that the surface coverage increased with ELP sequence length. The longer polypeptides (ELP2 and ELP4) also displayed higher specific dissipation values indicating that they established films with greater structural flexibility and associated water content than the shorter polypeptide, ELP1. Additionally, the stability of the ELP coating was lower with the shorter polypeptides. This study highlights the different surface properties of the ELP coatings as well as the dynamic nature of the ELP adsorbed layer wherein the conformational state may be an important factor contributing to their blood response.\n",
      "Abstract: We report on a boy presenting submucous cleft palate, hydronephrosis, ventriculoseptal defect, aniridia, and developmental delay. Additional material on 11p13 was cytogenetically visible and array analyses identified a duplicated segment on 15q25-26 chromosome region; further, array analyses revealed a small deletion (49 kb) at 11p13 region involving the ELP4 gene and a duplication at 8p23.1. Results were confirmed with both molecular and molecular cytogenetics techniques. Possibilities for etiological basis of clinical phenotype are discussed.\n",
      "Abstract: The Elongator complex is composed of 6 subunits (Elp1-Elp6) and promotes RNAPII transcript elongation through histone acetylation in the nucleus as well as tRNA modification in the cytoplasm. This acetyltransferase complex directly or indirectly regulates numerous biological processes ranging from exocytosis and resistance to heat shock in yeast to cell migration and neuronal differentiation in higher eukaryotes. The identity of human ELP1 through ELP4 has been reported but human ELP5 and ELP6 have remained uncharacterized. Here, we report that DERP6 (ELP5) and C3ORF75 (ELP6) encode these subunits of human Elongator. We further investigated the importance and function of these two subunits by a combination of biochemical analysis and cellular assays. Our results show that DERP6/ELP5 is required for the integrity of Elongator and directly connects ELP3 to ELP4. Importantly, the migration and tumorigenicity of melanoma-derived cells are significantly decreased upon Elongator depletion through ELP1 or ELP3. Strikingly, DERP6/ELP5 and C3ORF75/ELP6-depleted melanoma cells have similar defects, further supporting the idea that DERP6/ELP5 and C3ORF75/ELP6 are essential for Elongator function. Together, our data identify DERP6/ELP5 and C3ORF75/ELP6 as key players for migration, invasion and tumorigenicity of melanoma cells, as integral subunits of Elongator.\n",
      "Abstract: BECTS is considered to be the most common childhood epileptic syndrome. Multifactorial inheritance is the most important model accounting for the genetic behavior of the common epilepsies. In recent years, different mutations in genes that control the excitability of neurons have been described. Recent reports on the involvement of the BDNF and ELP4 genes with possible roles in cell motility, migration, and adhesion have provided first insights into the complex molecular bases of childhood focal epilepsies. However, in the most common idiopathic benign childhood epilepsies (BECTS and occipital epilepsies), major breakthroughs are still awaited.\n",
      "Abstract: Clinical observations confirm that the idiopathic epilepsies are neurodevelopmental disorders strongly influenced by genetic factors. Genetic epidemiological studies prove major genetic influences on two common forms of idiopathic epilepsy to be discussed here: rolandic epilepsy (RE) and juvenile myoclonic epilepsy (JME). The common forms of these idiopathic syndromes have a complex genetic inheritance and this fact complicates finding and elucidating the susceptibility genes as well as proving their pathogenic role. Genetic heterogeneity and phenotype definition are much more serious factors in studying common complex epilepsies than they are in the study of densely affected epilepsy pedigrees showing Mendelian inheritance. In complex disorders, several genes and sometimes environmental factors are believed to contribute to disease etiology; as a result, proving a causative role for any one gene, in a genetic model involving multiple genes, can present a challenge. In this chapter, we first review the methodological issues associated with analyzing complex genetic disorders, then explain the strategy that we have used in investigating these two particular syndromes. Genomewide linkage analyses and fine-mapping have resulted in the identification of susceptibility genes ELP4 in Rolandic epilepsy and BRD2 in juvenile myoclonic epilepsy. We discuss the evidence in favor of a pathogenic role for each gene and the further steps that are required to establish causality.\n",
      "Abstract: Elongator is a multiprotein complex composed of two subcomplexes, Elp1-3 and Elp4-6. Elongator is highly conserved between yeast and humans and plays an important role in RNA polymerase II-mediated transcriptional elongation and many other processes, including cytoskeleton organization, exocytosis, and tRNA modification. Here, we determined the crystal structure of the Elp4-6 subcomplex of yeast. The overall structure of Elp4-6 revealed that Elp6 acts as a bridge to assemble Elp4 and Elp5. Detailed structural and sequence analyses revealed that each subunit in the Elp4-6 subcomplex forms a RecA-ATPase-like fold, although it lacks the key sequence signature of ATPases. Site-directed mutagenesis and biochemical analyses indicated that the Elp4-6 subcomplex can assemble into a hexameric ring-shaped structure in vitro and in vivo. Furthermore, GST pulldown assays showed that the ring-shaped assembly of the Elp4-6 subcomplex is important for its specific histone H3 binding. Our results may shed light on the substrate recognition and assembly of the holo-Elongator complex.\n",
      "Abstract: In vascular tissue, elastin is an essential extracellular matrix protein that plays an important biomechanical and biological signalling role. Native elastin is insoluble and is difficult to extract from tissues, which results in its relatively rare use for the fabrication of vascular tissue engineering scaffolds. Recombinant elastin-like polypeptide-4 (ELP4), which mimics the structure and function of native tropoelastin, represents a practical alternative to the native elastic fibre for vascular applications. In this study, electrospinning was utilized to fabricate fibrous scaffolds which were subsequently surface modified with ELP4 and used as substrates for smooth muscle cell culture. ELP4 surface modified materials demonstrated enhanced smooth muscle cell (SMC) adhesion and maintenance of cell numbers over a 1-week period relative to controls. SMCs seeded on the ELP4 surface modified materials were also shown to exhibit the cell morphology and biological markers of a contractile phenotype including a spindle-like morphology, actin filament organization and smooth muscle myosin heavy chain expression. Competitive inhibition experiments demonstrated that the elastin-laminin cell surface receptor and its affinity for the VGVAPG peptide sequence on ELP4 molecules are likely involved in the initial SMC contact with the ELP4 modified materials. Elastin-like polypeptides show promise as surface modifiers for candidate scaffolds for engineering contractile vascular tissues.\n",
      "Abstract: Elongator was initially described as an RNA polymerase II-associated factor but has since been associated with a broad range of cellular activities. It has also attracted clinical attention because of its role in certain neurodegenerative diseases. Here we describe the crystal structure of the Saccharomyces cerevisiae subcomplex of Elongator proteins 4, 5 and 6 (Elp456). The subunits each show almost identical RecA folds that form a heterohexameric ring-like structure resembling hexameric RecA-like ATPases. This structural finding is supported by different complementary in vitro and in vivo approaches, including the specific binding of the hexameric Elp456 subcomplex to tRNAs in a manner regulated by ATP. Our results support a role of Elongator in tRNA modification, explain the importance of each of the Elp4, Elp5 and Elp6 subunits for complex integrity and suggest a model for the overall architecture of the holo-Elongator complex.\n",
      "Abstract: PURPOSE: The size of the optic nerve head, referred to as disc area (DA), and the vertical cup-disc ratio (VCDR), are clinically relevant parameters for glaucomatous optic neuropathy. Although these measures have a high heritability, little is known about the underlying genes. Previously, the genes SALL1 and SIX1 were found to be genome-wide significantly associated with DA and VCDR. The purpose of the present study was to investigate whether genes encoding protein known to interact with protein encoded by SALL1 and SIX1 are also associated with either DA or VCDR. METHODS: A total of 38 candidate genes were chosen covering all known proteins interacting with SALL1 and SIX1. These were initially studied in the Rotterdam Study (RS)-I, including 5312 Caucasian subjects characterized for DA and VCDR. Positive findings were further investigated in two independent cohorts (RS-II and RS-III) and finally replicated in a fourth population (ERF). Bonferroni correction was applied to the meta-analyses. RESULTS: Three loci were found to be associated with DA. The only locus significant after correcting for multiple testing is located on chromosome 11p13. Three single nucleotide polymorphisms (SNPs) in ELP4, a gene which neighbors and plays a crucial role in the expression of PAX6, show association in meta-analysis of the four cohorts yielding P values of respectively 4.79 x 10(-6), 3.92 x 10(-6), and 4.88 x 10(-6) which is below the threshold dictated by the most conservative Bonferroni correction (P = 5.2 x 10(-6)). CONCLUSIONS: This study suggests that the ELP4-PAX6 region plays a role in the DA. Further research to confirm this finding is needed.\n",
      "Abstract: Platelet adhesion and activation are important early markers of biomaterial blood compatibility, while surfaces that promote enhanced endothelial cell adhesion and eNOS expression are strategic targets for long term vascular graft applications. Materials surface modified with fluorinated surface modifiers, containing peptides inspired from elastin cross-linking domains, have been used for the cross-linking of elastin-like polypeptide 4 (ELP4) macromolecules onto polyurethane surfaces. In the present study, ELP4 modified polyurethanes were evaluated in vitro to assess platelet adhesion, microparticle formation and bulk platelet activation following blood-material interactions. Reduced platelet adhesion and bulk platelet activation were observed following contact between reconstituted human blood and the ELP4 materials, relative to the uncoated base polyurethane controls. ELP4 modified materials also promoted endothelial cell adhesion and retention over a period of one week and showed that the endothelial cells exhibited an organized actin cytoskeleton and enhanced endothelial nitric oxide synthase (eNOS) expression relative to the control surfaces. These results indicate that polyurethane elastomers modified with ELP4 covalently bound to fluorinated surface modifiers provide a promising approach for endowing synthetic elastomers with both reduced blood platelet activation properties and enhanced endothelial cell adhesion for potential use in vascular graft applications.\n",
      "Abstract: PURPOSE: To describe the clinical and genetic findings in two Chinese families with aniridia and other ocular abnormalities. METHODS: Two unrelated families were examined clinically. After informed consent was obtained, genomic DNA was extracted from the venous blood of all participants. Mutation screening of all exons of the PAX6 (paired box gene 6) gene was performed by direct sequencing of PCR-amplified DNA fragments. Multiplex ligation-dependent probe amplification (MLPA) was performed to detect large deletions. Linkage analysis was used to validate the large deletions revealed by MLPA in all available family members. RESULTS: Clinical examination and pedigree analysis revealed one four-generation family (85) and one three- generation family (86) with total aniridia, congenital cataracts, foveal hypoplasia, and glaucoma. No mutation in PAX6 was identified after PCR-sequencing. Through MLPA analysis, a large deletion including the whole PAX6 gene, DKFZp686k1684 (hypothetical LOC440034), and the RCN1 (reticulocalbin 1) gene was detected in family 85; a 3' deletion to the PAX6 gene including the ELP4 (elongator complex protein 4) and the DCDC1 (doublecortin domain containing 1) gene was identified in family 86.The two large deletions were confirmed with linkage analysis and the \"loss of heterozygous\" in the different PAX6 regions were co-segregated with the phenotype of the two families, respectively. CONCLUSIONS: Patients with the PAX6 contiguous gene deletion, including the RCN1 gene, presented more severe vision impairments than those carrying the PAX6 3' deletion. Large deletions may account for several Chinese families and sporadic cases with aniridia and screening for these kinds of alterations should be included in aniridia patients' analyses.\n",
      "Abstract: PURPOSE: The paired box gene 6 (PAX6) on human chromosome 11p13 is an essential transcription factor for eye formation in animals. Mutations in PAX6 can lead to varieties of autosomal-dominant ocular malformations with aniridia as the major clinical signs. Known genetic alterations causing haplo-insufficiency of PAX6 include nonsense mutations, frame-shift mutations, splicing errors, or genomic deletions. The purpose of this study was to identify genetic defects as the underlying cause of familial aniridia in a large Chinese family. METHODS: All exons of PAX6 in the proband were sequenced by the Sanger sequencing technique. The genome of the proband was evaluated by a microarray-based comparative genomic hybridization (aCGH). Quantitative real-time PCR was applied to verify the abnormal aCGH findings in the proband and to test five other family members. RESULTS: There were no detectable pathogenic mutations in the exons of PAX6 in the proband. The aCGH analysis showed two copies of PAX6 but revealed a 566 kb hemizygous deletion of chromosome 11p13, including four annotated genes doublecortin domain containing 1 (DCDC1), DnaJ homolog subfamily C member 24 (DNAJC24), IMP1 inner mitochondrial membrane(IMMP1L), andelongation factor protein 4 (ELP4) downstream of PAX6. Quantitative real-time PCR verified the deletion in the proband and further identified the deletion in a blind fashion in four affected family members but not in the one with a normal phenotype. CONCLUSIONS: The 566 kb hemizygous deletion of chromosome 11p13 downstream of PAX6 should be the cause of the familial aniridia in this Chinese family, although two copies of PAX6 are intact. aCGH evaluation should be applied if there is a negative result for the mutation detection of PAX6 in patients with aniridia.\n",
      "Abstract: Elastin-like polypeptide (ELP) surface modification represents a valuable approach for the development of biomaterials in a wide range of medical applications. In this study, ELP surface modification has been achieved through the use of elastin cross-linking peptide (ECP) bioactive fluorinated surface modifiers (ECP-BFSMs). The synthesis of low molecular weight fluorinated additives was described and their subsequent blending with a base polycarbonate urethane (PCNU) was shown to successfully enrich the surface to allow for ELP surface cross-linking via lysine moieties on the peptide segments of the ECP-BFSMs. The kinetics for the surface migration of fluorescent ECP-BFSMs was studied over a 2-week period by two-photon confocal microscopy. A decrease in advancing contact angle from 87.9 degrees to 75.3 degrees was observed for ECP-BFSM modified PCNU and was associated with the presence of ECP peptides on the surface. X-ray photoelectron spectroscopy demonstrated an increase in surface atomic percent of fluorine (from 0.2 to 7.2%) and nitrogen (from 1.0 to 3.0%) associated with the surface localization of fluoro groups and amide groups associated with the peptides in the ECP-BFSMs. A further increase in surface atomic percent of nitrogen (from 3.0 to 8.3%) was observed after ELP surface cross-linking. These ELP-modified surfaces were shown to promote increased smooth muscle cell adhesion, spreading and retention over a 7-day culture period relative to their non-ELP4 analogs. This novel surface modifying additive approach may be used for various biomimetic applications since it generates a stable ECM-like surface retained onto a relatively inert fluorinated background.\n",
      "Abstract: We recently showed genomewide linkage of centrotemporal sharp waves (CTS) in classic Rolandic epilepsy (RE) families to chromosome 11p13, and fine-mapped this locus to variants in the ELP4 gene. Speech sound disorder (SSD) is a common comorbidity in RE subjects, of unknown etiology, which co-aggregates in family members in a manner that could hypothetically be explained by shared underlying genetic risk with CTS. Furthermore, the neural mechanism of SSD is unknown, although individuals with rare, Mendelian forms of RE are described with severe verbal and oromotor apraxia. We therefore first performed genomewide linkage analysis for SSD, operationally defined as clinical history consistent with ICD-10 speech articulation disorder, in 38 families singly ascertained through a proband with RE. We tested the hypothesis of shared genetic risk with CTS at the 11p13 locus. In the second part of the study we used computerized acoustic analysis of recorded speech to test the hypothesis of dyspraxia as a mechanism for SSD in a smaller subset of RE probands and relatives. In two-point and multipoint LOD score analysis, we found that evidence for linkage to the 11p13 locus increased substantially when the phenotype was broadened from CTS to CTS/SSD. In multipoint analysis, the LOD score rose by 3.2 to HLOD 7.54 at D11S914 for CTS/SSD, the same marker at which multipoint linkage maximized for CTS alone. Non-parametric, affected-only methods in a sub-set of the data provide further confirmatory evidence for pleiotropy. In acoustic analysis there were voice-onset time abnormalities in 10/18 RE probands, 8/16 siblings and 5/15 parents, providing evidence of breakdown in the spatial/temporal properties of speech articulation consistent with a dyspraxic mechanism. The results from genetic and physiological studies suggest a pleiotropic role for the 11p13 locus in the development of both SSD and CTS, and also indicate a dyspraxic mechanism for the SSD linked to 11p13. Taken together, these data strongly support a neurodevelopmental origin for classic RE.\n",
      "Abstract: Previous work in our laboratory showed the potential of using a human recombinant elastin-like polypeptide (ELP) as a thromboresistant coating. In this work we investigate the use of three particular ELPs (ELP1, ELP2 and ELP4), that differ by molecular weight and number of repeating hydrophobic and cross-linking domains, as coatings to improve blood-contacting properties. All three ELPs were passively adsorbed on Mylar surfaces. Differences in water contact angle and surface concentration were found among the three ELP coatings, with the shortest polypeptide, ELP1, being the most hydrophilic and abundant on the surface (55 degrees , 0.76 mug/cm(2)), followed by ELP2 (55 degrees , 0.35 mug/cm(2)) and ELP4, the longest of the three (66 degrees , 0.25 mug/cm(2)), respectively. The blood interactions of the ELP coatings were investigated by measuring fibrinogen adsorption and platelet adhesion in whole blood under laminar flow in a cone and plate viscometer configuration. In general, platelet adhesion to the ELP-coated surfaces was found to correlate with fibrinogen adsorption. Decreases in fibrinogen accretion and platelet adhesion were observed for ELP-coated compared to uncoated surfaces. The magnitude of the decreases was found to depend on the ELP sequence length, with ELP4 exhibiting the lowest levels of fibrinogen adsorption and platelet adhesion at 43 +/- 24 ng/cm(2) and 113 +/- 77 platelets/mm(2), respectively.\n",
      "Abstract: Investigators performing genetic association studies grapple with how to measure strength of association evidence, choose sample size, and adjust for multiple testing. We apply the evidential paradigm (EP) to genetic association studies, highlighting its strengths. The EP uses likelihood ratios (LRs), as opposed to P-values or Bayes' factors, to measure strength of association evidence. We derive EP methodology to estimate sample size, adjust for multiple testing, and provide informative graphics for drawing inferences, as illustrated with a Rolandic Epilepsy (RE) fine-mapping study. We focus on controlling the probability of observing weak evidence for or against association (W) rather than type I errors (M). For example, for LR> or =32 representing strong evidence, at one locus with n=200 cases, n=200 controls, W=0.134, whereas M=0.005. For n=300 cases and controls, W=0.039 and M=0.004. These calculations are based on detecting an OR=1.5. Despite the common misconception, one is not tied to this planning value for analysis; rather one calculates the likelihood at all possible values to assess evidence for association. We provide methodology to adjust for multiple tests across m loci, which adjusts M and W for m. We do so for (a) single-stage designs, (b) two-stage designs, and (c) simultaneously controlling family-wise error rate (FWER) and W. Method (c) chooses larger sample sizes than (a) or (b), whereas (b) has smaller bounds on the FWER than (a). The EP, using our innovative graphical display, identifies important SNPs in elongator protein complex 4 (ELP4) associated with RE that may not have been identified using standard approaches.\n",
      "Abstract: In eukaryotes, transcription of protein-encoding genes is strongly regulated by posttranslational modifications of histones that affect the accessibility of the DNA by RNA polymerase II (RNAPII). The Elongator complex was originally identified in yeast as a histone acetyltransferase (HAT) complex that activates RNAPII-mediated transcription. In Arabidopsis thaliana, the Elongator mutants elo1, elo2, and elo3 with decreased leaf and primary root growth due to reduced cell proliferation identified homologs of components of the yeast Elongator complex, Elp4, Elp1, and Elp3, respectively. Here we show that the Elongator complex was purified from plant cell cultures as a six-component complex. The role of plant Elongator in transcription elongation was supported by colocalization of the HAT enzyme, ELO3, with euchromatin and the phosphorylated form of RNAPII, and reduced histone H3 lysine 14 acetylation at the coding region of the SHORT HYPOCOTYL 2 auxin repressor and the LAX2 auxin influx carrier gene with reduced expression levels in the elo3 mutant. Additional auxin-related genes were down-regulated in the transcriptome of elo mutants but not targeted by the Elongator HAT activity showing specificity in target gene selection. Biological relevance was apparent by auxin-related phenotypes and marker gene analysis. Ethylene and jasmonic acid signaling and abiotic stress responses were up-regulated in the elo transcriptome and might contribute to the pleiotropic elo phenotype. Thus, although the structure of Elongator and its substrate are conserved, target gene selection has diverged, showing that auxin signaling and influx are under chromatin control.\n",
      "Abstract: The life cycle of mammals begins when a sperm enters an egg. Immediately after fertilization, both the maternal and paternal genomes undergo dramatic reprogramming to prepare for the transition from germ cell to somatic cell transcription programs. One of the molecular events that takes place during this transition is the demethylation of the paternal genome. Despite extensive efforts, the factors responsible for paternal DNA demethylation have not been identified. To search for such factors, we developed a live cell imaging system that allows us to monitor the paternal DNA methylation state in zygotes. Through short-interfering-RNA-mediated knockdown in mouse zygotes, we identified Elp3 (also called KAT9), a component of the elongator complex, to be important for paternal DNA demethylation. We demonstrate that knockdown of Elp3 impairs paternal DNA demethylation as indicated by reporter binding, immunostaining and bisulphite sequencing. Similar results were also obtained when other elongator components, Elp1 and Elp4, were knocked down. Importantly, injection of messenger RNA encoding the Elp3 radical SAM domain mutant, but not the HAT domain mutant, into MII oocytes before fertilization also impaired paternal DNA demethylation, indicating that the SAM radical domain is involved in the demethylation process. Our study not only establishes a critical role for the elongator complex in zygotic paternal genome demethylation, but also indicates that the demethylation process may be mediated through a reaction that requires an intact radical SAM domain.\n",
      "Abstract: Elongator complex has been associated with hyperphosphorylated RNA polymerase II and is known to play critical roles in transcriptional elongation, as well as in tRNA modification and exocytosis. However, the specific mechanism of how human Elongator complex regulates cell growth and cell cycle remains unclear. To investigate the composition of human Elongator complex and its effects on cell growth, 293T cells were established that stably overexpressed Flag-Elp3 and Flag-Elp4. By using anti-Flag M2 antibody-bound resin, a core Elongator complex was purified from cells that stably overexpressed Flag-Elp3. No Elongator complex was purified from cells stably transfected with pFlagCMV4-Elp4. Interestingly, the cell growth was inhibited in 293T cells transfected with pFlagCMV4- Elp3. Flow cytometry analysis showed that most of the cells stably overexpressing Flag-Elp3 were found in G1 stage, indicating a role of the core Elongator in the G1 checkpoint for the regulation of cell cycle. We observed increased basal transcription and remarkably enhanced transcription stimulated by VP16 in 293T cells overexpressing Flag-Elp3. The transcription could also be synergistically activated by overexpressing both Elp3 and Elp4. Taken together, our results suggested that the core Elongator complex formed by Elp1, Elp2, and Elp3 was rather stable, whereas Elp4, Elp5, and Elp6 might loosely contact and work together with the core Elongator to regulate cell functions.\n",
      "Abstract: Epilepsy is a frequent neurologic disease in childhood, characterized by recurrent seizures and sometimes with major effects on social, behavioral, and cognitive development. Childhood focal epilepsies particularly are age-related diseases mainly occurring during developmental critical periods. A complex interplay between brain development and maturation processes and susceptibility genes may contribute to the development of various childhood epileptic syndromes associated with language and cognitive deficits. Indeed, the Landau-Kleffner syndrome (LKS), the continuous spike-and-waves during sleep syndrome (CSWS), and the benign childhood epilepsy with centrotemporal spikes (BCECTS) or benign rolandic epilepsy, are different entities that are considered as part of a single continuous spectrum of disorders. Genetic predisposition with simple to complex modes of inheritance has long been suspected for this wide group of childhood focal epilepsies. Recent reports on the involvement of the SRPX2 and ELP4 genes with possible roles in cell motility, migration, and adhesion have provided first insights into the complex molecular bases of childhood focal epilepsies.\n",
      "Abstract: Elongator is a histone acetyl-transferase complex consisting of six subunits, and is highly conserved in eukaryotic organisms. Here, we isolated two novel mutants, elp2 and elp6, during a genetic screening for ABA-hypersensitive Arabidopsis mutants. Map-based cloning identified ELP2 and ELP6, which encode the orthologs of the yeast Elongator subunits, ELP2 and ELP6, respectively. Another Elongator subunit mutant, elp4/elo1, was obtained from the SALK T-DNA collection. The elp1/abo1/elo2 mutant was isolated in a previous study. All four of the Elongator mutants had narrow leaves, reduced root growth, ABA hypersensitivity and an increased accumulation of anthocyanins. Mutations in the core subcomplex subunits ELP1/ABO1 and ELP2, but not in the accessory subcomplex subunits ELP4/ELO1 and ELP6, caused stomatal closing to be supersensitive to ABA. In addition, the four mutants were all more resistant than the wild type to oxidative stress produced by methyl viologen, and to CsCl. Gene expression analysis indicated that the four mutants had increased transcript levels of CAT3 under normal conditions, increased transcript levels of ZAT10 when treated with ABA and reduced transcript levels of MYBL2, which encodes a single-repeat MYB protein, acting as a negative regulator of anthocyanin biosynthesis. Our results suggest that Elongator plays crucial roles in regulating plant responses to ABA, oxidative stress resistance and anthocyanin biosynthesis in Arabidopsis.\n",
      "Abstract: Rolandic epilepsy (RE) is the most common human epilepsy, affecting children between 3 and 12 years of age, boys more often than girls (3:2). Focal sharp waves in the centrotemporal area define the electroencephalographic (EEG) trait for the syndrome, are a feature of several related childhood epilepsies and are frequently observed in common developmental disorders (eg, speech dyspraxia, attention deficit hyperactivity disorder and developmental coordination disorder). Here we report the first genome-wide linkage scan in RE for the EEG trait, centrotemporal sharp waves (CTS), with genome-wide linkage of CTS to 11p13 (HLOD 4.30). Pure likelihood statistical analysis refined our linkage peak by fine mapping CTS to variants in Elongator Protein Complex 4 (ELP4) in two independent data sets; the strongest evidence was with rs986527 in intron 9 of ELP4, providing a likelihood ratio of 629:1 (P=0.0002) in favor of an association. Resequencing of ELP4 coding, flanking and promoter regions revealed no significant exonic polymorphisms. This is the first report of a gene implicated in a common focal epilepsy and the first human disease association of ELP4. ELP4 is a component of the Elongator complex, involved in transcription and tRNA modification. Elongator depletion results in the brain-specific downregulation of genes implicated in cell motility and migration. We hypothesize that a non-coding mutation in ELP4 impairs brain-specific Elongator-mediated interaction of genes implicated in brain development, resulting in susceptibility to seizures and neurodevelopmental disorders.\n",
      "Abstract: Recombinantly-engineered elastin-like polypeptides (ELPs) possess many of the favourable attributes of the native elastin protein, making them an attractive option for designing biomaterials for tissue-engineering applications. The focus of this study was to synthesize and characterise the bulk material properties of two ELP sequences, ELP2 and ELP4, cross-linked with lysine diisocyanate (LDI). The two distinct ELPs consist of repeating hydrophobic and hydrophilic cross-linking domains in a block co-polymer structure, however, differ by the number of respective domains. Depending on the conditions sets for the cross-linking reactions, two different ELP-based materials were synthesized: a gel-like relatively non-porous material and a porous foam-like material, from both ELP sequences. The physical material properties were characterised by scanning electron microscopy, compression testing, differential calorimetry analysis and swelling analysis. The bulk material properties were found to vary depending on the ELP sequence investigated. ELP gels were also found to have a more dense solidified morphology, lower compressive moduli, higher melting temperature and greater swelling capacity than the porous ELP foams. These novel cross-linked bio-elastomeric materials show promising properties for soft tissue replacement, particularly in load-bearing applications.\n",
      "Abstract: Gene duplication is a major driver of evolutionary divergence. In most vertebrates a single PAX6 gene encodes a transcription factor required for eye, brain, olfactory system, and pancreas development. In zebrafish, following a postulated whole-genome duplication event in an ancestral teleost, duplicates pax6a and pax6b jointly fulfill these roles. Mapping of the homozygously viable eye mutant sunrise identified a homeodomain missense change in pax6b, leading to loss of target binding. The mild phenotype emphasizes role-sharing between the co-orthologues. Meticulous mapping of isolated BACs identified perturbed synteny relationships around the duplicates. This highlights the functional conservation of pax6 downstream (3') control sequences, which in most vertebrates reside within the introns of a ubiquitously expressed neighbour gene, ELP4, whose pax6a-linked exons have been lost in zebrafish. Reporter transgenic studies in both mouse and zebrafish, combined with analysis of vertebrate sequence conservation, reveal loss and retention of specific cis-regulatory elements, correlating strongly with the diverged expression of co-orthologues, and providing clear evidence for evolution by subfunctionalization.\n",
      "Abstract: PURPOSE: PAX6 mutations cause aniridia as well as other various congenital eye abnormalities. Aniridia can be due to both point mutations and chromosomal deletions/rearrangements. Therefore, a complete search for PAX6 gene alterations in aniridia subjects requires a technically complex approach involving the comprehension of fluorescence in situ hybridization (FISH) analysis. In the present study, an Italian casistic of aniridia patients has been investigated and a quantitative polymerase chain reaction (PCR) assay to detect PAX6 gene deletions was set up. METHODS: Twenty-one aniridia patients were screened for point mutations (missense, nonsense, splicing-affecting, and short insertion/deletion) by using single-stranded conformational polymorphism (SSCP) and denaturing high performance liquid chromatography (dHPLC). To reveal deletions not detectable by SSCP or dHPLC, a quantitative PCR approach was set up for the PAX6 structural gene and for regions 5' and 3' to it at the level of WT1 and ELP4, respectively. RESULTS: Point mutations were found in 7 out of 21 patients. Three out of twenty-one patients showed deletions at the level of the PAX6 structural gene. In addition, two familial cases showed an undamaged PAX6 gene but a deletion in the region 3' to it at level of the ELP4 gene. In one of the families, the presence of the deletion has been confirmed by linkage analysis of polymorphic markers. CONCLUSIONS: In our casistic, a significant fraction of familial aniridia patients appears to be caused by a 3' deletion to PAX6, suggesting that evaluation of this alteration should be included in routine procedures of aniridia patients analysis. The quantitative PCR assay described here represents a simple approach to accomplish this task.\n",
      "Abstract: The key enzyme for transcription of protein-encoding genes in eukaryotes is RNA polymerase II (RNAPII). The recruitment of this enzyme during transcription initiation and its passage along the template during transcription elongation is regulated through the association and dissociation of several complexes. Elongator is a histone acetyl transferase complex, consisting of six subunits (ELP1-ELP6), that copurifies with the elongating RNAPII in yeast and humans. We demonstrate that point mutations in three Arabidopsis thaliana genes, encoding homologs of the yeast Elongator subunits ELP1, ELP3 (histone acetyl transferase), and ELP4 are responsible for the phenotypes of the elongata2 (elo2), elo3, and elo1 mutants, respectively. The elo mutants are characterized by narrow leaves and reduced root growth that results from a decreased cell division rate. Morphological and molecular phenotypes show that the ELONGATA (ELO) genes function in the same biological process and the epistatic interactions between the ELO genes can be explained by the model of complex formation in yeast. Furthermore, the plant Elongator complex is genetically positioned in the process of RNAPII-mediated transcription downstream of Mediator. Our data indicate that the Elongator complex is evolutionarily conserved in structure and function but reveal that the mechanism by which it stimulates cell proliferation is different in yeast and plants.\n",
      "Abstract: In this study, we performed in vivo experiments to determine the function of human Elongator subunit Elp4 by using a yeast complementary system. Our results indicated that though human ELP4 was not able to complement the growth defects of the ELP4 deletion mutant strain to high concentration salt, it partially reduced the sensitivity of mutant strain to caffeine, high temperature and 6-AU. Gene expression analysis indicated that human ELP4 partially resumed the slow activation of the PHO5 gene caused by the deletion of yELP4 under the low phosphate concentration. Meanwhile, under the condition of heat shock treatment, hELP4 increased the expression level of SSA3 gene. All these data demonstrated that human ELP4 can partially complement the growth defects and restore the slow activation of certain genes of the yELP4 deletion strain. These results indicate that human Elp4 subunit has similar functions to that of the yeast.\n",
      "Abstract: The molecular architecture of six-subunit yeast holo-Elongator complex was investigated by the use of immunoprecipitation, two-hybrid interaction mapping, and in vitro studies of binary interactions between individual subunits. Surprisingly, Elp2 is dispensable for the integrity of the holo-Elongator complex, and a purified five-subunit elp2 Delta Elongator complex retains histone acetyltransferase activity in vitro. These results indicate that the WD40 repeats in Elp2 are required neither for subunit-subunit interactions within Elongator nor for Elongator interaction with histones during catalysis. Elp2 and Elp4 were largely dispensable for the association of Elongator with nascent RNA transcript in vivo. In contrast, Elongator-RNA interaction requires the Elp3 protein. Together, these data shed light on the structure-function relationship of the Elongator complex.\n",
      "Abstract: In response to the Kluyveromyces lactis zymocin, the gamma-toxin target (TOT) function of the Saccharomyces cerevisiae RNA polymerase II (pol II) Elongator complex prevents sensitive strains from cell cycle progression. Studying Elongator subunit communications, Tot1p (Elp1p), the yeast homologue of human IKK-associated protein, was found to be essentially involved in maintaining the structural integrity of Elongator. Thus, the ability of Tot2p (Elp2p) to interact with the HAT subunit Tot3p (Elp3p) of Elongator and with subunit Tot5p (Elp5p) is dependent on Tot1p (Elp1p). Also, the association of core-Elongator (Tot1-3p/Elp1-3p) with HAP (Elp4-6p/Tot5-7p), the second three-subunit subcomplex of Elongator, was found to be sensitive to loss of TOT1 (ELP1) gene function. Structural integrity of the HAP complex itself requires the ELP4/TOT7, ELP5/TOT5, and ELP6/TOT6 genes, and elp6Delta/tot6Delta as well as elp4Delta/tot7Delta cells can no longer promote interaction between Tot5p (Elp5p) and Tot2p (Elp2p). The association between Elongator and Tot4p (Kti12p), a factor that may modulate the TOT activity of Elongator, requires Tot1-3p (Elp1-3p) and Tot5p (Elp5p), indicating that this contact requires a preassembled holo-Elongator complex. Tot4p also binds pol II hyperphosphorylated at its C-terminal domain Ser(5) raising the possibility that Tot4p bridges the contact between Elongator and pol II.\n",
      "Abstract: Seventy-seven patients with aniridia, referred for cytogenetic analysis predominantly to assess Wilms tumor risk, were studied by fluorescence in situ hybridization (FISH), through use of a panel of cosmids encompassing the aniridia-associated PAX6 gene, the Wilms tumor predisposition gene WT1, and flanking markers, in distal chromosome 11p13. Thirty patients were found to be chromosomally abnormal. Cytogenetically visible interstitial deletions involving 11p13 were found in 13 patients, 11 of which included WT1. A further 13 patients had cryptic deletions detectable only by FISH, 3 of which included WT1. Six of these, with deletions <500 kb, share a similar proximal breakpoint within a cosmid containing the last 10 exons of PAX6 and part of the neighboring gene, ELP4. Two of these six patients were mosaic for the deletion. The remaining four had chromosomal rearrangements: an unbalanced translocation, t(11;13), with a deletion including the WAGR (Wilms' tumor, aniridia, genitourinary abnormalities, and mental retardation) region, and three balanced rearrangements with what appear to be position effect breakpoints 3' of PAX6: (a) a t(7;11) with the 11p13 breakpoint approximately 30 kb downstream of PAX6, (b) a dir ins(12;11) with a breakpoint >50 kb from PAX6, and (c) an inv(11)(p13q13) with a breakpoint >75 kb downstream of PAX6. The proportion and spectrum of chromosome anomalies in familial (4/14, or 28.5%) and sporadic (26/63, or 41%) cases are not significantly different. An unexpectedly high frequency of chromosomal rearrangements is associated with both sporadic and familial aniridia in this cohort.\n",
      "Abstract: TOT, the putative Kluyveromyces lactis zymocin target complex from Saccharomyces cerevisiae, is encoded by TOT1-7, six loci of which are isoallelic to RNA polymerase II (RNAPII) Elongator genes (ELP1-6). Unlike TOT1-3 (ELP1-3) and TOT5-7 (ELP5, ELP6 and ELP4 respectively), which display zymocin resistance when deleted, TOT4 (KTI12) also renders cells refractory to zymocin when maintained in multicopy or overexpressed from the GAL10 promoter. Elevated TOT4 copy number results in an intermediate tot phenotype, which includes mild sensitivities towards caffeine, Calcofluor white and elevated growth temperature, suggesting that TOT4 influences TOT/Elongator function. Tot4p interacts with Elongator, as shown by co-immunoprecipitation, and cell fractionation studies demonstrate partial co-migration with RNAPII and Elongator. As Elongator subunit interaction is not affected by either deletion of TOT4 or multicopy TOT4, Tot4p may not be a structural Elongator subunit but, rather, may regulate TOT/Elongator in a fashion that requires transient physical contact with TOT/Elongator. Consistent with a regulatory role, the presence of a potential P-loop motif conserved between yeast and human TOT4 homologues suggests capability of ATP or GTP binding and P-loop deletion renders Tot4p biologically inactive.\n",
      "Abstract: The elongating, hyperphosphorylated form of RNA polymerase II is associated with the Elongator complex, which has the histone acetyltransferase (HAT) Elp3 as a subunit. Here we show that, in contrast to the isolated Elp3 subunit, the activity of intact Elongator complex is directed specifically toward the amino-terminal tails of histone H3 and H4, and that Elongator can acetylate both core histones and nucleosomal substrates. The predominant acetylation sites are lysine-14 of histone H3 and lysine-8 of histone H4. The three smallest Elongator subunits--Elp4, Elp5, and Elp6--are required for HAT activity, and Elongator binds to both naked and nucleosomal DNA. By using chromatin immunoprecipitation, we show that the levels of multiply acetylated histone H3 and H4 in chromatin are decreased in vivo in yeast cells lacking ELP3.\n",
      "Abstract: The human eye anomaly aniridia is normally caused by intragenic mutations of PAX6. Several cases of aniridia are, however, associated with chromosomal rearrangements that leave the PAX6 gene intact. We have identified and characterized a novel gene, PAXNEB (C11orf19), downstream (telomeric) of PAX6. Sequence analysis, including interspecies comparisons, show this gene to consist of 10 exons, with an unusually large final intron spanning 134 kb in human and 18 kb in Fugu. This intron is disrupted by each chromosomal rearrangement. The 2-kb PAXNEB transcript, encoding a 424-amino acid protein, is expressed in all cell lines tested. The homologous mouse cDNA is broadly expressed in mouse embryos. PAXNEB is highly conserved from mammals to fish, with some regions of the protein showing conservation to invertebrates, yeast, and plants. The possible role of PAXNEB in aniridia was assessed. Using a transgenic mouse model, we show that the aniridia phenotype of the chromosomal rearrangement cases is not due to the heterozygous loss of PAXNEB function.\n",
      "Abstract: The putative Kluyveromyces lactis zymocin target complex, TOT, from Saccharomyces cerevisiae comprises five Tot proteins, four of which are RNA polymerase II (RNAP II) Elongator subunits. Recently, two more Elongator subunit genes, ELP6 (TOT6) and ELP4 (TOT7), have been identified. Deletions of both TOT6 and TOT7 result in the complex tot phenotype, including resistance to zymocin, thermosensitivity, slow growth and hypersensitivity towards drugs, thus reinforcing the notion that TOT/Elongator may be crucial in signalling zymocicity. Mutagenesis of ELP3/TOT3, the Elongator histone acetyltransferase (HAT) gene, revealed that zymocin sensitivity could be uncoupled from Elongator wild-type function, indicating that TOT interacts genetically with zymocin. To test the possibility that zymocin functions by affecting RNAP II activity in a TOT/Elongator-dependent manner, global poly(A)+ mRNA levels were found to decline drastically on zymocin treatment. Moreover, cells overexpressing Fcp1p, the RNAP II carboxy-terminal domain phosphatase, acquired partial zymocin resistance, whereas cells underproducing RNAP II became zymocin hypersensitive. This suggests that zymocin may convert TOT/Elongator into a cellular poison toxic for RNAP II function and eventually leading to the observed G1 cell cycle arrest.\n",
      "Abstract: The Elongator complex associated with elongating RNA polymerase II in Saccharomyces cerevisiae was originally reported to have three subunits, Elp1, Elp2, and Elp3. Using the tandem affinity purification (TAP) procedure, we have purified a six-subunit yeast Holo-Elongator complex containing three additional polypeptides, which we have named Elp4, Elp5, and Elp6. TAP tapping and subsequent purification of any one of the six subunits result in the isolation of all six components. Purification of Elongator in higher salt concentrations served to demonstrate that the complex could be separated into two subcomplexes: one consisted of Elp1, -2, and -3, and the other consisted of Elp4, -5, and -6. Deletions of the individual genes encoding the new Elongator subunits showed that only the ELP5 gene is essential for growth. Disruption of the two nonessential new Elongator-encoding genes, ELP4 and ELP6, caused the same phenotypes observed with knockouts of the original Elongator-encoding genes. Results of microarray analyses demonstrated that the gene expression profiles of strains containing deletions of genes encoding subunits of either Elongator subcomplex, in which we detected significantly altered mRNA expression levels for 96 genes, are very similar, implying that all the Elongator subunits likely function together to regulate a group of S. cerevisiae genes in vivo.\n",
      "Abstract: Elongator is a histone acetyltransferase complex that associates with the elongating form of RNA polymerase II. We purified Elongator to virtual homogeneity via a rapid three-step procedure based largely on affinity chromatography. The purified factor, holo-Elongator, is a labile six-subunit factor composed of two discrete subcomplexes: one comprised of the previously identified Elp1, Elp2, and Elp3 proteins and another comprised of three novel polypeptides, termed Elp4, Elp5, and Elp6. Disruption of the yeast genes encoding the new Elongator proteins confers phenotypes indistinguishable from those previously described for the other elp mutants, and concomitant disruption of genes encoding proteins in either subcomplex does not confer new phenotypes. Taken together, our results indicate that holo-Elongator is a functional entity in vitro as well as in vivo. Metazoan homologues of Elp1 and Elp3 have previously been reported. We cloned the human homologue of yeast ELP4 and show that this gene is ubiquitously expressed in human tissues.\n"
     ]
    }
   ],
   "source": [
    "for record in records:\n",
    "    print(\"Abstract:\", record.get(\"AB\", \"?\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "'StringElement' object has no attribute 'Abstract'",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-8-b3bca2e0cd7d>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m\u001b[0m\n\u001b[0;32m      3\u001b[0m \u001b[0martigos_para_reter\u001b[0m \u001b[1;33m=\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;33m]\u001b[0m \u001b[1;31m# lista dos IDs dos artigos que contém informação sobre o cancro ou mais especificamente, sobre o cancro da mama\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      4\u001b[0m \u001b[1;32mfor\u001b[0m \u001b[0ma\u001b[0m \u001b[1;32min\u001b[0m \u001b[0mlista_artigos\u001b[0m\u001b[1;33m:\u001b[0m \u001b[1;31m# o \"a\" é um ID da lista_artigos\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 5\u001b[1;33m     \u001b[1;32mif\u001b[0m \u001b[1;34m\"CANCER\"\u001b[0m \u001b[1;32min\u001b[0m \u001b[0ma\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mAbstract\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m      6\u001b[0m         \u001b[0martigos_para_reter\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0ma\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      7\u001b[0m \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0martigos_para_reter\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mAttributeError\u001b[0m: 'StringElement' object has no attribute 'Abstract'"
     ]
    }
   ],
   "source": [
    "#NÃO FUNCIONA!\n",
    "lista_artigos = idlist\n",
    "artigos_para_reter =[] # lista dos IDs dos artigos que contém informação sobre o cancro ou mais especificamente, sobre o cancro da mama \n",
    "for a in lista_artigos: # o \"a\" é um ID da lista_artigos\n",
    "    if \"CANCER\" in a.Abstract:\n",
    "        artigos_para_reter.append(a)\n",
    "print(artigos_para_reter)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "6.Com o código que se apresenta de seguida obtém-se num ficheiro com todos os artigos do PubMed relacionados com o gene ELP4, com todas as informações que eles contêm. Ou seja, obtém-se num ficheiro tudo o que se encontra em cada um dos 61 atigos e, posteriormente consegue-se se saber se algum dos artigos nos dá indicações sobre o facto de este gene estar relacionado com o cancro ou não (ou mais especificamente, se está relacionado com o cancro da mama ou não).É uma alternativa ao que foi realizado em parte no passo 3."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "Entrez.email = \"joanafigueiredo2014@gmail.com\"\n",
    "search_results = Entrez.read(Entrez.esearch(db=\"pubmed\", term=\"ELP4\", reldate=100000, datetype=\"pdat\", usehistory=\"y\"))\n",
    "count = int(search_results[\"Count\"])\n",
    "print(\"Found %i results\" % count)\n",
    "batch_size = 61\n",
    "out_handle = open(\"recent_ELP4_papers.txt\", \"w\")\n",
    "for start in range(0,count,batch_size):\n",
    "    end = min(count, start+batch_size)\n",
    "    print(\"Going to download record %i to %i\" % (start+1, end))\n",
    "    fetch_handle = Entrez.efetch(db=\"pubmed\",rettype=\"medline\",retmode=\"text\",retstart=start,retmax=batch_size,webenv=search_results[\"WebEnv\"],query_key=search_results[\"QueryKey\"])\n",
    "    data = fetch_handle.read()\n",
    "    fetch_handle.close()\n",
    "out_handle.write(data)\n",
    "out_handle.close()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
